--- ../../../downloads/openwrt_trunk-r35953/feeds/packages/libs/libjpeg-turbo/Makefile	2013-03-11 09:29:39.334073816 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libjpeg-turbo/Makefile	2013-03-11 10:57:16.745976988 +0400
@@ -19,6 +19,7 @@
 PKG_LICENSE_FILES:=README-turbo.txt
 
 PKG_BUILD_PARALLEL:=1
+PKG_BUILD_DEPENDS:=libjpeg
 
 include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
@@ -65,34 +66,34 @@
 #
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/jpeglib.h $(1)/usr/include/
-	$(CP) $(PKG_BUILD_DIR)/jpegint.h $(1)/usr/include/
-	$(CP) $(PKG_BUILD_DIR)/j{config,error,morecfg}.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libjpeg.{a,so*} $(1)/usr/lib/
-#	$(CP) $(PKG_BUILD_DIR)/.libs/libturbojpeg.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_BUILD_DIR)/jpeglib.h $(1)/opt/include/
+	$(CP) $(PKG_BUILD_DIR)/jpegint.h $(1)/opt/include/
+	$(CP) $(PKG_BUILD_DIR)/j{config,error,morecfg}.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libjpeg.{a,so*} $(1)/opt/lib/
+#	$(CP) $(PKG_BUILD_DIR)/.libs/libturbojpeg.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libjpeg-turbo/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libjpeg.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libjpeg.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/jpeg-tools/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/{c,d}jpeg $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/jpegtran $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/{rd,wr}jpgcom $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/{c,d}jpeg $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/jpegtran $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/{rd,wr}jpgcom $(1)/opt/bin/
 endef
 
 define Package/libjpeg-turbo-tests/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/tjbench $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/tjunittest $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/jcstest $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libturbojpeg.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/tjbench $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/tjunittest $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/jcstest $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libturbojpeg.{a,so*} $(1)/opt/lib/
 endef
 
 $(eval $(call HostBuild))
