--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/quicktun/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/quicktun/Makefile
@@ -22,7 +22,6 @@
 define Package/quicktun
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-tun
   TITLE:=QuickTun is a simple and secure VPN software
   URL:=http://wiki.ucis.nl/QuickTun
   SUBMENU:=VPN
@@ -32,16 +31,12 @@
  QuickTun is a simple and secure VPN software
 endef
 
-define Package/quicktun/conffiles
-/etc/config/quicktun
-endef
-
 define Build/Compile
 	( \
 		cd $(PKG_BUILD_DIR); \
 		mkdir -p obj out; \
 		\
-		export CPATH=$(STAGING_DIR)/usr/include/nacl; \
+		export CPATH=$(STAGING_DIR)/opt/include/nacl; \
 		\
 		$(TARGET_CC) $(TARGET_CFLAGS) $(TARGET_CPPFLAGS) -c -DCOMBINED_BINARY        src/proto.raw.c         -o obj/proto.raw.o; \
 		$(TARGET_CC) $(TARGET_CFLAGS) $(TARGET_CPPFLAGS) -c -DCOMBINED_BINARY        src/proto.nacl0.c       -o obj/proto.nacl0.o; \
@@ -56,14 +51,11 @@
 endef
 
 define Package/quicktun/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/out/quicktun.{combined,keypair} $(1)/usr/sbin/
-	$(LN) quicktun.combined $(1)/usr/sbin/quicktun
-
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) files/$(PKG_NAME).init $(1)/etc/init.d/$(PKG_NAME)
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) files/$(PKG_NAME).config $(1)/etc/config/$(PKG_NAME)
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/out/quicktun.{combined,keypair} $(1)/opt/sbin/
+	$(LN) quicktun.combined $(1)/opt/sbin/quicktun
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) files/S19quicktun $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,quicktun))
