--- ../../../openwrt_trunk.orig/feeds/packages/libs/flac/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/flac/Makefile
@@ -51,27 +51,27 @@
 TARGET_CFLAGS += $(FPIC)
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/include
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/FLAC \
-		$(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+		$(PKG_INSTALL_DIR)/opt/include/FLAC \
+		$(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libFLAC.{a,so*,la} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libFLAC.{a,so*,la} \
+		$(1)/opt/lib/
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/* \
-		$(1)/usr/lib/pkgconfig/
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/* \
+		$(1)/opt/lib/pkgconfig/
 
-	$(INSTALL_DIR) $(1)/usr/share/aclocal/
+	$(INSTALL_DIR) $(1)/opt/share/aclocal/
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/share/aclocal/* \
-		$(1)/usr/share/aclocal/
+		$(PKG_INSTALL_DIR)/opt/share/aclocal/* \
+		$(1)/opt/share/aclocal/
 endef
 
 define Package/libflac/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libFLAC.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libFLAC.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libflac))
