--- ../../../openwrt_trunk/feeds/packages/net/bind/Makefile.prev	2013-03-15 02:14:58.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/bind/Makefile	2013-03-17 22:46:35.000000000 +0400
@@ -81,6 +81,11 @@
   TITLE+= DNS excavation tool
 endef
 
+define Package/bind-nslookup
+  $(call Package/bind/Default)
+  TITLE+= DNS querying tool
+endef
+
 export BUILD_CC="$(TARGET_CC)"
 
 CONFIGURE_ARGS += \
@@ -187,6 +192,11 @@
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/dig $(1)/opt/bin/
 endef
 
+define Package/bind-nslookup/install
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/nslookup $(1)/opt/bin/
+endef
+
 $(eval $(call BuildPackage,bind-libs))
 $(eval $(call BuildPackage,bind-server))
 $(eval $(call BuildPackage,bind-client))
@@ -196,3 +206,4 @@
 $(eval $(call BuildPackage,bind-dnssec))
 $(eval $(call BuildPackage,bind-host))
 $(eval $(call BuildPackage,bind-dig))
+$(eval $(call BuildPackage,bind-nslookup))
