--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/nmeap/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/nmeap/Makefile
@@ -32,10 +32,10 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/inc/nmeap{,_def}.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/libnmeap.a $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_BUILD_DIR)/inc/nmeap{,_def}.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/libnmeap.a $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libnmeap))
