--- ../../../trunk_clean/feeds/packages/net/mtr/Makefile	2012-06-02 17:11:01.000000000 +0400
+++ ../../../trunk/feeds/packages/net/mtr/Makefile	2012-06-03 03:17:57.000000000 +0400
@@ -60,8 +60,8 @@
 endef
 
 define Package/mtr/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/mtr $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/mtr $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,mtr))
