diff -urx .svn ../../../downloads/openwrt_trunk-r43593/feeds/packages/lang/python/files/python-package.mk ../../../openwrt_trunk/feeds/packages/lang/python/files/python-package.mk
--- ../../../downloads/openwrt_trunk-r43593/feeds/packages/lang/python/files/python-package.mk	2014-12-10 12:26:32.878951683 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/python/files/python-package.mk	2014-12-11 11:58:03.017409293 +0300
@@ -8,12 +8,12 @@
 PYTHON_VERSION:=2.7
 PYTHON_VERSION_MICRO:=8
 
-PYTHON_DIR:=$(STAGING_DIR)/usr
+PYTHON_DIR:=$(STAGING_DIR)/opt
 PYTHON_BIN_DIR:=$(PYTHON_DIR)/bin
 PYTHON_INC_DIR:=$(PYTHON_DIR)/include/python$(PYTHON_VERSION)
 PYTHON_LIB_DIR:=$(PYTHON_DIR)/lib/python$(PYTHON_VERSION)
 
-PYTHON_PKG_DIR:=/usr/lib/python$(PYTHON_VERSION)/site-packages
+PYTHON_PKG_DIR:=/opt/lib/python$(PYTHON_VERSION)/site-packages
 
 PYTHON:=python$(PYTHON_VERSION)
 
@@ -91,7 +91,7 @@
 		CPPFLAGS="$(TARGET_CPPFLAGS) -I$(PYTHON_INC_DIR)" \
 		LDFLAGS="$(TARGET_LDFLAGS) -lpython$(PYTHON_VERSION)" \
 		_PYTHON_HOST_PLATFORM="linux-$(ARCH)" \
-		__PYVENV_LAUNCHER__="/usr/bin/$(PYTHON)" \
+		__PYVENV_LAUNCHER__="/opt/bin/$(PYTHON)" \
 		$(3) \
 		, \
 		./setup.py $(2) \
diff -urx .svn ../../../downloads/openwrt_trunk-r43593/feeds/packages/lang/python/Makefile ../../../openwrt_trunk/feeds/packages/lang/python/Makefile
--- ../../../downloads/openwrt_trunk-r43593/feeds/packages/lang/python/Makefile	2014-12-10 12:26:32.878951683 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/python/Makefile	2014-12-11 12:02:13.881401725 +0300
@@ -87,11 +87,11 @@
 endif
 
 CONFIGURE_ARGS+= \
-	--sysconfdir=/etc \
+	--sysconfdir=/opt/etc \
 	--enable-shared \
 	--without-cxx-main \
 	--with-threads \
-	--with-system-ffi="$(STAGING_DIR)/usr" \
+	--with-system-ffi="$(STAGING_DIR)/opt" \
 	--without-pymalloc \
 	$(ENABLE_IPV6) \
 	CONFIG_SITE="$(PKG_BUILD_DIR)/config.site" \
@@ -104,66 +104,66 @@
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(STAGING_DIR)/mk/
-	$(INSTALL_DIR) $(1)/usr/include/ $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/python$(PYTHON_VERSION)/
+	$(INSTALL_DIR) $(1)/opt/include/ $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/python$(PYTHON_VERSION)/
 	$(INSTALL_DATA) ./files/python-package.mk $(STAGING_DIR)/mk/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/python$(PYTHON_VERSION) \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/python$(PYTHON_VERSION) \
+		$(1)/opt/include/
 	$(CP) \
 		$(STAGING_DIR_HOST)/lib/python$(PYTHON_VERSION) \
-		$(PKG_INSTALL_DIR)/usr/lib/libpython$(PYTHON_VERSION).so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libpython$(PYTHON_VERSION).so* \
+		$(1)/opt/lib/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/python$(PYTHON_VERSION)/config \
-		$(1)/usr/lib/python$(PYTHON_VERSION)/
+		$(PKG_INSTALL_DIR)/opt/lib/python$(PYTHON_VERSION)/config \
+		$(1)/opt/lib/python$(PYTHON_VERSION)/
 endef
 
 define PyPackage/python-base/filespec
-+|/usr/bin/python$(PYTHON_VERSION)
-+|/usr/lib/python$(PYTHON_VERSION)/_abcoll.py
-+|/usr/lib/python$(PYTHON_VERSION)/_sysconfigdata.py
-+|/usr/lib/python$(PYTHON_VERSION)/_weakrefset.py
-+|/usr/lib/python$(PYTHON_VERSION)/abc.py
-+|/usr/lib/python$(PYTHON_VERSION)/copy_reg.py
-+|/usr/lib/python$(PYTHON_VERSION)/genericpath.py
-+|/usr/lib/python$(PYTHON_VERSION)/linecache.py
-+|/usr/lib/python$(PYTHON_VERSION)/posixpath.py
-+|/usr/lib/python$(PYTHON_VERSION)/os.py
-+|/usr/lib/python$(PYTHON_VERSION)/re.py
-+|/usr/lib/python$(PYTHON_VERSION)/site.py
-+|/usr/lib/python$(PYTHON_VERSION)/sre_compile.py
-+|/usr/lib/python$(PYTHON_VERSION)/sre_constants.py
-+|/usr/lib/python$(PYTHON_VERSION)/sre_parse.py
-+|/usr/lib/python$(PYTHON_VERSION)/sysconfig.py
-+|/usr/lib/python$(PYTHON_VERSION)/stat.py
-+|/usr/lib/python$(PYTHON_VERSION)/traceback.py
-+|/usr/lib/python$(PYTHON_VERSION)/types.py
-+|/usr/lib/python$(PYTHON_VERSION)/UserDict.py
-+|/usr/lib/python$(PYTHON_VERSION)/warnings.py
++|/opt/bin/python$(PYTHON_VERSION)
++|/opt/lib/python$(PYTHON_VERSION)/_abcoll.py
++|/opt/lib/python$(PYTHON_VERSION)/_sysconfigdata.py
++|/opt/lib/python$(PYTHON_VERSION)/_weakrefset.py
++|/opt/lib/python$(PYTHON_VERSION)/abc.py
++|/opt/lib/python$(PYTHON_VERSION)/copy_reg.py
++|/opt/lib/python$(PYTHON_VERSION)/genericpath.py
++|/opt/lib/python$(PYTHON_VERSION)/linecache.py
++|/opt/lib/python$(PYTHON_VERSION)/posixpath.py
++|/opt/lib/python$(PYTHON_VERSION)/os.py
++|/opt/lib/python$(PYTHON_VERSION)/re.py
++|/opt/lib/python$(PYTHON_VERSION)/site.py
++|/opt/lib/python$(PYTHON_VERSION)/sre_compile.py
++|/opt/lib/python$(PYTHON_VERSION)/sre_constants.py
++|/opt/lib/python$(PYTHON_VERSION)/sre_parse.py
++|/opt/lib/python$(PYTHON_VERSION)/sysconfig.py
++|/opt/lib/python$(PYTHON_VERSION)/stat.py
++|/opt/lib/python$(PYTHON_VERSION)/traceback.py
++|/opt/lib/python$(PYTHON_VERSION)/types.py
++|/opt/lib/python$(PYTHON_VERSION)/UserDict.py
++|/opt/lib/python$(PYTHON_VERSION)/warnings.py
 endef
 
 define PyPackage/python/filespec
-+|/usr/lib/python$(PYTHON_VERSION)
--|/usr/lib/python$(PYTHON_VERSION)/config
--|/usr/lib/python$(PYTHON_VERSION)/distutils/cygwinccompiler.py
--|/usr/lib/python$(PYTHON_VERSION)/distutils/command/wininst*
--|/usr/lib/python$(PYTHON_VERSION)/idlelib
--|/usr/lib/python$(PYTHON_VERSION)/lib2to3
--|/usr/lib/python$(PYTHON_VERSION)/lib-tk
--|/usr/lib/python$(PYTHON_VERSION)/lib-dynload/_testcapi.so
--|/usr/lib/python$(PYTHON_VERSION)/pdb.doc
--|/usr/lib/python$(PYTHON_VERSION)/test
--|/usr/lib/python$(PYTHON_VERSION)/webbrowser.py
--|/usr/lib/python$(PYTHON_VERSION)/*/test
--|/usr/lib/python$(PYTHON_VERSION)/*/tests
--|/usr/lib/python$(PYTHON_VERSION)/lib-dynload/readline.so
++|/opt/lib/python$(PYTHON_VERSION)
+-|/opt/lib/python$(PYTHON_VERSION)/config
+-|/opt/lib/python$(PYTHON_VERSION)/distutils/cygwinccompiler.py
+-|/opt/lib/python$(PYTHON_VERSION)/distutils/command/wininst*
+-|/opt/lib/python$(PYTHON_VERSION)/idlelib
+-|/opt/lib/python$(PYTHON_VERSION)/lib2to3
+-|/opt/lib/python$(PYTHON_VERSION)/lib-tk
+-|/opt/lib/python$(PYTHON_VERSION)/lib-dynload/_testcapi.so
+-|/opt/lib/python$(PYTHON_VERSION)/pdb.doc
+-|/opt/lib/python$(PYTHON_VERSION)/test
+-|/opt/lib/python$(PYTHON_VERSION)/webbrowser.py
+-|/opt/lib/python$(PYTHON_VERSION)/*/test
+-|/opt/lib/python$(PYTHON_VERSION)/*/tests
+-|/opt/lib/python$(PYTHON_VERSION)/lib-dynload/readline.so
 endef
 
 define PyPackage/python-base/install
-	$(LN) python$(PYTHON_VERSION) $(1)/usr/bin/python
-	$(LN) python$(PYTHON_VERSION) $(1)/usr/bin/python2
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpython$(PYTHON_VERSION).so* $(1)/usr/lib/
+	$(LN) python$(PYTHON_VERSION) $(1)/opt/bin/python
+	$(LN) python$(PYTHON_VERSION) $(1)/opt/bin/python2
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpython$(PYTHON_VERSION).so* $(1)/opt/lib/
 endef
 
 HOST_CONFIGURE_ARGS+= \
