--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/net/znc/Makefile	2015-09-29 08:23:28.728164410 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/znc/Makefile	2015-09-29 15:14:53.080164410 +0300
@@ -34,7 +34,7 @@
 
 define Package/znc
   $(Package/znc/default)
-  DEPENDS:=+libopenssl +libpthread +libstdcpp @GCC_VERSION_4_8||GCC_VERSION_4_9
+  DEPENDS:=+libopenssl +libpthread +libstdcpp
   MENU:=1
 endef
 
@@ -45,19 +45,19 @@
 endef
 
 define Package/znc/conffiles
-/etc/config/znc
+/opt/etc/config/znc
 endef
 
 
 define Package/znc/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/znc $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/znc.init $(1)/etc/init.d/znc
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/znc.conf $(1)/etc/config/znc
-	$(INSTALL_DIR) $(1)/usr/lib/znc/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/droproot.so $(1)/usr/lib/znc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/znc $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S99znc $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/etc/config
+	$(INSTALL_DATA) ./files/znc.conf $(1)/opt/etc/config/znc
+	$(INSTALL_DIR) $(1)/opt/lib/znc/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/droproot.so $(1)/opt/lib/znc/
 endef
 
 
@@ -76,12 +76,12 @@
   endef
 
   define Package/znc-mod-$(strip $(1))/install
-	$(INSTALL_DIR) $$(1)/usr/lib/znc/
-	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/$(subst -,_,$(strip $(1))).so $$(1)/usr/lib/znc/
+	$(INSTALL_DIR) $$(1)/opt/lib/znc/
+	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/$(subst -,_,$(strip $(1))).so $$(1)/opt/lib/znc/
 	# include webadmin page templates if existing
 	if [ -d $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) ]; then \
-		$(INSTALL_DIR) $$(1)/usr/share/znc/modules ;\
-		$(CP) -r $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) $$(1)/usr/share/znc/modules ;\
+		$(INSTALL_DIR) $$(1)/opt/share/znc/modules ;\
+		$(CP) -r $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) $$(1)/opt/share/znc/modules ;\
 	fi
   endef
 
@@ -101,12 +101,12 @@
   endef
 
   define Package/znc-mod-webadmin/install
-	$(INSTALL_DIR) $$(1)/usr/lib/znc/
-	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/webadmin.so $$(1)/usr/lib/znc/
-	$(INSTALL_DIR) $$(1)/usr/share/znc/modules
-	$(CP) -r $$(PKG_BUILD_DIR)/modules/data/webadmin $$(1)/usr/share/znc/modules
-	$(INSTALL_DIR) $$(1)/usr/share/znc/webskins/
-	$(CP) -r $$(PKG_BUILD_DIR)/webskins/_default_ $$(1)/usr/share/znc/webskins/
+	$(INSTALL_DIR) $$(1)/opt/lib/znc/
+	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/webadmin.so $$(1)/opt/lib/znc/
+	$(INSTALL_DIR) $$(1)/opt/share/znc/modules
+	$(CP) -r $$(PKG_BUILD_DIR)/modules/data/webadmin $$(1)/opt/share/znc/modules
+	$(INSTALL_DIR) $$(1)/opt/share/znc/webskins/
+	$(CP) -r $$(PKG_BUILD_DIR)/webskins/_default_ $$(1)/opt/share/znc/webskins/
   endef
 
   ZNC_MODULES += znc-mod-webadmin
@@ -125,8 +125,8 @@
   endef
 
   define Package/znc-webskin-$(strip $(1))/install
-	$(INSTALL_DIR) $$(1)/usr/share/znc/webskins/
-	$(CP) -r $$(PKG_BUILD_DIR)/webskins/$(strip $(1)) $$(1)/usr/share/znc/webskins/
+	$(INSTALL_DIR) $$(1)/opt/share/znc/webskins/
+	$(CP) -r $$(PKG_BUILD_DIR)/webskins/$(strip $(1)) $$(1)/opt/share/znc/webskins/
   endef
   ZNC_MODULES += znc-webskin-$(strip $(1))
 endef
@@ -270,10 +270,15 @@
 
 include $(INCLUDE_DIR)/package.mk
 
+DYNLINK=ld-linux.so.2
+ifeq ($(ARCH),arm)
+  DYNLINK:=ld-linux.so.3
+endif
+
 CONFIGURE_VARS += \
 	CXXFLAGS="$(TARGET_CFLAGS) -fno-builtin -fno-rtti" \
-	CPPFLAGS="-I$(STAGING_DIR)/usr/include -I$(STAGING_DIR)/include" \
-	LDFLAGS="-nodefaultlibs -lc -L$(STAGING_DIR)/usr/lib -L$(STAGING_DIR)/lib" \
+	CPPFLAGS="-I$(STAGING_DIR)/opt/include -I$(STAGING_DIR)/include" \
+	LDFLAGS="-nodefaultlibs -lc -L$(STAGING_DIR)/opt/lib -L$(STAGING_DIR)/lib -Wl,--dynamic-linker=/Apps/opt/lib/$(DYNLINK) -Wl,-rpath,/Apps/opt/lib" \
 	LIBS="-lstdc++ -lm -lssl -lcrypto $(LIBGCC_S) -lc"
 
 CONFIGURE_ARGS += \
