--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/znc/Makefile
+++ ../../../openwrt_trunk/feeds/packages/net/znc/Makefile
@@ -44,22 +44,29 @@
 endef
 
 define Package/znc/conffiles
-/etc/config/znc
+/opt/etc/config/znc
 endef
 
 
 define Package/znc/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/znc $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/znc.init $(1)/etc/init.d/znc
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/znc.conf $(1)/etc/config/znc
-	$(INSTALL_DIR) $(1)/usr/lib/znc/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/droproot.so $(1)/usr/lib/znc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/znc $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S99znc $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/lib/znc/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/droproot.so $(1)/opt/lib/znc/
+endef
+
+define Package/znc/postinst
+#!/bin/sh
+
+echo Before using ZNC do not forget to:
+echo a\) Generate new config with \"znc -d /opt/share/znc -c\"
+echo b\) Add a \"LoadModule = droproot nobody nogroup\" string at first string
+echo to /opt/share/znc/configs/znc.conf, where \"nobody\" and \"nogroup\" is
+echo some unprivileged user\\group on your system.
 endef
 
-
 ZNC_MODULES :=
 ZNC_MODULE_TARGETS := droproot.so
 
@@ -75,12 +82,12 @@
   endef
 
   define Package/znc-mod-$(strip $(1))/install
-	$(INSTALL_DIR) $$(1)/usr/lib/znc/
-	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/$(subst -,_,$(strip $(1))).so $$(1)/usr/lib/znc/
+	$(INSTALL_DIR) $$(1)/opt/lib/znc/
+	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/$(subst -,_,$(strip $(1))).so $$(1)/opt/lib/znc/
 	# include webadmin page templates if existing
 	if [ -d $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) ]; then \
-		$(INSTALL_DIR) $$(1)/usr/share/znc/modules ;\
-		$(CP) -r $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) $$(1)/usr/share/znc/modules ;\
+		$(INSTALL_DIR) $$(1)/opt/share/znc/modules ;\
+		$(CP) -r $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) $$(1)/opt/share/znc/modules ;\
 	fi
   endef
 
@@ -100,12 +107,12 @@
   endef
 
   define Package/znc-mod-webadmin/install
-	$(INSTALL_DIR) $$(1)/usr/lib/znc/
-	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/webadmin.so $$(1)/usr/lib/znc/
-	$(INSTALL_DIR) $$(1)/usr/share/znc/modules
-	$(CP) -r $$(PKG_BUILD_DIR)/modules/data/webadmin $$(1)/usr/share/znc/modules
-	$(INSTALL_DIR) $$(1)/usr/share/znc/webskins/
-	$(CP) -r $$(PKG_BUILD_DIR)/webskins/_default_ $$(1)/usr/share/znc/webskins/
+	$(INSTALL_DIR) $$(1)/opt/lib/znc/
+	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/webadmin.so $$(1)/opt/lib/znc/
+	$(INSTALL_DIR) $$(1)/opt/share/znc/modules
+	$(CP) -r $$(PKG_BUILD_DIR)/modules/data/webadmin $$(1)/opt/share/znc/modules
+	$(INSTALL_DIR) $$(1)/opt/share/znc/webskins/
+	$(CP) -r $$(PKG_BUILD_DIR)/webskins/_default_ $$(1)/opt/share/znc/webskins/
   endef
 
   ZNC_MODULES += znc-mod-webadmin
@@ -124,8 +131,8 @@
   endef
 
   define Package/znc-webskin-$(strip $(1))/install
-	$(INSTALL_DIR) $$(1)/usr/share/znc/webskins/
-	$(CP) -r $$(PKG_BUILD_DIR)/webskins/$(strip $(1)) $$(1)/usr/share/znc/webskins/
+	$(INSTALL_DIR) $$(1)/opt/share/znc/webskins/
+	$(CP) -r $$(PKG_BUILD_DIR)/webskins/$(strip $(1)) $$(1)/opt/share/znc/webskins/
   endef
   ZNC_MODULES += znc-webskin-$(strip $(1))
 endef
@@ -266,8 +273,8 @@
 
 CONFIGURE_VARS += \
 	CXXFLAGS="$(TARGET_CFLAGS) -fno-builtin -fno-rtti -nostdinc++" \
-	CPPFLAGS="-I$(STAGING_DIR)/usr/include -I$(STAGING_DIR)/include" \
-	LDFLAGS="-nodefaultlibs -lc -L$(STAGING_DIR)/usr/lib -L$(STAGING_DIR)/lib" \
+	CPPFLAGS="-I$(STAGING_DIR)/opt/include -I$(STAGING_DIR)/include" \
+	LDFLAGS="-nodefaultlibs -lc -L$(STAGING_DIR)/opt/lib -L$(STAGING_DIR)/lib -Wl,--dynamic-linker=/opt/lib/ld-uClibc.so.0 -Wl,-rpath,/opt/lib" \
 	LIBS="-luClibc++ -lm -lssl -lcrypto $(LIBGCC_S) -lc"
 
 CONFIGURE_ARGS += \
