--- ../../../openwrt_trunk/feeds/oldpackages/net/mini_snmpd/Makefile.orig
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mini_snmpd/Makefile
@@ -37,7 +37,7 @@
 TARGET_CFLAGS+= -DSYSLOG
 
 MAKE_FLAGS+= \
-	OFLAGS="$(TARGET_CFLAGS)" \
+	OFLAGS="$(TARGET_CFLAGS) $(TARGET_LDFLAGS)" \
 	STRIP="/bin/true" \
 	INSTALL_ROOT="$(PKG_INSTALL_DIR)"
 
@@ -46,16 +46,10 @@
 endef
 
 define Package/mini-snmpd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/sbin/mini_snmpd $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/mini_snmpd.config $(1)/etc/config/mini_snmpd
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/mini_snmpd.init $(1)/etc/init.d/mini_snmpd
-endef
-
-define Package/mini-snmpd/conffiles
-/etc/config/mini_snmpd
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/sbin/mini_snmpd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S50mini_snmpd $(1)/opt/etc/init.d/S50mini_snmpd
 endef
 
 $(eval $(call BuildPackage,mini-snmpd))
