--- ../../../downloads/openwrt_trunk-r34120/feeds/oldpackages/utils/bc/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/bc/Makefile
@@ -48,13 +48,13 @@
 CONFIGURE_ARGS += --with-readline
 
 define Package/bc/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bc/bc $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bc/bc $(1)/opt/bin/
 endef
 
 define Package/dc/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dc/dc $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dc/dc $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,bc))
