--- ../../../backup/trunk_initialized/trunk/package/libs/libiconv/Makefile
+++ ../../../openwrt_trunk/package/libs/libiconv/Makefile
@@ -41,19 +41,19 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/lib/libiconv-stub/lib
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/libiconv.a $(1)/usr/lib/libiconv-stub/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/libiconv-stub/lib
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/libiconv.a $(1)/opt/lib/libiconv-stub/lib/
 
-	$(INSTALL_DIR) $(1)/usr/lib/libiconv-stub/include
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/include/iconv.h $(1)/usr/lib/libiconv-stub/include/
+	$(INSTALL_DIR) $(1)/opt/lib/libiconv-stub/include
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/include/iconv.h $(1)/opt/lib/libiconv-stub/include/
 
-	$(INSTALL_DIR) $(1)/usr/share/aclocal
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/m4/* $(1)/usr/share/aclocal/
+	$(INSTALL_DIR) $(1)/opt/share/aclocal
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/m4/* $(1)/opt/share/aclocal/
 endef
 
 define Package/libiconv/install
-	$(INSTALL_DIR) $(1)/tmp
-	touch $(1)/tmp/.libiconv-placeholder
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.libiconv-placeholder
 endef
 
 define Host/Prepare
