--- a/Makefile	2012-02-01 15:12:59.021036276 +0400
+++ b/Makefile	2012-02-01 15:13:32.668614159 +0400
@@ -83,7 +83,7 @@
 
 $(OUTFILE): $(objects) $(subdirs)
 	@echo '*' Linking $(OUTFILE)
-	@$(CC) $(objects) $(subdirobjs) -o $(OUTFILE) $(LFLAGS) $(EFLAGS)
+	@$(CC) $(objects) $(subdirobjs) -o $(OUTFILE) -Wl,-rpath,/opt/lib,--dynamic-linker=/opt/lib/ld-uClibc.so.0 $(LFLAGS) $(EFLAGS)
 ifndef DEBUG
 	@echo '*' Stripping $(OUTFILE)
 	@-$(STRIP) $(OUTFILE)
