--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/ossp-js/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/ossp-js/Makefile
@@ -56,21 +56,22 @@
 define Build/Configure
 	(cd $(PKG_BUILD_DIR); rm -f config.cache; \
 		./configure \
-			--prefix=/usr \
-			--exec-prefix=/usr \
-			--bindir=/usr/bin \
-			--datadir=/usr/share \
-			--includedir=/usr/include \
-			--infodir=/usr/share/info \
-			--libdir=/usr/lib \
-			--libexecdir=/usr/lib \
-			--localstatedir=/var \
-			--mandir=/usr/share/man \
-			--sbindir=/usr/sbin \
-			--sysconfdir=/etc \
+			--prefix=/opt \
+			--exec-prefix=/opt \
+			--bindir=/opt/bin \
+			--datadir=/opt/share \
+			--includedir=/opt/include \
+			--infodir=/opt/share/info \
+			--libdir=/opt/lib \
+			--libexecdir=/opt/lib \
+			--localstatedir=/opt/var \
+			--mandir=/opt/share/man \
+			--sbindir=/opt/sbin \
+			--sysconfdir=/opt/etc \
 			$(DISABLE_NLS) \
 			--disable-shared \
 			--enable-static \
+			ac_cv_va_copy=C99 \
 	);
 	$(MAKE) -C $(PKG_BUILD_DIR) src/jsautocfg.h
 	$(call Build/Configure/Default)
@@ -79,40 +80,40 @@
 define Build/InstallDev
 	$(INSTALL_DIR) $(2)/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/js-config \
+		$(PKG_INSTALL_DIR)/opt/bin/js-config \
 		$(2)/bin/
 	$(SED) \
-		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/usr,g' \
+		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/opt,g' \
 		$(2)/bin/js-config
 
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/include
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/js \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/js \
+		$(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libjs.{la,so*} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libjs.{la,so*} \
+		$(1)/opt/lib/
 
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/js.pc \
-		$(1)/usr/lib/pkgconfig/
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/js.pc \
+		$(1)/opt/lib/pkgconfig/
 endef
 
 define Package/js/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/js \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/js \
+		$(1)/opt/bin/
 endef
 
 define Package/libjs/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libjs.so.* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libjs.so.* \
+		$(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,js))
