--- a/src/Makefile
+++ b/src/Makefile
@@ -164,7 +164,7 @@
 	$(CC) $(CFLAGS) $(OBJS_IV) -o $(@) $(LIBSSL)
 
 kstats$(EXE): kstats.o
-	$(CC) $(CFLAGS) kstats.o -o $(@)
+	$(CC) $(CFLAGS) $(LDFLAGS) kstats.o -o $(@)
 
 wesside-ng$(EXE): $(OBJS_WS) $(LIBOSD)
 	$(CC) $(CFLAGS) $(OBJS_WS) -o $(@) $(LIBS) $(LIBSSL) -lz
@@ -173,10 +173,10 @@
 	$(CC) $(CFLAGS) $(OBJS_ES) -o $(@) $(LIBS) -lz
 
 buddy-ng$(EXE): $(OBJS_BUDDY)
-	$(CC) $(CFLAGS) $(OBJS_BUDDY) -o $(@)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_BUDDY) -o $(@)
 
 makeivs-ng$(EXE): $(OBJS_MI)
-	$(CC) $(CFLAGS) $(OBJS_MI) -o $(@)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(OBJS_MI) -o $(@)
 
 airolib-ng$(EXE): $(OBJS_AL)
 	$(CC) $(CFLAGS) $(OBJS_AL) -o $(@) $(LIBSSL) -DHAVE_REGEXP $(LIBSQL)
@@ -185,7 +185,7 @@
 	$(CC) $(CFLAGS) $(OBJS_AB) -o $(@) $(LIBS) $(LIBSSL) -lpthread
 
 airdecloak-ng$(EXE): $(OBJS_AU)
-	$(CC) $(CFLAGS) $(OBJS_AU) -o $(@)
+	$(CC) $(CFLAGS)  $(LDFLAGS) $(OBJS_AU) -o $(@)
 
 tkiptun-ng$(EXE): $(OBJS_TT) $(LIBOSD)
 	$(CC) $(CFLAGS) $(OBJS_TT) -o $(@) $(LIBS) $(LIBSSL)
