--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/aircrack-ng/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/aircrack-ng/Makefile
@@ -38,15 +38,15 @@
 	CFLAGS="$(TARGET_CFLAGS) -Wall -Iinclude/ $(TARGET_CPPFLAGS) -D_REVISION=0" \
 	LDFLAGS="$(TARGET_LDFLAGS)" \
 	OSNAME=Linux \
-	prefix="/usr" \
+	prefix="/opt" \
 	sqlite="false" \
 	unstable="true"
 
 define Package/aircrack-ng/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/* $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(CP) $(PKG_INSTALL_DIR)/opt/sbin/* $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,aircrack-ng))
