diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/utils/logrotate/files/logrotate.conf ../../../openwrt_trunk/feeds/oldpackages/utils/logrotate/files/logrotate.conf
--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/logrotate/files/logrotate.conf	2012-01-20 19:21:02.000000000 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/logrotate/files/logrotate.conf	2012-01-20 19:45:23.183130786 +0400
@@ -10,7 +10,7 @@
 
 notifempty
 nomail
-#olddir /var/log/backup/
+#olddir /opt/var/log/backup/
 missingok
 #dateext
 
@@ -18,7 +18,7 @@
 #compress
 
 # packages can drop log rotation information into this directory
-include /etc/logrotate.d
+include /opt/etc/logrotate.d
 
 # no packages own lastlog or wtmp -- we'll rotate them here
 #/var/log/wtmp {
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/utils/logrotate/Makefile ../../../openwrt_trunk/feeds/oldpackages/utils/logrotate/Makefile
--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/logrotate/Makefile	2012-01-20 19:21:02.000000000 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/logrotate/Makefile	2012-01-21 13:15:38.404420293 +0400
@@ -37,7 +37,7 @@
 endef
 
 define Package/logrotate/conffiles
-/etc/logrotate.conf
+/opt/etc/logrotate.conf
 endef
 
 EXTRA_CFLAGS += $(TARGET_CPPFLAGS)
@@ -47,16 +47,21 @@
 	$(call Build/Compile/Default, \
 		RPM_OPT_FLAGS:="$(TARGET_CFLAGS) $(EXTRA_CFLAGS)" \
 		LDFLAGS="$(EXTRA_LDFLAGS)" \
+		DEFAULT_MAKE_COMMAND="/usr/sbin/sendmail" \
+		STATEFILE="/opt/var/lib/logrotate.status" \
 		logrotate \
 	)
 endef
 
 define Package/logrotate/install	
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) ${PKG_BUILD_DIR}/logrotate $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/logrotate.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/logrotate.d
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/logrotate $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DIR) $(1)/opt/etc/logrotate.d
+	$(INSTALL_DATA) ./files/logrotate.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/logrotate.d
+	$(INSTALL_DIR) $(1)/opt/var
+	$(INSTALL_DIR) $(1)/opt/var/lib
 endef
 
 $(eval $(call BuildPackage,logrotate))
