--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/libol/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libol/Makefile
@@ -42,28 +42,28 @@
 define Build/InstallDev
 	$(INSTALL_DIR) $(2)/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/libol-config \
+		$(PKG_INSTALL_DIR)/opt/bin/libol-config \
 		$(2)/bin/
 	$(SED) \
-		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/usr,g' \
+		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/opt,g' \
 		$(2)/bin/libol-config
 
-	$(INSTALL_DIR) $(1)/usr/include/libol
+	$(INSTALL_DIR) $(1)/opt/include/libol
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/include/libol/*.h \
-		$(1)/usr/include/libol/
+		$(PKG_INSTALL_DIR)/opt/include/libol/*.h \
+		$(1)/opt/include/libol/
 
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libol.{a,la,so*} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libol.{a,la,so*} \
+		$(1)/opt/lib/
 endef
 
 define Package/libol/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libol.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libol.so* \
+		$(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libol))
