--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/scripts/download.pl	2014-04-16 20:19:01.526912073 +0400
+++ ../../../openwrt_trunk/scripts/download.pl	2014-04-17 20:39:40.477315627 +0400
@@ -190,6 +190,9 @@
 #push @mirrors, 'http://mirror1.openwrt.org';
 push @mirrors, 'http://mirror2.openwrt.org/sources';
 push @mirrors, 'http://downloads.openwrt.org/sources';
+push @mirrors, 'http://x.vm0.ru/wl500g-repo/sources-mirror';
+push @mirrors, 'http://entware.wl500g.info/sources';
+push @mirrors, 'http://entware.dyndns.info/sources';
 
 while (!$ok) {
 	my $mirror = shift @mirrors;
