--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r38784/feeds/oldpackages/net/dsniff/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/dsniff/Makefile
@@ -22,7 +22,7 @@
 define Package/dsniff
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libpcap +libnids +libnet0 +libopenssl +libgdbm $(LIBRPC_DEPENDS)
+  DEPENDS:=+libpcap +libnids +libnet0 +libopenssl +libgdbm
   TITLE:=A collection of tools for network auditing and penetration testing
   URL:=http://www.monkey.org/~dugsong/dsniff/
 endef
@@ -31,15 +31,13 @@
 	A collection of tools for network auditing and penetration testing.
 endef
 
-TARGET_LDFLAGS += $(LIBRPC)
-
 define Build/Configure
 	$(call Build/Configure/Default, \
-		--with-gdbm="$(STAGING_DIR)/usr" \
-		--with-libnids="$(STAGING_DIR)/usr" \
-		--with-libpcap="$(STAGING_DIR)/usr" \
-		--with-libnet="$(STAGING_DIR)/usr/lib/libnet-1.0.x" \
-		--with-openssl="$(STAGING_DIR)/usr" \
+		--with-gdbm="$(STAGING_DIR)/opt" \
+		--with-libnids="$(STAGING_DIR)/opt" \
+		--with-libpcap="$(STAGING_DIR)/opt" \
+		--with-libnet="$(STAGING_DIR)/opt/lib/libnet-1.0.x" \
+		--with-openssl="$(STAGING_DIR)/opt" \
 		--without-db \
 		--without-x \
 	)
@@ -53,10 +51,10 @@
 endef
 
 define Package/dsniff/install	
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/* $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP)  $(PKG_INSTALL_DIR)/usr/lib/* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/* $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP)  $(PKG_INSTALL_DIR)/opt/lib/* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,dsniff))
