--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/libnet-1.0.x/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libnet-1.0.x/Makefile
@@ -58,17 +58,17 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/lib/libnet-1.0.x/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/libnet-config $(1)/usr/lib/libnet-1.0.x/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/libnet-1.0.x/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libnet* $(1)/usr/lib/libnet-1.0.x/include/
-	$(INSTALL_DIR) $(1)/usr/lib/libnet-1.0.x/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libnet.{a,so*} $(1)/usr/lib/libnet-1.0.x/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/libnet-1.0.x/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/libnet-config $(1)/opt/lib/libnet-1.0.x/bin/
+	$(INSTALL_DIR) $(1)/opt/lib/libnet-1.0.x/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libnet* $(1)/opt/lib/libnet-1.0.x/include/
+	$(INSTALL_DIR) $(1)/opt/lib/libnet-1.0.x/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libnet.{a,so*} $(1)/opt/lib/libnet-1.0.x/lib/
 endef
 
 define Package/libnet0/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libnet.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libnet.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libnet0))
