--- ../../../downloads/openwrt_trunk-r36151/feeds/packages/net/sslh/Makefile
+++ ../../../openwrt_trunk/feeds/packages/net/sslh/Makefile
@@ -29,25 +29,24 @@
 endef
 
 define Package/sslh/conffiles
-/etc/config/sslh
+/opt/etc/config/sslh
 endef
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		CC="$(TARGET_CC)" \
 		CFLAGS="$(TARGET_CFLAGS)" \
+		LDFLAGS="$(TARGET_LDFLAGS)" \
 		USELIBCONFIG= \
 		USELIBWRAP= \
 		all
 endef
 
 define Package/sslh/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sslh-fork $(1)/usr/sbin/sslh
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) files/$(PKG_NAME).init $(1)/etc/init.d/$(PKG_NAME)
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) files/$(PKG_NAME).config $(1)/etc/config/$(PKG_NAME)
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sslh-fork $(1)/opt/sbin/sslh
+	$(INSTALL_DIR) $(1)/opt/etc/init.d/
+	$(INSTALL_BIN) files/S14$(PKG_NAME) $(1)/opt/etc/init.d/
 endef
 
 $(eval $(call BuildPackage,sslh))
