--- ../../../downloads/openwrt_trunk-r46338/feeds/packages/utils/sysstat/Makefile	2015-07-30 13:10:50.538896362 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/sysstat/Makefile	2015-08-10 12:22:45.099175130 +0300
@@ -9,7 +9,7 @@
 
 PKG_NAME:=sysstat
 PKG_VERSION:=11.0.4
-PKG_RELEASE:=2
+PKG_RELEASE:=2a
 
 PKG_LICENSE:=GPL-2.0
 PKG_MAINTAINER:=Marko Ratkaj <marko.ratkaj@sartura.hr>
@@ -35,32 +35,34 @@
 endef
 
 define Package/sysstat/conffiles
-/etc/sysstat/sysstat.ioconf
-/etc/sysstat/sysstat
+/opt/etc/sysstat/sysstat.ioconf
+/opt/etc/sysstat/sysstat
 endef
 
 CONFIGURE_VARS+= \
-	sa_lib_dir="/usr/lib/sysstat" \
-	sa_dir="/var/log/sysstat" \
-	conf_dir="/etc/sysstat"
+	sa_lib_dir="/opt/lib/sysstat" \
+	sa_dir="/opt/var/log/sysstat" \
+	conf_dir="/opt/etc/sysstat"
 
 CONFIGURE_ARGS+= \
 	--disable-documentation \
 	--disable-largefile \
 	--disable-sensors \
+	--disable-sensors \
 	--disable-nls
 
+TARGET_CFLAGS += $(TARGET_LDFLAGS)
+
 define Package/sysstat/install
-	$(INSTALL_DIR) $(1)/usr/lib/sysstat
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/sysstat/{sadc,sa1,sa2} $(1)/usr/lib/sysstat/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/{sar,sadf,iostat,mpstat,pidstat} $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/sysstat
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/sysstat/sysstat.ioconf $(1)/etc/sysstat/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/sysstat.config $(1)/etc/config/sysstat
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/sysstat.init $(1)/etc/init.d/sysstat
+	$(INSTALL_DIR) $(1)/opt/lib/sysstat
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/sysstat/{sadc,sa1,sa2} $(1)/opt/lib/sysstat/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/{sar,sadf,iostat,mpstat,pidstat} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/sysstat
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/sysstat/sysstat.ioconf $(1)/opt/etc/sysstat/
+	$(INSTALL_DIR) $(1)/opt/var/log/sysstat
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S99sysstat $(1)/opt/etc/init.d/
 endef
 
 $(eval $(call BuildPackage,sysstat))
