--- ../../../backup/trunk_clean/feeds/packages/libs/id3lib/Makefile	2011-11-28 10:57:20.000000000 +0400
+++ ../../../trunk/feeds/packages/libs/id3lib/Makefile	2011-12-02 13:47:27.731896201 +0400
@@ -39,17 +39,17 @@
 	LIBS="-nodefaultlibs -lz -luClibc++ -Wl,-Bstatic,-lstdc++,-Bdynamic -lm" \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/id3 $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/id3.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3.{a,so} $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3-*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/id3 $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/id3.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3.{a,so} $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3-*.so.* $(1)/opt/lib/
 endef
 
 define Package/id3lib/install
-	$(INSTALL_DIR) $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3-*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3-*.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,id3lib))
