--- ../../../openwrt_trunk.orig/feeds/packages/net/tcpdump/Makefile	2011-12-22 10:24:16.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/tcpdump/Makefile	2011-12-22 10:30:23.271379053 +0400
@@ -69,8 +69,8 @@
 
 
 define Package/tcpdump/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/tcpdump $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/tcpdump $(1)/opt/sbin/
 endef
 
 Package/tcpdump-mini/install = $(Package/tcpdump/install)
