--- ../../../backup/trunk_clean/feeds/packages/libs/libvorbisidec/Makefile	2011-12-05 10:46:44.000000000 +0400
+++ ../../../trunk/feeds/packages/libs/libvorbisidec/Makefile	2011-12-09 14:09:34.827499133 +0400
@@ -38,15 +38,15 @@
 CONFIGURE_ARGS += --enable-shared --enable-static
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/tremor $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libvorbisidec.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/tremor $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libvorbisidec.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libvorbisidec/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libvorbisidec.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libvorbisidec.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libvorbisidec))
