--- ../../../backup/trunk_clean/feeds/packages/libs/libsndfile/Makefile	2011-12-05 10:46:44.000000000 +0400
+++ ../../../trunk/feeds/packages/libs/libsndfile/Makefile	2011-12-09 14:05:00.449738263 +0400
@@ -54,8 +54,8 @@
 endef
 
 define Package/libsndfile/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsndfile.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libsndfile.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libsndfile))
