--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/ucl/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/ucl/Makefile
@@ -41,16 +41,16 @@
 	--disable-asm
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/ucl
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/ucl/ucl{,conf}.h $(1)/usr/include/ucl/
+	$(INSTALL_DIR) $(1)/opt/include/ucl
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/ucl/ucl{,conf}.h $(1)/opt/include/ucl/
 
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libucl.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libucl.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libucl/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libucl.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libucl.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libucl))
