diff -u -r a/lib/curl_transport/curlmulti.c b/lib/curl_transport/curlmulti.c
--- a/lib/curl_transport/curlmulti.c	2009-12-31 06:59:35.000000000 +0300
+++ b/lib/curl_transport/curlmulti.c	2011-12-22 14:04:44.481601302 +0400
@@ -19,7 +19,6 @@
 #endif
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 #include <curl/multi.h>
 
diff -u -r a/lib/curl_transport/curltransaction.c b/lib/curl_transport/curltransaction.c
--- a/lib/curl_transport/curltransaction.c	2010-12-30 21:24:56.000000000 +0300
+++ b/lib/curl_transport/curltransaction.c	2011-12-22 14:04:32.725745632 +0400
@@ -17,7 +17,6 @@
 #include "version.h"
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 
 #include "curlversion.h"
diff -u -r a/lib/curl_transport/xmlrpc_curl_transport.c b/lib/curl_transport/xmlrpc_curl_transport.c
--- a/lib/curl_transport/xmlrpc_curl_transport.c	2010-09-26 20:20:55.000000000 +0400
+++ b/lib/curl_transport/xmlrpc_curl_transport.c	2011-12-22 14:04:23.997856096 +0400
@@ -85,7 +85,6 @@
 #include "xmlrpc-c/time_int.h"
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 #include <curl/multi.h>
 
