--- a/util/Assert.c
+++ b/util/Assert.c
@@ -19,10 +19,6 @@
 #include <stdlib.h>
 #include <stdarg.h>
 
-#ifdef __GLIBC__
-#  include <execinfo.h>
-#endif
-
 Gcc_PRINTF(1, 2)
 void Assert_failure(const char* format, ...)
 {
@@ -33,21 +29,6 @@
     va_start(args, format);
     vfprintf(stderr, format, args);
     fflush(stderr);
-
-#ifdef __GLIBC__
-    void *array[20];
-    size_t size = backtrace(array, 20);
-    char **strings = backtrace_symbols(array, size);
-
-    fprintf(stderr, "Backtrace (%zd frames):\n", size);
-    for (size_t i = 0; i < size; i++)
-    {
-        fprintf(stderr, "  %s\n", strings[i]);
-    }
-    fflush(stderr);
-    free(strings);
-
-#endif
     abort();
     va_end(args);
 }
