diff -ur ../../../downloads/openwrt_trunk-r40859/feeds/oldpackages/net/pptpd/files/options.pptpd ../../../openwrt_trunk/feeds/oldpackages/net/pptpd/files/options.pptpd
--- ../../../downloads/openwrt_trunk-r40859/feeds/oldpackages/net/pptpd/files/options.pptpd	2014-05-26 16:48:25.135190400 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/pptpd/files/options.pptpd	2014-05-26 16:48:32.559190265 +0400
@@ -19,4 +19,4 @@
 refuse-pap
 #ms-dns 172.16.1.1
 #plugin radius.so
-#radius-config-file /etc/radius.conf
+#radius-config-file /opt/etc/radius.conf
diff -ur ../../../downloads/openwrt_trunk-r40859/feeds/oldpackages/net/pptpd/files/pptpd.conf ../../../openwrt_trunk/feeds/oldpackages/net/pptpd/files/pptpd.conf
--- ../../../downloads/openwrt_trunk-r40859/feeds/oldpackages/net/pptpd/files/pptpd.conf	2014-05-26 16:48:25.135190400 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/pptpd/files/pptpd.conf	2014-05-26 16:48:32.559190265 +0400
@@ -1,5 +1,5 @@
 #debug
-option /etc/ppp/options.pptpd
+option /opt/etc/ppp/options.pptpd
 speed 115200
 stimeout 10
 #localip & remoteip are not needed, ip management is done by pppd
diff -ur ../../../downloads/openwrt_trunk-r40859/feeds/oldpackages/net/pptpd/Makefile ../../../openwrt_trunk/feeds/oldpackages/net/pptpd/Makefile
--- ../../../downloads/openwrt_trunk-r40859/feeds/oldpackages/net/pptpd/Makefile	2014-05-26 16:48:25.135190400 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/pptpd/Makefile	2014-05-27 10:32:40.198025515 +0400
@@ -15,6 +15,8 @@
 PKG_SOURCE_URL:=@SF/poptop
 PKG_MD5SUM:=36f9f45c6ffa92bc3b6e24ae2d053505
 
+PKG_BUILD_DEPENDS:=ppp
+
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
@@ -22,7 +24,6 @@
 define Package/pptpd
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-ppp +kmod-gre +kmod-mppe +ppp
   TITLE:=PopTop pptp server
   URL:=http://www.poptop.org/
   SUBMENU:=VPN
@@ -35,32 +36,30 @@
 	ac_cv_header_libintl_h=no \
 
 MAKE_FLAGS += \
-	COPTS="$(TARGET_CFLAGS)" \
+	COPTS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/opt/include" \
 	INSTALL="install" \
 
 define Package/pptpd/conffiles
-/etc/pptpd.conf
-/etc/ppp/options.pptpd
-/etc/config/pptpd
+/opt/etc/pptpd.conf
+/opt/etc/ppp/options.pptpd
+/opt/etc/config/pptpd
 endef
 
 define Package/pptpd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/sbin/bcrelay \
-		$(PKG_INSTALL_DIR)/usr/sbin/pptpctrl \
-		$(PKG_INSTALL_DIR)/usr/sbin/pptpd \
-		$(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib/pptpd
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pptpd/* $(1)/usr/lib/pptpd/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/pptpd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/pptpd.init $(1)/etc/init.d/pptpd
-	$(INSTALL_DIR) $(1)/etc/ppp
-	$(INSTALL_DATA) ./files/options.pptpd $(1)/etc/ppp/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/pptpd.config $(1)/etc/config/pptpd
+		$(PKG_INSTALL_DIR)/opt/sbin/bcrelay \
+		$(PKG_INSTALL_DIR)/opt/sbin/pptpctrl \
+		$(PKG_INSTALL_DIR)/opt/sbin/pptpd \
+		$(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib/pptpd
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pptpd/* $(1)/opt/lib/pptpd/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/pptpd.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S20poptop $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/etc/ppp
+	$(INSTALL_DATA) ./files/options.pptpd $(1)/opt/etc/ppp/
 endef
 
 $(eval $(call BuildPackage,pptpd))
