--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/multimedia/tvheadend/Makefile	2014-04-09 16:17:24.318286480 +0400
+++ ../../../openwrt_trunk/feeds/packages/multimedia/tvheadend/Makefile	2014-04-09 16:53:37.394246894 +0400
@@ -8,12 +8,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tvheadend
-PKG_VERSION:=3.2
-PKG_RELEASE:=0.1
+PKG_VERSION:=3.5-20131011
+PKG_RELEASE:=1
 
+PKG_REV:=f5c5ffd779ce6dbd233b8f98927d57244fcaeac5
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://github.com/downloads/tvheadend/tvheadend/
-PKG_MD5SUM:=4f7faa75c4ea24942c5a12a77ebdcdf0
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/tvheadend/tvheadend.git
+PKG_SOURCE_VERSION:=$(PKG_REV)
+
+PKG_INSTALL:=1
 
 PKG_USE_MIPS16:=0
 
@@ -64,17 +69,24 @@
 
 CONFIGURE_ARGS += \
 	--release \
-	--enable-bundle
+	--disable-imagecache \
+	--disable-libav \
+	--disable-zlib \
+
+define Build/Prepare
+	$(call Build/Prepare/Default)
+	$(SED) "s|0.0.0~unknown|$(PKG_VERSION)|g" $(PKG_BUILD_DIR)/support/version
+endef
 
 define Package/tvheadend/install
-	$(INSTALL_DIR) $(1)/etc/tvheadend
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/tvheadend.init $(1)/etc/init.d/tvheadend
-	$(INSTALL_DIR) $(1)/lib/upgrade/keep.d
-	$(INSTALL_DATA) ./files/tvheadend.upgrade $(1)/lib/upgrade/keep.d/tvheadend
-	
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/build.linux/tvheadend $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/{init.d,tvheadend}
+	$(INSTALL_BIN) ./files/S71tvheadend $(1)/opt/etc/init.d/
+	$(INSTALL_CONF) ./files/tv_grab_wl500g_info.{cfg,offset} $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/tvheadend $(1)/opt/bin/
+	$(INSTALL_BIN) ./files/tv_grab_wl500g_info{,_toffset} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share/tvheadend
+	$(CP) -r $(PKG_INSTALL_DIR)/opt/share/tvheadend/* $(1)/opt/share/tvheadend/
 endef
 
 $(eval $(call BuildPackage,tvheadend))
