--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/id3lib/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/id3lib/Makefile
@@ -36,17 +36,17 @@
 	--enable-static
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/id3 $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/id3.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3.{a,so} $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3-*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/id3 $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/id3.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3.{a,so} $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3-*.so.* $(1)/opt/lib/
 endef
 
 define Package/id3lib/install
-	$(INSTALL_DIR) $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3-*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3-*.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,id3lib))
