--- ../../../openwrt_trunk.orig/feeds/packages/libs/id3lib/Makefile	2012-08-06 09:54:12.075011780 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/id3lib/Makefile	2012-08-06 09:55:39.831010184 +0400
@@ -36,17 +36,17 @@
 	--enable-static
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/id3 $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/id3.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3.{a,so} $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3-*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/id3 $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/id3.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3.{a,so} $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3-*.so.* $(1)/opt/lib/
 endef
 
 define Package/id3lib/install
-	$(INSTALL_DIR) $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3-*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3-*.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,id3lib))
