--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/ncdu/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/ncdu/Makefile
@@ -39,8 +39,8 @@
 endef
 
 define Package/ncdu/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ncdu $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ncdu $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,ncdu))
