--- ../../../downloads/openwrt_trunk-r46338/feeds/packages/utils/ncdu/Makefile	2015-07-14 10:55:19.834308174 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/ncdu/Makefile	2015-07-14 10:55:49.970308675 +0300
@@ -42,8 +42,8 @@
 CONFIGURE_ARGS += --with-ncurses
 
 define Package/ncdu/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ncdu $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ncdu $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,ncdu))
