--- ../../../downloads/openwrt_trunk-r44497/feeds/packages/libs/libshout/Makefile	2015-02-20 12:39:13.373134249 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/libshout/Makefile	2015-02-21 17:26:11.798143129 +0300
@@ -45,24 +45,24 @@
 	--enable-static \
 
 CONFIGURE_VARS += \
-	VORBIS_CFLAGS="-I$(STAGING_DIR)/usr/include/tremor/" \
+	VORBIS_CFLAGS="-I$(STAGING_DIR)/opt/include/tremor/" \
 	VORBIS_LIBS="$(TARGET_LDFLAGS) -lvorbis -lvorbisidec" \
 
-TARGET_CFLAGS += $(FPIC) -Wl,-rpath-link="$(STAGING_DIR)/usr/lib"
+TARGET_CFLAGS += $(FPIC) -Wl,-rpath-link="$(STAGING_DIR)/opt/lib"
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/shout $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libshout.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/shout.pc $(1)/usr/lib/pkgconfig/
-	$(SED) 's| -I/usr/include||' $(1)/usr/lib/pkgconfig/shout.pc
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/shout $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libshout.{a,so*} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/shout.pc $(1)/opt/lib/pkgconfig/
+	$(SED) 's| -I/opt/include||' $(1)/opt/lib/pkgconfig/shout.pc
 endef
 
 define Package/libshout/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libshout.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libshout.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libshout))
