--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/libshout/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libshout/Makefile
@@ -20,7 +20,7 @@
 
 include $(INCLUDE_DIR)/package.mk
 
-TARGET_CFLAGS += $(FPIC) -Wl,-rpath-link=$(STAGING_DIR)/usr/lib
+TARGET_CFLAGS += $(FPIC)
 
 define Package/libshout
   SECTION:=libs
@@ -40,7 +40,7 @@
 
 define Build/Configure
 	$(call Build/Configure/Default, \
-		VORBIS_CFLAGS="-I$(STAGING_DIR)/usr/include/tremor/" \
+		VORBIS_CFLAGS="-I$(STAGING_DIR)/opt/include/tremor/" \
 		VORBIS_LIBS="$(TARGET_LDFLAGS) -lvorbis -lvorbisidec" \
 		--enable-shared \
 		--enable-static \
@@ -48,19 +48,19 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/shout
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/shout/shout.h $(1)/usr/include/shout
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libshout.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(SED) 's|/usr/include|$(STAGING_DIR)/usr/include|' \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/shout.pc
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/shout.pc $(1)/usr/lib/pkgconfig/		
+	$(INSTALL_DIR) $(1)/opt/include/shout
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/shout/shout.h $(1)/opt/include/shout
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libshout.{a,so*} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(SED) 's|/opt/include|$(STAGING_DIR)/opt/include|' \
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/shout.pc
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/shout.pc $(1)/opt/lib/pkgconfig/		
 endef
 
 define Package/libshout/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libshout.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libshout.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libshout))
