--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/libs/elfutils/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/elfutils/Makefile
@@ -14,7 +14,6 @@
 PKG_SOURCE_URL:=http://fedorahosted.org/releases/e/l/$(PKG_NAME)/$(PKG_VERSION)
 PKG_MD5SUM:=163a5712b86f6bdfebdf233cc6e2192d
 
-PKG_INSTALL:=1
 PKG_USE_MIPS16:=0
 
 PKG_BUILD_DEPENDS:=USE_UCLIBC:argp-standalone
@@ -25,14 +24,13 @@
 define Package/elfutils/Default
   SECTION:=libs
   CATEGORY:=Libraries
-  DEPENDS:=$(INTL_DEPENDS)
-  TITLE:=ELF manipulation libraries
+  DEPENDS:=+libelf
+  TITLE:=ELF manipulation libraries, dummy package
   URL:=https://fedorahosted.org/elfutils/
 endef
 
 define Package/libdw
   $(call Package/elfutils/Default)
-  DEPENDS:=libelf1 +zlib +libbz2
   TITLE+= (libdw)
 endef
 
@@ -54,22 +52,23 @@
 	--disable-werror \
 	--without-lzma
 
-define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libdw/libdw.{a,so*} $(1)/usr/lib/
-	$(CP) $(PKG_BUILD_DIR)/libelf/libelf.{a,so*} $(1)/usr/lib/
+define Build/Prepare
+endef
+
+define Build/Configure
+endef
+
+define Build/Compile
 endef
 
 define Package/libdw/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libdw/libdw.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.libdw.placeholder
 endef
 
 define Package/libelf1/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libelf/libelf.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.libelf1.placeholder
 endef
 
 $(eval $(call BuildPackage,libdw))
