--- ../../../downloads/openwrt_trunk-q44497/package/libs/elfutils/Makefile	2015-02-25 12:52:24.036486673 +0300
+++ ../../../openwrt_trunk/package/libs/elfutils/Makefile	2015-03-01 08:59:25.079876772 +0300
@@ -69,27 +69,27 @@
 	--without-lzma
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libasm/libasm.{a,so*} $(1)/usr/lib/
-	$(CP) $(PKG_BUILD_DIR)/libdw/libdw.{a,so*} $(1)/usr/lib/
-	$(CP) $(PKG_BUILD_DIR)/libelf/libelf.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/include/* $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libasm/libasm.{a,so*} $(1)/opt/lib/
+	$(CP) $(PKG_BUILD_DIR)/libdw/libdw.{a,so*} $(1)/opt/lib/
+	$(CP) $(PKG_BUILD_DIR)/libelf/libelf.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libasm/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libasm/libasm.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libasm/libasm.so* $(1)/opt/lib/
 endef
 
 define Package/libdw/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libdw/libdw.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libdw/libdw.so* $(1)/opt/lib/
 endef
 
 define Package/libelf1/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libelf/libelf.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libelf/libelf.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libasm))
