diff -urx .svn ../../../downloads/openwrt_trunk-r47008/feeds/packages/libs/libsigc++/Makefile ../../../openwrt_trunk/feeds/packages/libs/libsigc++/Makefile
--- ../../../downloads/openwrt_trunk-r47008/feeds/packages/libs/libsigc++/Makefile	2015-09-21 12:01:28.359056705 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/libsigc++/Makefile	2015-07-30 13:10:52.000000000 +0300
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libsigc++
-PKG_VERSION:=2.5.4
+PKG_VERSION:=2.4.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=@GNOME/libsigc++/2.5
-PKG_MD5SUM:=0b74492da5f640ab69888a984c6520d7
+PKG_SOURCE_URL:=@GNOME/libsigc++/2.4
+PKG_MD5SUM:=55945ba6e1652f89999e910f6b52047c
 PKG_MAINTAINER:=Steven Barth <cyrus@openwrt.org>
 PKG_LICENSE:=LGPL-2.1
 
@@ -46,18 +46,18 @@
 	--enable-static \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/sigc++-2.0 $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsigc-2.0.{a,so*} $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/sigc++-2.0 $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/sigc++-2.0.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/sigc++-2.0 $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libsigc-2.0.{a,so*} $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/sigc++-2.0 $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/sigc++-2.0.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libsigcxx/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsigc-2.0.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libsigc-2.0.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libsigcxx))
