diff -urx .svn ../../../trunk_clean/feeds/packages/net/xinetd/files/xinetd.conf ../../../trunk/feeds/packages/net/xinetd/files/xinetd.conf
--- ../../../trunk_clean/feeds/packages/net/xinetd/files/xinetd.conf	2012-06-01 10:50:38.828227277 +0400
+++ ../../../trunk/feeds/packages/net/xinetd/files/xinetd.conf	2012-06-01 12:04:20.713442017 +0400
@@ -3,4 +3,4 @@
 
 }
 
-includedir /etc/xinetd.d
+includedir /opt/etc/xinetd.d
diff -urx .svn ../../../trunk_clean/feeds/packages/net/xinetd/Makefile ../../../trunk/feeds/packages/net/xinetd/Makefile
--- ../../../trunk_clean/feeds/packages/net/xinetd/Makefile	2012-06-01 10:50:38.824227321 +0400
+++ ../../../trunk/feeds/packages/net/xinetd/Makefile	2012-06-01 19:04:33.710007827 +0400
@@ -35,7 +35,7 @@
 endef
 
 define Package/xinetd/conffiles
-/etc/xinetd.conf
+/opt/etc/xinetd.conf
 endef
 
 TARGET_CFLAGS += -DNO_RPC
@@ -50,13 +50,12 @@
 	ac_cv_header_rpc_rpcent_h=no
 
 define Package/xinetd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/xinetd $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/xinetd.conf $(1)/etc/xinetd.conf
-	$(INSTALL_DIR) $(1)/etc/xinetd.d
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/xinetd.init $(1)/etc/init.d/xinetd
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/xinetd $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_DATA) ./files/xinetd.conf $(1)/opt/etc/xinetd.conf
+	$(INSTALL_DIR) $(1)/opt/etc/xinetd.d
+	$(INSTALL_BIN) ./files/S10xinetd $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,xinetd))
