diff -urx .svn ../../../trunk_clean/feeds/packages/net/xinetd/files/xinetd.conf ../../../trunk/feeds/packages/net/xinetd/files/xinetd.conf
--- ../../../trunk_clean/feeds/packages/net/xinetd/files/xinetd.conf	2012-05-02 16:17:25.310401906 +0400
+++ ../../../trunk/feeds/packages/net/xinetd/files/xinetd.conf	2012-05-02 16:22:08.059334383 +0400
@@ -3,4 +3,4 @@
 
 }
 
-includedir /etc/xinetd.d
+includedir /opt/etc/xinetd.d
diff -urx .svn ../../../trunk_clean/feeds/packages/net/xinetd/Makefile ../../../trunk/feeds/packages/net/xinetd/Makefile
--- ../../../trunk_clean/feeds/packages/net/xinetd/Makefile	2012-05-02 16:17:25.310401906 +0400
+++ ../../../trunk/feeds/packages/net/xinetd/Makefile	2012-05-27 21:43:23.371565799 +0400
@@ -33,7 +33,7 @@
 endef
 
 define Package/xinetd/conffiles
-/etc/xinetd.conf
+/opt/etc/xinetd.conf
 endef
 
 TARGET_CFLAGS += -DNO_RPC
@@ -50,19 +50,18 @@
 endef
 
 define Build/Compile
-	$(MAKE) -C $(PKG_BUILD_DIR) \
+	$(MAKE_VARS) $(MAKE) -C $(PKG_BUILD_DIR) \
 		DESTDIR="$(PKG_INSTALL_DIR)" \
+		$(MAKE_FLAGS) \
 		build install
 endef
 
 define Package/xinetd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/xinetd $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/xinetd.conf $(1)/etc/xinetd.conf
-	$(INSTALL_DIR) $(1)/etc/xinetd.d
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/xinetd.init $(1)/etc/init.d/xinetd
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/xinetd $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/xinetd.conf $(1)/opt/etc/xinetd.conf
+	$(INSTALL_DIR) $(1)/opt/etc/xinetd.d
 endef
 
 $(eval $(call BuildPackage,xinetd))
