--- ../../../downloads/openwrt_trunk-r40423/feeds/oldpackages/net/httping/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/httping/Makefile
@@ -74,8 +74,8 @@
 endif
 
 define Package/httping/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/httping $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/httping $(1)/opt/bin/
 endef
 
 Package/httping-nossl/install = $(Package/httping/install)
