--- ../../../downloads/openwrt_trunk-r35474/package/libs/libusb/Makefile	2013-02-05 09:11:10.371045878 +0400
+++ ../../../openwrt_trunk/package/libs/libusb/Makefile	2013-02-05 09:48:13.679006164 +0400
@@ -42,8 +42,8 @@
 endef
 
 define Package/libusb-1.0/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libusb*.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libusb*.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libusb-1.0))
