--- ../../../downloads/openwrt_trunk-r44486/package/libs/libusb/Makefile	2015-02-18 12:16:57.840306327 +0300
+++ ../../../openwrt_trunk/package/libs/libusb/Makefile	2015-02-18 12:17:29.728305747 +0300
@@ -46,8 +46,8 @@
 endef
 
 define Package/libusb-1.0/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libusb*.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libusb*.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libusb-1.0))
