--- ../../../openwrt_trunk.orig/feeds/packages/utils/shadow/Makefile	2012-08-06 14:41:06.102698213 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/shadow/Makefile	2012-08-22 11:25:28.388577074 +0400
@@ -15,7 +15,7 @@
 PKG_SOURCE_URL:=http://pkg-shadow.alioth.debian.org/releases
 PKG_MD5SUM:=a00449aa439c69287b6d472191dc2247
 
-PKG_FIXUP:=autoreconf
+#PKG_FIXUP:=autoreconf
 
 PKG_INSTALL:=1
 
@@ -61,12 +61,13 @@
 endef
 
 define Package/shadow-common/conffiles
-/etc/login.defs
+/opt/etc/login.defs
 endef
 
 define Package/shadow-common/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/login.defs $(1)/etc/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/login.defs $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/var/log $(1)/opt/var/spool/mail
 endef
 
 
@@ -85,12 +86,12 @@
   endef
 
   define Package/shadow-$(1)/install
-	[ -x "$(PKG_INSTALL_DIR)/usr/sbin/$(1)" ] && { \
-		$(INSTALL_DIR) $$(1)/usr/sbin; \
-		$(CP) $(PKG_INSTALL_DIR)/usr/sbin/$(1) $$(1)/usr/sbin/; \
+	[ -x "$(PKG_INSTALL_DIR)/opt/sbin/$(1)" ] && { \
+		$(INSTALL_DIR) $$(1)/opt/sbin; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/sbin/$(1) $$(1)/opt/sbin/; \
 	} || { \
-		$(INSTALL_DIR) $$(1)/usr/bin; \
-		$(CP) $(PKG_INSTALL_DIR)/usr/bin/$(1) $$(1)/usr/bin/; \
+		$(INSTALL_DIR) $$(1)/opt/bin; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/bin/$(1) $$(1)/opt/bin/; \
 	}
   endef
 
