--- ../../../downloads/openwrt_trunk-r44486/feeds/oldpackages/devel/make/Makefile	2015-02-18 12:16:57.728306330 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/devel/make/Makefile	2015-02-19 14:29:42.682582599 +0300
@@ -20,7 +20,7 @@
 define Package/make
   SECTION:=devel
   CATEGORY:=Development
-  DEPENDS:=+libelf1
+  DEPENDS:=+libelf
   TITLE:=make
   URL:=http://www.gnu.org/software/make/
 endef
@@ -30,8 +30,8 @@
 endef
 
 define Package/make/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/make $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/make $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,make))
