--- ../../../backup/trunk_clean/feeds/packages/utils/screen/Makefile	2011-11-25 09:49:47.224984785 +0400
+++ ../../../trunk/feeds/packages/utils/screen/Makefile	2011-11-25 10:06:31.639943307 +0400
@@ -33,16 +33,16 @@
 
 define Build/Configure
 	$(call Build/Configure/Default,\
-		--with-sys-screenrc=/etc/screenrc,\
+		--with-sys-screenrc=/opt/etc/screenrc,\
 		LIBS="-lcrypt" \
 	)
 endef
 
 define Package/screen/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/screen $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/etc/screenrc $(1)/etc/screenrc
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/screen $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/etc/screenrc $(1)/opt/etc/screenrc
 endef
 
 define Package/screen/conffiles
