--- ./src/libs/zbxcomms/comms.c.orig	2013-11-14 10:40:28.735143496 +0400
+++ ./src/libs/zbxcomms/comms.c	2013-11-14 10:52:25.795131946 +0400
@@ -31,9 +31,7 @@
 #	define ZBX_SOCKLEN_T socklen_t
 #endif
 
-#if !defined(SOCK_CLOEXEC)
 #	define SOCK_CLOEXEC 0	/* SOCK_CLOEXEC is Linux-specific, available since 2.6.23 */
-#endif
 
 /******************************************************************************
  *                                                                            *
