Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_helper_mac80211.c
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_helper_mac80211.c	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_helper_mac80211.c	(revision 28892)
@@ -1,32 +0,0 @@
-#define _GNU_SOURCE
-#include <stdio.h>
-#include <string.h>
-#include <stdlib.h>
-
-int main(int argc, char *argv[]) {
-
-    if(argc == 3) {
-        char *phy = NULL;
-        char *stat = NULL;
-        char *filename = NULL;
-        FILE *f = NULL;
-        phy = basename(argv[1]);
-        stat = basename(argv[2]);
-        if(asprintf(&filename, "/sys/kernel/debug/ieee80211/%s/statistics/%s", phy, stat) > 0)
-            f = fopen(filename, "r");
-
-        if(f != NULL) {
-            char temp[256];
-            while (fgets(temp, 256, f) != NULL)
-                printf("%s",temp);
-
-            fclose(f);
-        }
-        free(filename);
-    } else {
-        fprintf(stderr, "Usage: %s PHY STAT\n",argv[0]);
-        fprintf(stderr, " cat /sys/kernel/debug/ieee80211/PHY/statistics/STAT as root\n");
-        return 1;
-    } 
-    return 0;
-}
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/mac80211
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/mac80211	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/mac80211	(revision 28892)
@@ -1,27 +0,0 @@
-#see http://wiki.openwrt.org/doc/howto/zabbix for ready to use templates
-
-# If you want to know the exact meaning of an UserParameter, you can search in the ieee80211 standard:
-# http://standards.ieee.org/getieee802/download/802.11-2012.pdf
-# exemple: for mac80211.ACKFailureCount search for dot11ACKFailureCount (page 2145)
-
-# mac80211 phy discovery (like 'phy0')
-# exemple: {"data":[{"{#PHY}":"phy0"}]}
-#
-UserParameter=mac80211.phydiscovery,for phy in $(ls /sys/kernel/debug/ieee80211/); do list="$list,"'{"{#PHY}":"'$phy'"}'; done; echo '{"data":['${list#,}']}'
-
-#phy statistics (you need {#PHY} as parameter)
-#
-UserParameter=mac80211.ACKFailureCount[*],zabbix_helper_mac80211 $1 dot11ACKFailureCount
-UserParameter=mac80211.FCSErrorCount[*],zabbix_helper_mac80211 $1 dot11FCSErrorCount
-UserParameter=mac80211.RTSFailureCount[*],zabbix_helper_mac80211 $1 dot11RTSFailureCount
-UserParameter=mac80211.RTSSuccessCount[*],zabbix_helper_mac80211 $1 dot11RTSSuccessCount
-UserParameter=mac80211.FailedCount[*],zabbix_helper_mac80211 $1 failed_count
-UserParameter=mac80211.FrameDuplicateCount[*],zabbix_helper_mac80211 $1 frame_duplicate_count
-UserParameter=mac80211.MulticastReceivedFrameCount[*],zabbix_helper_mac80211 $1 multicast_received_frame_count
-UserParameter=mac80211.MulticastTransmittedFrameCount[*],zabbix_helper_mac80211 $1 multicast_transmitted_frame_count
-UserParameter=mac80211.MultipleRetryCount[*],zabbix_helper_mac80211 $1 multiple_retry_count
-UserParameter=mac80211.ReceivedFragmentCount[*],zabbix_helper_mac80211 $1 received_fragment_count
-UserParameter=mac80211.RetryCount[*],zabbix_helper_mac80211 $1 retry_count
-UserParameter=mac80211.TransmittedFragmentCount[*],zabbix_helper_mac80211 $1 transmitted_fragment_count
-UserParameter=mac80211.TransmittedFrameCount[*],zabbix_helper_mac80211 $1 transmitted_frame_count
-
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/wifi
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/wifi	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/wifi	(revision 28892)
@@ -1,29 +0,0 @@
-#see http://wiki.openwrt.org/doc/howto/zabbix for ready to use templates
-
-# wifi interface discovery
-# exemple: {"data":[{"{#IF}":"wlan0", "{#MODE}":"ap", "{#SSID}":"Openwrt", "{#NET}":"lan", "{#DEV}":"radio0", "{#ENC}":"psk2+ccmp", "{#TYPE}":"mac80211", "{#HWMODE}":"11ng", "{#CHANNEL}":"11", "{#BSSID}":"xx:xx:xx:xx:xx:xx"}]}
-#
-UserParameter=wifi.ifdiscovery,lua -l uci -l iwinfo -e 'x = uci.cursor(nil, "/var/state");list = "{\"data\":[";x:foreach("wireless", "wifi-iface", function(s) list=list.."{\"{#IF}\":\""..s.ifname.."\", \"{#MODE}\":\""..s.mode.."\", \"{#SSID}\":\""..s.ssid.."\", \"{#NET}\":\""..s.network.."\", \"{#DEV}\":\""..s.device.."\", \"{#ENC}\":\""..(s.encryption or "?").."\", \"{#TYPE}\":\""..x:get("wireless",s.device,"type").."\", \"{#HWMODE}\":\""..(x:get("wireless",s.device,"hwmode") or "?").."\", \"{#CHANNEL}\":\""..x:get("wireless",s.device,"channel").."\", \"{#BSSID}\":\""..iwinfo[iwinfo.type(s.ifname)].bssid(s.ifname).."\"}," end); list=string.gsub(list,",$",""); print(list.."]}")'
-
-
-#iwinfo info (you need {#IF} as parameter, like 'wlan0')
-UserParameter=wifi.iwinfo.channel[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].channel('$1'))"
-UserParameter=wifi.iwinfo.frequency[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].frequency('$1'))"
-UserParameter=wifi.iwinfo.txpower[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].txpower('$1'))"
-UserParameter=wifi.iwinfo.bitrate[*],lua -l iwinfo -e "b = iwinfo[iwinfo.type('$1')].bitrate('$1'); print(b or '0')"
-UserParameter=wifi.iwinfo.signal[*],lua -l iwinfo -e "s = iwinfo[iwinfo.type('$1')].signal('$1'); print(s or '-255')"
-UserParameter=wifi.iwinfo.noise[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].noise('$1'))"
-UserParameter=wifi.iwinfo.quality[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].quality('$1'))"
-UserParameter=wifi.iwinfo.quality_max[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].quality_max('$1'))"
-UserParameter=wifi.iwinfo.mode[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].mode('$1'))"
-UserParameter=wifi.iwinfo.ssid[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].ssid('$1'))"
-UserParameter=wifi.iwinfo.bssid[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].bssid('$1'))"
-UserParameter=wifi.iwinfo.country[*],lua -l iwinfo -e "print(iwinfo[iwinfo.type('$1')].country('$1'))"
-UserParameter=wifi.iwinfo.nbusers[*],lua -l iwinfo -e "n = 0; for _,_ in pairs(iwinfo[iwinfo.type('$1')].assoclist('$1')) do n = n + 1 end; print(n)"
-UserParameter=wifi.iwinfo.encryption[*],lua -l iwinfo -e "e = iwinfo[iwinfo.type('$1')].encryption('$1'); print(e and e.description or 'None')"
-UserParameter=wifi.iwinfo.hwmode[*],lua -l iwinfo -e "x=iwinfo[iwinfo.type('$1')].hwmodelist('$1'); print((x.a and 'a' or '')..(x.b and 'b' or '')..(x.g and 'g' or '')..(x.n and 'n' or ''))"
-
-#uci info (you need {#DEV} as parameter, like 'radio0')
-UserParameter=wifi.uci.hwmode[*],lua -l uci -e "x = uci.cursor(nil, '/var/state'); print(x:get('wireless','$1','hwmode'))"
-UserParameter=wifi.uci.channel[*],lua -l uci -e "x = uci.cursor(nil, '/var/state'); print(x:get('wireless','$1','channel'))"
-UserParameter=wifi.uci.txpower[*],lua -l uci -e "x = uci.cursor(nil, '/var/state'); print(x:get('wireless','$1','txpower'))"
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/network
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/network	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/network	(revision 28892)
@@ -1,9 +0,0 @@
-#see http://wiki.openwrt.org/doc/howto/zabbix for ready to use templates
-
-# network interface discovery
-# exemple: {"data":[{"{#IF}":"lo", "{#NET}":"loopback"},{"{#IF}":"br-lan", "{#NET}":"lan"},{"{#IF}":"eth0.1", "{#NET}":"wan"}]}
-#
-UserParameter=netowrt.discovery,lua -l uci -e 'x = uci.cursor(nil, "/var/state");list = "{\"data\":[";x:foreach("network", "interface", function(s) list=list.."{\"{#IF}\":\""..s.ifname.."\", \"{#NET}\":\""..s[".name"].."\"}," end); list=string.gsub(list,",$",""); print(list.."]}")'
-
-
-
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_agentd.init
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_agentd.init	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_agentd.init	(revision 28892)
@@ -3,31 +3,23 @@
 
 START=60
 
-PROG=/usr/sbin/zabbix_agentd
-CONFIG=/etc/zabbix_agentd.conf
-SERVICE_PID_FILE=/var/run/zabbix_agentd.pid
+SERVICE_PID_FILE=/var/run/zabbix/zabbix_agentd.pid
 
 start() {
-	# Sometimes the agentd config was installed in /etc/zabbix/zabbix_agentd.conf
-	[ -f /etc/zabbix/zabbix_agentd.conf ] && mv /etc/zabbix/zabbix_agentd.conf ${CONFIG}
-
-	[ -f ${CONFIG} ] || return 1
-
-	grep -q "^PidFile=${SERVICE_PID_FILE}" ${CONFIG} || {
-		logger -s -t ${CONFIG} -p daemon.error "Only \"PidFile=${SERVICE_PID_FILE}\" supported"
-		return 1
+	[ -f /etc/zabbix/zabbix_agentd.conf ] || return 1
+	user_exists zabbix 53 || user_add zabbix 53
+	group_exists zabbix 53 || group_add zabbix 53
+	[ -d /var/log/zabbix ] || {
+		mkdir -m0755 -p /var/log/zabbix
+		chown zabbix:zabbix /var/log/zabbix
 	}
-
-	grep -q "^AllowRoot=1" ${CONFIG} || {
-		user_exists zabbix 53 || user_add zabbix 53
-		group_exists zabbix 53 || group_add zabbix 53
-		touch ${SERVICE_PID_FILE}
-		chown zabbix:zabbix ${SERVICE_PID_FILE}
+	[ -d /var/run/zabbix ] || {
+		mkdir -m0755 -p /var/run/zabbix
+		chown zabbix:zabbix /var/run/zabbix
 	}
-
-	service_start ${PROG} -c ${CONFIG}
+	service_start /usr/sbin/zabbix_agentd
 }
 
 stop() {
-	service_stop ${PROG}
+	service_stop /usr/sbin/zabbix_agentd
 }
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_agentd.conf
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_agentd.conf	(revision 0)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/files/zabbix_agentd.conf	(revision 28892)
@@ -0,0 +1,85 @@
+# This is config file for zabbix_agentd
+# To get more information about ZABBIX, go http://www.zabbix.com
+
+############ GENERAL PARAMETERS #################
+
+# List of comma delimited IP addresses (or hostnames) of ZABBIX servers. 
+# No spaces allowed. First entry is used for sending active checks.
+# Note that hostnames must resolve hostname->IP address and
+# IP address->hostname.
+
+Server=127.0.0.1
+
+# Server port for sending active checks
+
+#ServerPort=10051
+
+# Unique hostname. Required for active checks.
+
+Hostname=localhost
+
+# Listen port. Default is 10050
+
+#ListenPort=10050
+
+# IP address to bind agent
+# If missing, bind to all available IPs
+
+#ListenIP=127.0.0.1
+
+# Number of pre-forked instances of zabbix_agentd.
+# Default value is 5
+# This parameter must be between 1 and 16
+
+StartAgents=5
+
+# How often refresh list of active checks. 2 minutes by default.
+
+#RefreshActiveChecks=120
+
+# Disable active checks. The agent will work in passive mode listening server.
+
+#DisableActive=1
+
+# Enable remote commands for ZABBIX agent. By default remote commands disabled.
+
+#EnableRemoteCommands=1
+
+# Specifies debug level
+# 0 - debug is not created
+# 1 - critical information
+# 2 - error information
+# 3 - warnings
+# 4 - information (default)
+# 5 - for debugging (produces lots of information)
+
+DebugLevel=3
+
+# Name of PID file
+
+PidFile=/var/run/zabbix/zabbix_agentd.pid
+
+# Name of log file.
+# If not set, syslog will be used
+
+LogFile=/var/log/zabbix/zabbix_agentd.log
+
+# Spend no more than Timeout seconds on processing
+# Must be between 1 and 30
+
+Timeout=3
+
+####### USER-DEFINED MONITORED PARAMETERS #######
+# Format: UserParameter=<key>,<shell command>
+# Note that shell command must not return empty string or EOL only
+#UserParameter=system.test,who|wc -l
+### Set of parameter for monitoring MySQL server (v3.23.42 and later)
+### Change -u<username> and add -p<password> if required
+#UserParameter=mysql.ping,mysqladmin -uroot ping|grep alive|wc -l
+#UserParameter=mysql.uptime,mysqladmin -uroot status|cut -f2 -d":"|cut -f1 -d"T"
+#UserParameter=mysql.threads,mysqladmin -uroot status|cut -f3 -d":"|cut -f1 -d"Q"
+#UserParameter=mysql.questions,mysqladmin -uroot status|cut -f4 -d":"|cut -f1 -d"S"
+#UserParameter=mysql.slowqueries,mysqladmin -uroot status|cut -f5 -d":"|cut -f1 -d"O"
+#UserParameter=mysql.qps,mysqladmin -uroot status|cut -f9 -d":"
+#UserParameter=mysql.version,mysql -V
+
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/002-fix-res_send-on-uclibc.patch
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/002-fix-res_send-on-uclibc.patch	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/002-fix-res_send-on-uclibc.patch	(revision 28892)
@@ -1,35 +0,0 @@
---- a/configure.in	2013-02-14 18:29:28.145419003 +0100
-+++ b/configure.in	2013-02-14 18:28:42.873418092 +0100
-@@ -149,6 +149,10 @@
- 	AC_MSG_ERROR([Unable to do DNS lookups (libresolv check failed)])
- fi
- LIBS="${LIBS} ${RESOLV_LIBS}"
-+AC_SEARCH_LIBS([res_mkquery], [], [AC_DEFINE([HAVE_RES_MKQUERY], 1, [Define if res_mkquery exists])])
-+AC_SEARCH_LIBS([__res_mkquery], [], [AC_DEFINE([HAVE_RES_MKQUERY], 1, [Define if res_mkquery exists])])
-+AC_SEARCH_LIBS([res_send], [], [AC_DEFINE([HAVE_RES_SEND], 1, [Define if res_send exists])])
-+AC_SEARCH_LIBS([__res_send], [], [AC_DEFINE([HAVE_RES_SEND], 1, [Define if res_send exists])]) 
- 
- dnl *****************************************************************
- dnl *                                                               *
---- a/src/libs/zbxsysinfo/common/net.c	2012-06-21 19:49:46.610567432 +0000
-+++ b/src/libs/zbxsysinfo/common/net.c	2012-06-21 19:50:35.816628541 +0000
-@@ -418,6 +418,7 @@
- #else	/* not _WINDOWS */
- 	res_init();	/* initialize always, settings might have changed */
- 
-+#if defined(HAVE_RES_MKQUERY) && defined(HAVE_RES_SEND) 
- 	if (-1 == (res = res_mkquery(QUERY, zone, C_IN, type, NULL, 0, NULL, buf, sizeof(buf))))
- 		return SYSINFO_RET_FAIL;
- 
-@@ -436,6 +437,11 @@
- 	_res.retry = retry;
- 
- 	res = res_send(buf, res, answer.buffer, sizeof(answer.buffer));
-+#else /* defined(HAVE_RES_QUERY) && defined(HAVE_RES_SEND) */
-+	/* retrand and retry are ignored */
-+	if (-1 == (res = res_query(zone, C_IN, type, answer.buffer, sizeof(answer.buffer))))
-+	return SYSINFO_RET_FAIL;
-+#endif 
- 
- 	hp = (HEADER *)answer.buffer;
- 
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/010-change-agentd-config.patch
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/010-change-agentd-config.patch	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/010-change-agentd-config.patch	(revision 28892)
@@ -1,61 +0,0 @@
---- a/conf/zabbix_agentd.conf
-+++ b/conf/zabbix_agentd.conf
-@@ -3,12 +3,8 @@
- 
- ############ GENERAL PARAMETERS #################
- 
--### Option: PidFile
--#	Name of PID file.
--#
--# Mandatory: no
--# Default:
--# PidFile=/tmp/zabbix_agentd.pid
-+# Only /var/run/zabbix_agentd.pid supported
-+PidFile=/var/run/zabbix_agentd.pid
- 
- ### Option: LogFile
- #	Name of log file.
-@@ -18,8 +14,6 @@
- # Default:
- # LogFile=
- 
--LogFile=/tmp/zabbix_agentd.log
--
- ### Option: LogFileSize
- #	Maximum size of log file in MB.
- #	0 - disable automatic log rotation.
-@@ -105,6 +99,7 @@ Server=127.0.0.1
- # Range: 0-100
- # Default:
- # StartAgents=3
-+StartAgents=1
- 
- ##### Active checks related
- 
-@@ -120,8 +115,6 @@ Server=127.0.0.1
- # Default:
- # ServerActive=
- 
--ServerActive=127.0.0.1
--
- ### Option: Hostname
- #	Unique, case sensitive hostname.
- #	Required for active checks and must match hostname as configured on the server.
-@@ -131,8 +124,6 @@ ServerActive=127.0.0.1
- # Default:
- # Hostname=
- 
--Hostname=Zabbix server
--
- ### Option: HostnameItem
- #	Item used for generating Hostname if it is undefined.
- #	Ignored if Hostname is defined.
-@@ -213,7 +204,7 @@ Hostname=Zabbix server
- # Include=
- 
- # Include=/usr/local/etc/zabbix_agentd.userparams.conf
--# Include=/usr/local/etc/zabbix_agentd.conf.d/
-+Include=/etc/zabbix_agentd.conf.d/
- 
- ####### USER-DEFINED MONITORED PARAMETERS #######
- 
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/002-uclibc_loadavg.patch
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/002-uclibc_loadavg.patch	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/002-uclibc_loadavg.patch	(revision 28892)
@@ -1,6 +1,7 @@
---- a/src/libs/zbxsysinfo/linux/cpu.c	2013-02-12 12:27:22.000000000 +0100
-+++ b/src/libs/zbxsysinfo/linux/cpu.c	2013-02-14 18:38:02.517429349 +0100
-@@ -21,6 +21,45 @@
+diff -urN zabbix-1.6/src/libs/zbxsysinfo/linux/cpu.c zabbix-1.6.new/src/libs/zbxsysinfo/linux/cpu.c
+--- zabbix-1.6/src/libs/zbxsysinfo/linux/cpu.c	2008-09-18 22:15:26.000000000 +0200
++++ zabbix-1.6.new/src/libs/zbxsysinfo/linux/cpu.c	2008-09-30 19:32:54.000000000 +0200
+@@ -22,6 +22,45 @@
  #include "sysinfo.h"
  #include "stats.h"
  
@@ -45,4 +46,4 @@
 +
  int	SYSTEM_CPU_NUM(const char *cmd, const char *param, unsigned flags, AGENT_RESULT *result)
  {
- 	char	tmp[16];
+ 	char	mode[128];
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/001-cross_compile.patch
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/001-cross_compile.patch	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/001-cross_compile.patch	(revision 28892)
@@ -1,29 +1,13 @@
---- a/configure.in	2012-06-21 21:32:23.707912790 +0000
-+++ b/configure.in	2012-06-21 21:33:36.213554564 +0000
-@@ -802,25 +802,8 @@
+diff -urN zabbix-1.6/configure zabbix-1.6.new/configure
+--- zabbix-1.6/configure	2008-09-18 22:15:39.000000000 +0200
++++ zabbix-1.6.new/configure	2008-09-30 19:20:43.000000000 +0200
+@@ -4826,8 +4826,7 @@
+   { { echo "$as_me:$LINENO: error: cannot run test program while cross compiling
+ See \`config.log' for more details." >&5
+ echo "$as_me: error: cannot run test program while cross compiling
+-See \`config.log' for more details." >&2;}
+-   { (exit 1); exit 1; }; }
++See \`config.log' for more details." >&2;} }
+ else
+   cat >conftest.$ac_ext <<_ACEOF
  
- dnl Check for %qu format (FreeBSD 4.x)
- dnl FreeBSD 4.x does not support %llu
--AC_MSG_CHECKING(for long long format)
--AC_TRY_RUN(
--[
--#include <sys/types.h>
--int main()
--{
--        uint64_t i;
--
--        sscanf("200000000010020", "%qu", &i);
--
--        if (i == 200000000010020) return 0;
--        else return -1;
--}
--],
--AC_DEFINE(HAVE_LONG_LONG_QU, 1 ,[Define to 1 if format '%qu' exists.])
--AC_MSG_RESULT(yes),
--AC_MSG_RESULT(no))
--
- dnl option -rdynamic is needed for readable backtraces
-+
- AC_MSG_CHECKING(for -rdynamic linking option)
- saved_LDFLAGS="$LDFLAGS"
- LDFLAGS="-rdynamic $LDFLAGS"
Index: ../../../openwrt_trunk/feeds/packages/admin/zabbix/Makefile
===================================================================
--- ../../../openwrt_trunk/feeds/packages/admin/zabbix/Makefile	(revision 36889)
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/Makefile	(revision 28892)
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2013 OpenWrt.org
+# Copyright (C) 2006-2011 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
@@ -8,19 +8,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=zabbix
-PKG_VERSION:=2.0.6
-PKG_RELEASE:=1
+PKG_VERSION:=1.6
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@SF/zabbix
-PKG_MD5SUM:=f7261987731dd74b58cb1da890655ddc
+PKG_MD5SUM:=39d4c871439b1b4f0429964b4abbfc45
 
+PKG_BUILD_DEPENDS:=libsqlite3
 PKG_INSTALL:=1
 
-PKG_FIXUP:=autoreconf
-
 include $(INCLUDE_DIR)/package.mk
-include $(INCLUDE_DIR)/nls.mk
 
 define Package/zabbix/Default
   SECTION:=admin
@@ -28,7 +26,7 @@
   TITLE:=Zabbix
   URL:=http://www.zabbix.com/
   SUBMENU:=zabbix
-  MAINTAINER:=Mirko Vogt <mirko@openwrt.org>
+  DEPENDS:=+libcurl
 endef
 
 define Package/zabbix-agent
@@ -36,213 +34,64 @@
   TITLE+= agent
 endef
 
-define Package/zabbix-agentd
-  $(call Package/zabbix/Default)
-  TITLE+= agentd
-endef
-
-define Package/zabbix-extra-mac80211
-  $(call Package/zabbix/Default)
-  TITLE+= discovery/userparameters for mac80211
-  DEPENDS += +zabbix-agentd @PACKAGE_MAC80211_DEBUGFS @KERNEL_DEBUG_FS
-endef
-
-define Package/zabbix-extra-network
-  $(call Package/zabbix/Default)
-  TITLE+= discovery/userparameters for network
-  DEPENDS += +zabbix-agentd +libuci-lua +lua
-endef
-
-define Package/zabbix-extra-wifi
-  $(call Package/zabbix/Default)
-  TITLE+= discovery/userparameters for wifi
-  DEPENDS += +zabbix-agentd +libiwinfo-lua +libuci-lua +lua
-endef
-
 define Package/zabbix-sender
   $(call Package/zabbix/Default)
   TITLE+= sender
 endef
 
-define Package/zabbix-get
-  $(call Package/zabbix/Default)
-  TITLE+= get
-endef
-
 define Package/zabbix-server
   $(call Package/zabbix/Default)
   TITLE+= server
   DEPENDS += +libsqlite3
 endef
 
-define Package/zabbix-proxy
-  $(call Package/zabbix/Default)
-  TITLE+= proxy
-  DEPENDS += +libsqlite3
-endef
 
-define Package/zabbix-extra-mac80211/description
-An extra package for zabbix-agentd that adds a discovery rule for mac80211 wifi phy and many userparameters.
-It contains an suid helper to allow zabbix-agentd to still run as zabbix user and not as root.
-See http://wiki.openwrt.org/doc/howto/zabbix for ready to use zabbix templates.
-endef
+ifneq ($(SDK),)
+CONFIG_PACKAGE_zabbix-server:=m
+endif
 
-define Package/zabbix-extra-network/description
-An extra package for zabbix-agentd that adds a discovery rule for openwrt network interfaces.
-The idea here is to discover only interfaces listed in /etc/config/network (discover br-lan and not eth0.1 and wlan0)
-See http://wiki.openwrt.org/doc/howto/zabbix for ready to use zabbix templates.
-endef
-
-define Package/zabbix-extra-wifi/description
-An extra package for zabbix-agentd that adds a discovery rule for wifi interfaces and many userparameters.
-As it uses libiwinfo, it works with all wifi devices supported by openwrt.
-See http://wiki.openwrt.org/doc/howto/zabbix for ready to use zabbix templates.
-endef
-
-
-define Package/zabbix-agentd/config
-  select BUSYBOX_CONFIG_HOSTNAME if !PACKAGE_net-tools-hostname
-  select BUSYBOX_CONFIG_UNAME if !PACKAGE_coreutils-uname
-endef
-
 CONFIGURE_ARGS+= \
-	--enable-agent \
+	--bindir="/usr/sbin" \
+	--enable-agent
+
+ifneq ($(CONFIG_PACKAGE_zabbix-server),)
+  CONFIGURE_ARGS+= \
 	--enable-server \
-	--enable-proxy \
-	--disable-java \
 	--with-sqlite3="$(STAGING_DIR)/usr"
+endif
 
 MAKE_FLAGS += ARCH="linux"
 
-define Package/zabbix/install/sbin
-	$(INSTALL_DIR) \
-		$(1)/usr/sbin
-
-	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/sbin/zabbix_$(2) \
-		$(1)/usr/sbin/
-endef
-
-define Package/zabbix/install/bin
-	$(INSTALL_DIR) \
-		$(1)/usr/bin
-
-	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/zabbix_$(2) \
-		$(1)/usr/bin/
-endef
-
-define Package/zabbix/install/etc
-	$(INSTALL_DIR) \
-		$(1)/etc
-
-	$(INSTALL_CONF) \
-		$(PKG_INSTALL_DIR)/etc/zabbix_$(2).conf \
-		$(1)/etc/
-endef
-
-define Package/zabbix/install/init.d
-	$(INSTALL_DIR) \
-		$(1)/etc/init.d
-
-	$(INSTALL_BIN) \
-		./files/zabbix_$(2).init \
-		$(1)/etc/init.d/zabbix_$(2)
-endef
-
-define Package/zabbix/install/zabbix.conf.d
-	$(INSTALL_DIR) \
-		$(1)/etc/zabbix_$(2).conf.d
-
-	$(INSTALL_BIN) \
-		./files/$(3) \
-		$(1)/etc/zabbix_$(2).conf.d/$(3)
-endef
-
 define Package/zabbix-agent/conffiles
-/etc/zabbix_agent.conf
+/etc/zabbix/zabbix_agentd.conf
 endef
-define Package/zabbix-agentd/conffiles
-/etc/zabbix_agentd.conf
-endef
-define Package/zabbix-server/conffiles
-/etc/zabbix_server.conf
-endef
-define Package/zabbix-proxy/conffiles
-/etc/zabbix_proxy.conf
-endef
 
-ifdef CONFIG_PACKAGE_zabbix-extra-mac80211
-define Build/Prepare/zabbix-extra-mac80211
-	mkdir -p $(PKG_BUILD_DIR)/zabbix-extra-mac80211
-	$(CP) ./files/zabbix_helper_mac80211.c $(PKG_BUILD_DIR)/zabbix-extra-mac80211/
-endef
-
-define Build/Compile/zabbix-extra-mac80211
-	$(TARGET_CC) $(TARGET_CFLAGS) $(PKG_BUILD_DIR)/zabbix-extra-mac80211/zabbix_helper_mac80211.c -o $(PKG_BUILD_DIR)/zabbix-extra-mac80211/zabbix_helper_mac80211
-endef
-endif
-
-define Build/Prepare
-	$(call Build/Prepare/Default)
-	$(call Build/Prepare/zabbix-extra-mac80211)
-endef
-
-define Build/Compile
-	$(call Build/Compile/Default)
-	$(call Build/Compile/zabbix-extra-mac80211)
-endef
-
 define Package/zabbix-agent/install
-	$(call Package/zabbix/install/sbin,$(1),agent)
-	$(call Package/zabbix/install/etc,$(1),agent)
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/zabbix_agentd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/etc/zabbix
+	$(INSTALL_CONF) ./files/zabbix_agentd.conf $(1)/etc/zabbix/
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/zabbix_agentd.init $(1)/etc/init.d/zabbix_agentd
 endef
 
-define Package/zabbix-agentd/install
-	$(call Package/zabbix/install/sbin,$(1),agentd)
-	$(call Package/zabbix/install/etc,$(1),agentd)
-	$(call Package/zabbix/install/init.d,$(1),agentd)
-endef
-
-define Package/zabbix-extra-mac80211/install
-	$(call Package/zabbix/install/zabbix.conf.d,$(1),agentd,mac80211)
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/zabbix-extra-mac80211/zabbix_helper_mac80211 $(1)/usr/bin/
-	chmod 4755 $(1)/usr/bin/zabbix_helper_mac80211
-endef
-
-define Package/zabbix-extra-network/install
-	$(call Package/zabbix/install/zabbix.conf.d,$(1),agentd,network)
-endef
-
-define Package/zabbix-extra-wifi/install
-	$(call Package/zabbix/install/zabbix.conf.d,$(1),agentd,wifi)
-endef
-
 define Package/zabbix-sender/install
-	$(call Package/zabbix/install/bin,$(1),sender)
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/zabbix_sender $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/etc/zabbix
 endef
 
-define Package/zabbix-get/install
-	$(call Package/zabbix/install/bin,$(1),get)
+define Package/zabbix-server/conffiles
+/etc/zabbix/zabbix_server.conf
 endef
 
 define Package/zabbix-server/install
-	$(call Package/zabbix/install/sbin,$(1),server)
-	$(call Package/zabbix/install/etc,$(1),server)
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/zabbix_server $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/etc/zabbix
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/misc/conf/zabbix_server.conf $(1)/etc/zabbix/
 endef
 
-define Package/zabbix-proxy/install
-	$(call Package/zabbix/install/sbin,$(1),proxy)
-	$(call Package/zabbix/install/etc,$(1),proxy)
-endef
-
 $(eval $(call BuildPackage,zabbix-agent))
-$(eval $(call BuildPackage,zabbix-agentd))
-$(eval $(call BuildPackage,zabbix-extra-mac80211))
-$(eval $(call BuildPackage,zabbix-extra-network))
-$(eval $(call BuildPackage,zabbix-extra-wifi))
 $(eval $(call BuildPackage,zabbix-sender))
 $(eval $(call BuildPackage,zabbix-server))
-$(eval $(call BuildPackage,zabbix-proxy))
-$(eval $(call BuildPackage,zabbix-get))
