--- ../../../backup/trunk/include/cmake.mk	2011-11-10 14:44:28.000000000 +0400
+++ ../../../trunk/include/cmake.mk	2011-11-15 12:41:28.668795212 +0400
@@ -48,7 +48,7 @@
 			-DCMAKE_FIND_ROOT_PATH_MODE_LIBRARY=ONLY \
 			-DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=ONLY \
 			-DCMAKE_STRIP=: \
-			-DCMAKE_INSTALL_PREFIX=/usr \
+			-DCMAKE_INSTALL_PREFIX=/opt \
 			-DDL_LIBRARY=$(STAGING_DIR) \
 			$(CMAKE_OPTIONS) \
 		. \
