--- ../../../backup/trunk_clean/include/package-defaults.mk	2011-12-05 10:44:32.000000000 +0400
+++ ../../../trunk/include/package-defaults.mk	2011-12-05 14:11:05.658005506 +0400
@@ -55,7 +55,8 @@
   endef
 endif
 
-CONFIGURE_PREFIX:=/usr
+LINUX_DIR:=~/Projects/firmware_r3497/src/linux/linux-2.6.22.19
+CONFIGURE_PREFIX:=/opt
 CONFIGURE_ARGS = \
 		--target=$(GNU_TARGET_NAME) \
 		--host=$(GNU_TARGET_NAME) \
@@ -67,9 +68,9 @@
 		--bindir=$(CONFIGURE_PREFIX)/bin \
 		--sbindir=$(CONFIGURE_PREFIX)/sbin \
 		--libexecdir=$(CONFIGURE_PREFIX)/lib \
-		--sysconfdir=/etc \
+		--sysconfdir=$(CONFIGURE_PREFIX)/etc \
 		--datadir=$(CONFIGURE_PREFIX)/share \
-		--localstatedir=/var \
+		--localstatedir=$(CONFIGURE_PREFIX)/var \
 		--mandir=$(CONFIGURE_PREFIX)/man \
 		--infodir=$(CONFIGURE_PREFIX)/info \
 		$(DISABLE_NLS) \
