--- ../../../openwrt_trunk.orig/feeds/packages/libs/libevent/Makefile	2011-11-28 10:57:21.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libevent/Makefile	2011-11-29 14:35:19.715347712 +0400
@@ -51,15 +51,15 @@
 	CFLAGS="$(TARGET_CFLAGS)" \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.{a,so} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/*.{a,so} $(1)/opt/lib/
 endef
 
 define Package/libevent/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/*.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libevent))
