--- a/Makefile.in	2011-05-28 02:16:58.000000000 +0400
+++ b/Makefile.in	2012-06-16 15:14:18.859680735 +0400
@@ -54,10 +54,10 @@
 	$(INSTALL) -m 755 $(PROJM) @DESTDIR@
 
 $(PROJM): $(chan_donglem_so_OBJS) Makefile
-	$(LD) $(LDFLAGS) $(SOLINK) -o $@ $(chan_donglem_so_OBJS) $(LIBS)
+	$(CC) $(LDFLAGS) $(SOLINK) -o $@ $(chan_donglem_so_OBJS) $(LIBS)
 
 $(PROJS): $(chan_dongles_so_OBJS) Makefile
-	$(LD) $(LDFLAGS) $(SOLINK) -o $@ $(chan_dongles_so_OBJS) $(LIBS)
+	$(CC) $(LDFLAGS) $(SOLINK) -o $@ $(chan_dongles_so_OBJS) $(LIBS)
 	$(CHMOD) 755 $@
 	mv $@ chan_dongle.so
 
@@ -67,15 +67,15 @@
 tests: test/test1 test/parse
 
 test/test1: $(test1_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(test1_OBJS) $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ $(test1_OBJS) $(LIBS)
 
 test/parse: $(parse_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(parse_OBJS) $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ $(parse_OBJS) $(LIBS)
 
 tools: tools/discovery
 
 tools/discovery: $(discovery_OBJS)
-	$(LD) $(LDFLAGS) -o $@ $(discovery_OBJS) $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ $(discovery_OBJS) $(LIBS)
 
 clean:
 	$(RM) $(PROJM) $(PROJS) *.o *.core .*.d autom4te.cache test/test1 test/*.o tools/discovery test/*.o
