--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/libs/openldap/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/openldap/Makefile
@@ -47,7 +47,7 @@
 endef
 
 define Package/libopenldap/conffiles
-/etc/openldap/ldap.conf
+/opt/etc/openldap/ldap.conf
 endef
 
 define Package/openldap-utils
@@ -77,7 +77,7 @@
 endef
 
 define Package/openldap-server/conffiles
-/etc/openldap/slapd.conf
+/opt/etc/openldap/slapd.conf
 endef
 
 TARGET_CFLAGS += $(FPIC) -lpthread
@@ -102,7 +102,8 @@
 	--disable-bdb \
 	--disable-hdb \
 	--disable-monitor \
-	--disable-relay
+	--disable-relay \
+	ac_cv_func_memcmp_working=yes
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
@@ -112,37 +113,38 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/include/{lber,ldap}*.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{lber,ldap}*.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/include/{lber,ldap}*.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{lber,ldap}*.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libopenldap/install
-	$(INSTALL_DIR) $(1)/etc/openldap
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/openldap/ldap.conf $(1)/etc/openldap/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{lber,ldap}*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DIR) $(1)/opt/etc/openldap
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/openldap/ldap.conf $(1)/opt/etc/openldap/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{lber,ldap}*.so.* $(1)/opt/lib/
 endef
 
 define Package/openldap-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ldap* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/ldap* $(1)/opt/bin/
 endef
 
 define Package/openldap-server/install
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/ldap.init $(1)/etc/init.d/ldap
-	$(INSTALL_DIR) $(1)/etc/openldap/schema
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/openldap/schema/* $(1)/etc/openldap/schema/
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/openldap/slapd.conf $(1)/etc/openldap/
-	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/etc/openldap/schema
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/openldap/schema/* $(1)/opt/etc/openldap/schema/
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/openldap/slapd.conf $(1)/opt/etc/openldap/
+	$(INSTALL_DIR) $(1)/opt/sbin
 	# XXX: OpenLDAP installs slapd into libexecdir, not sbindir:
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/slapd $(1)/usr/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/slapd $(1)/opt/sbin/
 	# XXX: switch default backend to ldif, since bdb is disabled
 	$(SED) 's|^\(database\)\([ \t]\+\)bdb|\1\2ldif|g' \
 	    -e 's|^\(index\)|#\1|g' \
-	    $(1)/etc/openldap/slapd.conf
+	    $(1)/opt/etc/openldap/slapd.conf
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S58slapd $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,libopenldap))
