--- ../../../openwrt_trunk.orig/feeds/packages/libs/giflib/Makefile	2012-01-13 12:15:02.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/giflib/Makefile	2012-01-14 17:31:20.984859442 +0400
@@ -41,16 +41,16 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/{lib,include}
-	$(CP) $(PKG_BUILD_DIR)/lib/.libs/lib*so* $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/.libs/libgif.a $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/libgif.la $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/*.h $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/{lib,include}
+	$(CP) $(PKG_BUILD_DIR)/lib/.libs/lib*so* $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/.libs/libgif.a $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/libgif.la $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/*.h $(1)/opt/include/
 endef
 
 define Package/giflib/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/.libs/lib*so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/.libs/lib*so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,giflib))
