--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r38784/feeds/oldpackages/libs/libnids/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libnids/Makefile
@@ -34,8 +34,8 @@
 	$(call Build/Configure/Default, \
 		--enable-shared \
 		--enable-static \
-		--with-libnet="$(STAGING_DIR)/usr/lib/libnet-1.0.x" \
-		--with-libpcap="$(STAGING_DIR)/usr" \
+		--with-libnet="$(STAGING_DIR)/opt/lib/libnet-1.0.x" \
+		--with-libpcap="$(STAGING_DIR)/opt" \
 		--disable-libglib \
 		, \
 		CFLAGS="$$$$CFLAGS $$$$CPPFLAGS" \
@@ -46,22 +46,22 @@
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
-		LNETLIB="-L$(STAGING_DIR)/usr/lib/libnet-1.0.x/lib -lnet" \
+		LNETLIB="-L$(STAGING_DIR)/opt/lib/libnet-1.0.x/lib -lnet" \
 		PCAPLIB="$(TARGET_LDFLAGS) -lpcap" \
 		install_prefix="$(PKG_INSTALL_DIR)" \
 		all install
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/nids.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libnids.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/nids.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libnids.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libnids/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libnids.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libnids.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libnids))
