diff -aur libubox-2014-08-04.old//CMakeLists.txt libubox-2014-08-04.new//CMakeLists.txt
--- libubox-2014-08-04.old//CMakeLists.txt	2014-11-08 08:50:53.000000000 +0300
+++ libubox-2014-08-04.new//CMakeLists.txt	2014-12-01 15:17:35.000000000 +0300
@@ -3,7 +3,7 @@
 INCLUDE(CheckFunctionExists)
 
 PROJECT(ubox C)
-ADD_DEFINITIONS(-Os -Wall -Werror --std=gnu99 -g3 -Wmissing-declarations)
+ADD_DEFINITIONS(-Os -Wall --std=gnu99 -g3 -Wmissing-declarations)
 
 OPTION(BUILD_LUA "build Lua plugin" ON)
 
diff -aur libubox-2014-08-04.old//examples/CMakeLists.txt libubox-2014-08-04.new//examples/CMakeLists.txt
--- libubox-2014-08-04.old//examples/CMakeLists.txt	2014-11-08 08:50:53.000000000 +0300
+++ libubox-2014-08-04.new//examples/CMakeLists.txt	2014-12-01 15:18:05.000000000 +0300
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.6)
 
 PROJECT(ubox-examples C)
-ADD_DEFINITIONS(-O1 -Wall -Werror --std=gnu99 -g3)
+ADD_DEFINITIONS(-O1 -Wall --std=gnu99 -g3)
 
 IF(APPLE)
   INCLUDE_DIRECTORIES(/opt/local/include)
diff -aur libubox-2014-08-04.old//lua/CMakeLists.txt libubox-2014-08-04.new//lua/CMakeLists.txt
--- libubox-2014-08-04.old//lua/CMakeLists.txt	2014-11-08 08:50:53.000000000 +0300
+++ libubox-2014-08-04.new//lua/CMakeLists.txt	2014-12-01 15:17:55.000000000 +0300
@@ -15,7 +15,7 @@
 	ENDIF()
 ENDIF()
 
-ADD_DEFINITIONS(-Os -Wall -Werror --std=gnu99 -g3 -I.. ${LUA_CFLAGS})
+ADD_DEFINITIONS(-Os -Wall --std=gnu99 -g3 -I.. ${LUA_CFLAGS})
 LINK_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/..)
 
 IF(APPLE)
diff -aur libubox-2014-08-04.old//utils.h libubox-2014-08-04.new//utils.h
--- libubox-2014-08-04.old//utils.h	2014-11-08 08:50:53.000000000 +0300
+++ libubox-2014-08-04.new//utils.h	2014-12-01 15:41:08.000000000 +0300
@@ -105,7 +105,7 @@
 	return ((val >> 8) & 0xffu) | ((val & 0xffu) << 8);
 }
 
-#if _GNUC_MIN_VER(4, 2)
+#if _GNUC_MIN_VER(4, 3)
 #define __u_bswap32(x) __builtin_bswap32(x)
 #define __u_bswap64(x) __builtin_bswap64(x)
 #else
