--- ../../../openwrt_trunk.orig/feeds/packages/libs/libusb-1.0/Makefile	2011-11-28 10:57:22.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libusb-1.0/Makefile	2011-11-28 14:46:19.054031697 +0400
@@ -33,19 +33,19 @@
 
 define Build/InstallDev
 	$(MAKE) install
-	$(INSTALL_DIR) $(1)/usr/include
-	$(INSTALL_DIR) $(1)/usr/include/libusb-1.0
-	$(CP) $(PKG_BUILD_DIR)/libusb/libusb.h $(1)/usr/include/libusb-1.0/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libusb/.libs/libusb*.{a,so*} $(1)/usr/lib/
-	$(CP) $(PKG_BUILD_DIR)/libusb/libusb*.la $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_BUILD_DIR)/libusb*.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(INSTALL_DIR) $(1)/opt/include/libusb-1.0
+	$(CP) $(PKG_BUILD_DIR)/libusb/libusb.h $(1)/opt/include/libusb-1.0/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libusb/.libs/libusb*.{a,so*} $(1)/opt/lib/
+	$(CP) $(PKG_BUILD_DIR)/libusb/libusb*.la $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_BUILD_DIR)/libusb*.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libusb-1.0/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libusb/.libs/libusb*.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libusb/.libs/libusb*.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libusb-1.0))
