--- ../../../openwrt_trunk.orig/feeds/packages/libs/libusb-1.0/Makefile	2012-07-10 18:59:22.285231228 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libusb-1.0/Makefile	2012-07-11 12:14:16.539245900 +0400
@@ -36,21 +36,21 @@
 TARGET_CFLAGS += $(FPIC)
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/libusb-1.0
+	$(INSTALL_DIR) $(1)/opt/include/libusb-1.0
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/libusb-1.0/libusb.h \
-		$(1)/usr/include/libusb-1.0/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libusb*.{a,la,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+		$(PKG_INSTALL_DIR)/opt/include/libusb-1.0/libusb.h \
+		$(1)/opt/include/libusb-1.0/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libusb*.{a,la,so*} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libusb*.pc \
-		$(1)/usr/lib/pkgconfig/
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/libusb*.pc \
+		$(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libusb-1.0/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libusb*.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libusb*.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libusb-1.0))
