--- ../../../downloads/openwrt_trunk-q44497/feeds/packages/utils/less/Makefile	2015-03-01 22:34:30.757562210 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/less/Makefile	2015-03-20 07:22:09.765537214 +0300
@@ -9,7 +9,7 @@
 
 PKG_NAME:=less
 PKG_VERSION:=458
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.greenwoodsoftware.com/less
@@ -69,19 +69,19 @@
 endif
 
 define Package/less/install
-	$(INSTALL_DIR) $(1)/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/less $(1)/bin/less
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/less $(1)/opt/bin/less
 endef
 
 define Package/less/postinst
 #!/bin/sh
-[ -L "$${IPKG_INSTROOT}/usr/bin/less" ] && rm -f "$${IPKG_INSTROOT}/usr/bin/less"
+[ -L "$${IPKG_INSTROOT}/opt/bin/less" ] && rm -f "$${IPKG_INSTROOT}/opt/bin/less"
 exit 0
 endef
 
 define Package/less/postrm
 #!/bin/sh
-/bin/busybox less -h 2>&1 | grep -q BusyBox && ln -sf ../../bin/busybox /usr/bin/less
+/Apps/opt/bin/busybox less -h 2>&1 | grep -q BusyBox && ln -sf ../../bin/busybox /Apps/opt/bin/less
 exit 0
 endef
 
