--- ../../../openwrt_trunk.orig/feeds/packages/utils/coreutils/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/coreutils/Makefile
@@ -23,7 +23,7 @@
 
 COREUTILS_APPLETS := \
 	base64 basename cat chcon chgrp chmod chown chroot cksum comm cp csplit \
-	cut date dd dir dircolors dirname du echo env expand expr factor	\
+	cut date dd df dir dircolors dirname du echo env expand expr factor	\
 	false fmt fold groups head hostid id install join kill link ln logname	\
 	ls md5sum mkdir mkfifo mknod mktemp mv nice nl nohup nproc od paste	\
 	pathchk pinky pr printenv printf ptx pwd readlink realpath rm rmdir	\
@@ -74,7 +74,8 @@
 CONFIGURE_VARS += \
 	gl_cv_func_mbrtowc_incomplete_state=yes \
 	gl_cv_func_mbrtowc_retval=yes \
-	gl_cv_func_wcrtomb_retval=yes
+	gl_cv_func_wcrtomb_retval=yes \
+	fu_cv_sys_stat_statfs2_bsize=yes
 
 ifneq ($(CONFIG_USE_UCLIBC),)
   CONFIGURE_VARS += \
@@ -98,8 +99,8 @@
 
 define BuildPlugin
   define Package/$(1)/install
-	$(INSTALL_DIR) $$(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(2) $$(1)/usr/bin/
+	$(INSTALL_DIR) $$(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/$(2) $$(1)/opt/bin/
   endef
 
   $$(eval $$(call BuildPackage,$(1)))
