diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/apccontrol ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/apccontrol
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/apccontrol	2012-05-02 16:17:28.210365448 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/apccontrol	2012-05-02 16:18:17.813741849 +0400
@@ -15,14 +15,14 @@
 #
 # These variables are needed for set up the autoconf other variables.
 #
-prefix=/usr
-exec_prefix=/usr
+prefix=/opt
+exec_prefix=/opt
 
-APCPID=/var/run/apcupsd.pid
-APCUPSD=/usr/sbin/apcupsd
-SHUTDOWN=/sbin/shutdown
+APCPID=/opt/var/run/apcupsd.pid
+APCUPSD=/opt/sbin/apcupsd
+SHUTDOWN=/sbin/halt
 SCRIPTSHELL=/bin/sh
-SCRIPTDIR=/etc/apcupsd
+SCRIPTDIR=/opt/etc/apcupsd
 WALL=true
 
 #
@@ -83,7 +83,7 @@
     ;;
     mainsback)
 	echo "Power has returned on UPS ${2}..." | ${WALL}
-	if [ -f /etc/powerfail ] ; then
+	if [ -f /opt/etc/powerfail ] ; then
 	   printf "Continuing with shutdown."  | ${WALL}
 	fi
     ;;
@@ -124,8 +124,8 @@
 	THEPID=`cat ${APCPID}`
 	kill ${THEPID}
 	rm -f ${APCPID}
-	rm -f /etc/powerfail
-	rm -f /etc/nologin
+	rm -f /opt/etc/powerfail
+	rm -f /opt/etc/nologin
 	sleep 5
 	`${APCUPSD}`
 	echo "apcupsd"
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/apcupsd.conf ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/apcupsd.conf
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/apcupsd.conf	2012-05-02 16:17:28.210365448 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/apcupsd.conf	2012-05-02 16:18:17.813741849 +0400
@@ -78,23 +78,23 @@
 
 # LOCKFILE <path to lockfile>
 #   Path for device lock file. Not used on Win32.
-LOCKFILE /var/lock
+LOCKFILE /opt/var/lock
 
 # SCRIPTDIR <path to script directory>
 #   Directory in which apccontrol and event scripts are located.
-SCRIPTDIR /etc/apcupsd
+SCRIPTDIR /opt/etc/apcupsd
 
 # PWRFAILDIR <path to powerfail directory>
 #   Directory in which to write the powerfail flag file. This file
 #   is created when apcupsd initiates a system shutdown and is
 #   checked in the OS halt scripts to determine if a killpower
 #   (turning off UPS output power) is required.
-PWRFAILDIR /etc
+PWRFAILDIR /opt/etc
 
 # NOLOGINDIR <path to nologin directory>
 #   Directory in which to write the nologin file. The existence
 #   of this flag file tells the OS to disallow new logins.
-NOLOGINDIR /etc
+NOLOGINDIR /opt/etc
 
 
 #
@@ -191,7 +191,7 @@
 
 # If you want the last few EVENTS to be available over the network
 # by the network information server, you must define an EVENTSFILE.
-EVENTSFILE /var/log/apcupsd.events
+EVENTSFILE /opt/var/log/apcupsd.events
 
 # EVENTSFILEMAX <kilobytes>
 #  By default, the size of the EVENTSFILE will be not be allowed to exceed
@@ -231,7 +231,7 @@
 STATTIME 0
 
 # Location of STATUS file (written to only if STATTIME is non-zero)
-STATFILE /var/log/apcupsd.status
+STATFILE /opt/var/log/apcupsd.status
 
 # LOGSTATS [ on | off ] on enables, off disables
 # Note! This generates a lot of output, so if         
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/apcupsd_mail.conf ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/apcupsd_mail.conf
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/apcupsd_mail.conf	2012-05-02 16:17:28.210365448 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/apcupsd_mail.conf	2012-05-02 16:18:17.817741799 +0400
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-MAILPROG="/usr/sbin/smtp"
+MAILPROG="/usr/sbin/sendmail"
 MAILHOST="mail:25"
 FROM="OpenWrt"
 TO="apcups@example.com"
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/changeme ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/changeme
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/changeme	2012-05-02 16:17:28.210365448 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/changeme	2012-05-02 16:18:17.817741799 +0400
@@ -5,13 +5,13 @@
 # detects that the battery should be replaced.
 # We send an email message to root to notify him.
 #
-. /etc/apcupsd/apcupsd_mail.conf
+. /opt/etc/apcupsd/apcupsd_mail.conf
 
 MSG="$HOSTNAME UPS battery needs changing NOW."
 #
 (
    echo "$MSG"
    echo " "
-   /usr/sbin/apcaccess status
+   /opt/sbin/apcaccess status
 ) | $MAILPROG -h $MAILHOST -f $FROM -s "$MSG" $TO
 exit 0
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/commfailure ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/commfailure
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/commfailure	2012-05-02 16:17:28.214365397 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/commfailure	2012-05-02 16:18:17.817741799 +0400
@@ -5,13 +5,13 @@
 # loses contact with the UPS (i.e. the serial connection is not responding).
 # We send an email message to root to notify him.
 #
-. /etc/apcupsd/apcupsd_mail.conf
+. /opt/etc/apcupsd/apcupsd_mail.conf
 
 MSG="$HOSTNAME Communications with UPS lost"
 #
 (
    echo "$MSG"
    echo " "
-   /usr/sbin/apcaccess status
+   /opt/sbin/apcaccess status
 ) | $MAILPROG -h $MAILHOST -f $FROM -s "$MSG" $TO
 exit 0
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/commok ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/commok
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/commok	2012-05-02 16:17:28.210365448 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/commok	2012-05-02 16:18:17.817741799 +0400
@@ -5,13 +5,13 @@
 # restores contact with the UPS (i.e. the serial connection is restored).
 # We send an email message to root to notify him.
 #
-. /etc/apcupsd/apcupsd_mail.conf
+. /opt/etc/apcupsd/apcupsd_mail.conf
 
 MSG="$HOSTNAME Communications with UPS restored"
 #
 (
    echo "$MSG"
    echo " "
-   /usr/sbin/apcaccess status
+   /opt/sbin/apcaccess status
 ) | $MAILPROG -h $MAILHOST -f $FROM -s "$MSG" $TO
 exit 0
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/offbattery ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/offbattery
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/offbattery	2012-05-02 16:17:28.210365448 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/offbattery	2012-05-02 16:18:17.817741799 +0400
@@ -5,13 +5,13 @@
 # UPS goes back on to the mains after a power failure.
 # We send an email message to root to notify him.
 #
-. /etc/apcupsd/apcupsd_mail.conf
+. /opt/etc/apcupsd/apcupsd_mail.conf
 
 MSG="$HOSTNAME Power has returned"
 #
 (
    echo "$MSG"
    echo " "
-   /usr/sbin/apcaccess status
+   /opt/sbin/apcaccess status
 ) | $MAILPROG -h $MAILHOST -f $FROM -s "$MSG" $TO
 exit 0
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/onbattery ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/onbattery
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/files/onbattery	2012-05-02 16:17:28.210365448 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/files/onbattery	2012-05-02 16:18:17.817741799 +0400
@@ -5,13 +5,13 @@
 # goes on batteries.
 # We send an email message to root to notify him.
 #
-. /etc/apcupsd/apcupsd_mail.conf
+. /opt/etc/apcupsd/apcupsd_mail.conf
 
 MSG="$HOSTNAME Power Failure !!!"
 #
 (
    echo "$MSG"
    echo " "
-   /usr/sbin/apcaccess status
+   /opt/sbin/apcaccess status
 ) | $MAILPROG -h $MAILHOST -f $FROM -s "$MSG" $TO
 exit 0
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/Makefile ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/Makefile
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/apcupsd/Makefile	2012-05-02 16:17:28.210365448 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/apcupsd/Makefile	2012-05-31 14:15:46.759572579 +0400
@@ -29,7 +29,7 @@
 	$(CP) $(SCRIPT_DIR)/config.* $(PKG_BUILD_DIR)/autoconf/
 	$(call Build/Configure/Default, \
 		--with-distname=unknown \
-		--sysconfdir=/etc/apcupsd \
+		--sysconfdir=/opt/etc/apcupsd \
 		--enable-usb \
 		--without-x \
 	)
@@ -43,32 +43,33 @@
 endef
 
 define Package/apcupsd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/apcupsd $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/smtp $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/apctest $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/apcaccess $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/apcupsd
-	$(INSTALL_CONF) ./files/apcupsd.conf $(1)/etc/apcupsd/
-	$(INSTALL_CONF) ./files/apcupsd_mail.conf $(1)/etc/apcupsd/
-	$(INSTALL_BIN) ./files/changeme $(1)/etc/apcupsd/
-	$(INSTALL_BIN) ./files/commfailure $(1)/etc/apcupsd/
-	$(INSTALL_BIN) ./files/commok $(1)/etc/apcupsd/
-	$(INSTALL_BIN) ./files/offbattery $(1)/etc/apcupsd/
-	$(INSTALL_BIN) ./files/onbattery $(1)/etc/apcupsd/
-	$(INSTALL_BIN) ./files/apccontrol $(1)/etc/apcupsd/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN)	./files/apcupsd.init $(1)/etc/init.d/apcupsd
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/apcupsd $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/smtp $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/apctest $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/apcaccess $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/apcupsd
+	$(INSTALL_CONF) ./files/apcupsd.conf $(1)/opt/etc/apcupsd/
+	$(INSTALL_CONF) ./files/apcupsd_mail.conf $(1)/opt/etc/apcupsd/
+	$(INSTALL_BIN) ./files/changeme $(1)/opt/etc/apcupsd/
+	$(INSTALL_BIN) ./files/commfailure $(1)/opt/etc/apcupsd/
+	$(INSTALL_BIN) ./files/commok $(1)/opt/etc/apcupsd/
+	$(INSTALL_BIN) ./files/offbattery $(1)/opt/etc/apcupsd/
+	$(INSTALL_BIN) ./files/onbattery $(1)/opt/etc/apcupsd/
+	$(INSTALL_BIN) ./files/apccontrol $(1)/opt/etc/apcupsd/
+	$(INSTALL_DIR) $(1)/opt/var/lock $(1)/opt/var/log
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S17apcupsd $(1)/opt/etc/init.d
 endef
 
 define Package/apcupsd/conffiles
-/etc/apcupsd/apcupsd.conf
-/etc/apcupsd/apcupsd_mail.conf
-/etc/apcupsd/changeme
-/etc/apcupsd/commfailure
-/etc/apcupsd/commok
-/etc/apcupsd/offbattery
-/etc/apcupsd/onbattery
+/opt/etc/apcupsd/apcupsd.conf
+/opt/etc/apcupsd/apcupsd_mail.conf
+/opt/etc/apcupsd/changeme
+/opt/etc/apcupsd/commfailure
+/opt/etc/apcupsd/commok
+/opt/etc/apcupsd/offbattery
+/opt/etc/apcupsd/onbattery
 endef
 
 $(eval $(call BuildPackage,apcupsd))
