--- ../../../trunk_clean/feeds/packages/net/ipset/Makefile	2012-06-01 10:50:37.924238650 +0400
+++ ../../../trunk/feeds/packages/net/ipset/Makefile	2012-06-01 10:51:33.199543808 +0400
@@ -29,7 +29,7 @@
 $(call Package/ipset/Default)
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS+= +iptables-mod-ipset +kmod-ipt-ipset +libmnl
+  DEPENDS+=+libmnl
   TITLE:=IPset administration utility
   URL:=http://ipset.netfilter.org/
 endef
@@ -61,10 +61,10 @@
 endef
 
 define Package/ipset/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/ipset $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libipset*.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(CP) $(PKG_INSTALL_DIR)/opt/sbin/ipset $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libipset*.so* $(1)/opt/lib/
 endef
 
 define KernelPackage/ipt-ipset
