--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/libassuan/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libassuan/Makefile
@@ -40,25 +40,25 @@
 CONFIGURE_ARGS += \
 	--enable-shared \
 	--enable-static \
-	--with-gpg-error-prefix="$(STAGING_DIR)/usr" \
+	--with-gpg-error-prefix="$(STAGING_DIR)/opt" \
 	--without-pth-test \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/assuan.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libassuan.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/share/aclocal
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/aclocal/libassuan.m4 $(1)/usr/share/aclocal/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/libassuan-config $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/assuan.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libassuan.{a,so*} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/share/aclocal
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/aclocal/libassuan.m4 $(1)/opt/share/aclocal/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/libassuan-config $(1)/opt/bin/
 	$(INSTALL_DIR) $(2)/bin
-	$(LN) -nsf ../../usr/bin/libassuan-config $(2)/bin/
+	$(LN) -nsf ../../opt/bin/libassuan-config $(2)/bin/
 endef
 
 define Package/libassuan/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libassuan*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libassuan*.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libassuan))
