--- ../../../openwrt_trunk/feeds/oldpackages/ipv6/wide-dhcpv6/Makefile.orig
+++ ../../../openwrt_trunk/feeds/oldpackages/ipv6/wide-dhcpv6/Makefile
@@ -22,7 +22,6 @@
   CATEGORY:=Network
   TITLE:=WIDE-DHCPv6
   SUBMENU:=wide-dhcpv6
-  DEPENDS:=+kmod-ipv6 
   URL:=https://sourceforge.net/projects/wide-dhcpv6/
   MAINTAINER:=Andrew Byrne <openwrt@andy.id.au>
 endef
@@ -82,7 +81,7 @@
 TARGET_CFLAGS+=-D_GNU_SOURCE
 
 define Build/Configure
-	$(call Build/Configure/Default,--with-localdbdir=/var)
+	$(call Build/Configure/Default,--with-localdbdir=/opt/var)
 endef
 
 define Build/Compile	
@@ -93,50 +92,35 @@
 endef
 
 define Package/wide-dhcpv6-client/conffiles
-/etc/config/dhcp6c
+/opt/etc/config/dhcp6c
 endef
 
 define Package/wide-dhcpv6-client/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dhcp6c $(1)/usr/sbin
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/dhcp6c.config $(1)/etc/config/dhcp6c
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/dhcp6c.init $(1)/etc/init.d/dhcp6c
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	$(INSTALL_DATA) ./files/dhcp6c.hotplug $(1)/etc/hotplug.d/iface/40-dhcp6c
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/dhcp6c
-	$(INSTALL_DATA) ./files/dhcp6c-dnsmasq.hotplug $(1)/etc/hotplug.d/dhcp6c/10-dnsmasq
-	$(INSTALL_DATA) ./files/dhcp6c-radvd.hotplug $(1)/etc/hotplug.d/dhcp6c/20-radvd
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) ./files/dhcp6c-state $(1)/usr/bin/dhcp6c-state
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dhcp6c $(1)/opt/sbin
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) ./files/dhcp6c-state $(1)/opt/bin/dhcp6c-state
 endef
 
 define Package/wide-dhcpv6-server/conffiles
-/etc/config/dhcp6s
+/opt/etc/config/dhcp6s
 endef
 
 define Package/wide-dhcpv6-server/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dhcp6s $(1)/usr/sbin
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/dhcp6s.conf.sample $(1)/etc
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/dhcp6s.config $(1)/etc/config/dhcp6s
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/dhcp6s.init $(1)/etc/init.d/dhcp6s
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	$(INSTALL_DATA) ./files/dhcp6s.hotplug $(1)/etc/hotplug.d/iface/50-dhcp6s
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dhcp6s $(1)/opt/sbin
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/dhcp6s.conf.sample $(1)/opt/etc
 endef
 
 define Package/wide-dhcpv6-relay/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dhcp6relay $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dhcp6relay $(1)/opt/sbin
 endef
 
 define Package/wide-dhcpv6-control/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dhcp6ctl $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dhcp6ctl $(1)/opt/sbin
 endef
 
 $(eval $(call BuildPackage,wide-dhcpv6-client))
