--- a/libpam/pam_private.h
+++ b/libpam/pam_private.h
@@ -24,9 +24,9 @@
 
 /* the Linux-PAM configuration file */
 
-#define PAM_CONFIG    "/etc/pam.conf"
-#define PAM_CONFIG_D  "/etc/pam.d"
-#define PAM_CONFIG_DF "/etc/pam.d/%s"
+#define PAM_CONFIG    "/opt/etc/pam.conf"
+#define PAM_CONFIG_D  "/opt/etc/pam.d"
+#define PAM_CONFIG_DF "/opt/etc/pam.d/%s"
 
 #define PAM_DEFAULT_SERVICE        "other"     /* lower case */
 #define PAM_DEFAULT_SERVICE_FILE   PAM_CONFIG_D "/" PAM_DEFAULT_SERVICE
