--- a/libpam/pam_private.h
+++ b/libpam/pam_private.h
@@ -24,11 +24,11 @@
 
 /* the Linux-PAM configuration file */
 
-#define PAM_CONFIG         "/etc/pam.conf"
-#define PAM_CONFIG_D       "/etc/pam.d"
-#define PAM_CONFIG_DF      "/etc/pam.d/%s"
-#define PAM_CONFIG_DIST_D  "/usr/lib/pam.d"
-#define PAM_CONFIG_DIST_DF "/usr/lib/pam.d/%s"
+#define PAM_CONFIG         "/Apps/opt/etc/pam.conf"
+#define PAM_CONFIG_D       "/Apps/opt/etc/pam.d"
+#define PAM_CONFIG_DF      "/Apps/opt/etc/pam.d/%s"
+#define PAM_CONFIG_DIST_D  "/Apps/opt/lib/pam.d"
+#define PAM_CONFIG_DIST_DF "/Apps/opt/lib/pam.d/%s"
 
 #define PAM_DEFAULT_SERVICE        "other"     /* lower case */
 #define PAM_DEFAULT_SERVICE_FILE   PAM_CONFIG_D "/" PAM_DEFAULT_SERVICE
