diff -urx .svn ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/libs/slang2/files/ncurses5-config ../../../openwrt_trunk/feeds/oldpackages/libs/slang2/files/ncurses5-config
--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/libs/slang2/files/ncurses5-config	2014-04-09 16:17:24.302286481 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/slang2/files/ncurses5-config	2014-04-09 16:53:33.770246961 +0400
@@ -1,6 +1,6 @@
 #!/bin/sh
 
 if [ "$1" == "--terminfo" ]; then
-	echo "/usr/share/terminfo"
+	echo "/opt/share/terminfo"
 fi
 
diff -urx .svn ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/libs/slang2/Makefile ../../../openwrt_trunk/feeds/oldpackages/libs/slang2/Makefile
--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/libs/slang2/Makefile	2014-04-09 16:17:24.302286481 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/slang2/Makefile	2014-04-11 15:14:06.007208548 +0400
@@ -73,24 +73,24 @@
 endef
 
 define Package/libslang2/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libslang.so* $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/slang/ $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libslang.so* $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/slang/ $(1)/opt/lib/
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/*.h $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/*.h $(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libslang.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libslang.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/slsh/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/slsh $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/usr/share
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/slsh/ $(1)/usr/share/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/slsh $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/share
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/slsh/ $(1)/opt/share/
 endef
 
 $(eval $(call BuildPackage,libslang2))
