--- ../../../openwrt_trunk.orig/package/utils/fuse/Makefile
+++ ../../../openwrt_trunk/package/utils/fuse/Makefile
@@ -6,7 +6,6 @@
 #
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=fuse
 PKG_VERSION:=2.9.0
@@ -53,7 +52,7 @@
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE+= (library)
-  DEPENDS:=+kmod-fuse +libpthread
+  DEPENDS:=+libpthread
   SUBMENU:=Filesystem
 endef
 
@@ -77,29 +76,29 @@
 
 define Build/InstallDev
 	@echo "--> Build/InstallDev enter"
-	mkdir -p $(1)/usr/include
-	$(CP)	$(PKG_INSTALL_DIR)/usr/include/{fuse,*.h} $(1)/usr/include/
-	mkdir -p $(1)/usr/lib
-	$(CP)	$(PKG_INSTALL_DIR)/usr/lib/*.{a,so*} $(1)/usr/lib/
-	mkdir -p $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/fuse.pc $(1)/usr/lib/pkgconfig/
-	$(SED) 's,-I$$$${includedir}/fuse,,g' $(1)/usr/lib/pkgconfig/fuse.pc
-	$(SED) 's,-L$$$${libdir},,g' $(1)/usr/lib/pkgconfig/fuse.pc
+	mkdir -p $(1)/opt/include
+	$(CP)	$(PKG_INSTALL_DIR)/opt/include/{fuse,*.h} $(1)/opt/include/
+	mkdir -p $(1)/opt/lib
+	$(CP)	$(PKG_INSTALL_DIR)/opt/lib/*.{a,so*} $(1)/opt/lib/
+	mkdir -p $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/fuse.pc $(1)/opt/lib/pkgconfig/
+	$(SED) 's,-I$$$${includedir}/fuse,,g' $(1)/opt/lib/pkgconfig/fuse.pc
+	$(SED) 's,-L$$$${libdir},,g' $(1)/opt/lib/pkgconfig/fuse.pc
 	@echo "--> Build/InstallDev leave"
 endef
 
 define Package/fuse-utils/install
 	@echo "--> Package/fuse-utils/install enter"
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	# use cp and keep fusermount sticky bit
-	$(FIND) $(PKG_INSTALL_DIR)/usr/bin/ -type f -exec $(CP) -a {} $(1)/usr/bin/ \;
+	$(FIND) $(PKG_INSTALL_DIR)/opt/bin/ -type f -exec $(CP) -a {} $(1)/opt/bin/ \;
 	@echo "--> Package/fuse-utils/install leave"
 endef
 
 define Package/libfuse/install
 	@echo "--> Package/libfuse/install enter"
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/*.so.* $(1)/opt/lib/
 	@echo "--> Package/libfuse/install leave"
 endef
 
