--- a/l2tp.h
+++ b/l2tp.h
@@ -37,14 +37,14 @@
 #include "common.h"
 #include "ipsecmast.h"
 
-#define CONTROL_PIPE "/var/run/xl2tpd/l2tp-control"
+#define CONTROL_PIPE "/opt/tmp/xl2tpd/l2tp-control"
 #define CONTROL_PIPE_MESSAGE_SIZE 1024
 
 #define BINARY "xl2tpd"
 #define SERVER_VERSION "xl2tpd-1.3.6"
 #define VENDOR_NAME "xelerance.com"
 #ifndef PPPD
-#define PPPD		"/usr/sbin/pppd"
+#define PPPD		"/opt/sbin/pppd"
 #endif
 #define CALL_PPP_OPTS "defaultroute"
 #define FIRMWARE_REV	0x0690  /* Revision of our firmware (software, in this case) */
