--- ../../../downloads/openwrt_trunk-r36151/feeds/packages/mail/xmail/Makefile	2013-04-02 14:27:44.953540141 +0400
+++ ../../../openwrt_trunk/feeds/packages/mail/xmail/Makefile	2013-04-30 13:29:59.604773890 +0400
@@ -9,7 +9,7 @@
 
 PKG_NAME:=xmail
 PKG_VERSION:=1.26
-PKG_RELEASE:=3
+PKG_RELEASE:=3a
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.xmailserver.org/
@@ -36,24 +36,18 @@
 	CFLAGS="$(TARGET_CFLAGS)" \
 	$(MAKE) -C $(PKG_BUILD_DIR) -f Makefile.lnx \
 		$(TARGET_CONFIGURE_OPTS) \
+		CPPFLAGS="$(TARGET_CPPFLAGS)" \
+		LDFLAGS="$(TARGET_LDFLAGS) -lpthread -lssl" \
 		LD="$(TARGET_CXX)" \
 		all
 	tar -cjf $(PKG_BUILD_DIR)/MailRoot.tar.bz2 -C $(PKG_BUILD_DIR)/MailRoot --exclude bin .
 endef
 
 define Package/xmail/install
-	$(INSTALL_DIR) $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/* $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/MailRoot.tar.bz2 $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/xmail.config $(1)/etc/config/xmail
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) ./files/xmail.init $(1)/etc/init.d/xmail
-endef
-
-define Package/xmail/conffiles
-/etc/config/xmail
+	$(INSTALL_DIR) $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/* $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/MailRoot.tar.bz2 $(1)/opt/etc/
 endef
 
 $(eval $(call BuildPackage,xmail))
