diff -urx .svn ../../../downloads/mipsel-hardfloat/openwrt_trunk-r39354/package/utils/busybox/Makefile ../../../openwrt_trunk/package/utils/busybox/Makefile
--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r39354/package/utils/busybox/Makefile	2014-01-20 16:36:36.614313258 +0400
+++ ../../../openwrt_trunk/package/utils/busybox/Makefile	2014-01-20 23:11:07.225887179 +0400
@@ -9,7 +9,7 @@
 
 PKG_NAME:=busybox
 PKG_VERSION:=1.19.4
-PKG_RELEASE:=7
+PKG_RELEASE:=7a
 PKG_FLAGS:=essential
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
@@ -54,7 +54,7 @@
 
 LDLIBS:=m crypt
 ifdef CONFIG_BUSYBOX_CONFIG_FEATURE_MOUNT_NFS
-  TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
+  TARGET_CFLAGS += -I$(STAGING_DIR)/opt/include
   export LDFLAGS=$(TARGET_LDFLAGS)
   LDLIBS += rpc
 endif
@@ -90,6 +90,7 @@
 		CROSS_COMPILE="$(TARGET_CROSS)" \
 		KBUILD_HAVE_NLS=no \
 		EXTRA_CFLAGS="$(TARGET_CFLAGS)" \
+		EXTRA_LDFLAGS="$(TARGET_LDFLAGS)" \
 		ARCH="$(ARCH)" \
 		SKIP_STRIP=y \
 		LDLIBS="$(LDLIBS)" \
@@ -101,6 +102,7 @@
 		CC="$(TARGET_CC)" \
 		CROSS_COMPILE="$(TARGET_CROSS)" \
 		EXTRA_CFLAGS="$(TARGET_CFLAGS)" \
+		EXTRA_LDFLAGS="$(TARGET_LDFLAGS)" \
 		ARCH="$(ARCH)" \
 		CONFIG_PREFIX="$(PKG_INSTALL_DIR)" \
 		LDLIBS="$(LDLIBS)" \
@@ -109,12 +111,44 @@
 endef
 
 define Package/busybox/install
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
-	$(INSTALL_BIN) ./files/cron $(1)/etc/init.d/cron
-	$(INSTALL_BIN) ./files/telnet $(1)/etc/init.d/telnet
-	$(INSTALL_BIN) ./files/sysntpd $(1)/etc/init.d/sysntpd
-	-rm -rf $(1)/lib64
+	$(INSTALL_DIR) $(1)/opt/{bin,sbin}
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/busybox $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share
+	(cd $(PKG_INSTALL_DIR); \
+	    find ./opt -lname "*" | sed "s|^./opt/||g" > $(1)/opt/share/busybox_applets.lst.dist)
+endef
+
+define Package/busybox/postinst
+#!/bin/sh
+
+cd /opt
+rm -f share/busybox_applets.lst
+for applet in $$(cat share/busybox_applets.lst.dist)
+do
+    if [ ! -e $$applet ]
+    then
+	ln -s /opt/bin/busybox $$applet
+	echo "$$applet" >> share/busybox_applets.lst
+    else
+	echo "Warning: '/opt/$$applet' is already exists, skipping."
+    fi
+done
+endef
+
+define Package/busybox/postrm
+#!/bin/sh
+
+cd /opt
+for applet in $$(cat share/busybox_applets.lst)
+do
+    if [ -L $$applet ]
+    then
+	rm -f $$applet
+    else
+	echo "Warning: '/opt/$$applet' has been deleted\modified before, skipping."
+    fi
+done
+rm -f share/busybox_applets.lst
 endef
 
 $(eval $(call BuildPackage,busybox))
diff -urx .svn ../../../downloads/mipsel-hardfloat/openwrt_trunk-r39354/package/utils/busybox/patches/200-etc_crontabs.patch ../../../openwrt_trunk/package/utils/busybox/patches/200-etc_crontabs.patch
--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r39354/package/utils/busybox/patches/200-etc_crontabs.patch	2014-01-20 16:36:36.610313259 +0400
+++ ../../../openwrt_trunk/package/utils/busybox/patches/200-etc_crontabs.patch	2014-01-20 16:36:57.998312867 +0400
@@ -5,7 +5,7 @@
  
  #define TMPDIR          CONFIG_FEATURE_CROND_DIR
 -#define CRONTABS        CONFIG_FEATURE_CROND_DIR "/crontabs"
-+#define CRONTABS        "/etc/crontabs"
++#define CRONTABS        "/opt/etc/crontabs"
  #ifndef SENDMAIL
  # define SENDMAIL       "sendmail"
  #endif
@@ -16,7 +16,7 @@
  #include "libbb.h"
  
 -#define CRONTABS        CONFIG_FEATURE_CROND_DIR "/crontabs"
-+#define CRONTABS        "/etc/crontabs"
++#define CRONTABS        "/opt/etc/crontabs"
  #ifndef CRONUPDATE
  #define CRONUPDATE      "cron.update"
  #endif
