--- ../../../downloads/openwrt_trunk-r42803/package/libs/libreadline/Makefile
+++ ../../../openwrt_trunk/package/libs/libreadline/Makefile
@@ -60,15 +60,15 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/readline $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.{a,so,so.6,so.6.3} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/readline $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{history,readline}.{a,so,so.6,so.6.3} $(1)/opt/lib/
 endef
 
 define Package/libreadline/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.{so,so.6,so.6.3} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{history,readline}.{so,so.6,so.6.3} $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libreadline))
