--- ../../../downloads/openwrt_trunk-q44497/feeds/packages/multimedia/minidlna/Makefile	2015-02-25 12:52:24.000426160 +0300
+++ ../../../openwrt_trunk/feeds/packages/multimedia/minidlna/Makefile	2015-02-27 17:23:01.210124069 +0300
@@ -41,15 +41,15 @@
 endef
 
 define Package/minidlna/conffiles
-/etc/config/minidlna
+/opt/etc/minidlna.conf
 endef
 
 TARGET_CPPFLAGS += \
-	-I$(STAGING_DIR)/usr/include \
-	-I$(STAGING_DIR)/usr/include/FLAC \
-	-I$(STAGING_DIR)/usr/include/libexif \
-	-I$(STAGING_DIR)/usr/include/uuid \
-	-I$(STAGING_DIR)/usr/include/vorbis \
+	-I$(STAGING_DIR)/opt/include \
+	-I$(STAGING_DIR)/opt/include/FLAC \
+	-I$(STAGING_DIR)/opt/include/libexif \
+	-I$(STAGING_DIR)/opt/include/uuid \
+	-I$(STAGING_DIR)/opt/include/vorbis \
 	-I$(ICONV_PREFIX)/include \
 	-I$(INTL_PREFIX)/include \
 	-D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
@@ -70,16 +70,16 @@
 	--with-os-name="OpenWrt Linux" \
 	--with-os-version="$(LINUX_VERSION)" \
 	--with-os-url="http://openwrt.org/" \
-	--with-db-path="/var/run/minidlna" \
-	--with-log-path="/var/log" \
+	--with-db-path="/Apps/opt/var/minidlna" \
+	--with-log-path="/Apps/opt/var/minidlna" \
 
 define Package/minidlna/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/minidlnad $(1)/usr/bin/minidlna
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/minidlna.init $(1)/etc/init.d/minidlna
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/minidlna.config $(1)/etc/config/minidlna
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/minidlnad $(1)/opt/bin/minidlna
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S90minidlna $(1)/opt/etc/init.d
+	$(INSTALL_CONF) ./files/minidlna.conf $(1)/opt/etc
+	$(INSTALL_DIR) $(1)/opt/var/minidlna
 endef
 
 $(eval $(call BuildPackage,minidlna))
