--- ../../../openwrt_trunk.orig/feeds/packages/libs/libgpg-error/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/libgpg-error/Makefile
@@ -44,36 +44,36 @@
 	--disable-rpath
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(2)/bin $(1)/usr/bin
+	$(INSTALL_DIR) $(2)/bin $(1)/opt/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/gpg-error-config \
+		$(PKG_INSTALL_DIR)/opt/bin/gpg-error-config \
 		$(2)/bin/
 	$(SED) \
-		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/usr,g' \
+		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/opt,g' \
 		$(2)/bin/gpg-error-config
-	ln -sf $(STAGING_DIR)/host/bin/gpg-error-config $(1)/usr/bin/gpg-error-config
+	ln -sf $(STAGING_DIR)/host/bin/gpg-error-config $(1)/opt/bin/gpg-error-config
 
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/include
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/include/gpg-error.h \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/gpg-error.h \
+		$(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libgpg-error.{la,a,so*} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libgpg-error.{la,a,so*} \
+		$(1)/opt/lib/
 
-	$(INSTALL_DIR) $(1)/usr/share/aclocal
+	$(INSTALL_DIR) $(1)/opt/share/aclocal
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/share/aclocal/gpg-error.m4 \
-		$(1)/usr/share/aclocal/
+		$(PKG_INSTALL_DIR)/opt/share/aclocal/gpg-error.m4 \
+		$(1)/opt/share/aclocal/
 endef
 
 define Package/libgpg-error/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libgpg-error.so.* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libgpg-error.so.* \
+		$(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libgpg-error))
