--- ../../../backup/trunk_clean/feeds/packages/lang/python-imaging-library/Makefile	2012-01-13 12:14:28.000000000 +0400
+++ ../../../trunk/feeds/packages/lang/python-imaging-library/Makefile	2012-01-14 16:30:54.018455319 +0400
@@ -45,7 +45,7 @@
 
 define Build/Compile
 	$(call Build/Compile/PyMod,., \
-		install --prefix="/usr" --root="$(PKG_INSTALL_DIR)", \
+		install --prefix="/opt" --root="$(PKG_INSTALL_DIR)", \
 	)
 endef
 
