--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/utils/procps/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/procps/Makefile
@@ -63,17 +63,17 @@
 	LDFLAGS="$(TARGET_LDFLAGS)" \
 
 define Package/procps/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/proc/libproc-$(PKG_VERSION).so $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/proc/libproc-$(PKG_VERSION).so $(1)/opt/lib/
 endef
 
 AUXDIR_ps := "ps/"
 
 define BuildPlugin
   define Package/$(1)/install
-	$(INSTALL_DIR) $$(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(AUXDIR_$(2))$(2) $$(1)/usr/bin/
+	$(INSTALL_DIR) $$(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(AUXDIR_$(2))$(2) $$(1)/opt/bin/
   endef
 
   $$(eval $$(call BuildPackage,$(1)))
