--- ../../../trunk_clean/feeds/packages/utils/lsof/Makefile	2012-05-02 16:17:34.986280264 +0400
+++ ../../../trunk/feeds/packages/utils/lsof/Makefile	2012-05-28 09:44:37.525258936 +0400
@@ -20,13 +20,13 @@
 
 include $(INCLUDE_DIR)/package.mk
 
-TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
-TARGET_LDFLAGS += $(LIBRPC)
+TARGET_CFLAGS += -I$(STAGING_DIR)/opt/include
+#TARGET_LDFLAGS += $(LIBRPC)
 
 define Package/lsof
   SECTION:=utils
   CATEGORY:=Utilities
-  DEPENDS:=$(LIBRPC_DEPENDS)
+#  DEPENDS:=$(LIBRPC_DEPENDS)
   TITLE:=LiSt Open Files - a diagnostic tool
   URL:=http://people.freebsd.org/~abe/
 endef
@@ -45,12 +45,12 @@
 		AR="$(TARGET_CROSS)ar cr" \
 		RANLIB="$(TARGET_CROSS)ranlib" \
 		DEBUG="$(TARGET_CFLAGS)" \
-		LDFLAGS="$(TARGET_LDFLAGS)" 
+		CFGL="-L./lib -llsof $(TARGET_LDFLAGS)" 
 endef
 
 define Package/lsof/install	
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/lsof $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/lsof $(1)/opt/bin
 endef
 
 $(eval $(call BuildPackage,lsof))
