--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/darkstat/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/darkstat/Makefile
@@ -33,13 +33,13 @@
 over HTTP.
 endef
 
-define Package/darkstat/conffiles
-/etc/config/darkstat
-endef
+#define Package/darkstat/conffiles
+#/opt/etc/config/darkstat
+#endef
 
 CONFIGURE_ARGS += \
 	--disable-debug \
-	--with-chroot-dir=/var/empty
+	--with-chroot-dir=/opt/var/empty
 
 TARGET_CFLAGS += -std=gnu99
 
@@ -50,12 +50,11 @@
 endef
 
 define Package/darkstat/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/darkstat $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/darkstat.init $(1)/etc/init.d/darkstat
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/darkstat.config $(1)/etc/config/darkstat
+	$(INSTALL_DIR) $(1)/opt/var/empty
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/darkstat $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S54darkstat $(1)/opt/etc/init.d/
 endef
 
 $(eval $(call BuildPackage,darkstat))
