--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r39354/feeds/oldpackages/net/nzbget/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/nzbget/Makefile
@@ -8,26 +8,22 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nzbget
-PKG_VERSION:=0.8.0
+PKG_VERSION:=12.0
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=@SF/nzbget
-PKG_MD5SUM:=c47f464fe988deeb4fc5381506c1fbe3
+PKG_SOURCE_URL:=@SF/nzbget/nzbget-stable/$(PKG_VERSION)
+PKG_MD5SUM:=a04e31f4b6aeb3bc14166f9f5751eef3
 
-PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
-TARGET_LDFLAGS+=-Wl,-rpath-link="$(STAGING_DIR)/usr/lib" -lcrypto -lsigc-2.0
-TARGET_CFLAGS += $(FPIC)
-
 define Package/nzbget
   SUBMENU:=NNTP
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libxml2 +libpar2 +libopenssl +libpthread +libncurses
+  DEPENDS:=+libxml2 +libpar2 +libopenssl +libncurses
   URL:=http://nzbget.sourceforge.net/
   TITLE:=Binary newsgrabber for nzb-files
   MAINTAINER:=Artur Wronowski <arteqw@gmail.com>
@@ -37,33 +33,44 @@
 A command-line client/server based binary newsgrabber for nzb-files.
 endef
 
+TARGET_LDFLAGS += \
+	-lsigc-2.0 \
+
 CONFIGURE_ARGS += \
-	--disable-nls \
+	--disable-libpar2-bugfixes-check \
 	--with-tlslib=OpenSSL \
-	--with-libpar2-includes=$(STAGING_DIR)/usr/include/libpar2 \
-	--with-libpar2-libraries=$(STAGING_DIR)/usr/lib \
-	--with-libsigc-includes=$(STAGING_DIR)/usr/include/sigc++-2.0 \
-	--with-libsigc-libraries=$(STAGING_DIR)/usr/lib \
-	--with-openssl-includes=$(STAGING_DIR)/usr/include \
-	--with-openssl-libraries=$(STAGING_DIR)/usr/lib \
-	--with-libcurses-includes=$(STAGING_DIR)/usr/include \
-	--with-libcurses-libraries=$(STAGING_DIR)/usr/lib \
-	, \
-	PKG_CONFIG_PATH="$(STAGING_DIR)/usr/lib/pkgconfig"
-
-CONFIGURE_VARS += \
-	ac_cv_header_regex_h=no \
-	CPPFLAGS="-I$(STAGING_DIR)/usr/include/libxml2 $$$$CPPFLAGS"
+	--with-libpar2-includes=$(STAGING_DIR)/opt/include/libpar2 \
+	--with-libpar2-libraries=$(STAGING_DIR)/opt/lib \
+	--with-libsigc-includes=$(STAGING_DIR)/opt/include/sigc++-2.0 \
+	--with-libsigc-libraries=$(STAGING_DIR)/opt/lib \
+	--with-openssl-includes=$(STAGING_DIR)/opt/include \
+	--with-openssl-libraries=$(STAGING_DIR)/opt/lib \
+	--with-libcurses-includes=$(STAGING_DIR)/opt/include \
+	--with-libcurses-libraries=$(STAGING_DIR)/opt/lib \
+	--with-zlib-includes=$(STAGING_DIR)/opt/include \
+	--with-zlib-libraries=$(STAGING_DIR)/opt/lib \
+
+define Build/Configure
+	$(call Build/Configure/Default)
+	$(SED) 's|-I/usr/include||g' $(PKG_BUILD_DIR)/Makefile
+endef
+
 
 define Package/nzbget/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/nzbget $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/nzbget.conf.example $(1)/etc/nzbget.conf
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/nzbget* $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/nzbgetd $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S75nzbget $(1)/opt/etc/init.d/
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/share/nzbget/nzbget*.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/share/nzbget
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/nzbget/* $(1)/opt/share/nzbget/
 endef
 
 define Package/nzbget/conffiles
-/etc/nzbget.conf
+/opt/etc/nzbget.conf
+/opt/etc/nzbget-postprocess.conf
 endef
 
 $(eval $(call BuildPackage,nzbget))
