--- ../../../downloads/openwrt_trunk-r44497/feeds/oldpackages/utils/dstat/Makefile	2015-02-21 21:16:28.053891450 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/dstat/Makefile	2015-02-23 20:10:05.618809695 +0300
@@ -9,7 +9,7 @@
 
 PKG_NAME:=dstat
 PKG_VERSION:=0.7.2
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=http://dag.wieers.com/home-made/dstat/
@@ -37,11 +37,12 @@
 endef
 
 define Package/dstat/install
-	$(INSTALL_DIR) $(1)/usr/share/dstat
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dstat $(1)/usr/share/dstat/dstat.py
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/plugins/dstat_*.py $(1)/usr/share/dstat/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(LN) -sf ../share/dstat/dstat.py $(1)/usr/bin/dstat
+	$(INSTALL_DIR) $(1)/opt/share/dstat
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dstat $(1)/opt/share/dstat/dstat.py
+	$(SED) 's!/usr/share/dstat/!/opt/share/dstat/!' $(1)/opt/share/dstat/dstat.py
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/plugins/dstat_*.py $(1)/opt/share/dstat/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(LN) -sf ../share/dstat/dstat.py $(1)/opt/bin/dstat
 endef
 
 $(eval $(call BuildPackage,dstat))
