--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/dstat/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/dstat/Makefile
@@ -39,7 +39,7 @@
 define Package/dstat/install
-	$(INSTALL_DIR) $(1)/usr/share/dstat
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dstat $(1)/usr/share/dstat/dstat.py
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/plugins/dstat_*.py $(1)/usr/share/dstat/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(LN) -sf ../share/dstat/dstat.py $(1)/usr/bin/dstat
+	$(INSTALL_DIR) $(1)/opt/share/dstat
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dstat $(1)/opt/share/dstat/dstat.py
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/plugins/dstat_*.py $(1)/opt/share/dstat/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(LN) -sf ../share/dstat/dstat.py $(1)/opt/bin/dstat
 endef
