--- ../../../downloads/openwrt_trunk-r40423/feeds/oldpackages/net/srelay/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/srelay/Makefile
@@ -27,23 +27,23 @@
 endef
 
 define Package/srelay/conffiles
-/etc/srelay.conf
+/opt/etc/srelay.conf
 endef
 
 CONFIGURE_ARGS += \
 	--disable-thread \
-	--with-libwrap="$(STAGING_DIR)/usr" \
+	--with-libwrap="$(STAGING_DIR)/opt" \
 
 CONFIGURE_VARS += \
 	CPPFLAGS="-DLINUX $$$$CPPFLAGS" \
 
 define Package/srelay/install	
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(PKG_NAME) $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) files/$(PKG_NAME).conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) files/$(PKG_NAME).init $(1)/etc/init.d/$(PKG_NAME)
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(PKG_NAME) $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) files/$(PKG_NAME).conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) files/S26$(PKG_NAME) $(1)/opt/etc/init.d/
 endef
 
 $(eval $(call BuildPackage,srelay))
