--- ../../../openwrt_trunk.orig/feeds/oldpackages/lang/pybluez/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/pybluez/Makefile
@@ -39,7 +39,7 @@
 define Build/Compile
 	$(if $(Build/Compile/PyMod),,@echo Python packaging code not found.; false)
 	$(call Build/Compile/PyMod,., \
-		install --prefix="$(PKG_INSTALL_DIR)/usr", \
+		install --prefix="$(PKG_INSTALL_DIR)/opt", \
 	)
 endef
 
