--- ../../../downloads/openwrt_trunk-r45513/feeds/oldpackages/lang/pybluez/Makefile	2015-04-21 10:37:38.280588031 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/pybluez/Makefile	2015-04-22 13:37:48.142816557 +0300
@@ -26,7 +26,7 @@
   CATEGORY:=Languages
   TITLE:=Python wrapper for the BlueZ Bluetooth stack
   URL:=http://code.google.com/p/pybluez/
-  DEPENDS:=+python-mini +bluez-libs
+  DEPENDS:=+python +bluez3-libs
 endef
 
 define Package/python-psycopg/description
@@ -39,7 +39,7 @@
 define Build/Compile
 	$(if $(Build/Compile/PyMod),,@echo Python packaging code not found.; false)
 	$(call Build/Compile/PyMod,., \
-		install --prefix="$(PKG_INSTALL_DIR)/usr", \
+		install --prefix="$(PKG_INSTALL_DIR)/opt", \
 	)
 endef
 
