diff -r -u -x .svn ../../../backup/trunk_clean/trunk/feeds/packages/libs/gettext-full//Makefile ../../../trunk/feeds/packages/libs/gettext-full//Makefile
--- ../../../backup/trunk_clean/trunk/feeds/packages/libs/gettext-full//Makefile	2011-11-15 14:03:59.000000000 +0400
+++ ../../../trunk/feeds/packages/libs/gettext-full//Makefile	2011-11-15 17:10:27.204793723 +0400
@@ -41,23 +41,24 @@
 	--disable-openmp \
 	--with-included-gettext \
 	--without-libintl-prefix \
+	--disable-curses \
 	--without-libexpat-prefix \
 	--without-emacs
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/lib/libintl-full/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libintl.h $(1)/usr/lib/libintl-full/include/
+	$(INSTALL_DIR) $(1)/opt/lib/libintl-full/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libintl.h $(1)/opt/lib/libintl-full/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib/libintl-full/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libintl.{a,so*} $(1)/usr/lib/libintl-full/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/libintl-full/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libintl.{a,so*} $(1)/opt/lib/libintl-full/lib/
 
-	$(INSTALL_DIR) $(1)/usr/share/aclocal
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/share/aclocal/* $(1)/usr/share/aclocal/
+	$(INSTALL_DIR) $(1)/opt/share/aclocal
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/share/aclocal/* $(1)/opt/share/aclocal/
 endef
 
 define Package/libintl-full/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/libintl.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/lib/libintl.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libintl-full))
