--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r38800/feeds/oldpackages/libs/gpgme/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/gpgme/Makefile
@@ -37,33 +37,34 @@
   for CMS (S/MIME).
 endef
 
-ifdef CONFIG_USE_UCLIBC
 CONFIGURE_VARS += \
     LIBS="-largp"
-endif
 
 CONFIGURE_ARGS += \
 	--enable-shared \
 	--enable-static \
-	--with-gpg=/"usr/bin/gpg" \
-	--with-gpg-error-prefix="$(STAGING_DIR)/usr" \
-	--with-libassuan-prefix="$(STAGING_DIR)/usr" \
+	--with-gpg="/opt/bin/gpg" \
+	--with-gpg-error-prefix="$(STAGING_DIR)/opt" \
+	--with-libassuan-prefix="$(STAGING_DIR)/opt" \
 	--without-g13 \
 	--without-gpgconf \
 	--without-gpgsm \
 	--without-pth \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/gpgme.h $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/gpgme.h $(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libgpgme{,-pthread}.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libgpgme{,-pthread}.{a,so*} $(1)/opt/lib/
+	
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin
 endef
 
 define Package/libgpgme/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libgpgme{,-pthread}.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libgpgme{,-pthread}.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libgpgme))
