--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/lang/php5/Makefile
+++ ../../../openwrt_trunk/feeds/packages/lang/php5/Makefile
@@ -136,13 +136,13 @@
 	--disable-debug \
 	--without-pear \
 	\
-	--with-config-file-path=/etc \
-	--with-config-file-scan-dir=/etc/php5 \
+	--with-config-file-path=/opt/etc \
+	--with-config-file-scan-dir=/opt/etc/php5 \
 	--disable-short-tags \
 	\
-	--with-zlib="$(STAGING_DIR)/usr" \
-	  --with-zlib-dir="$(STAGING_DIR)/usr" \
-	--with-pcre-regex="$(STAGING_DIR)/usr" \
+	--with-zlib="$(STAGING_DIR)/opt" \
+	  --with-zlib-dir="$(STAGING_DIR)/opt" \
+	--with-pcre-regex="$(STAGING_DIR)/opt" \
 	--disable-phar
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-calendar),)
@@ -158,7 +158,7 @@
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-curl),)
-  CONFIGURE_ARGS+= --with-curl=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --with-curl=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-curl
 endif
@@ -170,7 +170,7 @@
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-gettext),)
-  CONFIGURE_ARGS+= --with-gettext=shared,"$(STAGING_DIR)/usr/lib/libintl-full"
+  CONFIGURE_ARGS+= --with-gettext=shared,"$(STAGING_DIR)/opt/lib/libintl-full"
 else
   CONFIGURE_ARGS+= --without-gettext
 endif
@@ -197,8 +197,8 @@
   CONFIGURE_ARGS+= \
 	--with-gd=shared \
 	--without-freetype-dir \
-	--with-jpeg-dir="$(STAGING_DIR)/usr" \
-	--with-png-dir="$(STAGING_DIR)/usr" \
+	--with-jpeg-dir="$(STAGING_DIR)/opt" \
+	--with-png-dir="$(STAGING_DIR)/opt" \
 	--without-xpm-dir \
 	--without-t1lib \
 	--enable-gd-native-ttf \
@@ -208,7 +208,7 @@
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-gmp),)
-  CONFIGURE_ARGS+= --with-gmp=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --with-gmp=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-gmp
 endif
@@ -233,8 +233,8 @@
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-ldap),)
   CONFIGURE_ARGS+= \
-	--with-ldap=shared,"$(STAGING_DIR)/usr" \
-	--with-ldap-sasl="$(STAGING_DIR)/usr"
+	--with-ldap=shared,"$(STAGING_DIR)/opt" \
+	--with-ldap-sasl="$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-ldap
 endif
@@ -246,28 +246,28 @@
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-mcrypt),)
-  CONFIGURE_ARGS+=  --with-mcrypt=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+=  --with-mcrypt=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-mcrypt
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-mysql),)
-  CONFIGURE_ARGS+= --with-mysql=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --with-mysql=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-mysql
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-mysqli),)
-  CONFIGURE_ARGS+= --with-mysqli=shared,"$(STAGING_DIR)/usr/bin/mysql_config"
+  CONFIGURE_ARGS+= --with-mysqli=shared,"$(STAGING_DIR)/opt/bin/mysql_config"
 else
   CONFIGURE_ARGS+= --without-mysqli
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-openssl),)
   CONFIGURE_ARGS+= \
-	--with-openssl=shared,"$(STAGING_DIR)/usr" \
+	--with-openssl=shared,"$(STAGING_DIR)/opt" \
 	--with-kerberos=no \
-	--with-openssl-dir="$(STAGING_DIR)/usr"
+	--with-openssl-dir="$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-openssl
 endif
@@ -281,17 +281,17 @@
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-pdo),)
   CONFIGURE_ARGS+= --enable-pdo=shared
   ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-pdo-mysql),)
-    CONFIGURE_ARGS+= --with-pdo-mysql=shared,"$(STAGING_DIR)/usr"
+    CONFIGURE_ARGS+= --with-pdo-mysql=shared,"$(STAGING_DIR)/opt"
   else
     CONFIGURE_ARGS+= --without-pdo-mysql
   endif
   ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-pdo-pgsql),)
-    CONFIGURE_ARGS+= --with-pdo-pgsql=shared,"$(STAGING_DIR)/usr"
+    CONFIGURE_ARGS+= --with-pdo-pgsql=shared,"$(STAGING_DIR)/opt"
   else
     CONFIGURE_ARGS+= --without-pdo-pgsql
   endif
   ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-pdo-sqlite),)
-    CONFIGURE_ARGS+= --with-pdo-sqlite=shared,"$(STAGING_DIR)/usr"
+    CONFIGURE_ARGS+= --with-pdo-sqlite=shared,"$(STAGING_DIR)/opt"
   else
     CONFIGURE_ARGS+= --without-pdo-sqlite
   endif
@@ -300,7 +300,7 @@
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-pgsql),)
-  CONFIGURE_ARGS+= --with-pgsql=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --with-pgsql=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-pgsql
 endif
@@ -336,13 +336,13 @@
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-sqlite),)
-  CONFIGURE_ARGS+= --with-sqlite=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --with-sqlite=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-sqlite
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-sqlite3),)
-  CONFIGURE_ARGS+= --with-sqlite3=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --with-sqlite3=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --without-sqlite3
 endif
@@ -372,24 +372,24 @@
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-xml),)
-  CONFIGURE_ARGS+= --enable-xml=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --enable-xml=shared,"$(STAGING_DIR)/opt"
   ifneq ($(CONFIG_PHP5_LIBXML),)
-    CONFIGURE_ARGS+= --with-libxml-dir="$(STAGING_DIR)/usr/include/libxml2"
+    CONFIGURE_ARGS+= --with-libxml-dir="$(STAGING_DIR)/opt/include/libxml2"
   else
-    CONFIGURE_ARGS+= --with-libexpat-dir="$(STAGING_DIR)/usr"
+    CONFIGURE_ARGS+= --with-libexpat-dir="$(STAGING_DIR)/opt"
   endif
 else
   CONFIGURE_ARGS+= --disable-xml
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-xmlreader),)
-  CONFIGURE_ARGS+= --enable-xmlreader=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --enable-xmlreader=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --disable-xmlreader
 endif
 
 ifneq ($(SDK)$(CONFIG_PACKAGE_php5-mod-xmlwriter),)
-  CONFIGURE_ARGS+= --enable-xmlwriter=shared,"$(STAGING_DIR)/usr"
+  CONFIGURE_ARGS+= --enable-xmlwriter=shared,"$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= --disable-xmlwriter
 endif
@@ -408,7 +408,7 @@
 
 ifneq ($(SDK)$(CONFIG_PHP5_LIBXML),)
   CONFIGURE_ARGS+= --enable-libxml
-  CONFIGURE_ARGS+= --with-libxml-dir="$(STAGING_DIR)/usr/include/libxml2"
+  CONFIGURE_ARGS+= --with-libxml-dir="$(STAGING_DIR)/opt/include/libxml2"
 else
   CONFIGURE_ARGS+= --disable-libxml
 endif
@@ -426,31 +426,27 @@
 	ac_cv_php_xml2_config_path="$(STAGING_DIR)/host/bin/xml2-config" \
 
 define Package/php5/conffiles
-/etc/php.ini
+/opt/etc/php.ini
 endef
 
 define Package/php5/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/php.ini $(1)/etc/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/php.ini $(1)/opt/etc/
 endef
 
 define Package/php5-cli/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_BUILD_DIR)/sapi/cli/php $(1)/usr/bin/php-cli
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_BUILD_DIR)/sapi/cli/php $(1)/opt/bin/php-cli
 endef
 
 define Package/php5-cgi/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_BUILD_DIR)/sapi/cgi/php-cgi $(1)/usr/bin/php-cgi
-	ln -sf php-cgi $(1)/usr/bin/php-fcgi
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_BUILD_DIR)/sapi/cgi/php-cgi $(1)/opt/bin/php-cgi
+	ln -sf php-cgi $(1)/opt/bin/php-fcgi
 endef
 
 define Package/php5-fastcgi/install
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/php5-fastcgi.config $(1)/etc/config/php5-fastcgi
-
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/php5-fastcgi.init $(1)/etc/init.d/php5-fastcgi
+	true
 endef
 
 define Build/Prepare
@@ -459,13 +455,13 @@
 endef
 
 define Build/InstallDev
-	mkdir -p $(PKG_BUILD_DIR)/staging/usr/bin
+	mkdir -p $(PKG_BUILD_DIR)/staging/opt/bin
 	make -C $(PKG_BUILD_DIR) install INSTALL_ROOT=$(PKG_BUILD_DIR)/staging
-	rm -f $(PKG_BUILD_DIR)/staging/usr/bin/php
+	rm -f $(PKG_BUILD_DIR)/staging/opt/bin/php
 	$(CP) $(PKG_BUILD_DIR)/staging/* $(STAGING_DIR_HOST)
-	sed -i -e "s#prefix='/usr'#prefix='$(STAGING_DIR_HOST)/usr'#" $(STAGING_DIR_HOST)/usr/bin/phpize
-	sed -i -e "s#exec_prefix=\"\`eval echo /usr\`\"#exec_prefix='$(STAGING_DIR_HOST)/usr'#" $(STAGING_DIR_HOST)/usr/bin/phpize
-	sed -i -e "s#prefix=\"/usr\"#prefix=\"$(STAGING_DIR_HOST)/usr\"#" $(STAGING_DIR_HOST)/usr/bin/php-config
+	sed -i -e "s#prefix='/opt'#prefix='$(STAGING_DIR_HOST)/opt'#" $(STAGING_DIR_HOST)/opt/bin/phpize
+	sed -i -e "s#exec_prefix=\"\`eval echo /opt\`\"#exec_prefix='$(STAGING_DIR_HOST)/opt'#" $(STAGING_DIR_HOST)/opt/bin/phpize
+	sed -i -e "s#prefix=\"/opt\"#prefix=\"$(STAGING_DIR_HOST)/opt\"#" $(STAGING_DIR_HOST)/opt/bin/php-config
 endef
 
 define BuildModule
@@ -481,10 +477,10 @@
   endef
 
   define Package/php5-mod-$(1)/install
-	$(INSTALL_DIR) $$(1)/usr/lib/php
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/$(subst -,_,$(1)).so $$(1)/usr/lib/php/
-	$(INSTALL_DIR) $$(1)/etc/php5
-	echo "extension=$(subst -,_,$(1)).so" > $$(1)/etc/php5/$(subst -,_,$(1)).ini
+	$(INSTALL_DIR) $$(1)/opt/lib/php
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/$(subst -,_,$(1)).so $$(1)/opt/lib/php/
+	$(INSTALL_DIR) $$(1)/opt/etc/php5
+	echo "extension=$(subst -,_,$(1)).so" > $$(1)/opt/etc/php5/$(subst -,_,$(1)).ini
   endef
 
   $$(eval $$(call BuildPackage,php5-mod-$(1)))
@@ -508,7 +504,7 @@
 $(eval $(call BuildModule,gd,GD graphics,+PACKAGE_php5-mod-gd:libjpeg +PACKAGE_php5-mod-gd:libpng))
 $(eval $(call BuildModule,gmp,GMP,+PACKAGE_php5-mod-gmp:libgmp))
 $(eval $(call BuildModule,hash,Hash))
-$(eval $(call BuildModule,iconv,iConv,+PACKAGE_php5-mod-iconv:libiconv))
+$(eval $(call BuildModule,iconv,iConv,+PACKAGE_php5-mod-iconv:libiconv-full))
 $(eval $(call BuildModule,json,JSON))
 $(eval $(call BuildModule,ldap,LDAP,+PACKAGE_php5-mod-ldap:libopenldap +PACKAGE_php5-mod-ldap:libsasl2))
 $(eval $(call BuildModule,mbstring,MBString))
@@ -533,6 +529,6 @@
 $(eval $(call BuildModule,sysvshm,System V semaphore))
 $(eval $(call BuildModule,tokenizer,Tokenizer))
 $(eval $(call BuildModule,xml,XML,+PHP5_LIBXML:libxml2 +!PHP5_LIBXML:libexpat))
-$(eval $(call BuildModule,xmlreader,XMLReader,+@PHP5_LIBXML +PACKAGE_php5-mod-xmlreader:libxml2 +PACKAGE_php5-mod-xmlreader:libiconv))
-$(eval $(call BuildModule,xmlwriter,XMLWriter,+@PHP5_LIBXML +PACKAGE_php5-mod-xmlwriter:libxml2 +PACKAGE_php5-mod-xmlwriter:libiconv))
+$(eval $(call BuildModule,xmlreader,XMLReader,+@PHP5_LIBXML +PACKAGE_php5-mod-xmlreader:libxml2 +PACKAGE_php5-mod-xmlreader:libiconv-full))
+$(eval $(call BuildModule,xmlwriter,XMLWriter,+@PHP5_LIBXML +PACKAGE_php5-mod-xmlwriter:libxml2 +PACKAGE_php5-mod-xmlwriter:libiconv-full))
 $(eval $(call BuildModule,zip,ZIP,+PACKAGE_php5-mod-zip:zlib))
