diff -urx .svn ../../../trunk_clean/package/dropbear/Makefile ../../../trunk/package/dropbear/Makefile
--- ../../../trunk_clean/package/dropbear/Makefile	2012-05-02 16:17:35.674271615 +0400
+++ ../../../trunk/package/dropbear/Makefile	2012-05-23 12:23:08.252101713 +0400
@@ -37,9 +37,9 @@
 endef
 
 define Package/dropbear/conffiles
-/etc/dropbear/dropbear_rsa_host_key
-/etc/dropbear/dropbear_dss_host_key 
-/etc/config/dropbear 
+/opt/etc/dropbear/dropbear_rsa_host_key
+/opt/etc/dropbear/dropbear_dss_host_key
+/opt/etc/dropbear/authorized_keys
 endef
 
 define Package/dropbearconvert
@@ -66,12 +66,11 @@
 	--disable-zlib \
 	--enable-bundled-libtom
 
-TARGET_CFLAGS += -DARGTYPE=3 -ffunction-sections -fdata-sections
-TARGET_LDFLAGS += -Wl,--gc-sections
-
 define Build/Configure
 	$(SED) 's,^/\* #define PKG_MULTI.*,#define PKG_MULTI,g' $(PKG_BUILD_DIR)/options.h
 	$(SED) 's,^#define DO_HOST_LOOKUP,/* & */,g' $(PKG_BUILD_DIR)/options.h
+	$(SED) 's,/etc/dropbear/dropbear_,/opt/etc/dropbear/dropbear_,g' $(PKG_BUILD_DIR)/options.h
+	$(SED) 's,^#define MOTD_FILENAME "/etc/motd",#define MOTD_FILENAME "/opt/etc/motd",g' $(PKG_BUILD_DIR)/options.h
 	$(call Build/Configure/Default)
 endef
 
@@ -88,26 +87,31 @@
 endef
 
 define Package/dropbear/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dropbearmulti $(1)/usr/sbin/dropbear
-	$(INSTALL_DIR) $(1)/usr/bin
-	ln -sf ../sbin/dropbear $(1)/usr/bin/scp
-	ln -sf ../sbin/dropbear $(1)/usr/bin/ssh
-	ln -sf ../sbin/dropbear $(1)/usr/bin/dbclient
-	ln -sf ../sbin/dropbear $(1)/usr/bin/dropbearkey
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/dropbear.config $(1)/etc/config/dropbear
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/dropbear.init $(1)/etc/init.d/dropbear
-	$(INSTALL_DIR) $(1)/usr/lib/opkg/info
-	$(INSTALL_DIR) $(1)/etc/dropbear
-	touch $(1)/etc/dropbear/dropbear_rsa_host_key
-	touch $(1)/etc/dropbear/dropbear_dss_host_key
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dropbearmulti $(1)/opt/sbin/dropbear
+	$(INSTALL_DIR) $(1)/opt/bin
+	ln -sf ../sbin/dropbear $(1)/opt/bin/scp
+	ln -sf ../sbin/dropbear $(1)/opt/bin/ssh
+	ln -sf ../sbin/dropbear $(1)/opt/bin/dbclient
+	ln -sf ../sbin/dropbear $(1)/opt/bin/dropbearkey
+	$(INSTALL_DIR) $(1)/opt/etc/dropbear
+endef
+
+define Package/dropbear/postinst
+#!/bin/sh
+
+if [ ! -f /opt/etc/dropbear/dropbear_rsa_host_key ]; then
+    /opt/bin/dropbearkey -t rsa -f /opt/etc/dropbear/dropbear_rsa_host_key
+fi
+
+if [ ! -f /opt/etc/dropbear/dropbear_dss_host_key ]; then
+    /opt/bin/dropbearkey -t dss -f /opt/etc/dropbear/dropbear_dss_host_key
+fi
 endef
 
 define Package/dropbearconvert/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dropbearconvert $(1)/usr/bin/dropbearconvert
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dropbearconvert $(1)/opt/bin/dropbearconvert
 endef
 
 $(eval $(call BuildPackage,dropbear))
diff -urx .svn ../../../trunk_clean/package/dropbear/patches/100-pubkey_path.patch ../../../trunk/package/dropbear/patches/100-pubkey_path.patch
--- ../../../trunk_clean/package/dropbear/patches/100-pubkey_path.patch	2012-05-02 16:17:35.674271615 +0400
+++ ../../../trunk/package/dropbear/patches/100-pubkey_path.patch	2012-05-23 09:53:03.000878094 +0400
@@ -28,7 +28,7 @@
 +		/* open the file */
 +		authfile = fopen(filename, "r");
 +	} else {
-+		authfile = fopen("/etc/dropbear/authorized_keys","r");
++		authfile = fopen("/opt/etc/dropbear/authorized_keys","r");
 +	}
  	if (authfile == NULL) {
  		goto out;
@@ -58,10 +58,10 @@
 -	if (checkfileperm(filename) != DROPBEAR_SUCCESS) {
 -		goto out;
 +	if (ses.authstate.pw_uid == 0) {
-+		if (checkfileperm("/etc/dropbear") != DROPBEAR_SUCCESS) {
++		if (checkfileperm("/opt/etc/dropbear") != DROPBEAR_SUCCESS) {
 +			goto out;
 +		}
-+		if (checkfileperm("/etc/dropbear/authorized_keys") != DROPBEAR_SUCCESS) {
++		if (checkfileperm("/opt/etc/dropbear/authorized_keys") != DROPBEAR_SUCCESS) {
 +			goto out;
 +		}
 +	} else {
