--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/vnstat/Makefile
+++ ../../../openwrt_trunk/feeds/packages/net/vnstat/Makefile
@@ -56,21 +56,21 @@
 endef
 
 define Package/vnstat/conffiles
-/etc/vnstat.conf
-/etc/config/vnstat
+/opt/etc/vnstat.conf
 endef
 
 define Build/Compile/vnstat
 	+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) \
 		$(TARGET_CONFIGURE_OPTS) \
-		CFLAGS="$(TARGET_CFLAGS)"
+		CFLAGS="$(TARGET_CFLAGS)" \
+		LDFLAGS="$(TARGET_LDFLAGS)"
 endef
 
 define Build/Compile/vnstati
 	+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) \
 		$(TARGET_CONFIGURE_OPTS) \
-		CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/usr/include" \
-		LDFLAGS="$(TARGET_LDFLAGS) -Wl,-rpath-link,$(STAGING_DIR)/usr/lib" \
+		CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/opt/include" \
+		LDFLAGS="$(TARGET_LDFLAGS)" \
 		all
 endef
 
@@ -80,29 +80,17 @@
 endef
 
 define Package/vnstat/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/vnstat $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/vnstatd $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/vnstat.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/vnstat.config $(1)/etc/config/vnstat
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/vnstat.init $(1)/etc/init.d/vnstat
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(INSTALL_BIN) ./files/vnstat.defaults $(1)/etc/uci-defaults/vnstat
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/vnstat $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/vnstatd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/var/lib/vnstat
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S32vnstat $(1)/opt/etc/init.d
 endef
 
 define Package/vnstati/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/vnstati $(1)/usr/bin/
-endef
-
-define Package/vnstat/postinst
-#!/bin/sh
-[ -n "$${IPKG_INSTROOT}" ] || (. /etc/uci-defaults/vnstat) && rm -f /etc/uci-defaults/vnstat
-exit 0
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/bin/vnstati $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,vnstat))
