--- ../../../downloads/openwrt_trunk-r40423/feeds/telephony/net/asterisk-1.8.x-chan-sccp-b/Makefile	2014-05-15 15:17:03.203017679 +0400
+++ ../../../openwrt_trunk/feeds/telephony/net/asterisk-1.8.x-chan-sccp-b/Makefile	2014-05-15 20:21:40.814684490 +0400
@@ -38,25 +38,26 @@
 endef
 
 define Build/Configure
-	$(call Build/Configure/Default,--with-asterisk=$(STAGING_DIR)/usr/include/asterisk-1.8)
+	$(call Build/Configure/Default,--with-asterisk=$(STAGING_DIR)/opt/include/asterisk-1.8)
 endef
 
 define Build/Compile
-	$(MAKE) -C "$(PKG_BUILD_DIR)" \
+	(export echo=echo; \
+		$(MAKE) -C "$(PKG_BUILD_DIR)" \
 		CFLAGS="$(CFLAGS) -I$(PKG_BUILD_DIR)/src -DLOW_MEMORY" \
 		DESTDIR="$(PKG_INSTALL_DIR)" \
-		all install
+		all install)
 endef
 
 define Package/asterisk18-chan-sccp-b/conffiles
-/etc/asterisk/sccp.conf
+/opt/etc/asterisk/sccp.conf
 endef
 
 define Package/asterisk18-chan-sccp-b/install
-	$(INSTALL_DIR) $(1)/etc/asterisk
-	$(CP)  ./files/sccp.openwrt.conf $(1)/etc/asterisk/sccp.conf
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/modules
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/asterisk/modules/chan_sccp.so $(1)/usr/lib/asterisk/modules/
+	$(INSTALL_DIR) $(1)/opt/etc/asterisk
+	$(CP)  ./files/sccp.openwrt.conf $(1)/opt/etc/asterisk/sccp.conf
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/modules
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/asterisk/modules/chan_sccp.so $(1)/opt/lib/asterisk/modules/
 endef
 
 $(eval $(call BuildPackage,asterisk18-chan-sccp-b))
