--- ../../../downloads/openwrt_trunk-r44486/feeds/packages/lang/simplejson/Makefile	2015-02-18 12:16:57.580306331 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/simplejson/Makefile	2015-02-20 11:44:19.409201518 +0300
@@ -35,7 +35,7 @@
 endef
 
 define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix=/usr --root=$(PKG_INSTALL_DIR))
+	$(call Build/Compile/PyMod,,install --prefix=/opt --root=$(PKG_INSTALL_DIR))
 endef
 
 define Package/simplejson/install
