--- ../../../openwrt_trunk-r40859/feeds/oldpackages/net/amule/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/amule/Makefile
@@ -26,7 +26,7 @@
   CATEGORY:=Network
   TITLE:=A multi-platform eMule-like ed2k client
   URL:=http://www.amule.org/
-  DEPENDS:=+libpng +libpthread +libncurses +libreadline +libwxbase +libcryptoxx
+  DEPENDS:=+libpng +libpthread +libncurses +libreadline +libwxbase +libcryptoxx $(ICONV_DEPENDS)
 endef
 
 CONFIGURE_ARGS+= \
@@ -51,11 +51,11 @@
 	--disable-plasmamule \
 	--without-wxdebug \
 	\
-	--with-zlib="$(STAGING_DIR)/usr" \
-	--with-gdlib-prefix="$(STAGING_DIR)/usr" \
-	--with-libpng-prefix="$(STAGING_DIR)/usr" \
-	--with-wx-prefix="$(STAGING_DIR)/usr" \
-	--with-crypto-prefix="$(STAGING_DIR)/usr" \
+	--with-zlib="$(STAGING_DIR)/opt" \
+	--with-gdlib-prefix="$(STAGING_DIR)/opt" \
+	--with-libpng-prefix="$(STAGING_DIR)/opt" \
+	--with-wx-prefix="$(STAGING_DIR)/opt" \
+	--with-crypto-prefix="$(STAGING_DIR)/opt" \
 	--with-libiconv-prefix="$(ICONV_PREFIX)" \
 	--with-libintl-prefix="$(INTL_PREFIX)" \
 	--without-x \
@@ -63,6 +63,9 @@
 TARGET_LDFLAGS += \
 	-liconv
 
+TARGET_CFLAGS += \
+	-O1
+
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		HOSTCC="$(HOSTCC)" \
@@ -71,10 +74,13 @@
 endef
 
 define Package/amule/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/amule{cmd,d,web} $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/share
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/amule $(1)/usr/share/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/amule{cmd,d,web} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/amule $(1)/opt/share/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S57amuled $(1)/opt/etc/init.d
+	$(INSTALL_DIR) $(1)/opt/var/amule
 endef
 
 $(eval $(call BuildPackage,amule))
