--- ../../../openwrt_trunk/feeds/oldpackages/utils/lirc/Makefile.orig
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/lirc/Makefile
@@ -6,7 +6,6 @@
 #
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=lirc
 PKG_VERSION:=0.8.7
@@ -68,7 +67,7 @@
 	--disable-static
 
 define Package/lirc/conffiles
-/etc/lircd.conf
+/opt/etc/lircd.conf
 endef
 
 MAKE_FLAGS += \
@@ -76,10 +75,10 @@
 	SUBDIRS="daemons tools"
 
 define Package/lirc/install
-	$(INSTALL_DIR) $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/daemons/lircd $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/lircd.conf $(1)/etc/
+	$(INSTALL_DIR) $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/daemons/lircd $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/lircd.conf $(1)/opt/etc/
 endef
 #todo:
 #	$(CP) $(PKG_BUILD_DIR)/drivers/lirc_dev/lirc_dev.ko
@@ -87,16 +86,19 @@
 #depends on the selected module
 
 define Package/lircdaemonadd/install
-	$(INSTALL_DIR) $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/daemons/irrecord $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/daemons/lircmd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/daemons/irrecord $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/daemons/lircmd $(1)/opt/sbin/
 endef
 
 define Package/lirctools/install
-	$(INSTALL_DIR) $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/irsend $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/irw $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/mode2 $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/irsend $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/irw $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/mode2 $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tools/.libs/irexec $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib/
+	$(CP) $(PKG_BUILD_DIR)/tools/.libs/liblirc_client.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,lirc))
