--- ../../../downloads/openwrt_trunk-r43593/package/libs/polarssl/Makefile	2014-12-10 12:26:33.286951674 +0300
+++ ../../../openwrt_trunk/package/libs/polarssl/Makefile	2014-12-10 12:49:08.662926986 +0300
@@ -55,15 +55,15 @@
 	-DENABLE_PROGRAMS:Bool=OFF \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/polarssl $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpolarssl.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/polarssl $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpolarssl.so* $(1)/opt/lib/
 endef
 
 define Package/libpolarssl/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpolarssl.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpolarssl.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libpolarssl))
