--- ../../../downloads/openwrt_trunk-r40859/feeds/packages/net/lftp/Makefile	2014-05-27 14:18:46.717780077 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/lftp/Makefile	2014-05-31 17:28:30.239438341 +0400
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lftp
-PKG_VERSION:=4.3.6
-PKG_RELEASE:=2
+PKG_VERSION:=4.4.16
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=http://lftp.cybermirror.org \
-  http://lftp.cybermirror.org/old
-PKG_MD5SUM:=ffcdbaec5d1416cb6dd754efd43fa61a
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://lftp.yar.ru/ftp \
+	http://lftp.yar.ru/ftp/old
+PKG_MD5SUM:=d3337046233cc9b3c66673dcd29458cf
 
 PKG_INSTALL:=1
 
@@ -25,7 +25,7 @@
   SUBMENU:=File Transfer
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libncurses +libopenssl +libreadline $(CXX_DEPENDS) +libexpat
+  DEPENDS:=+libncurses +libgnutls +zlib +libreadline $(CXX_DEPENDS) +libexpat
   TITLE:=Sophisticated file transfer program
   URL:=http://lftp.yar.ru/
 endef
@@ -40,21 +40,19 @@
 CONFIGURE_ARGS += \
 	--without-libiconv-prefix \
 	--without-libintl-prefix \
-	--without-gnutls \
 	--without-libresolv \
-	--with-openssl="$(STAGING_DIR)/usr" \
 	--disable-static
 
 CONFIGURE_VARS += \
 	LIBS="-lz -lutil -lcurses -ldl" \
-	i_cv_posix_fallocate_works=no
+	i_cv_posix_fallocate_works=yes \
 
 MAKE_VARS += \
 	LD="$(TARGET_CXX)"
 
 define Package/lftp/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/lftp $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/lftp $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,lftp))
