diff -ur ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/net-snmp/files/snmpd.default ../../../openwrt_trunk/feeds/packages/net/net-snmp/files/snmpd.default
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/net-snmp/files/snmpd.default	2014-10-06 15:46:23.753250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/net-snmp/files/snmpd.default	2014-10-07 11:59:31.647924729 +0400
@@ -1 +1 @@
-OPTIONS="-Lf /dev/null -p /var/run/snmpd.pid"
+OPTIONS="-Lf /dev/null -p /opt/var/run/snmpd.pid"
diff -ur ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/net-snmp/files/snmpd.init ../../../openwrt_trunk/feeds/packages/net/net-snmp/files/snmpd.init
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/net-snmp/files/snmpd.init	2014-10-06 15:46:23.753250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/net-snmp/files/snmpd.init	2014-10-07 11:59:31.647924729 +0400
@@ -2,10 +2,10 @@
 # Copyright (C) 2008 OpenWrt.org
 START=50
 
-DEFAULT=/etc/default/snmpd
-LIB_D=/var/lib/snmp
-LOG_D=/var/log
-RUN_D=/var/run
+DEFAULT=/opt/etc/default/snmpd
+LIB_D=/opt/var/lib/snmp
+LOG_D=/opt/var/log
+RUN_D=/opt/var/run
 PID_F=$RUN_D/snmpd.pid
 RUN_C=$RUN_D/snmpd.conf
 
@@ -138,7 +138,7 @@
 	config_foreach snmpd_exec_add exec
 
 	[ -f $DEFAULT ] && . $DEFAULT
-	$DEBUG /usr/sbin/snmpd $OPTIONS
+	$DEBUG /opt/sbin/snmpd $OPTIONS
 }
 stop() {
 	[ -f $PID_F ] && kill $(cat $PID_F)
diff -ur ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/net-snmp/Makefile ../../../openwrt_trunk/feeds/packages/net/net-snmp/Makefile
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/net-snmp/Makefile	2014-10-06 15:46:23.753250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/net-snmp/Makefile	2014-10-07 11:59:31.651924729 +0400
@@ -156,31 +156,32 @@
 	--enable-shared \
 	--enable-static \
 	--with-endianness=$(if $(CONFIG_BIG_ENDIAN),big,little) \
-	--with-logfile=/var/log/snmpd.log \
-	--with-persistent-directory=/usr/lib/snmp/ \
+	--with-logfile=/opt/var/log/snmpd.log \
+	--with-persistent-directory=/opt/lib/snmp/ \
 	--with-default-snmp-version=1 \
 	--with-sys-contact=root@localhost \
 	--with-sys-location=Unknown \
 	--enable-applications \
 	--disable-debugging \
 	--disable-manuals \
-	--disable-mibs \
 	--disable-scripts \
-	--with-out-mib-modules="$(SNMP_MIB_MODULES_EXCLUDED)" \
-	--with-mib-modules="$(SNMP_MIB_MODULES_INCLUDED)" \
-	--with-out-transports="$(SNMP_TRANSPORTS_EXCLUDED)" \
-	--with-transports="$(SNMP_TRANSPORTS_INCLUDED)" \
 	--without-openssl \
 	--without-libwrap \
 	--without-rpm \
 	--without-zlib \
 	 $(call autoconf_bool,CONFIG_IPV6,ipv6) \
 
+#	--with-out-mib-modules="$(SNMP_MIB_MODULES_EXCLUDED)" \
+#	--with-mib-modules="$(SNMP_MIB_MODULES_INCLUDED)" \
+#	--with-out-transports="$(SNMP_TRANSPORTS_EXCLUDED)" \
+#	--with-transports="$(SNMP_TRANSPORTS_INCLUDED)" \
+#	--disable-mibs \
+
 ifeq ($(CONFIG_IPV6),y)
 SNMP_TRANSPORTS_INCLUDED+= UDPIPv6
 endif
 
-TARGET_LDFLAGS += -L$(TOOLCHAIN_DIR)/usr/lib
+#TARGET_LDFLAGS += -L$(TOOLCHAIN_DIR)/opt/lib
 
 ifdef CONFIG_PACKAGE_snmpd-static
   define Build/Compile/static
@@ -201,59 +202,45 @@
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(2)/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/net-snmp-config $(2)/bin/
-	$(SED) 's,=/usr,=$(STAGING_DIR)/usr,g' $(2)/bin/net-snmp-config
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/net-snmp-config $(2)/bin/
+	$(SED) 's,=/opt,=$(STAGING_DIR)/opt,g' $(2)/bin/net-snmp-config
 
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/net-snmp $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libnetsnmp{,agent,helpers,mibs}.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/net-snmp $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libnetsnmp{,agent,helpers,mibs,trapd}.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libnetsnmp/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libnetsnmp{,agent,helpers,mibs}.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libnetsnmp{,agent,helpers,mibs,trapd}.so.* $(1)/opt/lib/
 endef
 
 define Package/snmpd/conffiles
-/etc/default/snmpd
-/etc/config/snmpd
+/opt/etc/snmp/snmpd.config
 endef
 
 define Package/snmpd/install
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/snmpd.conf $(1)/etc/config/snmpd
-	$(INSTALL_DIR) $(1)/etc/snmp
-	ln -sf /var/run/snmpd.conf $(1)/etc/snmp/
-	$(INSTALL_DIR) $(1)/etc/default
-	$(INSTALL_DATA) ./files/snmpd.default $(1)/etc/default/snmpd
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/snmpd.init $(1)/etc/init.d/snmpd
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/snmpd $(1)/usr/sbin/snmpd
+	$(INSTALL_DIR) $(1)/opt/etc/snmp
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/snmpd $(1)/opt/sbin/snmpd
 endef
 
 define Package/snmpd-static/conffiles
-/etc/default/snmpd
-/etc/config/snmpd
+/opt/etc/snmp/snmpd.config
 endef
 
 define Package/snmpd-static/install
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/snmpd.conf $(1)/etc/config/snmpd
-	$(INSTALL_DIR) $(1)/etc/snmp
-	ln -sf /var/run/snmpd.conf $(1)/etc/snmp/
-	$(INSTALL_DIR) $(1)/etc/default
-	$(INSTALL_DATA) ./files/snmpd.default $(1)/etc/default/snmpd
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/snmpd.init $(1)/etc/init.d/snmpd
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/agent/snmpd-static $(1)/usr/sbin/snmpd
+	$(INSTALL_DIR) $(1)/opt/etc/snmp
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/agent/snmpd-static $(1)/opt/sbin/snmpd
 endef
 
 define Package/snmp-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/snmp{get,set,status,test,trap,walk} $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/snmp{get,set,status,test,trap,walk} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share/snmp/mibs
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/snmp/mibs/*.txt $(1)/opt/share/snmp/mibs
 endef
 
 $(eval $(call BuildPackage,libnetsnmp))
diff -ur ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/net-snmp/patches/120-debian-searchdirs.patch ../../../openwrt_trunk/feeds/packages/net/net-snmp/patches/120-debian-searchdirs.patch
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/net-snmp/patches/120-debian-searchdirs.patch	2014-10-06 15:46:23.753250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/net-snmp/patches/120-debian-searchdirs.patch	2014-10-07 11:59:31.659924729 +0400
@@ -6,9 +6,9 @@
  }
 -push @def_search_dirs, "/usr/local/share/snmp/";
 -push @def_search_dirs, "/usr/local/share/snmp/mib2c-data";
-+push @def_search_dirs, "/etc/snmp/";
-+push @def_search_dirs, "/usr/share/snmp/";
-+push @def_search_dirs, "/usr/share/snmp/mib2c-data";
++push @def_search_dirs, "/opt/etc/snmp/";
++push @def_search_dirs, "/opt/share/snmp/";
++push @def_search_dirs, "/opt/share/snmp/mib2c-data";
  push @def_search_dirs, "./mib2c-conf.d";
  
  sub usage {
