*** ../backup/trunk_initialized/trunk/include/package-defaults.mk	2011-11-10 14:44:28.000000000 +0400
--- ../trunk/include/package-defaults.mk	2011-11-11 12:11:34.574342748 +0400
***************
*** 55,61 ****
    endef
  endif
  
! CONFIGURE_PREFIX:=/usr
  CONFIGURE_ARGS = \
  		--target=$(GNU_TARGET_NAME) \
  		--host=$(GNU_TARGET_NAME) \
--- 55,61 ----
    endef
  endif
  
! CONFIGURE_PREFIX:=/opt
  CONFIGURE_ARGS = \
  		--target=$(GNU_TARGET_NAME) \
  		--host=$(GNU_TARGET_NAME) \
***************
*** 67,75 ****
  		--bindir=$(CONFIGURE_PREFIX)/bin \
  		--sbindir=$(CONFIGURE_PREFIX)/sbin \
  		--libexecdir=$(CONFIGURE_PREFIX)/lib \
! 		--sysconfdir=/etc \
  		--datadir=$(CONFIGURE_PREFIX)/share \
! 		--localstatedir=/var \
  		--mandir=$(CONFIGURE_PREFIX)/man \
  		--infodir=$(CONFIGURE_PREFIX)/info \
  		$(DISABLE_NLS) \
--- 67,75 ----
  		--bindir=$(CONFIGURE_PREFIX)/bin \
  		--sbindir=$(CONFIGURE_PREFIX)/sbin \
  		--libexecdir=$(CONFIGURE_PREFIX)/lib \
! 		--sysconfdir=$(CONFIGURE_PREFIX)/etc \
  		--datadir=$(CONFIGURE_PREFIX)/share \
! 		--localstatedir=$(CONFIGURE_PREFIX)/var \
  		--mandir=$(CONFIGURE_PREFIX)/man \
  		--infodir=$(CONFIGURE_PREFIX)/info \
  		$(DISABLE_NLS) \
