*** ../backup/trunk/package/cyassl/Makefile	2011-10-23 21:23:47.000000000 +0400
--- ../trunk/package/cyassl/Makefile	2011-10-25 11:33:17.514286077 +0400
***************
*** 41,56 ****
  	--enable-singleThreaded
  
  define Build/InstallDev
! 	$(INSTALL_DIR) $(1)/usr/include
! 	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
  
! 	$(INSTALL_DIR) $(1)/usr/lib
! 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcyassl.{a,so*,la} $(1)/usr/lib/
  endef
  
  define Package/libcyassl/install
! 	$(INSTALL_DIR) $(1)/usr/lib
! 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcyassl.so* $(1)/usr/lib/
  endef
  
  $(eval $(call BuildPackage,libcyassl))
--- 41,56 ----
  	--enable-singleThreaded
  
  define Build/InstallDev
! 	$(INSTALL_DIR) $(1)/opt/include
! 	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
  
! 	$(INSTALL_DIR) $(1)/opt/lib
! 	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcyassl.{a,so*,la} $(1)/opt/lib/
  endef
  
  define Package/libcyassl/install
! 	$(INSTALL_DIR) $(1)/opt/lib
! 	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcyassl.so* $(1)/opt/lib/
  endef
  
  $(eval $(call BuildPackage,libcyassl))
