*** ../backup/trunk/package/gdb/Makefile	2011-10-23 21:23:47.000000000 +0400
--- ../trunk/package/gdb/Makefile	2011-10-25 11:33:18.074279038 +0400
***************
*** 69,81 ****
  endef
  
  define Package/gdb/install
! 	$(INSTALL_DIR) $(1)/usr/bin
! 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/gdb $(1)/usr/bin/
  endef
  
  define Package/gdbserver/install
! 	$(INSTALL_DIR) $(1)/usr/bin
! 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/gdbserver $(1)/usr/bin/
  endef
  
  $(eval $(call BuildPackage,gdb))
--- 69,81 ----
  endef
  
  define Package/gdb/install
! 	$(INSTALL_DIR) $(1)/opt/bin
! 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/gdb $(1)/opt/bin/
  endef
  
  define Package/gdbserver/install
! 	$(INSTALL_DIR) $(1)/opt/bin
! 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/gdbserver $(1)/opt/bin/
  endef
  
  $(eval $(call BuildPackage,gdb))
