diff -urx .svn ../../../trunk_clean/feeds/packages/net/lighttpd/files/lighttpd.conf ../../../trunk/feeds/packages/net/lighttpd/files/lighttpd.conf
--- ../../../trunk_clean/feeds/packages/net/lighttpd/files/lighttpd.conf	2012-06-01 10:50:36.492245404 +0400
+++ ../../../trunk/feeds/packages/net/lighttpd/files/lighttpd.conf	2012-06-01 10:51:34.183531437 +0400
@@ -26,10 +26,10 @@
 
 ## a static document-root, for virtual-hosting take look at the 
 ## server.virtual-* options
-server.document-root = "/www/"
+server.document-root = "/opt/share/www/"
 
 ## where to send error-messages to
-#server.errorlog = "/var/log/lighttpd/error.log"
+#server.errorlog = "/opt/var/log/lighttpd/error.log"
 
 ## files to check for if .../ is requested
 index-file.names = ( "index.html", "default.html", "index.htm", "default.htm" )
@@ -121,7 +121,7 @@
 ## change uid to <uid> (default: don't care)
 #server.username = "nobody"
 #
-server.upload-dirs = ( "/tmp" )
+server.upload-dirs = ( "/opt/tmp" )
 
 ## change uid to <uid> (default: don't care)
 #server.groupname = "nobody"
@@ -146,14 +146,14 @@
 #fastcgi.server = (
 #	".php" => (
 #		"localhost" => (
-#			"socket" => "/tmp/php-fastcgi.socket",
-#			"bin-path" => "/usr/local/bin/php"
+#			"socket" => "/opt/tmp/php-fastcgi.socket",
+#			"bin-path" => "/opt/bin/php"
 #		)
 #	)
 #)
 
 #### CGI module
-#cgi.assign = ( ".pl"  => "/usr/bin/perl", ".cgi" => "/usr/bin/perl" )
+#cgi.assign = ( ".pl"  => "/opt/bin/perl", ".cgi" => "/opt/bin/perl" )
 
 #### SSL engine
 #ssl.engine = "enable"
@@ -214,8 +214,8 @@
 #index-file.names += (foo + ".php")
 
 #### include
-#include /etc/lighttpd/lighttpd-inc.conf
-## same as above if you run: "lighttpd -f /etc/lighttpd/lighttpd.conf"
+#include /opt/etc/lighttpd/lighttpd-inc.conf
+## same as above if you run: "lighttpd -f /opt/etc/lighttpd/lighttpd.conf"
 #include "lighttpd-inc.conf"
 
 #### include_shell
@@ -227,5 +227,5 @@
 #$HTTP["url"] =~ "^/webdav($|/)" {
 # webdav.activate = "enable"
 # webdav.is-readonly = "enable"
-# webdav.sqlite-db-name = "/var/run/lighttpd-webdav-lock.db"
+# webdav.sqlite-db-name = "/opt/var/run/lighttpd-webdav-lock.db"
 #}
diff -urx .svn ../../../trunk_clean/feeds/packages/net/lighttpd/Makefile ../../../trunk/feeds/packages/net/lighttpd/Makefile
--- ../../../trunk_clean/feeds/packages/net/lighttpd/Makefile	2012-06-01 10:50:36.492245404 +0400
+++ ../../../trunk/feeds/packages/net/lighttpd/Makefile	2012-06-01 20:22:46.442705660 +0400
@@ -220,8 +220,8 @@
 endef
 
 CONFIGURE_ARGS+= \
-	--libdir=/usr/lib/lighttpd \
-	--sysconfdir=/etc/lighttpd \
+	--libdir=/opt/lib/lighttpd \
+	--sysconfdir=/opt/etc/lighttpd \
 	--enable-shared \
 	--enable-static \
 	--disable-rpath \
@@ -242,7 +242,7 @@
 
 ifneq ($(strip $(CONFIG_LIGHTTPD_SSL)),)
   CONFIGURE_ARGS+= \
-	--with-openssl="$(STAGING_DIR)/usr"
+	--with-openssl="$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= \
 	--without-openssl
@@ -269,27 +269,27 @@
 endef
 
 define Package/lighttpd/conffiles
-/etc/lighttpd/lighttpd.conf
+/opt/etc/lighttpd/lighttpd.conf
 endef
 
 define Package/lighttpd/install
-	$(INSTALL_DIR) $(1)/etc/lighttpd
-	$(INSTALL_DATA) ./files/lighttpd.conf $(1)/etc/lighttpd/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/lighttpd.init $(1)/etc/init.d/lighttpd
-	$(INSTALL_DIR) $(1)/usr/lib/lighttpd
+	$(INSTALL_DIR) $(1)/opt/etc/lighttpd
+	$(INSTALL_DATA) ./files/lighttpd.conf $(1)/opt/etc/lighttpd/
+	$(INSTALL_DIR) $(1)/opt/lib/lighttpd
 	for m in dirlisting indexfile staticfile; do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/lighttpd/mod_$$$${m}.so $(1)/usr/lib/lighttpd/ ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/lighttpd/mod_$$$${m}.so $(1)/opt/lib/lighttpd/ ; \
 	done
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/lighttpd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/lighttpd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S80lighttpd $(1)/opt/etc/init.d
 endef
 
 define BuildPlugin
   define Package/$(1)/install
-	[ -z "$(2)" ] || $(INSTALL_DIR) $$(1)/usr/lib/lighttpd
+	[ -z "$(2)" ] || $(INSTALL_DIR) $$(1)/opt/lib/lighttpd
 	for m in $(2); do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/lighttpd/mod_$$$$$$$${m}.so $$(1)/usr/lib/lighttpd/ ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/lighttpd/mod_$$$$$$$${m}.so $$(1)/opt/lib/lighttpd/ ; \
 	done
   endef
 
