--- ../../../downloads/openwrt_trunk-r46338/feeds/packages/net/ipsec-tools/Makefile	2015-07-14 10:55:19.782308173 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/ipsec-tools/Makefile	2015-07-14 12:12:10.078384907 +0300
@@ -7,7 +7,6 @@
 #
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=ipsec-tools
 PKG_VERSION:=0.8.2
@@ -30,7 +29,7 @@
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=VPN
-  DEPENDS:=+libopenssl +kmod-ipsec
+  DEPENDS:=+libopenssl
   TITLE:=IPsec management tools
   URL:=http://ipsec-tools.sourceforge.net/
   MAINTAINER:=Noah Meyerhans <frodo@morgul.net>
@@ -41,7 +40,7 @@
 	--enable-static \
 	--with-kernel-headers="$(LINUX_DIR)/include" \
 	--without-readline \
-	--with-openssl="$(STAGING_DIR)/usr" \
+	--with-openssl="$(STAGING_DIR)/opt" \
 	--without-libradius \
 	--without-libpam \
 	--enable-dpd \
@@ -76,27 +75,16 @@
 endef
 
 define Package/ipsec-tools/install
-	$(INSTALL_DIR) $(1)/etc/racoon
-	$(INSTALL_CONF) ./files/functions.sh $(1)/etc/racoon/
-	$(INSTALL_BIN) ./files/p1client-up $(1)/etc/racoon/
-	$(INSTALL_BIN) ./files/p1client-down $(1)/etc/racoon/
-	$(INSTALL_BIN) ./files/vpnctl $(1)/etc/racoon/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/racoon.init $(1)/etc/init.d/racoon
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/racoon $(1)/etc/config/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libipsec.so.* $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libracoon.so.* $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/plainrsa-gen $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/racoon $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/racoonctl $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/setkey $(1)/usr/sbin/
-endef
-
-define Package/ipsec-tools/conffiles
-/etc/config/racoon
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S49racoon $(1)/opt/etc/init.d
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libipsec.so.* $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libracoon.so.* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/plainrsa-gen $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/racoon $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/racoonctl $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/setkey $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,ipsec-tools))
