--- ../../../openwrt_trunk.orig/feeds/telephony/rtpproxy/Makefile	2012-05-02 16:17:27.522374074 +0400
+++ ../../../openwrt_trunk/feeds/telephony/rtpproxy/Makefile	2012-05-31 23:10:01.238648734 +0400
@@ -31,8 +31,10 @@
 # uses GNU configure
 
 define Package/rtpproxy/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/rtpproxy $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/rtpproxy $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S18rtpproxy $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,rtpproxy))
