--- ../../../downloads/openwrt_trunk-q44497/feeds/oldpackages/utils/rrdtool/Makefile	2015-02-25 12:52:24.504485917 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/rrdtool/Makefile	2015-03-01 00:24:41.306454681 +0300
@@ -74,7 +74,8 @@
 	--disable-tcl \
 	--disable-ruby \
 	--without-x \
-	--with-rrd-default-font="/usr/share/rrdtool/fonts/DejaVuSansMono-Roman.ttf" \
+	--with-rrd-default-font="/opt/share/rrdtool/fonts/DejaVuSansMono-Roman.ttf" \
+	CPPFLAGS="-I $(STAGING_DIR)/opt/include/freetype2" \
 
 CONFIGURE_VARS += \
 	rd_cv_ieee_works=yes \
@@ -102,30 +103,30 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/lib/rrdtool-1.2/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/rrd.h $(1)/usr/lib/rrdtool-1.2/include/
-	$(INSTALL_DIR) $(1)/usr/lib/rrdtool-1.2/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/librrd{,_th}.{a,so*} $(1)/usr/lib/rrdtool-1.2/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/rrdtool-1.2/include
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/include/rrd.h $(1)/opt/lib/rrdtool-1.2/include/
+	$(INSTALL_DIR) $(1)/opt/lib/rrdtool-1.2/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/librrd{,_th}.{a,so*} $(1)/opt/lib/rrdtool-1.2/lib/
 endef
 
 define Package/librrd/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/librrd.so.* $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/librrd_th.so.* $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/share/rrdtool/fonts
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/librrd.so.* $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/librrd_th.so.* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/share/rrdtool/fonts
 	$(CP) $(PKG_BUILD_DIR)/src/DejaVuSansMono-Roman.ttf \
-		$(1)/usr/share/rrdtool/fonts/
+		$(1)/opt/share/rrdtool/fonts/
 endef
 
 define Package/rrdcgi/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/rrdcgi $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/bin/rrdcgi $(1)/opt/bin/
 endef
 
 define Package/rrdtool/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/rrdtool $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/rrdupdate $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/bin/rrdtool $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/bin/rrdupdate $(1)/opt/bin/
 endef
 
 define PyPackage/python-rrd/install
