--- ../../../openwrt_trunk.orig/include/autotools.mk
+++ ../../../openwrt_trunk/include/autotools.mk
@@ -75,7 +75,7 @@
   $(strip $(call autoreconf, \
     $(PKG_BUILD_DIR), $(PKG_REMOVE_FILES), \
     $(PKG_AUTOMAKE_PATHS), $(PKG_LIBTOOL_PATHS), \
-    $(STAGING_DIR)/host/share/aclocal $(STAGING_DIR)/usr/share/aclocal $(PKG_MACRO_PATHS)))
+    $(STAGING_DIR)/host/share/aclocal $(STAGING_DIR)/opt/share/aclocal $(PKG_MACRO_PATHS)))
 endef
 
 define patch_libtool_target
