--- ../../../downloads/openwrt_trunk-r42753/include/cmake.mk
+++ ../../../openwrt_trunk/include/cmake.mk
@@ -56,7 +56,7 @@
 			-DCMAKE_FIND_ROOT_PATH_MODE_LIBRARY=ONLY \
 			-DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=ONLY \
 			-DCMAKE_STRIP=: \
-			-DCMAKE_INSTALL_PREFIX=/usr \
+			-DCMAKE_INSTALL_PREFIX=/opt \
 			-DDL_LIBRARY=$(STAGING_DIR) \
 			-DCMAKE_PREFIX_PATH=$(STAGING_DIR) \
 			$(CMAKE_OPTIONS) \
