--- ../../../openwrt_trunk-r40423/include/nls.mk
+++ ../../../openwrt_trunk/include/nls.mk
@@ -7,18 +7,18 @@
 
 # iconv full
 ifeq ($(CONFIG_BUILD_NLS),y)
-	ICONV_PREFIX:=$(STAGING_DIR)/usr/lib/libiconv-full
+	ICONV_PREFIX:=$(STAGING_DIR)/opt/lib/libiconv-full
 	ICONV_FULL:=1
 
-	INTL_PREFIX:=$(STAGING_DIR)/usr/lib/libintl-full
+	INTL_PREFIX:=$(STAGING_DIR)/opt/lib/libintl-full
 	INTL_FULL:=1
 
 # iconv stub
 else
-	ICONV_PREFIX:=$(STAGING_DIR)/usr/lib/libiconv-stub
+	ICONV_PREFIX:=$(STAGING_DIR)/opt/lib/libiconv-stub
 	ICONV_FULL:=
 
-	INTL_PREFIX:=$(STAGING_DIR)/usr/lib/libintl-stub
+	INTL_PREFIX:=$(STAGING_DIR)/opt/lib/libintl-stub
 	INTL_FULL:=
 endif
 
@@ -28,7 +28,8 @@
 ICONV_DEPENDS:=+BUILD_NLS:libiconv-full
 ICONV_CFLAGS:=-I$(ICONV_PREFIX)/include
 ICONV_CPPFLAGS:=-I$(ICONV_PREFIX)/include
-ICONV_LDFLAGS:=-L$(ICONV_PREFIX)/lib
+ICONV_LDFLAGS:=-L$(ICONV_PREFIX)/lib \
+	-Wl,-rpath-link=$(ICONV_PREFIX)/lib
 
 INTL_DEPENDS:=+BUILD_NLS:libintl-full
 INTL_CFLAGS:=-I$(INTL_PREFIX)/include
