--- ../../../downloads/openwrt_trunk-r44497/include/package-defaults.mk	2015-02-20 12:39:13.649134242 +0300
+++ ../../../openwrt_trunk/include/package-defaults.mk	2015-02-20 20:26:16.855521329 +0300
@@ -69,7 +69,8 @@
 EXTRA_CXXFLAGS = $(EXTRA_CFLAGS)
 DISABLE_NLS:=--disable-nls
 
-CONFIGURE_PREFIX:=/usr
+LINUX_DIR:=$(TOPDIR)/../linux-2.6.22.19
+CONFIGURE_PREFIX:=/opt
 CONFIGURE_ARGS = \
 		--target=$(GNU_TARGET_NAME) \
 		--host=$(GNU_TARGET_NAME) \
@@ -81,9 +82,9 @@
 		--bindir=$(CONFIGURE_PREFIX)/bin \
 		--sbindir=$(CONFIGURE_PREFIX)/sbin \
 		--libexecdir=$(CONFIGURE_PREFIX)/lib \
-		--sysconfdir=/etc \
+		--sysconfdir=$(CONFIGURE_PREFIX)/etc \
 		--datadir=$(CONFIGURE_PREFIX)/share \
-		--localstatedir=/var \
+		--localstatedir=$(CONFIGURE_PREFIX)/var \
 		--mandir=$(CONFIGURE_PREFIX)/man \
 		--infodir=$(CONFIGURE_PREFIX)/info \
 		$(DISABLE_NLS) \
@@ -130,6 +131,7 @@
 
 MAKE_INSTALL_FLAGS = \
 	$(MAKE_FLAGS) \
+	MKDIR_P="mkdir -p" \
 	DESTDIR="$(PKG_INSTALL_DIR)"
 
 MAKE_PATH = .
