--- ../../../downloads/openwrt_trunk-r45513/include/package-defaults.mk	2015-04-21 10:37:38.364588029 +0300
+++ ../../../openwrt_trunk/include/package-defaults.mk	2015-04-22 09:32:18.371079124 +0300
@@ -66,14 +66,10 @@
   endef
 endif
 
-EXTRA_CXXFLAGS = $(EXTRA_CFLAGS)
-ifeq ($(CONFIG_BUILD_NLS),y)
-    DISABLE_NLS:=
-else
-    DISABLE_NLS:=--disable-nls
-endif
+DISABLE_NLS:=--disable-nls
 
-CONFIGURE_PREFIX:=/usr
+LINUX_DIR:=$(TOPDIR)/../linux-2.6.22.19
+CONFIGURE_PREFIX:=/opt
 CONFIGURE_ARGS = \
 		--target=$(GNU_TARGET_NAME) \
 		--host=$(GNU_TARGET_NAME) \
@@ -85,9 +81,9 @@
 		--bindir=$(CONFIGURE_PREFIX)/bin \
 		--sbindir=$(CONFIGURE_PREFIX)/sbin \
 		--libexecdir=$(CONFIGURE_PREFIX)/lib \
-		--sysconfdir=/etc \
+		--sysconfdir=$(CONFIGURE_PREFIX)/etc \
 		--datadir=$(CONFIGURE_PREFIX)/share \
-		--localstatedir=/var \
+		--localstatedir=$(CONFIGURE_PREFIX)/var \
 		--mandir=$(CONFIGURE_PREFIX)/man \
 		--infodir=$(CONFIGURE_PREFIX)/info \
 		$(DISABLE_NLS) \
@@ -134,6 +130,7 @@
 
 MAKE_INSTALL_FLAGS = \
 	$(MAKE_FLAGS) \
+	MKDIR_P="mkdir -p" \
 	DESTDIR="$(PKG_INSTALL_DIR)"
 
 MAKE_PATH = .
