--- ../../../downloads/openwrt_trunk-r42803/include/package-ipkg.mk
+++ ../../../openwrt_trunk/include/package-ipkg.mk
@@ -11,7 +11,7 @@
 IPKG_BUILD:= \
   ipkg-build -c -o 0 -g 0
 
-IPKG_STATE_DIR:=$(TARGET_DIR)/usr/lib/opkg
+IPKG_STATE_DIR:=$(TARGET_DIR)/opt/lib/opkg
 
 define BuildIPKGVariable
 ifdef Package/$(1)/$(2)
@@ -117,8 +117,8 @@
 
     $(eval $(call BuildIPKGVariable,$(1),conffiles))
     $(eval $(call BuildIPKGVariable,$(1),preinst))
-    $(eval $(call BuildIPKGVariable,$(1),postinst,-pkg))
-    $(eval $(call BuildIPKGVariable,$(1),prerm,-pkg))
+    $(eval $(call BuildIPKGVariable,$(1),postinst))
+    $(eval $(call BuildIPKGVariable,$(1),prerm))
     $(eval $(call BuildIPKGVariable,$(1),postrm))
 
     $(STAGING_DIR_ROOT)/stamp/.$(1)_installed: $(STAMP_BUILT)
@@ -179,18 +179,6 @@
 		echo -n "Description: "; $(SH_FUNC) getvar $(call shvar,Package/$(1)/description) | sed -e 's,^[[:space:]]*, ,g'; \
  	) > $$(IDIR_$(1))/CONTROL/control
 	chmod 644 $$(IDIR_$(1))/CONTROL/control
-	( \
-		echo "#!/bin/sh"; \
-		echo "[ \"\$$$${IPKG_NO_SCRIPT}\" = \"1\" ] && exit 0"; \
-		echo ". \$$$${IPKG_INSTROOT}/lib/functions.sh"; \
-		echo "default_postinst \$$$$0 \$$$$@"; \
-	) > $$(IDIR_$(1))/CONTROL/postinst
-	( \
-		echo "#!/bin/sh"; \
-		echo ". \$$$${IPKG_INSTROOT}/lib/functions.sh"; \
-		echo "default_prerm \$$$$0 \$$$$@"; \
-	) > $$(IDIR_$(1))/CONTROL/prerm
-	chmod 0755 $$(IDIR_$(1))/CONTROL/prerm
 	$(SH_FUNC) (cd $$(IDIR_$(1))/CONTROL; \
 		$($(1)_COMMANDS) \
 	)
@@ -202,8 +190,8 @@
 				[ -f "$$(IDIR_$(1))/$$$$x" ] || keepfiles="$$$${keepfiles:+$$$$keepfiles }$$$$x"; \
 			done; \
 			[ -z "$$$$keepfiles" ] || { \
-				mkdir -p $$(IDIR_$(1))/lib/upgrade/keep.d; \
-				for x in $$$$keepfiles; do echo $$$$x >> $$(IDIR_$(1))/lib/upgrade/keep.d/$(1); done; \
+				mkdir -p $$(IDIR_$(1))/opt/lib/upgrade/keep.d; \
+				for x in $$$$keepfiles; do echo $$$$x >> $$(IDIR_$(1))/opt/lib/upgrade/keep.d/$(1); done; \
 			}; \
 		)
     endif
