--- ../../../openwrt_trunk-r40423/include/package-defaults.mk
+++ ../../../openwrt_trunk/include/package-defaults.mk
@@ -69,7 +69,8 @@
 EXTRA_CXXFLAGS = $(EXTRA_CFLAGS)
 DISABLE_NLS:=--disable-nls
 
-CONFIGURE_PREFIX:=/usr
+LINUX_DIR:=__LINUX_DIR_PATH__
+CONFIGURE_PREFIX:=/opt
 CONFIGURE_ARGS = \
 		--target=$(GNU_TARGET_NAME) \
 		--host=$(GNU_TARGET_NAME) \
@@ -81,9 +82,9 @@
 		--bindir=$(CONFIGURE_PREFIX)/bin \
 		--sbindir=$(CONFIGURE_PREFIX)/sbin \
 		--libexecdir=$(CONFIGURE_PREFIX)/lib \
-		--sysconfdir=/etc \
+		--sysconfdir=$(CONFIGURE_PREFIX)/etc \
 		--datadir=$(CONFIGURE_PREFIX)/share \
-		--localstatedir=/var \
+		--localstatedir=$(CONFIGURE_PREFIX)/var \
 		--mandir=$(CONFIGURE_PREFIX)/man \
 		--infodir=$(CONFIGURE_PREFIX)/info \
 		$(DISABLE_NLS) \
