--- ../../../openwrt_trunk/scripts/rstrip.sh.orig
+++ ../../../openwrt_trunk/scripts/rstrip.sh
@@ -34,7 +34,7 @@
 			old_rpath="$($PATCHELF --print-rpath $F)"; new_rpath=""
 			for path in $old_rpath; do
 				case "$path" in
-					/lib/[^/]*|/usr/lib/[^/]*|\$ORIGIN/*) new_rpath="${new_rpath:+$new_rpath:}$path" ;;
+					/opt/lib/[^/]*|/opt/usr/lib/[^/]*|\$ORIGIN/*) new_rpath="${new_rpath:+$new_rpath:}$path" ;;
 					*) echo "$SELF: $F: removing rpath $path" ;;
 				esac
 			done
