--- ../../../backup/trunk_clean/trunk/feeds/packages/libs/uclibc++/Makefile	2011-11-15 14:03:59.000000000 +0400
+++ ../../../trunk/feeds/packages/libs/uclibc++/Makefile	2011-11-24 11:24:46.092076942 +0400
@@ -65,12 +65,12 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(2)/bin $(1)/usr/include/uClibc++ $(1)/usr/lib
+	$(INSTALL_DIR) $(2)/bin $(1)/opt/include/uClibc++ $(1)/opt/lib
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/g++-uc $(2)/bin/
-	$(CP) $(PKG_INSTALL_DIR)/include/* $(1)/usr/include/uClibc++/
-	$(CP) $(PKG_INSTALL_DIR)/lib/libuClibc++*.{a,so}* $(1)/usr/lib/
-	$(SED) 's|-I/include/|-I$(STAGING_DIR)/usr/include/uClibc++/|g' $(2)/bin/g++-uc
-	$(SED) 's|-L/lib/|-L$(STAGING_DIR)/usr/lib/|g' $(2)/bin/g++-uc
+	$(CP) $(PKG_INSTALL_DIR)/include/* $(1)/opt/include/uClibc++/
+	$(CP) $(PKG_INSTALL_DIR)/lib/libuClibc++*.{a,so}* $(1)/opt/lib/
+	$(SED) 's|-I/include/|-I$(STAGING_DIR)/opt/include/uClibc++/|g' $(2)/bin/g++-uc
+	$(SED) 's|-L/lib/|-L$(STAGING_DIR)/opt/lib/|g' $(2)/bin/g++-uc
 # add another wrapper which links against both uClibc++ and libstdc++ 
 	$(INSTALL_BIN) $(2)/bin/g++-uc $(2)/bin/g++-uc+std
 	$(SED) 's|^WRAPPER_INCLUDEDIR=.*||g' $(2)/bin/g++-uc+std
@@ -79,9 +79,9 @@
 endef
 
 define Package/uclibcxx/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/lib/libuClibc++.so.* $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/lib/libuClibc++-*.so $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/lib/libuClibc++.so.* $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/lib/libuClibc++-*.so $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,uclibcxx))
