--- ../../../downloads/openwrt_trunk-r42803/package/devel/binutils//Makefile	2014-11-07 23:35:26.231309440 +0300
+++ ../../../openwrt_trunk/package/devel/binutils//Makefile	2014-11-26 13:57:24.774797310 +0300
@@ -62,21 +62,20 @@
 endef
 
 define Build/InstallDev
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/* $(1)/
 endef
 
 define Package/objdump/install
-	$(INSTALL_DIR) $(1)/usr/bin $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/objdump $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libopcodes*.so $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libbfd*.so $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/bin $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/objdump $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libopcodes*.so $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libbfd*.so $(1)/opt/lib/
 endef
 
 define Package/binutils/install
-	$(INSTALL_DIR) $(1)/usr $(1)/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ $(1)/usr/
-	mv $(1)/usr/bin/strings $(1)/bin/strings
-	rm -f $(1)/usr/bin/objdump
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/ $(1)/opt
+	rm -f $(1)/opt/bin/objdump
 endef
 
 $(eval $(call BuildPackage,binutils))
