--- ../../../downloads/openwrt_trunk-r42803/package/devel/binutils/Makefile
+++ ../../../openwrt_trunk/package/devel/binutils/Makefile
@@ -66,17 +66,16 @@
 endef
 
 define Package/objdump/install
-	$(INSTALL_DIR) $(1)/usr/bin $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/objdump $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libopcodes*.so $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libbfd*.so $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/bin $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/objdump $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libopcodes*.so $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libbfd*.so $(1)/opt/lib/
 endef
 
 define Package/binutils/install
-	$(INSTALL_DIR) $(1)/usr $(1)/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ $(1)/usr/
-	mv $(1)/usr/bin/strings $(1)/bin/strings
-	rm -f $(1)/usr/bin/objdump
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/ $(1)/opt
+	rm -f $(1)/opt/bin/objdump
 endef
 
 $(eval $(call BuildPackage,binutils))
