--- ../../../downloads/openwrt_trunk-r43718/package/devel/binutils/Makefile	2015-01-09 19:51:38.507174880 +0300
+++ ../../../openwrt_trunk/package/devel/binutils/Makefile	2015-01-09 23:21:43.624192454 +0300
@@ -25,6 +25,10 @@
 PKG_LICENSE:=GPL-3.0+
 PKG_BUILD_PARALLEL:=1
 
+# Dirty trick!!! gdb compilation fails, if compiled after binutils (system bug with include file ansidecl.h)
+# So this depenancy must help
+PKG_BUILD_DEPENDS:=gdb
+
 include $(INCLUDE_DIR)/package.mk
 
 define Package/binutils
@@ -62,21 +66,20 @@
 endef
 
 define Build/InstallDev
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/* $(1)/
 endef
 
 define Package/objdump/install
-	$(INSTALL_DIR) $(1)/usr/bin $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/objdump $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libopcodes*.so $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libbfd*.so $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/bin $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/objdump $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libopcodes*.so $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libbfd*.so $(1)/opt/lib/
 endef
 
 define Package/binutils/install
-	$(INSTALL_DIR) $(1)/usr $(1)/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ $(1)/usr/
-	mv $(1)/usr/bin/strings $(1)/bin/strings
-	rm -f $(1)/usr/bin/objdump
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/ $(1)/opt
+	rm -f $(1)/opt/bin/objdump
 endef
 
 $(eval $(call BuildPackage,binutils))
