--- ../../../trunk_clean/feeds/packages/utils/smartmontools/Makefile	2012-05-02 16:17:34.874281671 +0400
+++ ../../../trunk/feeds/packages/utils/smartmontools/Makefile	2012-06-01 08:49:19.325463359 +0400
@@ -54,7 +54,7 @@
 
 CONFIGURE_VARS += \
 	CXXFLAGS="$$$$CXXFLAGS -fno-builtin -fno-rtti -nostdinc++" \
-	CPPFLAGS="$$$$CPPFLAGS -I$(STAGING_DIR)/usr/include/uClibc++ -I$(LINUX_DIR)/include" \
+	CPPFLAGS="$$$$CPPFLAGS -I$(STAGING_DIR)/opt/include/uClibc++ -I$(LINUX_DIR)/include" \
 	LDFLAGS="$$$$LDFLAGS" \
 	LIBS="-nodefaultlibs -lc -luClibc++ -lm $(LIBGCC_S) -lc" \
 
@@ -71,21 +71,19 @@
 endef
 
 define Package/smartmontools/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/smartctl $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/smartctl $(1)/opt/bin/
 endef
 
 define Package/smartd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/smartd $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/smartd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/smartd.init $(1)/etc/init.d/smartd
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/smartd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S20smartmontools $(1)/opt/etc/init.d
 endef
 
 define Package/smartd/conffiles
-/etc/smartd.conf
+/opt/etc/smartd.conf
 endef
 
 $(eval $(call BuildPackage,smartmontools))
