diff -r -u -x .svn ../../../openwrt_trunk.orig/feeds/packages/sound/mpc/files/pls-handler.sh ../../../openwrt_trunk/feeds/packages/sound/mpc/files/pls-handler.sh
--- ../../../openwrt_trunk.orig/feeds/packages/sound/mpc/files/pls-handler.sh	2011-12-05 10:46:40.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/sound/mpc/files/pls-handler.sh	2011-12-09 14:30:36.960785917 +0400
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/bin/sh
 echo parameter: file.pls
 mpc clear
 grep '^File[0-9]*' $1 | sed -e 's/^File[0-9]*=//' | mpc add
diff -r -u -x .svn ../../../openwrt_trunk.orig/feeds/packages/sound/mpc/Makefile ../../../openwrt_trunk/feeds/packages/sound/mpc/Makefile
--- ../../../openwrt_trunk.orig/feeds/packages/sound/mpc/Makefile	2011-12-05 10:46:40.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/sound/mpc/Makefile	2011-12-09 14:30:55.328553086 +0400
@@ -44,15 +44,15 @@
 define Build/Compile
 $(call Build/Compile/Default,\
 		CFLAGS="$(TARGET_CFLAGS) -std=gnu99" \
-		prefix="/usr" \
+		prefix="/opt" \
 		all \
 )
 endef
 
 define Package/mpc/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/mpc $(1)/usr/bin/
-	$(INSTALL_BIN) ./files/pls-handler.sh $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/mpc $(1)/opt/bin/
+	$(INSTALL_BIN) ./files/pls-handler.sh $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,mpc))
