--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/tmux/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/tmux/Makefile
@@ -34,12 +34,12 @@
 endef
 
 MAKE_FLAGS += \
-	PREFIX="/usr" \
+	PREFIX="/opt" \
 	PLATFORM="linux"
 
 define Package/tmux/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/tmux $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/tmux $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,tmux))
