--- ../../../downloads/openwrt_trunk-r46338/feeds/packages/utils/tmux/Makefile	2015-07-14 10:55:19.834308174 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/tmux/Makefile	2015-07-14 10:55:54.918308758 +0300
@@ -37,12 +37,12 @@
 endef
 
 MAKE_FLAGS += \
-	PREFIX="/usr" \
+	PREFIX="/opt" \
 	PLATFORM="linux"
 
 define Package/tmux/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/tmux $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/tmux $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,tmux))
