--- ../../../openwrt_trunk.orig/feeds/packages/utils/tmux/Makefile	2012-02-09 12:02:01.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/tmux/Makefile	2012-02-09 12:06:19.212736323 +0400
@@ -32,12 +32,12 @@
 endef
 
 MAKE_FLAGS += \
-	PREFIX="/usr" \
+	PREFIX="/opt" \
 	PLATFORM="linux"
 
 define Package/tmux/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/tmux $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/tmux $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,tmux))
