--- ../../../downloads/openwrt_trunk-r46338/feeds/packages/utils/tmux/Makefile	2015-07-15 16:48:18.960098714 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/tmux/Makefile	2015-07-17 13:41:02.234786515 +0300
@@ -12,7 +12,7 @@
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=@SF/tmux
+PKG_SOURCE_URL:=https://github.com/tmux/tmux/releases/download/$(PKG_VERSION)/
 PKG_MD5SUM:=9fb6b443392c3978da5d599f1e814eaa
 PKG_MAINTAINER:=Maxim Storchak <m.storchak@gmail.com>
 
@@ -37,12 +37,12 @@
 endef
 
 MAKE_FLAGS += \
-	PREFIX="/usr" \
+	PREFIX="/opt" \
 	PLATFORM="linux"
 
 define Package/tmux/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/tmux $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/tmux $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,tmux))
