--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/libs/gnutls/Makefile	2015-04-20 10:09:28.898135362 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/gnutls/Makefile	2015-04-20 12:39:50.781967854 +0300
@@ -123,9 +123,9 @@
 	--disable-doc \
 	--disable-tests \
 	--disable-rsa-export \
-	--with-default-trust-store-dir=/etc/ssl/certs/ \
+	--with-default-trust-store-dir=/opt/etc/ssl/certs/ \
 	--disable-crywrap \
-	--with-librt-prefix="$(STAGING_DIR)/"
+	--with-librt-prefix="$(LIBRT_ROOT_DIR)/"
 
 ifneq ($(CONFIG_GNUTLS_EXT_LIBTASN1),y)
 CONFIGURE_ARGS += --with-included-libtasn1
@@ -178,69 +178,69 @@
 TARGET_CFLAGS += $(FPIC)
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib/pkgconfig
+	$(INSTALL_DIR) $(1)/opt/include $(1)/opt/lib/pkgconfig
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/gnutls \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/gnutls \
+		$(1)/opt/include/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libgnutls{,-openssl}.{a,so*} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libgnutls{,-openssl}.{a,so*} \
+		$(1)/opt/lib/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/gnutls.pc \
-		$(1)/usr/lib/pkgconfig/
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/gnutls.pc \
+		$(1)/opt/lib/pkgconfig/
 endef
 
 define Package/certtool/conffiles
-/etc/gnutls/certtool.cfg
+/opt/etc/gnutls/certtool.cfg
 endef
 
 define Package/certtool/install
-	$(INSTALL_DIR) $(1)/etc/gnutls
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/doc/certtool.cfg $(1)/etc/gnutls/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/certtool $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/gnutls
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/doc/certtool.cfg $(1)/opt/etc/gnutls/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/certtool $(1)/opt/bin/
 endef
 
 
 define Package/gnutls-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 ifeq ($(CONFIG_GNUTLS_OCSP),y)
 ifeq ($(CONFIG_GNUTLS_ANON),y)
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/gnutls-{cli,serv} \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/gnutls-{cli,serv} \
+		$(1)/opt/bin/
 endif
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/ocsptool \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/ocsptool \
+		$(1)/opt/bin/
 endif
 ifeq ($(CONFIG_GNUTLS_SRP),y)
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/srptool \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/srptool \
+		$(1)/opt/bin/
 endif
 ifeq ($(CONFIG_GNUTLS_PSK),y)
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/psktool \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/psktool \
+		$(1)/opt/bin/
 endif
 ifeq ($(CONFIG_GNUTLS_PKCS11),y)
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/p11tool \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/p11tool \
+		$(1)/opt/bin/
 endif
 endef
 
 
 define Package/libgnutls/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libgnutls.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libgnutls.so.* $(1)/opt/lib/
 endef
 
 
 define Package/libgnutls-openssl/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libgnutls-openssl.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libgnutls-openssl.so.* $(1)/opt/lib/
 endef
 
 
