--- ../../openwrt_trunk.orig/scripts/download.pl	2012-06-02 15:57:48.000000000 +0400
+++ ../../openwrt_trunk/scripts/download.pl	2012-06-02 16:15:12.000000000 +0400
@@ -194,6 +194,7 @@
 #push @mirrors, 'http://mirror1.openwrt.org';
 push @mirrors, 'http://mirror2.openwrt.org/sources';
 push @mirrors, 'http://downloads.openwrt.org/sources';
+push @mirrors, 'http://x.vm0.ru/wl500g-repo/sources-mirror';
 
 while (!$ok) {
 	my $mirror = shift @mirrors;
