--- ../../openwrt_trunk.orig/include/package-defaults.mk	2012-07-10 18:45:09.783948620 +0400
+++ ../../openwrt_trunk/include/package-defaults.mk	2012-07-10 18:51:48.806938029 +0400
@@ -63,7 +63,8 @@
 
 EXTRA_CXXFLAGS = $(EXTRA_CFLAGS)
 
-CONFIGURE_PREFIX:=/usr
+LINUX_DIR:=__LINUX_DIR_PATH__
+CONFIGURE_PREFIX:=/opt
 CONFIGURE_ARGS = \
 		--target=$(GNU_TARGET_NAME) \
 		--host=$(GNU_TARGET_NAME) \
@@ -75,9 +76,9 @@
 		--bindir=$(CONFIGURE_PREFIX)/bin \
 		--sbindir=$(CONFIGURE_PREFIX)/sbin \
 		--libexecdir=$(CONFIGURE_PREFIX)/lib \
-		--sysconfdir=/etc \
+		--sysconfdir=$(CONFIGURE_PREFIX)/etc \
 		--datadir=$(CONFIGURE_PREFIX)/share \
-		--localstatedir=/var \
+		--localstatedir=$(CONFIGURE_PREFIX)/var \
 		--mandir=$(CONFIGURE_PREFIX)/man \
 		--infodir=$(CONFIGURE_PREFIX)/info \
 		$(DISABLE_NLS) \
