--- ../../openwrt_trunk.orig/include/package-ipkg.mk	2012-10-08 17:25:13.751248572 +0400
+++ ../../openwrt_trunk/include/package-ipkg.mk	2012-10-08 17:26:05.407247631 +0400
@@ -9,7 +9,7 @@
 IPKG_BUILD:= \
   ipkg-build -c -o 0 -g 0
 
-IPKG_STATE_DIR:=$(TARGET_DIR)/usr/lib/opkg
+IPKG_STATE_DIR:=$(TARGET_DIR)/opt/lib/opkg
 
 define BuildIPKGVariable
 ifdef Package/$(1)/$(2)
@@ -170,8 +170,8 @@
 				[ -f "$$(IDIR_$(1))/$$$$x" ] || keepfiles="$$$${keepfiles:+$$$$keepfiles }$$$$x"; \
 			done; \
 			[ -z "$$$$keepfiles" ] || { \
-				mkdir -p $$(IDIR_$(1))/lib/upgrade/keep.d; \
-				for x in $$$$keepfiles; do echo $$$$x >> $$(IDIR_$(1))/lib/upgrade/keep.d/$(1); done; \
+				mkdir -p $$(IDIR_$(1))/opt/lib/upgrade/keep.d; \
+				for x in $$$$keepfiles; do echo $$$$x >> $$(IDIR_$(1))/opt/lib/upgrade/keep.d/$(1); done; \
 			}; \
 		)
     endif
