--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/openssh/Makefile
+++ ../../../openwrt_trunk/feeds/packages/net/openssh/Makefile
@@ -57,7 +57,7 @@
 endef
 
 define Package/openssh-client/conffiles
-/etc/ssh/ssh_config
+/opt/etc/ssh/ssh_config
 endef
 
 define Package/openssh-client-utils
@@ -90,7 +90,7 @@
 endef
 
 define Package/openssh-server/conffiles
-/etc/ssh/sshd_config
+/opt/etc/ssh/sshd_config
 endef
 
 define Package/openssh-server-pam
@@ -105,9 +105,9 @@
 endef
 
 define Package/openssh-server-pam/conffiles
-/etc/pam.d/sshd
-/etc/security/access-sshd-local.conf
-/etc/ssh/sshd_config
+/opt/etc/pam.d/sshd
+/opt/etc/security/access-sshd-local.conf
+/opt/etc/ssh/sshd_config
 endef
 
 define Package/openssh-sftp-client
@@ -131,9 +131,9 @@
 
 CONFIGURE_ARGS+= \
 	$(DISABLE_NLS) \
-	--sysconfdir=/etc/ssh \
+	--sysconfdir=/opt/etc/ssh \
 	--with-privsep-user=sshd \
-	--with-privsep-path=/var/empty \
+	--with-privsep-path=/opt/var/empty \
 	--enable-shared \
 	--disable-static \
 	--disable-debug \
@@ -183,78 +183,60 @@
 		all install
 endef
 
-define Package/openssh-client/preinst
-#!/bin/sh
-if [ -L $${IPKG_INSTROOT}/usr/bin/ssh ] && [ -L $${IPKG_INSTROOT}/usr/bin/scp ]; then
-	rm -f $${IPKG_INSTROOT}/usr/bin/ssh $${IPKG_INSTROOT}/usr/bin/scp;
-fi
-exit 0
-endef
-
-define Package/openssh-client/postrm
-#!/bin/sh
-rm -f $${IPKG_INSTROOT}/usr/bin/ssh $${IPKG_INSTROOT}/usr/bin/scp;
-if [ -x $${IPKG_INSTROOT}/usr/sbin/dropbear ] ; then
-	ln -s /usr/sbin/dropbear $${IPKG_INSTROOT}/usr/bin/ssh;
-	ln -s /usr/sbin/dropbear $${IPKG_INSTROOT}/usr/bin/scp;
-fi
-exit 0
-endef
-
 define Package/openssh-moduli/install
-	$(INSTALL_DIR) $(1)/etc/ssh
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/etc/ssh/moduli $(1)/etc/ssh/
+	$(INSTALL_DIR) $(1)/opt/etc/ssh
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/etc/ssh/moduli $(1)/opt/etc/ssh/
 endef
 
 define Package/openssh-client/install
-	$(INSTALL_DIR) $(1)/etc/ssh
-	chmod 0700 $(1)/etc/ssh
-	$(CP) $(PKG_INSTALL_DIR)/etc/ssh/ssh_config $(1)/etc/ssh/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ssh $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/scp $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/ssh
+	chmod 0700 $(1)/opt/etc/ssh
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/ssh/ssh_config $(1)/opt/etc/ssh/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ssh $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/scp $(1)/opt/bin/
 endef
 
 define Package/openssh-client-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(foreach bin,add agent keyscan keysign,$(PKG_BUILD_DIR)/ssh-$(bin)) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(foreach bin,add agent keyscan keysign,$(PKG_BUILD_DIR)/ssh-$(bin)) $(1)/opt/bin/
 endef
 
 define Package/openssh-keygen/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ssh-keygen $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ssh-keygen $(1)/opt/bin/
 endef
 
 define Package/openssh-server/install
-	$(INSTALL_DIR) $(1)/etc/ssh
-	chmod 0700 $(1)/etc/ssh
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/etc/ssh/sshd_config $(1)/etc/ssh/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/sshd.init $(1)/etc/init.d/sshd
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/sshd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/ssh
+	chmod 0700 $(1)/opt/etc/ssh
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/etc/ssh/sshd_config $(1)/opt/etc/ssh/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S40sshd $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/sshd $(1)/opt/sbin/
 endef
 
 define Package/openssh-server-pam/install
 	$(call Package/openssh-server/install,$(1))
-	sed -i 's,#PasswordAuthentication yes,PasswordAuthentication no,g' $(1)/etc/ssh/sshd_config
-	sed -i 's,#UsePAM no,UsePAM yes,g' $(1)/etc/ssh/sshd_config
-	$(INSTALL_DIR) $(1)/etc/pam.d
-	$(INSTALL_DATA) ./files/sshd.pam $(1)/etc/pam.d/sshd
-	$(INSTALL_DIR) $(1)/etc/security
-	$(INSTALL_DATA) ./files/sshd.pam-access $(1)/etc/security/access-sshd-local.conf
+	sed -i 's,#PasswordAuthentication yes,PasswordAuthentication no,g' $(1)/opt/etc/ssh/sshd_config
+	sed -i 's,#UsePAM no,UsePAM yes,g' $(1)/opt/etc/ssh/sshd_config
+	$(INSTALL_DIR) $(1)/opt/etc/pam.d
+	$(INSTALL_DATA) ./files/sshd.pam $(1)/opt/etc/pam.d/sshd
+	$(INSTALL_DIR) $(1)/opt/etc/security
+	$(INSTALL_DATA) ./files/sshd.pam-access $(1)/opt/etc/security/access-sshd-local.conf
 endef
 
 define Package/openssh-sftp-client/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/sftp $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/sftp $(1)/opt/bin/
 endef
 
 define Package/openssh-sftp-server/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/sftp-server $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/libexec
-	ln -sf ../lib/sftp-server $(1)/usr/libexec/sftp-server
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/sftp-server $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/libexec
+	ln -sf ../lib/sftp-server $(1)/opt/libexec/sftp-server
 endef
 
 $(eval $(call BuildPackage,openssh-client))
