--- ../../../downloads/openwrt_trunk-r43718/package/libs/toolchain/Makefile	2015-01-05 20:26:58.701408722 +0300
+++ ../../../openwrt_trunk/package/libs/toolchain/Makefile	2015-01-08 11:08:09.831129632 +0300
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2007-2014 OpenWrt.org
+# Copyright (C) 2007-2012 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
@@ -7,19 +7,15 @@
 
 include $(TOPDIR)/rules.mk
 PKG_NAME:=toolchain
-PKG_RELEASE:=1
+PKG_RELEASE:=8a
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@openwrt.org>
 PKG_LICENSE:=GPL-3.0-with-GCC-exception
 
 include $(INCLUDE_DIR)/package.mk
 
-ifneq ($(DUMP),1)
-  LIBGCC_VERSION:=$(GCC_VERSION)
-else
-  LIBC_VERSION:=<LIBC_VERSION>
-  LIBGCC_VERSION:=<LIBGCC_VERSION>
-endif
+LIBC_VERSION:=2.20
+LIBGCC_VERSION:=4.8.3
 
 define Package/gcc/Default
   SECTION:=libs
@@ -28,6 +24,62 @@
   VERSION:=$(LIBGCC_VERSION)-$(PKG_RELEASE)
 endef
 
+define Package/locales
+$(call Package/gcc/Default)
+  TITLE:=Locales support (independent of system lib)
+  VERSION:=$(LIBC_VERSION)-$(PKG_RELEASE)
+endef
+
+define Package/locales/config
+	menu "Configuration"
+		depends on EXTERNAL_TOOLCHAIN && PACKAGE_locales
+
+	config TOOLCHAIN_USR_DIR
+		string
+		prompt "External toolchain usr directory"
+		depends on EXTERNAL_TOOLCHAIN && PACKAGE_locales
+
+	config LOCALE_PROGS_SPEC
+		string
+		prompt "programs to work with locales"
+		depends on EXTERNAL_TOOLCHAIN && PACKAGE_locales
+		default "./bin/locale ./bin/localedef"
+
+	config i18N_LOCALES
+		string
+		prompt "locale sources to install"
+		depends on EXTERNAL_TOOLCHAIN && PACKAGE_locales
+
+	config i18N_CHARMAPS
+		string
+		prompt "charmaps to install"
+		depends on EXTERNAL_TOOLCHAIN && PACKAGE_locales
+
+	endmenu
+endef
+
+define Package/locales/postinst
+#!/bin/sh
+
+echo "qnapware uses separate locale-archive file independent from main system"
+if ! [ -f '/Apps/opt/usr/lib/locale/locale-archive' ]
+then
+	mkdir -p /Apps/opt/usr/lib/locale
+        echo "Creating locale archive - /Apps/opt/usr/lib/locale/locale-archive"
+        echo "Adding en_EN.UTF-8"
+        /Apps/opt/bin/localedef.new -c -f UTF-8 -i en_US en_US.UTF-8
+        echo "Adding ru_RU.UTF-8"
+        /Apps/opt/bin/localedef.new -c -f UTF-8 -i ru_RU ru_RU.UTF-8
+fi
+if [ -f '/Apps/opt/usr/lib/locale/locale-archive' ]
+then
+        echo "/Apps/opt/usr/lib/locale/locale-archive found"
+        echo "You can download locale sources from http://qnapware.zyxmon.org/sources/i18n.tar.gz"
+        echo "You can add new locales for qnapware using /Apps/opt/bin/localedef.new"
+fi
+endef
+
+
 define Package/libgcc
 $(call Package/gcc/Default)
   TITLE:=GCC support library
@@ -81,7 +133,6 @@
 
 define Package/libssp
 $(call Package/gcc/Default)
-  DEPENDS+=@SSP_SUPPORT
   TITLE:=GCC support library
 endef
 
@@ -139,7 +190,6 @@
   VERSION:=$(LIBC_VERSION)-$(PKG_RELEASE)
   DEPENDS:=+libgcc
   URL:=$(LIBC_URL)
-  PKG_FLAGS:=hold essential
 endef
 
 
@@ -402,7 +452,7 @@
 
   define Package/libatomic/install
 	$(INSTALL_DIR) $(1)/lib
-	$(if $(CONFIG_GCC_VERSION_4_6)$(CONFIG_TARGET_avr32)$(CONFIG_TARGET_coldfire),,$(CP) $(TOOLCHAIN_DIR)/lib/libatomic.so.* $(1)/lib/)
+	$(if $(CONFIG_TARGET_avr32)$(CONFIG_TARGET_coldfire),,$(CP) $(TOOLCHAIN_DIR)/lib/libatomic.so.* $(1)/lib/)
   endef
 
   define Package/libgfortran/install
@@ -427,7 +477,7 @@
   define Package/eglibc/install
 	$(CP) ./eglibc-files/* $(1)/
 	rm -f $(1)/etc/localtime
-	$(LN) /tmp/localtime $(1)/etc/localtime
+	ln -sf /tmp/localtime $(1)/etc/localtime
 	$(INSTALL_DIR) $(1)/lib
 	$(CP) \
 		$(TOOLCHAIN_DIR)/lib/ld*.so.* \
@@ -452,7 +502,7 @@
 		$(TOOLCHAIN_DIR)/lib/libc.so.* \
 		$(TOOLCHAIN_DIR)/lib/libuClibc-$(LIBC_SO_VERSION).so \
 		$(1)/lib/
-	for file in libcrypt libdl libm libutil; do \
+	for file in libatomic libcrypt libdl libm libutil; do \
 		$(CP) \
 			$(TOOLCHAIN_DIR)/lib/$$$$file.so.* \
 			$(TOOLCHAIN_DIR)/lib/$$$$file-$(LIBC_SO_VERSION).so \
@@ -466,17 +516,17 @@
 		$(1)/lib/
   endef
 
-  LD_MUSL_NAME = $(notdir $(firstword $(wildcard $(TOOLCHAIN_DIR)/lib/libc.so*)))
+  LD_MUSL_NAME = $(notdir $(firstword $(wildcard $(TOOLCHAIN_DIR)/lib/ld-musl-*.so*)))
 
   define Package/musl/install
-	$(INSTALL_DIR) $(1)/lib $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/lib
 	$(CP) \
 		$(TOOLCHAIN_DIR)/lib/ld-musl-*.so* \
 		$(1)/lib/
 	$(CP) \
 		$(TOOLCHAIN_DIR)/lib/libc.so* \
 		$(1)/lib/
-	$(LN) ../../lib/$(LD_MUSL_NAME) $(1)/usr/bin/ldd
+	$(if $(findstring -sf.so,$(LD_MUSL_NAME)),ln -s $(LD_MUSL_NAME) $(1)/lib/$(subst -sf.so,.so,$(LD_MUSL_NAME)))
   endef
 
   define Package/libc/install
@@ -545,8 +595,8 @@
   define Package/libgcc/install
 	for file in $(call qstrip,$(CONFIG_LIBGCC_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBGCC_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+		$(CP) $(call qstrip,$(CONFIG_LIBGCC_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
 	done ; \
 	exit 0
   endef
@@ -554,16 +604,16 @@
   define Package/libgfortran/install
 	for file in $(call qstrip,$(CONFIG_LIBGFORTRAN_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBGFORTRAN_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+		$(CP) $(call qstrip,$(CONFIG_LIBGFORTRAN_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
 	done
   endef
 
   define Package/libssp/install
 	for file in $(call qstrip,$(CONFIG_LIBSSP_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBSSP_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+		$(CP) $(call qstrip,$(CONFIG_LIBSSP_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
 	done ; \
 	exit 0
   endef
@@ -571,17 +621,18 @@
   define Package/libstdcpp/install
 	for file in $(call qstrip,$(CONFIG_LIBSTDCPP_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBSTDCPP_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+		$(CP) $(call qstrip,$(CONFIG_LIBSTDCPP_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
 	done ; \
+	rm -f $(1)/opt/$$$$dir/*.py ; \
 	exit 0
   endef
 
   define Package/libc/install
 	for file in $(call qstrip,$(CONFIG_LIBC_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBC_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+		$(CP) $(call qstrip,$(CONFIG_LIBC_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
 	done ; \
 	exit 0
   endef
@@ -589,17 +640,22 @@
   define Package/libpthread/install
 	for file in $(call qstrip,$(CONFIG_LIBPTHREAD_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBPTHREAD_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+		$(CP) $(call qstrip,$(CONFIG_LIBPTHREAD_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
 	done ; \
 	exit 0
   endef
 
+  define Package/libthread-db/install
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(call qstrip,$(CONFIG_LIBPTHREAD_ROOT_DIR))/lib/libthread_db{-*.so,.so.*} $(1)/opt/lib/
+  endef
+
   define Package/librt/install
 	for file in $(call qstrip,$(CONFIG_LIBRT_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBRT_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+		$(CP) $(call qstrip,$(CONFIG_LIBRT_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
 	done ; \
 	exit 0
   endef
@@ -607,17 +663,41 @@
   define Package/ldd/install
 	for file in $(call qstrip,$(CONFIG_LDD_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LDD_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/bin ; \
+		$(CP) $(call qstrip,$(CONFIG_LDD_ROOT_DIR))/$$$$file $(1)/opt/bin/ ; \
 	done ; \
+	$(SED) 's!/bin/bash!/bin/sh!g' $(1)/opt/bin/ldd ; \
+	$(SED) 's!/lib/ld-linux.so.3!/Apps/opt/lib/ld-linux.so.3!g' $(1)/opt/bin/ldd ; \
 	exit 0
   endef
 
   define Package/ldconfig/install
 	for file in $(call qstrip,$(CONFIG_LDCONFIG_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LDCONFIG_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/bin ; \
+		$(CP) $(call qstrip,$(CONFIG_LDCONFIG_ROOT_DIR))/$$$$file $(1)/opt/bin/ ; \
+	done ; \
+	exit 0
+  endef
+
+  define Package/locales/install
+	$(INSTALL_DIR) $(1)/opt/usr/lib/locale; \
+	for file in $(call qstrip,$(CONFIG_LOCALE_PROGS_SPEC)); do \
+		dir=`dirname $$$$file` ; \
+		$(INSTALL_DIR) $(1)/opt/bin ; \
+		$(CP) $(call qstrip,$(CONFIG_TOOLCHAIN_USR_DIR))/$$$$file $(1)/opt/bin/ ; \
+	done ; \
+	mv $(1)/opt/bin/locale $(1)/opt/bin/locale.new; \
+	mv $(1)/opt/bin/localedef $(1)/opt/bin/localedef.new; \
+	for file in $(call qstrip,$(CONFIG_i18N_LOCALES)); do \
+		dir=`dirname $$$$file` ; \
+		$(INSTALL_DIR) $(1)/opt/usr/share/i18n/locales; \
+		$(CP) $(call qstrip,$(CONFIG_TOOLCHAIN_USR_DIR))/$$$$file $(1)/opt/usr/share/i18n/locales ; \
+	done ; \
+	for file in $(call qstrip,$(CONFIG_i18N_CHARMAPS)); do \
+		dir=`dirname $$$$file` ; \
+		$(INSTALL_DIR) $(1)/opt/usr/share/i18n/charmaps; \
+		$(CP) $(call qstrip,$(CONFIG_TOOLCHAIN_USR_DIR))/$$$$file $(1)/opt/usr/share/i18n/charmaps ; \
 	done ; \
 	exit 0
   endef
@@ -635,3 +715,4 @@
 $(eval $(call BuildPackage,libgfortran))
 $(eval $(call BuildPackage,ldd))
 $(eval $(call BuildPackage,ldconfig))
+$(eval $(call BuildPackage,locales))
