--- ../../../downloads/openwrt_trunk-r44486/package/network/utils/comgt/Makefile	2015-02-18 12:16:58.292306319 +0300
+++ ../../../openwrt_trunk/package/network/utils/comgt/Makefile	2015-02-18 16:54:35.176002899 +0300
@@ -31,14 +31,13 @@
 define Package/comgt
 $(call Package/comgt/Default)
   TITLE:=Option/Vodafone 3G/GPRS control tool
-  DEPENDS:=+chat
   URL:=http://www.pharscape.org/comgt.html
 endef
 
 define Package/comgt-directip
 $(call Package/comgt/Default)
   TITLE:=Sierra Wireless Direct-IP support
-  DEPENDS:=+comgt +kmod-usb-serial +kmod-usb-serial-sierrawireless +kmod-usb-net +kmod-usb-net-sierrawireless
+  DEPENDS:=+comgt
 endef
 
 define Package/comgt-ncm
@@ -57,48 +56,39 @@
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		$(TARGET_CONFIGURE_OPTS) \
 		CFLAGS="$(TARGET_CFLAGS)" \
-		LDFLAGS="" \
+		LDFLAGS="$(TARGET_LDFLAGS)" \
 		comgt
 endef
 
 define Package/comgt/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/comgt $(1)/usr/bin/
-	ln -s comgt $(1)/usr/bin/gcom
-	$(INSTALL_DIR) $(1)/etc/chatscripts
-	$(INSTALL_DATA) ./files/3g.chat $(1)/etc/chatscripts/3g.chat
-	$(INSTALL_DATA) ./files/evdo.chat $(1)/etc/chatscripts/evdo.chat
-	$(INSTALL_DIR) $(1)/etc/gcom
-	$(INSTALL_DATA) ./files/setpin.gcom $(1)/etc/gcom/setpin.gcom
-	$(INSTALL_DATA) ./files/setmode.gcom $(1)/etc/gcom/setmode.gcom
-	$(INSTALL_DATA) ./files/getcardinfo.gcom $(1)/etc/gcom/getcardinfo.gcom
-	$(INSTALL_DATA) ./files/getstrength.gcom $(1)/etc/gcom/getstrength.gcom
-	$(INSTALL_DATA) ./files/getcarrier.gcom $(1)/etc/gcom/getcarrier.gcom
-	$(INSTALL_DATA) ./files/getcnum.gcom $(1)/etc/gcom/getcnum.gcom
-	$(INSTALL_DATA) ./files/getimsi.gcom $(1)/etc/gcom/getimsi.gcom
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/tty
-	$(INSTALL_DATA) ./files/3g.usb $(1)/etc/hotplug.d/tty/30-3g
-	$(INSTALL_DIR) $(1)/lib/netifd/proto
-	$(INSTALL_BIN) ./files/3g.sh $(1)/lib/netifd/proto/3g.sh
-	$(INSTALL_BIN) ./files/directip.sh $(1)/lib/netifd/proto/directip.sh
-	$(INSTALL_DIR) $(1)/lib/network/wwan/
-	$(INSTALL_DATA) ./files/data/* $(1)/lib/network/wwan/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/comgt $(1)/opt/bin/
+	ln -s comgt $(1)/opt/bin/gcom
+	$(INSTALL_DIR) $(1)/opt/etc/chatscripts
+	$(INSTALL_DATA) ./files/3g.chat $(1)/opt/etc/chatscripts/3g.chat
+	$(INSTALL_DATA) ./files/evdo.chat $(1)/opt/etc/chatscripts/evdo.chat
+	$(INSTALL_DIR) $(1)/opt/etc/gcom
+	$(INSTALL_DATA) ./files/setpin.gcom $(1)/opt/etc/gcom/setpin.gcom
+	$(INSTALL_DATA) ./files/setmode.gcom $(1)/opt/etc/gcom/setmode.gcom
+	$(INSTALL_DATA) ./files/getcardinfo.gcom $(1)/opt/etc/gcom/getcardinfo.gcom
+	$(INSTALL_DATA) ./files/getstrength.gcom $(1)/opt/etc/gcom/getstrength.gcom
+	$(INSTALL_DATA) ./files/getcarrier.gcom $(1)/opt/etc/gcom/getcarrier.gcom
+	$(INSTALL_DATA) ./files/getcnum.gcom $(1)/opt/etc/gcom/getcnum.gcom
+	$(INSTALL_DATA) ./files/getimsi.gcom $(1)/opt/etc/gcom/getimsi.gcom
+	$(INSTALL_DIR) $(1)/opt/lib/network/wwan/
+	$(INSTALL_DATA) ./files/data/* $(1)/opt/lib/network/wwan/
 endef
 
 define Package/comgt-directip/install
-	$(INSTALL_DIR) $(1)/etc/gcom
-	$(INSTALL_DATA) ./files/directip.gcom $(1)/etc/gcom/directip.gcom
-	$(INSTALL_DATA) ./files/directip-stop.gcom $(1)/etc/gcom/directip-stop.gcom
-	$(INSTALL_DIR) $(1)/lib/netifd/proto
-	$(INSTALL_BIN) ./files/directip.sh $(1)/lib/netifd/proto/directip.sh
+	$(INSTALL_DIR) $(1)/opt/etc/gcom
+	$(INSTALL_DATA) ./files/directip.gcom $(1)/opt/etc/gcom/directip.gcom
+	$(INSTALL_DATA) ./files/directip-stop.gcom $(1)/opt/etc/gcom/directip-stop.gcom
 endef
 
 define Package/comgt-ncm/install
-	$(INSTALL_DIR) $(1)/etc/gcom
-	$(INSTALL_DATA) ./files/ncm.json $(1)/etc/gcom/ncm.json
-	$(INSTALL_DATA) ./files/runcommand.gcom $(1)/etc/gcom/runcommand.gcom
-	$(INSTALL_DIR) $(1)/lib/netifd/proto
-	$(INSTALL_BIN) ./files/ncm.sh $(1)/lib/netifd/proto/ncm.sh
+	$(INSTALL_DIR) $(1)/opt/etc/gcom
+	$(INSTALL_DATA) ./files/ncm.json $(1)/opt/etc/gcom/ncm.json
+	$(INSTALL_DATA) ./files/runcommand.gcom $(1)/opt/etc/gcom/runcommand.gcom
 endef
 
 $(eval $(call BuildPackage,comgt))
