--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/mail/msmtp/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/mail/msmtp/Makefile
@@ -9,12 +9,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=msmtp
-PKG_VERSION:=1.4.27
+PKG_VERSION:=1.4.32
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=@SF/msmtp
-PKG_MD5SUM:=2d6d10d9c59ed2b2635554ed35fb9226
+PKG_MD5SUM:=aa60e22211118f222470439cea60f795
 
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
@@ -27,6 +27,7 @@
   SECTION:=mail
   CATEGORY:=Mail
   TITLE:=Simple sendmail SMTP forwarding
+  DEPENDS:= +libidn
   URL:=http://msmtp.sourceforge.net/
 endef
 
@@ -46,7 +47,7 @@
 endef
 
 define Package/msmtp/conffiles
-/etc/msmtprc
+/opt/etc/msmtprc
 endef
 
 define Package/msmtp/description
@@ -80,7 +81,6 @@
 	--disable-rpath \
 	--without-libintl-prefix \
 	--without-libgsasl \
-	--without-libidn
 
 MAKE_FLAGS :=
 
@@ -95,16 +95,16 @@
 endif
 
 define Package/msmtp/install
-	$(INSTALL_DIR) $(1)/etc
+	$(INSTALL_DIR) $(1)/opt/etc
 	$(INSTALL_CONF) $(PKG_BUILD_DIR)/doc/msmtprc-system.example \
-		$(1)/etc/msmtprc
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/msmtp $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/sbin
+		$(1)/opt/etc/msmtprc
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/msmtp $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/sbin
 endef
 
 define Package/msmtp/postinst
-	ln -sf ../bin/msmtp $${IPKG_INSTROOT}/usr/sbin/sendmail
+	ln -sf ../bin/msmtp $${IPKG_INSTROOT}/opt/sbin/sendmail
 endef
 
 Package/msmtp-nossl/conffiles = $(Package/msmtp/conffiles)
@@ -112,9 +112,9 @@
 Package/msmtp-nossl/postinst = $(Package/msmtp/postinst)
 
 define Package/msmtp-queue/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/msmtpq/msmtp{q,-queue} $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/msmtpqueue/msmtp-{en,list,run}queue.sh $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/msmtpq/msmtp{q,-queue} $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/msmtpqueue/msmtp-{en,list,run}queue.sh $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,msmtp))
