--- ../../../downloads/openwrt_trunk-r44497/feeds/packages/mail/msmtp/Makefile	2015-02-24 12:08:55.725768266 +0300
+++ ../../../openwrt_trunk/feeds/packages/mail/msmtp/Makefile	2015-03-04 10:46:13.130566000 +0300
@@ -10,7 +10,7 @@
 
 PKG_NAME:=msmtp
 PKG_VERSION:=1.6.1
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@SF/msmtp
@@ -39,7 +39,7 @@
  an SMTP server (for example at a free mail provider) which does the
  delivery. To use this program with your mail user agent (MUA), create
  a configuration file with your mail account(s) and tell your MUA to
- call msmtp instead of /usr/sbin/sendmail.
+ call msmtp instead of sendmail.
 endef
 
 define Package/msmtp
@@ -50,7 +50,7 @@
 endef
 
 define Package/msmtp/conffiles
-/etc/msmtprc
+/opt/etc/msmtprc
 endef
 
 define Package/msmtp/description
@@ -85,6 +85,7 @@
 	--without-libintl-prefix \
 	--without-libgsasl \
 	--without-libidn \
+	--disable-gai-idn \
 	--without-gnome-keyring \
 	--without-macosx-keyring \
 
@@ -101,34 +102,35 @@
 endif
 
 define Package/msmtp/install
-	$(INSTALL_DIR) $(1)/etc
+	$(INSTALL_DIR) $(1)/opt/etc
 	$(INSTALL_CONF) $(PKG_BUILD_DIR)/doc/msmtprc-system.example \
-		$(1)/etc/msmtprc
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/msmtp $(1)/usr/bin/
+		$(1)/opt/etc/msmtprc
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/msmtp $(1)/opt/bin/
 endef
 
 define Package/msmtp/postinst
-	[ -e $${IPKG_INSTROOT}/usr/sbin/sendmail ] || {
-		mkdir -p $${IPKG_INSTROOT}/usr/sbin
-		ln -sf ../bin/msmtp $${IPKG_INSTROOT}/usr/sbin/sendmail
+	[ -e $${IPKG_INSTROOT}/opt/sbin/sendmail ] || {
+		mkdir -p $${IPKG_INSTROOT}/opt/sbin
+		ln -sf ../bin/msmtp $${IPKG_INSTROOT}/opt/sbin/sendmail
 	}
 endef
 
 define Package/msmtp/prerm
-	[ "../bin/msmtp" = "$(readlink -qs $${IPKG_INSTROOT}/usr/sbin/sendmail)" ] && {
-		rm -f $${IPKG_INSTROOT}/usr/sbin/sendmail
+	[ "../bin/msmtp" = "$(readlink -qs $${IPKG_INSTROOT}/opt/sbin/sendmail)" ] && {
+		rm -f $${IPKG_INSTROOT}/opt/sbin/sendmail
 	}
 endef
 
 Package/msmtp-nossl/conffiles = $(Package/msmtp/conffiles)
 Package/msmtp-nossl/install = $(Package/msmtp/install)
 Package/msmtp-nossl/postinst = $(Package/msmtp/postinst)
+Package/msmtp-nossl/prerm = $(Package/msmtp/prerm)
 
 define Package/msmtp-queue/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/msmtpq/msmtp{q,-queue} $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/msmtpqueue/msmtp-{en,list,run}queue.sh $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/msmtpq/msmtp{q,-queue} $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/msmtpqueue/msmtp-{en,list,run}queue.sh $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,msmtp))
