--- ../../../downloads/openwrt_trunk-r42803/package/devel/strace/Makefile
+++ ../../../openwrt_trunk/package/devel/strace/Makefile
@@ -6,7 +6,6 @@
 #
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=strace
 
@@ -39,12 +38,15 @@
 system calls a program makes while it is running.
 endef
 
+TARGET_CFLAGS += \
+	-I$(LINUX_DIR)/include
+
 MAKE_FLAGS := \
 	CCOPT="$(TARGET_CFLAGS)"
 
 define Package/strace/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/strace $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/strace $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,strace))
