--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/bash-completion/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/bash-completion/Makefile
@@ -42,12 +42,12 @@
 endef
 
 define Package/bash-completion/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bash_completion $(1)/etc/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bash_completion $(1)/opt/etc/
 endef
 
 define Package/bash-completion/conffiles
-/etc/bash_completion
+/opt/etc/bash_completion
 endef
 
 $(eval $(call BuildPackage,bash-completion))
