--- ../../../openwrt_trunk.orig/package/libs/libtool/Makefile
+++ ../../../openwrt_trunk/package/libs/libtool/Makefile
@@ -39,14 +39,14 @@
 		prefix="$(2)" \
 		exec_prefix="$(2)" \
 		install
-	$(INSTALL_DIR) $(1)/usr/lib $(1)/usr/include
-	mv $(2)/lib/* $(1)/usr/lib/
-	mv $(2)/include/* $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/lib $(1)/opt/include
+	mv $(2)/lib/* $(1)/opt/lib/
+	mv $(2)/include/* $(1)/opt/include/
 endef
 
 define Package/libltdl/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libltdl/.libs/libltdl.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libltdl/.libs/libltdl.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libltdl))
