--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/utils/zsh/Makefile	2014-04-09 16:17:24.230286480 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/zsh/Makefile	2014-04-11 16:14:49.227139471 +0400
@@ -23,7 +23,7 @@
   SECTION:=utils
   CATEGORY:=Utilities
   TITLE:=The Z shell
-  DEPENDS:=+libncurses +librt
+  DEPENDS:=+libncurses +librt +libgdbm
   URL:=http://www.zsh.org/
 endef
 
@@ -38,8 +38,9 @@
 
 define Build/Configure
 	$(call Build/Configure/Default, \
-		--bindir=/bin \
-		--disable-etcdir \
+		--bindir=/opt/bin \
+		--enable-etcdir=/opt/etc \
+		--enable-multibyte \
 		--disable-gdbm \
 		--with-term-lib="ncurses", \
 		zsh_cv_sys_nis=no \
@@ -60,17 +61,11 @@
 	$(MAKE) -C $(PKG_BUILD_DIR) DESTDIR="$(PKG_INSTALL_DIR)" prep
 endef
 
-define Package/zsh/postinst
-#!/bin/sh
-grep zsh $${IPKG_INSTROOT}/etc/shells || \
-	echo "/bin/zsh" >> $${IPKG_INSTROOT}/etc/shells
-endef
-
 define Package/zsh/install
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/zsh $(1)/bin/
-	$(INSTALL_DIR) $(1)/usr/share/zsh/$(PKG_VERSION)
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/zsh/$(PKG_VERSION)/* $(1)/usr/share/zsh/$(PKG_VERSION)/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/zsh $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share/zsh/$(PKG_VERSION)
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/zsh/$(PKG_VERSION)/* $(1)/opt/share/zsh/$(PKG_VERSION)/
 endef
 
 $(eval $(call BuildPackage,zsh))
