--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/zsh/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/zsh/Makefile
@@ -25,7 +25,7 @@
   SECTION:=utils
   CATEGORY:=Utilities
   TITLE:=The Z shell
-  DEPENDS:=+libncurses +libncursesw +libpcre +librt
+  DEPENDS:=+libncurses +libncursesw +libpcre +librt +libgdbm
   URL:=http://www.zsh.org/
 endef
 
@@ -40,8 +40,9 @@
 
 define Build/Configure
 	$(call Build/Configure/Default, \
-		--bindir=/bin \
-		--disable-etcdir \
+		--bindir=/opt/bin \
+		--enable-etcdir=/opt/etc \
+		--enable-multibyte \
 		--disable-gdbm \
 		--enable-pcre \
 		--enable-multibyte \
@@ -66,17 +67,11 @@
 	$(MAKE) -C $(PKG_BUILD_DIR) DESTDIR="$(PKG_INSTALL_DIR)" prep
 endef
 
-define Package/zsh/postinst
-#!/bin/sh
-grep zsh $${IPKG_INSTROOT}/etc/shells || \
-	echo "/bin/zsh" >> $${IPKG_INSTROOT}/etc/shells
-endef
-
 define Package/zsh/install
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/zsh $(1)/bin/
-	$(INSTALL_DIR) $(1)/usr/share/zsh/$(PKG_VERSION)
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/zsh/$(PKG_VERSION)/* $(1)/usr/share/zsh/$(PKG_VERSION)/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/zsh $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share/zsh/$(PKG_VERSION)
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/zsh/$(PKG_VERSION)/* $(1)/opt/share/zsh/$(PKG_VERSION)/
 endef
 
 $(eval $(call BuildPackage,zsh))
