--- ../../../openwrt_trunk.orig/feeds/packages/libs/libmad/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/libmad/Makefile
@@ -65,15 +65,15 @@
 	CFLAGS="$(TARGET_CFLAGS)" \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/mad.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libmad.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/mad.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libmad.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libmad/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libmad.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libmad.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libmad))
