--- ../../../downloads/openwrt_trunk-r46940/feeds/packages/sound/madplay/Makefile	2015-09-15 12:24:40.786451662 +0300
+++ ../../../openwrt_trunk/feeds/packages/sound/madplay/Makefile	2015-09-21 10:22:46.626958144 +0300
@@ -56,8 +56,8 @@
 endef
 
 define Package/madplay/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/madplay $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/madplay $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,madplay))
