--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/proftpd/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/proftpd/Makefile
@@ -30,7 +30,7 @@
 endef
 
 define Package/proftpd/conffiles
-/etc/proftpd.conf
+/opt/etc/proftpd.conf
 endef
 
 TARGET_CPPFLAGS += \
@@ -48,16 +48,15 @@
 	ac_cv_header_ncurses_h=no
 
 define Package/proftpd/install
-	$(INSTALL_DIR) $(1)/etc
-	$(if $(CONFIG_IPV6),,$(SED) 's/^UseIPv6/#UseIPv6/' $(PKG_INSTALL_DIR)/etc/proftpd.conf)
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/proftpd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/proftpd.init $(1)/etc/init.d/proftpd
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ftp{count,top,who} $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/{ftpshut,proftpd} $(1)/usr/sbin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/in.proftpd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(if $(CONFIG_IPV6),,$(SED) 's/^UseIPv6/#UseIPv6/' $(PKG_INSTALL_DIR)/opt/etc/proftpd.conf)
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/proftpd.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ftp{count,top,who} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/{ftpshut,proftpd} $(1)/opt/sbin/
+	$(CP) $(PKG_INSTALL_DIR)/opt/sbin/in.proftpd $(1)/opt/sbin/
+	$(INSTALL_BIN) ./files/S66proftpd $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,proftpd))
