--- ../../../openwrt_trunk/package/system/utils/e2fsprogs/Makefile.prev	2013-05-10 18:57:51.000000000 +0400
+++ ../../../openwrt_trunk/package/system/utils/e2fsprogs/Makefile	2013-06-10 02:23:11.240280748 +0400
@@ -12,3 +12,3 @@
 PKG_MD5SUM:=b6e296f210d642361b7394437ff0f318
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
@@ -124,3 +124,3 @@
 define Package/e2fsprogs/install
-	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin $(1)/opt/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/e2fsck $(1)/opt/sbin/
@@ -128,2 +128,4 @@
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/filefrag $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/chattr $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/lsattr $(1)/opt/bin/
 	ln -sf mke2fs $(1)/opt/sbin/mkfs.ext2
