--- ../../../openwrt_trunk.orig/package/system/utils/e2fsprogs/Makefile	2012-07-10 18:59:26.209181897 +0400
+++ ../../../openwrt_trunk/package/system/utils/e2fsprogs/Makefile	2012-07-10 19:00:03.080718359 +0400
@@ -122,43 +122,44 @@
 endef
 
 define Package/e2fsprogs/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/e2fsck $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/mke2fs $(1)/usr/sbin/
-	ln -sf mke2fs $(1)/usr/sbin/mkfs.ext2
-	ln -sf mke2fs $(1)/usr/sbin/mkfs.ext3
-	ln -sf mke2fs $(1)/usr/sbin/mkfs.ext4
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libe2p.so.* $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_DIR) $(1)/lib/functions/fsck
-	$(INSTALL_DATA) ./files/e2fsck.sh $(1)/lib/functions/fsck/
-	$(INSTALL_DATA) ./files/e2fsck.conf $(1)/etc/e2fsck.conf
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/e2fsck $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/mke2fs $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/filefrag $(1)/opt/sbin/
+	ln -sf mke2fs $(1)/opt/sbin/mkfs.ext2
+	ln -sf mke2fs $(1)/opt/sbin/mkfs.ext3
+	ln -sf mke2fs $(1)/opt/sbin/mkfs.ext4
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libe2p.so.* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DIR) $(1)/opt/lib/functions/fsck
+	$(INSTALL_DATA) ./files/e2fsck.sh $(1)/opt/lib/functions/fsck/
+	$(INSTALL_DATA) ./files/e2fsck.conf $(1)/opt/etc/
 endef
 
 define Package/libcom_err/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcom_err.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcom_err.so.* $(1)/opt/lib/
 endef
 
 define Package/libext2fs/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libext2fs.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libext2fs.so.* $(1)/opt/lib/
 endef
 
 define Package/tune2fs/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/tune2fs $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/tune2fs $(1)/opt/sbin/
 endef
 
 define Package/resize2fs/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/resize2fs $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/resize2fs $(1)/opt/sbin/
 endef
 
 define Package/badblocks/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/badblocks $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/badblocks $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,e2fsprogs))
