--- a/options.c	2012-01-20 14:48:59.689834389 +0400
+++ b/options.c	2012-01-20 14:50:23.046441596 +0400
@@ -96,7 +96,7 @@
     }
 
     while(nameindex[j].if_index != 0) {
-        if (strcmp(nameindex[j].if_name, "lo") != 0 && !is_bad_interface_name(nameindex[j].if_name)) {
+        if (strcmp(nameindex[j].if_name, "eth") == 0 && !is_bad_interface_name(nameindex[j].if_name)) {
             i = xstrdup(nameindex[j].if_name);
             break;
         }
