--- ../../../openwrt_trunk/feeds/packages/libs/libtorrent/Makefile.orig	2013-08-09 12:56:11.274797756 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libtorrent/Makefile	2013-08-14 15:47:52.510752006 +0400
@@ -47,7 +47,8 @@
 	--enable-static \
 	--enable-aligned \
 	--disable-debug \
-	--enable-openssl
+	--enable-openssl \
+	--with-posix-fallocate
 
 define Build/Configure
 	( cd $(PKG_BUILD_DIR); ./autogen.sh );
@@ -55,17 +56,17 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/torrent $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libtorrent.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libtorrent.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/torrent $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libtorrent.so* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/libtorrent.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libtorrent/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libtorrent.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libtorrent.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libtorrent))
