--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/libs/libtorrent/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/libtorrent/Makefile
@@ -43,11 +43,11 @@
 TARGET_LDFLAGS += $(LIBGCC_S)
 
 CONFIGURE_ARGS+= \
-	--enable-shared \
-	--enable-static \
 	--enable-aligned \
 	--disable-debug \
 	--enable-openssl \
+	--with-posix-fallocate \
+	--enable-ipv6 \
 	--disable-instrumentation
 
 define Build/Configure
@@ -56,17 +56,17 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/torrent $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libtorrent.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libtorrent.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/torrent $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libtorrent.so* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/libtorrent.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libtorrent/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libtorrent.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libtorrent.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libtorrent))
