--- ../../../downloads/openwrt_trunk-r44486/feeds/packages/libs/libtorrent/Makefile	2015-02-18 12:16:57.584306331 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/libtorrent/Makefile	2015-02-19 15:09:18.218542118 +0300
@@ -49,20 +49,22 @@
 	--disable-debug \
 	--enable-openssl \
 	--disable-instrumentation \
-	--with-zlib=$(STAGING_DIR)/usr
+	--with-posix-fallocate \
+	--enable-ipv6 \
+	--with-zlib=$(STAGING_DIR)/opt
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/torrent $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libtorrent.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libtorrent.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/torrent $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libtorrent.so* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/libtorrent.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libtorrent/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libtorrent.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libtorrent.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libtorrent))
