--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/matrixssl/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/matrixssl/Makefile
@@ -42,16 +42,16 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/matrixSsl
-	$(CP) $(PKG_BUILD_DIR)/matrixSsl.h $(1)/usr/include/matrixSsl/
-	ln -sf matrixSsl/matrixSsl.h $(1)/usr/include/matrixSsl.h
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/src/libmatrixssl.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include/matrixSsl
+	$(CP) $(PKG_BUILD_DIR)/matrixSsl.h $(1)/opt/include/matrixSsl/
+	ln -sf matrixSsl/matrixSsl.h $(1)/opt/include/matrixSsl.h
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/src/libmatrixssl.so* $(1)/opt/lib/
 endef
 
 define Package/libmatrixssl/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/src/libmatrixssl.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/src/libmatrixssl.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libmatrixssl))
