--- ../../../openwrt_trunk.orig/feeds/telephony/kamailio/Makefile	2012-04-02 11:00:40.148141446 +0400
+++ ../../../openwrt_trunk/feeds/telephony/kamailio/Makefile	2012-04-03 12:18:13.367660790 +0400
@@ -58,8 +58,8 @@
 endef
 
 define Package/kamailio-example/conffiles
-/etc/default/kamailio
-/etc/kamailio/kamailio.cfg
+/opt/etc/default/kamailio
+/opt/etc/kamailio/kamailio.cfg
 endef
 
 
@@ -70,7 +70,7 @@
 endef
 
 define Package/kamailio-tools/conffiles
-/etc/kamailio/kamctlrc
+/opt/etc/kamailio/kamctlrc
 endef
 
 
@@ -89,11 +89,11 @@
 	snmpstats \
 
 PKG_MAKE_ARGS:= \
-	cfg-dir="etc/kamailio/" \
-	bin-dir="usr/sbin/" \
-	data-dir="var/lib/kamailio/" \
-	lib-dir="usr/lib/kamailio/" \
-	modules-dir="usr/lib/kamailio/modules/" \
+	cfg-dir="opt/etc/kamailio/" \
+	bin-dir="opt/sbin/" \
+	data-dir="opt/var/lib/kamailio/" \
+	lib-dir="opt/lib/kamailio/" \
+	modules-dir="opt/lib/kamailio/modules/" \
 	exclude_modules="$(PKG_EXCL_MODULES)" \
 
 
@@ -104,9 +104,10 @@
 		CC="$(TARGET_CC)" \
 		ARCH="$(ARCH)"  \
 		CFLAGS="$(TARGET_CFLAGS)" \
+		LD_EXTRA_OPTS="$(TARGET_LDFLAGS)" \
 		ISSMP="no" \
-		LOCALBASE="$(STAGING_DIR)/usr" \
-		SYSBASE="$(STAGING_DIR)/usr" \
+		LOCALBASE="$(STAGING_DIR)/opt" \
+		SYSBASE="$(STAGING_DIR)/opt" \
 		PCREDEFS:="$(TARGET_CPPFLAGS)" \
 		PCRELIBS:="$(TARGET_LDFLAGS)" \
 		CROSS_COMPILE=1 \
@@ -117,42 +118,42 @@
 endef
 
 define Package/kamailio/install
-	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/sbin/kamailio \
-		$(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib/kamailio/modules
+		$(PKG_INSTALL_DIR)/opt/sbin/kamailio \
+		$(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib/kamailio/modules
 endef
 
 define Package/kamailio-example/install
-	$(INSTALL_DIR) $(1)/etc/kamailio
+	$(INSTALL_DIR) $(1)/opt/etc/kamailio
 	$(CP) \
-		$(PKG_INSTALL_DIR)/etc/kamailio/kamailio.cfg \
-		$(1)/etc/kamailio/
+		$(PKG_INSTALL_DIR)/opt/etc/kamailio/kamailio.cfg \
+		$(1)/opt/etc/kamailio/
 	$(SED) 's,$(PKG_INSTALL_DIR),,g' \
-		$(1)/etc/kamailio/kamailio.cfg
-	$(INSTALL_DIR) $(1)/etc/default
-	$(INSTALL_DATA) ./files/kamailio.default $(1)/etc/default/kamailio
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/kamailio.init $(1)/etc/init.d/kamailio
+		$(1)/opt/etc/kamailio/kamailio.cfg
+#	$(INSTALL_DIR) $(1)/opt/etc/default
+#	$(INSTALL_DATA) ./files/kamailio.default $(1)/opt/etc/default/kamailio
+#	$(INSTALL_DIR) $(1)/opt/etc/init.d
+#	$(INSTALL_BIN) ./files/kamailio.init $(1)/opt/etc/init.d/kamailio
 endef
 
 define Package/kamailio-tools/install
-	$(INSTALL_DIR) $(1)/etc/kamailio
+	$(INSTALL_DIR) $(1)/opt/etc/kamailio
 	$(CP) \
-		$(PKG_INSTALL_DIR)/etc/kamailio/kamctlrc \
-		$(1)/etc/kamailio/
-	$(INSTALL_DIR) $(1)/usr/sbin
+		$(PKG_INSTALL_DIR)/opt/etc/kamailio/kamctlrc \
+		$(1)/opt/etc/kamailio/
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/sbin/kam{ctl,dbctl,unix} \
-		$(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib/kamailio
+		$(PKG_INSTALL_DIR)/opt/sbin/kam{ctl,dbctl,unix} \
+		$(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib/kamailio
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/kamailio/kamctl \
-		$(1)/usr/lib/kamailio/
+		$(PKG_INSTALL_DIR)/opt/lib/kamailio/kamctl \
+		$(1)/opt/lib/kamailio/
 	$(SED) 's,/bin/bash,/bin/sh,g' \
 		-e 's,$(PKG_INSTALL_DIR),,g' \
-		$(1)/usr/sbin/kam{ctl,dbctl}
+		$(1)/opt/sbin/kam{ctl,dbctl}
 endef
 
 define BuildPlugin
@@ -163,9 +164,9 @@
   endef
 
   define Package/kamailio-mod-$(1)/install
-	[ -z "$(2)" ] || $(INSTALL_DIR) $$(1)/usr/lib/kamailio/modules/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/kamailio/modules/$(2).so \
-			$$(1)/usr/lib/kamailio/modules/ ;
+	[ -z "$(2)" ] || $(INSTALL_DIR) $$(1)/opt/lib/kamailio/modules/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/kamailio/modules/$(2).so \
+			$$(1)/opt/lib/kamailio/modules/ ;
   endef
 
   $$(eval $$(call BuildPackage,kamailio-mod-$(1)))
