--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/giflib/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/giflib/Makefile
@@ -44,16 +44,16 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/{lib,include}
-	$(CP) $(PKG_BUILD_DIR)/lib/.libs/lib*so* $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/.libs/libgif.a $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/libgif.la $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/*.h $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/{lib,include}
+	$(CP) $(PKG_BUILD_DIR)/lib/.libs/lib*so* $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/.libs/libgif.a $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/libgif.la $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/*.h $(1)/opt/include/
 endef
 
 define Package/giflib/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/.libs/lib*so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/.libs/lib*so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,giflib))
