--- ../../../openwrt_trunk.orig/feeds/packages/lang/erlang/Makefile	2012-03-21 16:34:38.866039807 +0400
+++ ../../../openwrt_trunk/feeds/packages/lang/erlang/Makefile	2012-03-28 13:30:34.501833342 +0400
@@ -250,7 +250,7 @@
 	--disable-smp-support \
 	--disable-threads \
 	--without-java \
-	--with-ssl="$(STAGING_DIR)/usr" \
+	--with-ssl="$(STAGING_DIR)/opt" \
 	--enable-dynamic-ssl-lib
 
 CONFIGURE_VARS += \
@@ -274,33 +274,33 @@
 endef
 
 define Package/erlang/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	for f in epmd erl erlc escript run_erl; do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/bin/$$$$f $(1)/usr/bin/ ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/bin/$$$$f $(1)/opt/bin/ ; \
 	done
-	$(INSTALL_DIR) $(1)/usr/lib/erlang/bin
+	$(INSTALL_DIR) $(1)/opt/lib/erlang/bin
 	for f in erl erlc escript run_erl start start.boot start.script start_clean.boot start_erl start_sasl.boot to_erl; do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/bin/$$$$f $(1)/usr/lib/erlang/bin/ ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/bin/$$$$f $(1)/opt/lib/erlang/bin/ ; \
 	done
-	$(INSTALL_DIR) $(1)/usr/lib/erlang/lib
+	$(INSTALL_DIR) $(1)/opt/lib/erlang/lib
 	for m in erts kernel sasl stdlib; do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/lib/$$$$m-* $(1)/usr/lib/erlang/lib/ ; \
-		rm -rf $(1)/usr/lib/erlang/lib/$$$$m-*/examples ; \
-		rm -rf $(1)/usr/lib/erlang/lib/$$$$m-*/src ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/lib/$$$$m-* $(1)/opt/lib/erlang/lib/ ; \
+		rm -rf $(1)/opt/lib/erlang/lib/$$$$m-*/examples ; \
+		rm -rf $(1)/opt/lib/erlang/lib/$$$$m-*/src ; \
 	done
-	$(INSTALL_DIR) $(1)/usr/lib/erlang
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/erts-* $(1)/usr/lib/erlang/
-	rm -rf $(1)/usr/lib/erlang/erts-*/{doc,include,lib,man,src}
-	rm -rf $(1)/usr/lib/erlang/erts-*/bin/*.src
-	$(INSTALL_DIR) $(1)/usr/lib/erlang/releases
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/releases/* $(1)/usr/lib/erlang/releases/
-	$(SED) 's,%ERL_ROOT%,/usr/lib/erlang,g' \
-		$(1)/usr/lib/erlang/releases/RELEASES.src
-	mv -f $(1)/usr/lib/erlang/releases/RELEASES.src \
-		$(1)/usr/lib/erlang/releases/RELEASES
+	$(INSTALL_DIR) $(1)/opt/lib/erlang
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/erts-* $(1)/opt/lib/erlang/
+	rm -rf $(1)/opt/lib/erlang/erts-*/{doc,include,lib,man,src}
+	rm -rf $(1)/opt/lib/erlang/erts-*/bin/*.src
+	$(INSTALL_DIR) $(1)/opt/lib/erlang/releases
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/releases/* $(1)/opt/lib/erlang/releases/
+	$(SED) 's,%ERL_ROOT%,/opt/lib/erlang,g' \
+		$(1)/opt/lib/erlang/releases/RELEASES.src
+	mv -f $(1)/opt/lib/erlang/releases/RELEASES.src \
+		$(1)/opt/lib/erlang/releases/RELEASES
 	for f in bin/erl bin/start erts-*/bin/erl erts-*/bin/start; do \
-		$(SED) 's,^\(ROOTDIR\)=.*,\1=/usr/lib/erlang,g' \
-			$(1)/usr/lib/erlang/$$$$f ; \
+		$(SED) 's,^\(ROOTDIR\)=.*,\1=/opt/lib/erlang,g' \
+			$(1)/opt/lib/erlang/$$$$f ; \
 	done
 endef
 
@@ -308,10 +308,10 @@
 define BuildModule
 
   define Package/erlang-$(1)/install
-	$(INSTALL_DIR) $$(1)/usr/lib/erlang/lib
+	$(INSTALL_DIR) $$(1)/opt/lib/erlang/lib
 	for m in $(2); do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/lib/$$$$$$$$m-* $$(1)/usr/lib/erlang/lib/ ; \
-		rm -rf $$(1)/usr/lib/erlang/lib/$$$$$$$$m-*/{examples,priv/obj,src} ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/lib/$$$$$$$$m-* $$(1)/opt/lib/erlang/lib/ ; \
+		rm -rf $$(1)/opt/lib/erlang/lib/$$$$$$$$m-*/{examples,priv/obj,src} ; \
 	done
   endef
 
