--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/libs/postgresql/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/postgresql/Makefile
@@ -68,18 +68,18 @@
 define Build/Configure
 	(cd $(PKG_BUILD_DIR); rm -f config.cache; \
 		./configure \
-			--prefix=/usr \
-			--exec-prefix=/usr \
-			--bindir=/usr/bin \
-			--datadir=/usr/share \
-			--includedir=/usr/include \
-			--infodir=/usr/share/info \
-			--libdir=/usr/lib \
-			--libexecdir=/usr/lib \
-			--localstatedir=/var \
-			--mandir=/usr/share/man \
-			--sbindir=/usr/sbin \
-			--sysconfdir=/etc \
+			--prefix=/opt \
+			--exec-prefix=/opt \
+			--bindir=/opt/bin \
+			--datadir=/opt/share \
+			--includedir=/opt/include \
+			--infodir=/opt/share/info \
+			--libdir=/opt/lib \
+			--libexecdir=/opt/lib \
+			--localstatedir=/opt/var \
+			--mandir=/opt/share/man \
+			--sbindir=/opt/sbin \
+			--sysconfdir=/opt/etc \
 			$(DISABLE_NLS) \
 			--enable-shared \
 			--enable-static \
@@ -98,7 +98,7 @@
 			--without-tk \
 			--with-zlib="yes" \
 			--enable-depend \
-			--with-system-timezone=/tmp \
+			--with-system-timezone=/opt/tmp \
 	);
 	$(MAKE) -C $(PKG_BUILD_DIR)/src/interfaces/ecpg/preproc CC="$(HOSTCC)" CFLAGS+="-I../../libpq"
 	mv $(PKG_BUILD_DIR)/src/interfaces/ecpg/preproc/ecpg \
@@ -123,18 +123,18 @@
 			--build=$(GNU_HOST_NAME) \
 			--program-prefix="" \
 			--program-suffix="" \
-			--prefix=/usr \
-			--exec-prefix=/usr \
-			--bindir=/usr/bin \
-			--datadir=/usr/share \
-			--includedir=/usr/include \
-			--infodir=/usr/share/info \
-			--libdir=/usr/lib \
-			--libexecdir=/usr/lib \
-			--localstatedir=/var \
-			--mandir=/usr/share/man \
-			--sbindir=/usr/sbin \
-			--sysconfdir=/etc \
+			--prefix=/opt \
+			--exec-prefix=/opt \
+			--bindir=/opt/bin \
+			--datadir=/opt/share \
+			--includedir=/opt/include \
+			--infodir=/opt/share/info \
+			--libdir=/opt/lib \
+			--libexecdir=/opt/lib \
+			--localstatedir=/opt/var \
+			--mandir=/opt/share/man \
+			--sbindir=/opt/sbin \
+			--sysconfdir=/opt/etc \
 			$(DISABLE_NLS) \
 			$(DISABLE_LARGEFILE) \
 			--enable-shared \
@@ -173,80 +173,74 @@
 endef
 
 define Package/libpq/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpq.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpq.so.* $(1)/opt/lib/
 endef
 
 define Package/pgsql-cli/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/psql $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/psql $(1)/opt/bin/
 endef
 
 define Package/pgsql-server/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/usr/share/postgresql
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin
-	ln -sf postgres $(1)/usr/bin/postmaster
-
-	$(INSTALL_BIN) ./files/postgresql.init $(1)/etc/init.d/postgresql
-
-	$(CP) -r $(PKG_INSTALL_DIR)/usr/share/postgresql/* \
-		$(1)/usr/share/postgresql
-
-	$(INSTALL_DATA) ./files/postgresql.config $(1)/etc/config/postgresql
-
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/share/postgresql
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin
+	ln -sf postgres $(1)/opt/bin/postmaster
+	$(CP) -r $(PKG_INSTALL_DIR)/opt/share/postgresql/* \
+		$(1)/opt/share/postgresql
 	$(CP) -r \
-		$(PKG_INSTALL_DIR)/usr/lib/postgresql \
-		$(1)/usr/lib
+		$(PKG_INSTALL_DIR)/opt/lib/postgresql \
+		$(1)/opt/lib
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S98postgresql  $(1)/opt/etc/init.d
 endef
 
 define Package/pgsql-server/conffiles
-/etc/config/postgresql
+/opt/etc/config/postgresql
 endef
 
-define Package/pgsql-server/postinst
+#define Package/pgsql-server/postinst
 #!/bin/sh
-grep -q '^postgres:' /etc/passwd && exit 0
-group=$$(grep '^postgres:' /etc/group | cut -f3 -d:)
-if [ -z "$${group}" ] ; then
-	group=1000
-	tst=$$(cat /etc/group | grep ":$${group}:")
-	while [ -n "$${tst}" ] ; do
-		group=$$(($${group}+1))
-		tst=$$(cat /etc/group | grep ":$${group}:")
-	done
-	echo "postgres:x:$${group}:" >>/etc/group
-fi
-
-user=$$(cat /etc/passwd | grep "^postgres:")
-if [ -z "$${user}" ] ; then
-	num="$${group}"
-	tst=$$(cat /etc/passwd | grep ":.*:$${num}:")
-	while [ -n "$${tst}" ] ; do
-		num=$$(($${num}+1))
-		tst=$$(cat /etc/passwd | grep ":.*:$${num}:")
-	done
-	echo "postgres:*:$${num}:$${group}:PostgreSQL administrator:/tmp:/bin/ash" >>/etc/passwd
-fi
-endef
+#grep -q '^postgres:' /etc/passwd && exit 0
+#group=$$(grep '^postgres:' /etc/group | cut -f3 -d:)
+#if [ -z "$${group}" ] ; then
+#	group=1000
+#	tst=$$(cat /etc/group | grep ":$${group}:")
+#	while [ -n "$${tst}" ] ; do
+#		group=$$(($${group}+1))
+#		tst=$$(cat /etc/group | grep ":$${group}:")
+#	done
+#	echo "postgres:x:$${group}:" >>/etc/group
+#fi
+
+#user=$$(cat /etc/passwd | grep "^postgres:")
+#if [ -z "$${user}" ] ; then
+#	num="$${group}"
+#	tst=$$(cat /etc/passwd | grep ":.*:$${num}:")
+#	while [ -n "$${tst}" ] ; do
+#		num=$$(($${num}+1))
+#		tst=$$(cat /etc/passwd | grep ":.*:$${num}:")
+#	done
+#	echo "postgres:*:$${num}:$${group}:PostgreSQL administrator:/tmp:/bin/ash" >>/etc/passwd
+#fi
+#endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/bin/pg_config/pg_config.host $(1)/usr/bin/pg_config
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libpq $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libpq-fe.h $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/pg_config.h $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/pg_config_manual.h $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/postgres_ext.h $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/postgresql $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpq.{a,so*} $(1)/usr/lib/
-	$(CP) $(PKG_BUILD_DIR)/src/interfaces/ecpg/preproc/ecpg.host $(1)/usr/bin/ecpg
-	$(CP) $(PKG_BUILD_DIR)/host-zic $(1)/usr/bin/zic
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/bin/pg_config/pg_config.host $(1)/opt/bin/pg_config
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libpq $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libpq-fe.h $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/pg_config.h $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/pg_config_manual.h $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/postgres_ext.h $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/postgresql $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpq.{a,so*} $(1)/opt/lib/
+	$(CP) $(PKG_BUILD_DIR)/src/interfaces/ecpg/preproc/ecpg.host $(1)/opt/bin/ecpg
+	$(CP) $(PKG_BUILD_DIR)/host-zic $(1)/opt/bin/zic
 endef
 
 $(eval $(call BuildPackage,libpq))
