--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/cutter/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/cutter/Makefile
@@ -31,12 +31,12 @@
 endef
 
 define Build/Compile
-	$(TARGET_CC) $(TARGET_CFLAGS) $(PKG_BUILD_DIR)/cutter.c -o $(PKG_BUILD_DIR)/cutter
+	$(TARGET_CC) $(TARGET_CFLAGS) $(TARGET_LDFLAGS) $(PKG_BUILD_DIR)/cutter.c -o $(PKG_BUILD_DIR)/cutter
 endef
 
 define Package/cutter/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/cutter $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/cutter $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,cutter))
