--- ../../../backup/trunk_clean/feeds/packages/lang/php-pear-db/Makefile	2012-01-30 15:02:12.000000000 +0400
+++ ../../../trunk/feeds/packages/lang/php-pear-db/Makefile	2012-02-02 12:51:01.691636254 +0400
@@ -35,9 +35,9 @@
 endef
 
 define Package/php-pear-db/install
-	$(INSTALL_DIR) $(1)/usr/lib/php
+	$(INSTALL_DIR) $(1)/opt/lib/php
 	$(CP)	$(PKG_BUILD_DIR)/DB.php $(PKG_BUILD_DIR)/DB \
-		$(1)/usr/lib/php/
+		$(1)/opt/lib/php/
 endef
 
 $(eval $(call BuildPackage,php-pear-db))
