--- ../../../openwrt_trunk/feeds/packages/net/elinks/Makefile.my0	2013-03-17 21:56:53.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/elinks/Makefile	2013-03-17 22:09:59.000000000 +0400
@@ -10,3 +10,3 @@
 PKG_NAME:=elinks
-PKG_VERSION:=0.12pre5
+PKG_VERSION:=0.12pre6
 PKG_RELEASE:=1
@@ -15,3 +15,3 @@
 PKG_SOURCE_URL:=http://elinks.or.cz/download
-PKG_MD5SUM:=92790144290131ac5e63b44548b45e08
+PKG_MD5SUM:=3d0962f4b65fb039342fefd3ada050a9
 
