--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/elinks/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/elinks/Makefile
@@ -8,19 +8,19 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=elinks
-PKG_VERSION:=0.11.7
+PKG_VERSION:=0.12pre6
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=http://elinks.or.cz/download
-PKG_MD5SUM:=fcd087a6d2415cd4c6fd1db53dceb646
+PKG_MD5SUM:=3d0962f4b65fb039342fefd3ada050a9
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/elinks
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libopenssl
+  DEPENDS:=+libopenssl +zlib +libidn
   TITLE:=A text based web browser
   URL:=http://elinks.or.cz/
 endef
@@ -47,18 +47,18 @@
                         --build=$(GNU_HOST_NAME) \
                         --program-prefix="" \
                         --program-suffix="" \
-                        --prefix=/usr \
-                        --exec-prefix=/usr \
-                        --bindir=/usr/bin \
-                        --datadir=/usr/share \
-                        --includedir=/usr/include \
-                        --infodir=/usr/share/info \
-                        --libdir=/usr/lib \
-                        --libexecdir=/usr/lib \
-                        --localstatedir=/var \
-                        --mandir=/usr/share/man \
-                        --sbindir=/usr/sbin \
-                        --sysconfdir=/etc \
+                        --prefix=/opt \
+                        --exec-prefix=/opt \
+                        --bindir=/opt/bin \
+                        --datadir=/opt/share \
+                        --includedir=/opt/include \
+                        --infodir=/opt/share/info \
+                        --libdir=/opt/lib \
+                        --libexecdir=/opt/lib \
+                        --localstatedir=/opt/var \
+                        --mandir=/opt/share/man \
+                        --sbindir=/opt/sbin \
+                        --sysconfdir=/opt/etc \
                         $(DISABLE_NLS) \
                         --enable-small\
                         --enable-256-colors\
@@ -73,20 +73,20 @@
                         --disable-xbel \
                         --without-bzlib \
                         --without-gpm \
-                        --without-idn \
+                        --with-idn \
                         --without-gnutls \
                         --without-libiconv \
                         --without-lua \
-                        --with-openssl="$(STAGING_DIR)/usr" \
+                        --with-openssl="$(STAGING_DIR)/opt" \
                         --without-spidermonkey\
                         --without-x\
-                        --without-zlib \
+                        --with-zlib \
  	);
 endef
 
 define Package/elinks/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/elinks $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/elinks $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,elinks))
