--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/psmisc/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/psmisc/Makefile
@@ -34,32 +34,19 @@
 endef
 
 CONFIGURE_ARGS += \
-	--disable-harden-flags
+	--disable-harden-flags \
+	--localedir=/opt/share/locale
 
 MAKE_FLAGS += \
 	CFLAGS="$(TARGET_CFLAGS)" \
 	CPPFLAGS="$(TARGET_CPPFLAGS)" \
 	LDFLAGS="$(TARGET_LDFLAGS)"
 
-define Package/psmisc/preinst
-#!/bin/sh
-if [ -e $${IPKG_INSTROOT}/usr/bin/killall ]; then
-  rm $${IPKG_INSTROOT}/usr/bin/killall;
-fi
-endef
-
 define Package/psmisc/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/{fuser,killall,prtstat,pstree} \
-		$(1)/usr/bin/
-endef
-
-define Package/psmisc/postrm
-#!/bin/sh
-ln -sf ../../bin/busybox $${IPKG_INSTROOT}/usr/bin/killall
-$${IPKG_INSTROOT}/usr/bin/killall 2>&1 | grep 'applet not found' > /dev/null 2>&1 && rm $${IPKG_INSTROOT}/usr/bin/killall
-exit 0
+		$(PKG_INSTALL_DIR)/opt/bin/{fuser,killall,prtstat,pstree,peekfd} \
+		$(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,psmisc))
