--- ../../../openwrt_trunk.orig/feeds/packages/utils/gnupg/Makefile	2012-07-18 00:48:35.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/gnupg/Makefile	2012-07-24 13:06:59.000000000 +0400
@@ -23,7 +23,7 @@
 define Package/gnupg
   SECTION:=utils
   CATEGORY:=Utilities
-  DEPENDS:=+zlib +libncurses +libreadline
+  DEPENDS:=+zlib +libncurses +libreadline +libbz2
   TITLE:=GNU privacy guard - a free PGP replacement
   URL:=http://www.gnupg.org/
 endef
@@ -44,12 +44,10 @@
 	--disable-gnupg-iconv \
 	--disable-card-support \
 	--disable-agent-support \
-	--disable-bzip2 \
 	--disable-exec \
 	--disable-ldap \
 	--disable-hkp \
 	--disable-finger \
-	--disable-ftp \
 	--disable-dns-srv \
 	--enable-fake-curl \
 	--disable-regex \
@@ -58,8 +56,8 @@
 	SUBDIRS="m4 intl zlib util mpi cipher tools g10 keyserver ${checks}" \
 
 define Package/gnupg/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/gpg $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/gpg $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,gnupg))
