--- ../../../openwrt_trunk.orig/feeds/oldpackages/multimedia/ushare/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/multimedia/ushare/Makefile
@@ -40,14 +40,14 @@
 	# this is *NOT* GNU configure
 	( cd $(PKG_BUILD_DIR); \
 		CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS)" \
-		LDFLAGS="$(TARGET_LDFLAGS) -Wl,-rpath-link=$(STAGING_DIR)/usr/lib" \
-		PKG_CONFIG_PATH="$(STAGING_DIR)/usr/lib/pkgconfig" \
-		PKG_CONFIG_LIBDIR="$(STAGING_DIR)/usr/lib/pkgconfig" \
+		LDFLAGS="$(TARGET_LDFLAGS)" \
+		PKG_CONFIG_PATH="$(STAGING_DIR)/opt/lib/pkgconfig" \
+		PKG_CONFIG_LIBDIR="$(STAGING_DIR)/opt/lib/pkgconfig" \
 		./configure \
 		--cross-compile \
 		--cross-prefix="$(TARGET_CROSS)" \
-		--prefix="/usr" \
-		--sysconfdir="/etc" \
+		--prefix="/opt" \
+		--sysconfdir="/opt/etc" \
 		--enable-dlna \
 		--disable-nls \
 		--disable-debug \
@@ -57,19 +57,15 @@
 endef
 
 define Package/ushare/conffiles
-/etc/config/ushare
-/etc/ushare.conf
+/opt/etc/ushare.conf
 endef
 
 define Package/ushare/install
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/ushare.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ushare $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/$(PKG_NAME).config $(1)/etc/config/$(PKG_NAME)
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/$(PKG_NAME).init $(1)/etc/init.d/$(PKG_NAME)
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/ushare.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/ushare $(1)/opt/bin/
+	$(INSTALL_BIN) ./files/S99ushare $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,ushare))
