--- a/packages/utils/bluez-utils/Makefile
+++ ../../../trunk/feeds/packages/utils/bluez-utils/Makefile
@@ -10,5 +10,5 @@
 PKG_NAME:=bluez-utils
 PKG_VERSION:=3.36
-PKG_RELEASE:=11
+PKG_RELEASE:=12
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz

--- a/packages/utils/bluez-utils/files/bluez-utils.init
+++ ../../../trunk/feeds/packages/utils/bluez-utils/files/bluez-utils.init
@@ -115,4 +115,5 @@
 
 start() {
+        config_load bluetooth
         config_foreach hcid_config hcid
         config_foreach hciattach_config hciattach
