--- ../../../downloads/openwrt_trunk-r44486/feeds/packages/net/rtorrent/Makefile	2015-02-18 12:16:57.620306332 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/rtorrent/Makefile	2015-02-18 12:17:37.444305607 +0300
@@ -67,12 +67,13 @@
  This package is built with xmlrpc support
 endef
 
-TARGET_LDFLAGS += -lz -lpthread -Wl,-rpath-link=$(STAGING_DIR)/usr/lib
+TARGET_LDFLAGS += -lz -lpthread
 
 CONFIGURE_ARGS+= \
 	--enable-shared \
 	--disable-static \
-	--disable-debug
+	--disable-debug \
+	--enable-ipv6
 
 ifeq ($(BUILD_VARIANT),rpc)
 	CONFIGURE_ARGS += \
@@ -81,8 +82,8 @@
 
 
 define Package/rtorrent/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/rtorrent $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/rtorrent $(1)/opt/bin/
 endef
 
 Package/rtorrent-rpc/install = $(Package/rtorrent/install)
