--- ../../../downloads/openwrt_trunk-r35342/feeds/packages/net/znc/Makefile	2013-02-04 16:57:54.440109831 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/znc/Makefile	2013-01-28 14:43:41.435273452 +0400
@@ -41,29 +41,22 @@
 endef
 
 define Package/znc/conffiles
-/opt/etc/config/znc
+/etc/config/znc
 endef
 
 
 define Package/znc/install
-	$(INSTALL_DIR) $(1)/opt/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/znc $(1)/opt/bin/
-	$(INSTALL_DIR) $(1)/opt/etc/init.d
-	$(INSTALL_BIN) ./files/S99znc $(1)/opt/etc/init.d/
-	$(INSTALL_DIR) $(1)/opt/lib/znc/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/droproot.so $(1)/opt/lib/znc/
-endef
-
-define Package/znc/postinst
-#!/bin/sh
-
-echo Before using ZNC do not forget to:
-echo a\) Generate new config with \"znc -d /opt/share/znc -c\"
-echo b\) Add a \"LoadModule = droproot nobody nogroup\" string at first string
-echo to /opt/share/znc/configs/znc.conf, where \"nobody\" and \"nogroup\" is
-echo some unprivileged user\\group on your system.
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/znc $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/znc.init $(1)/etc/init.d/znc
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_DATA) ./files/znc.conf $(1)/etc/config/znc
+	$(INSTALL_DIR) $(1)/usr/lib/znc/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/droproot.so $(1)/usr/lib/znc/
 endef
 
+
 ZNC_MODULES :=
 ZNC_MODULE_TARGETS := droproot.so
 
@@ -79,12 +72,12 @@
   endef
 
   define Package/znc-mod-$(strip $(1))/install
-	$(INSTALL_DIR) $$(1)/opt/lib/znc/
-	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/$(subst -,_,$(strip $(1))).so $$(1)/opt/lib/znc/
+	$(INSTALL_DIR) $$(1)/usr/lib/znc/
+	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/modules/$(subst -,_,$(strip $(1))).so $$(1)/usr/lib/znc/
 	# include webadmin page templates if existing
 	if [ -d $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) ]; then \
-		$(INSTALL_DIR) $$(1)/opt/share/znc/modules ;\
-		$(CP) -r $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) $$(1)/opt/share/znc/modules ;\
+		$(INSTALL_DIR) $$(1)/usr/share/znc/modules ;\
+		$(CP) -r $$(PKG_BUILD_DIR)/modules/data/$(subst -,_,$(strip $(1))) $$(1)/usr/share/znc/modules ;\
 	fi
   endef
 
@@ -268,7 +261,7 @@
 CONFIGURE_VARS += \
 	CXXFLAGS="$(TARGET_CFLAGS) -fno-builtin -fno-rtti -nostdinc++" \
 	CPPFLAGS="-I$(STAGING_DIR)/opt/include -I$(STAGING_DIR)/include" \
-	LDFLAGS="-nodefaultlibs -lc -L$(STAGING_DIR)/opt/lib -L$(STAGING_DIR)/lib" \
+	LDFLAGS="-nodefaultlibs -lc -L$(STAGING_DIR)/opt/lib -L$(STAGING_DIR)/lib -Wl,--dynamic-linker=/opt/lib/ld-uClibc.so.0" \
 	LIBS="-luClibc++ -lm -lssl -lcrypto $(LIBGCC_S) -lc"
 
 CONFIGURE_ARGS += \
