--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/lmbench/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/lmbench/Makefile
@@ -15,8 +15,6 @@
 PKG_SOURCE_URL:=@SF/lmbench/
 PKG_MD5SUM:=b3351a3294db66a72e2864a199d37cbf
 
-PKG_BUILD_DEPENDS:=librpc
-
 include $(INCLUDE_DIR)/package.mk
 
 define Package/lmbench
@@ -42,7 +40,7 @@
 		AR="$(TARGET_CROSS)ar" \
 		OS="$(ARCH)" \
 		BASE="$(PKG_INSTALL_DIR)" \
-		LDLIBS="-lrpc" \
+		LDLIBS="" \
 		COMPILE="$(TARGET_CC) $(TARGET_CFLAGS) $(TARGET_CPPFLAGS) $(TARGET_LDFLAGS)" \
 		lmbench install
 endef
@@ -53,14 +51,14 @@
   CATEGORY:=Utilities
   TITLE:=$(1) microbenchmark
   URL:=http://sourceforge.net/projects/lmbench
-  DEPENDS:=lmbench +librpc
+  DEPENDS:=lmbench
 endef
 endef
 
 define PartInstall
 define Package/lmbench-$(subst _,-,$(1))/install
-	$(INSTALL_DIR) $$(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/$(1) $$(1)/usr/sbin/$(1)
+	$(INSTALL_DIR) $$(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/$(1) $$(1)/opt/sbin/$(1)
 endef
 endef
 
