--- a/Makefile
+++ b/Makefile
@@ -61,16 +61,16 @@
 
 DOCS=edge.8.gz supernode.1.gz n2n_v2.7.gz
 
-all: $(APPS) $(DOCS)
+all: $(APPS)
 
 edge: edge.c $(N2N_LIB) n2n_wire.h n2n.h Makefile
-	$(CC) $(CFLAGS) edge.c $(N2N_LIB) $(LIBS_EDGE) -o edge
+	$(CC) $(CFLAGS) edge.c $(N2N_LIB) $(LIBS_EDGE) $(LDFLAGS) -o edge
 
 test: test.c $(N2N_LIB) n2n_wire.h n2n.h Makefile
 	$(CC) $(CFLAGS) test.c $(N2N_LIB) $(LIBS_EDGE) -o test
 
 supernode: sn.c $(N2N_LIB) n2n.h Makefile
-	$(CC) $(CFLAGS) sn.c $(N2N_LIB) $(LIBS_SN) -o supernode
+	$(CC) $(CFLAGS) sn.c $(N2N_LIB) $(LIBS_SN) $(LDFLAGS) -o supernode
 
 benchmark: benchmark.c $(N2N_LIB) n2n_wire.h n2n.h Makefile
 	$(CC) $(CFLAGS) benchmark.c $(N2N_LIB) $(LIBS_SN) -o benchmark
