diff -urx .svn ../../../downloads/openwrt_trunk-r45513/feeds/packages/net/ntpd/files/ntp.conf ../../../openwrt_trunk/feeds/packages/net/ntpd/files/ntp.conf
--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/net/ntpd/files/ntp.conf	2015-04-20 10:09:28.934135359 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/ntpd/files/ntp.conf	2015-04-20 10:09:48.778134998 +0300
@@ -6,7 +6,7 @@
 
 restrict 127.0.0.1
 
-driftfile  /var/lib/ntp/ntp.drift
+driftfile  /opt/var/spool/ntp/ntp.drift
 
 server 0.openwrt.pool.ntp.org iburst
 server 1.openwrt.pool.ntp.org iburst
diff -urx .svn ../../../downloads/openwrt_trunk-r45513/feeds/packages/net/ntpd/files/ntpd.hotplug ../../../openwrt_trunk/feeds/packages/net/ntpd/files/ntpd.hotplug
--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/net/ntpd/files/ntpd.hotplug	2015-04-20 10:09:28.934135359 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/ntpd/files/ntpd.hotplug	2015-04-20 13:40:09.417905791 +0300
@@ -1,10 +1,10 @@
 NAME=ntpd
-CONFIG=/etc/ntp.conf
-COMMAND=/sbin/$NAME
+CONFIG=/opt/etc/ntp.conf
+COMMAND=/opt/sbin/$NAME
 
 [ "$ACTION" = "ifup" -a "$INTERFACE" = "wan" ] && {
         [ -x $COMMAND ] && [ -r $CONFIG ] && {
 		killall ntpd
-		/etc/init.d/ntpd start
+		/opt/etc/init.d/S77ntpd start
         } &
 }
diff -urx .svn ../../../downloads/openwrt_trunk-r45513/feeds/packages/net/ntpd/Makefile ../../../openwrt_trunk/feeds/packages/net/ntpd/Makefile
--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/net/ntpd/Makefile	2015-04-20 10:09:28.934135359 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/ntpd/Makefile	2015-04-20 10:09:48.778134998 +0300
@@ -87,7 +87,7 @@
 endef
 
 define Package/ntpd/conffiles
-/etc/ntp.conf
+/opt/etc/ntp.conf
 endef
 
 CONFIGURE_VARS += \
@@ -106,49 +106,38 @@
 	--enable-linuxcaps \
 	--with-yielding-select=yes \
 	--with-crypto \
-	--with-openssl-incdir="$(STAGING_DIR)/usr/include" \
-	--with-openssl-libdir="$(STAGING_DIR)/usr/lib"
+	--with-openssl-incdir="$(STAGING_DIR)/opt/include" \
+	--with-openssl-libdir="$(STAGING_DIR)/opt/lib"
 
 define Package/ntpd/install
-	$(INSTALL_DIR) $(1)/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpd/ntpd $(1)/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) ./files/ntp.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/ntpd.init $(1)/etc/init.d/ntpd
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	$(INSTALL_BIN) ./files/ntpd.hotplug $(1)/etc/hotplug.d/iface/20-ntpd
-endef
-
-define Package/ntpd/postinst
-#!/bin/sh
-[ -L "$${IPKG_INSTROOT}/usr/sbin/ntpd" ] && rm -f "$${IPKG_INSTROOT}/usr/sbin/ntpd"
-exit 0
-endef
-
-define Package/ntpd/postrm
-#!/bin/sh
-/bin/busybox ntpd -h 2>&1 | grep -q BusyBox && ln -sf ../../bin/busybox /usr/sbin/ntpd
-exit 0
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpd/ntpd $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) ./files/ntp.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S77ntpd $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/etc/hotplug.d/iface
+	$(INSTALL_BIN) ./files/ntpd.hotplug $(1)/opt/etc/hotplug.d/iface/20-ntpd
+	$(INSTALL_DIR) $(1)/opt/var/spool/ntp
 endef
 
 define Package/ntpdate/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpdate/ntpdate $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/ntpdate.init $(1)/etc/init.d/ntpdate
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpdate/ntpdate $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S77ntpdate $(1)/opt/etc/init.d/
 endef
 
 define Package/ntp-utils/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpdc/ntpdc $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpq/ntpq $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/util/ntptime $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpdc/ntpdc $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpq/ntpq $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/util/ntptime $(1)/opt/sbin/
 endef
 
 define Package/ntp-keygen/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/util/ntp-keygen $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/util/ntp-keygen $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,ntpd))
