--- ../../../openwrt_trunk.orig/feeds/packages/net/spawn-fcgi/Makefile	2012-01-30 14:19:59.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/spawn-fcgi/Makefile	2012-01-30 14:57:27.935216142 +0400
@@ -32,8 +32,8 @@
 endef
 
 define Package/spawn-fcgi/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/spawn-fcgi $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/spawn-fcgi $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,spawn-fcgi))
