--- ../../../downloads/openwrt_trunk-r36569/package/libs/libpcap/Makefile	2013-05-07 10:43:21.241939125 +0400
+++ ../../../openwrt_trunk/package/libs/libpcap/Makefile	2013-05-22 18:11:35.584469457 +0400
@@ -9,7 +9,7 @@
 
 PKG_NAME:=libpcap
 PKG_VERSION:=1.3.0
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.tcpdump.org/release/
@@ -78,16 +78,16 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/pcap* $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/pcap* $(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpcap.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpcap.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libpcap/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpcap.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpcap.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libpcap))
