--- a/includes/dhcpd.h
+++ b/includes/dhcpd.h
@@ -1416,15 +1416,15 @@
 #else /* !DEBUG */
 
 #ifndef _PATH_DHCPD_CONF
-#define _PATH_DHCPD_CONF	"/etc/dhcpd.conf"
+#define _PATH_DHCPD_CONF	"/opt/etc/dhcpd.conf"
 #endif /* DEBUG */
 
 #ifndef _PATH_DHCPD_DB
-#define _PATH_DHCPD_DB		LOCALSTATEDIR"/db/dhcpd.leases"
+#define _PATH_DHCPD_DB		"/opt/var/db/dhcpd.leases"
 #endif
 
 #ifndef _PATH_DHCPD6_DB
-#define _PATH_DHCPD6_DB		LOCALSTATEDIR"/db/dhcpd6.leases"
+#define _PATH_DHCPD6_DB		"/opt/var/db/dhcpd6.leases"
 #endif
 
 #ifndef _PATH_DHCPD_PID
@@ -1438,11 +1438,11 @@
 #endif /* DEBUG */
 
 #ifndef _PATH_DHCLIENT_CONF
-#define _PATH_DHCLIENT_CONF	"/etc/dhclient.conf"
+#define _PATH_DHCLIENT_CONF	"/opt/etc/dhclient.conf"
 #endif
 
 #ifndef _PATH_DHCLIENT_SCRIPT
-#define _PATH_DHCLIENT_SCRIPT	"/sbin/dhclient-script"
+#define _PATH_DHCLIENT_SCRIPT	"/opt/sbin/dhclient-script"
 #endif
 
 #ifndef _PATH_DHCLIENT_PID
@@ -1454,11 +1454,11 @@
 #endif
 
 #ifndef _PATH_DHCLIENT_DB
-#define _PATH_DHCLIENT_DB	LOCALSTATEDIR"/db/dhclient.leases"
+#define _PATH_DHCLIENT_DB	"/opt/var/db/dhclient.leases"
 #endif
 
 #ifndef _PATH_DHCLIENT6_DB
-#define _PATH_DHCLIENT6_DB	LOCALSTATEDIR"/db/dhclient6.leases"
+#define _PATH_DHCLIENT6_DB	"/opt/var/db/dhclient6.leases"
 #endif
 
 #ifndef _PATH_RESOLV_CONF
