--- ../../../downloads/openwrt_trunk-r43708/feeds/packages/utils/joe/Makefile	2014-12-15 12:19:58.048388959 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/joe/Makefile	2014-12-15 14:51:08.092223838 +0300
@@ -47,10 +47,10 @@
 endef
 
 define Package/joe/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/joe $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/joe $(1)/opt/bin/
 	for i in jmacs jstar rjoe jpico; do \
-	  ln -sf joe $(1)/usr/bin/$$$$i; \
+	  ln -sf joe $(1)/opt/bin/$$$$i; \
 	done
 	$(INSTALL_DIR) $(1)/etc/joe
 	$(INSTALL_CONF) ./files/joerc $(1)/etc/joe/joerc
