--- ../../../downloads/openwrt_trunk-r44486/feeds/packages/libs/libftdi/Makefile	2015-02-18 12:16:57.596306332 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/libftdi/Makefile	2015-02-19 15:03:21.642550549 +0300
@@ -40,25 +40,25 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/ftdi.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libftdi.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libftdi.pc $(1)/usr/lib/pkgconfig/libftdi.pc
+	$(INSTALL_DIR) $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/ftdi.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libftdi.{a,so*} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/libftdi.pc $(1)/opt/lib/pkgconfig/libftdi.pc
 	$(SED) \
-		's,/usr/include,$$$${prefix}/include,g' \
-		 $(1)/usr/lib/pkgconfig/libftdi.pc
+		's,/opt/include,$$$${prefix}/include,g' \
+		 $(1)/opt/lib/pkgconfig/libftdi.pc
 	$(SED) \
-		's,/usr/lib,$$$${prefix}/lib,g' \
-		 $(1)/usr/lib/pkgconfig/libftdi.pc
+		's,/opt/lib,$$$${prefix}/lib,g' \
+		 $(1)/opt/lib/pkgconfig/libftdi.pc
 endef
 
 define Package/libftdi/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/libftdi-config $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libftdi.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/libftdi-config $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libftdi.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libftdi))
