--- ../../../openwrt_trunk.orig/feeds/packages/ipv6/aiccu/Makefile
+++ ../../../openwrt_trunk/feeds/packages/ipv6/aiccu/Makefile
@@ -22,7 +22,7 @@
 define Package/aiccu
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libpthread +ip +kmod-sit +kmod-tun
+  DEPENDS:=+libpthread
   TITLE:=SixXS Automatic IPv6 Connectivity Client Utility
   URL:=http://www.sixxs.net/tools/aiccu/
 endef
@@ -40,12 +40,12 @@
 endef
 
 define Package/aiccu/conffiles
-/etc/config/aiccu
+/opt/etc/config/aiccu
 endef
 
 define Package/aiccu/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/unix-console/$(PKG_NAME) $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/unix-console/$(PKG_NAME) $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,aiccu))
