--- ../../../openwrt_trunk.orig/feeds/packages/utils/upx/Makefile	2012-07-15 21:54:16.725949870 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/upx/Makefile	2012-07-21 19:17:16.136325281 +0400
@@ -44,8 +44,8 @@
 	-DUCL_NO_ASM \
 
 define Package/upx/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/upx $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/upx $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,upx))
