--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/utils/hplip/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/hplip/Makefile
@@ -8,15 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=hplip
-PKG_VERSION:=3.9.12
-PKG_RELEASE:=4
+PKG_VERSION:=3.14.4
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@SF/hplip
-PKG_MD5SUM:=2b0906b4a57b437988829557ca62adfd
+PKG_MD5SUM:=6e8d53fb2284271505d2e1706c01f6c5
 
-PKG_BUILD_DEPENDS := python
 PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -25,7 +25,7 @@
   CATEGORY:=Utilities
   TITLE:=HP Linux Imaging and Printing
   URL:=http://sourceforge.net/projects/hplip/
-  DEPENDS+=+libjpeg +libtiff +libusb-compat +cups +sane-libs
+  DEPENDS+=+libjpeg +libtiff +libusb-compat +cups +sane-libs +dbus +python +libcupsimage
 endef
 
 define Package/hplip/description
@@ -39,7 +39,13 @@
 	--disable-pp-build \
 	--disable-doc-build \
 	--disable-foomatic-xml-install \
-	--disable-dbus-build
+	--with-cupsbackenddir=/opt/lib/cups/backend \
+	--with-cupsfilterdir=/opt/lib/cups/filter \
+	--with-icondir=/opt/var \
+	--with-systraydir=/opt/var
+
+TARGET_CFLAGS += \
+	-I$(STAGING_DIR)/opt/include/libusb-1.0
 
 define Build/Configure
 	$(call Build/Configure/Default,\
@@ -50,31 +56,61 @@
 endef
 
 define Package/hplip/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libhpip.so* $(1)/usr/lib/
-	$(CP) $(PKG_BUILD_DIR)/.libs/libhpmud.so* $(1)/usr/lib/
+	find $(PKG_INSTALL_DIR)/opt/share/hplip -name *.py | xargs $(SED) 's|/usr/bin/env python|/opt/bin/python|;s|/usr/bin/python|/opt/bin/python|'
+	find $(PKG_INSTALL_DIR)/opt -name *.py | xargs $(SED) 's|/usr/|/opt/|g'
+	$(SED) 's|/usr/|/opt/|g' $(PKG_INSTALL_DIR)/opt/lib/cups/filter/{pstotiff,hpps}
+	$(SED) 's|/bin/env python|/bin/python|' $(PKG_INSTALL_DIR)/opt/lib/cups/filter/pstotiff
+	find $(PKG_INSTALL_DIR)/opt/etc/udev/rules.d -name *.rules | xargs $(SED) 's|/usr/bin/|/opt/bin/|g'
+
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin
+
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libhp{ip,mud}.so* $(1)/opt/lib/
+
+	$(INSTALL_DIR) $(1)/opt/lib/cups/{backend,filter}
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/cups/backend/hp $(1)/opt/lib/cups/backend
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/cups/filter/{hpcups,hpps,pstotiff} $(1)/opt/lib/cups/filter
+
+	$(INSTALL_DIR) $(1)/opt/lib/sane
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/sane/libsane-hpaio.so* $(1)/opt/lib/sane
+
+	$(INSTALL_DIR) $(1)/opt/lib/python2.7/site-packages
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/python2.7/site-packages/{cups,hpmud,pcard,scan}ext.so $(1)/opt/lib/python2.7/site-packages
+
+	$(INSTALL_DIR) $(1)/opt/etc/hp
+	$(SED) 's|/usr/|/opt/|g' $(PKG_INSTALL_DIR)/opt/etc/hp/hplip.conf
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/hp/hplip.conf $(1)/opt/etc/hp
+
+	$(INSTALL_DIR) $(1)/opt/etc/sane.d/
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/sane.d/dll.conf $(1)/opt/etc/sane.d
+
+	$(INSTALL_DIR) $(1)/opt/share/hal/fdi/preprobe/10osvendor
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/share/hal/fdi/preprobe/10osvendor/20-hplip-devices.fdi $(1)/opt/share/hal/fdi/preprobe/10osvendor
+
+	$(INSTALL_DIR) $(1)/opt/share/cups/drv/hp/
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/share/cups/drv/hp/hpcups.drv  $(1)/opt/share/cups/drv/hp/
 
-	$(INSTALL_DIR) $(1)/usr/lib/cups/backend
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/hp $(1)/usr/lib/cups/backend
+	$(INSTALL_DIR) $(1)/opt/share/ppd/HP
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/ppd/HP/hp-*.ppd.gz $(1)/opt/share/ppd/HP
 
-	$(INSTALL_DIR) $(1)/usr/lib/sane
-	$(CP) $(PKG_BUILD_DIR)/.libs/libsane-hpaio.so* $(1)/usr/lib/sane
+	$(INSTALL_DIR) $(1)/opt/share/hplip
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/hplip/* $(1)/opt/share/hplip
 
-	$(INSTALL_DIR) $(1)/etc/hp
-	$(CP) $(PKG_BUILD_DIR)/hplip.conf $(1)/etc/hp/hplip.conf
+	$(INSTALL_DIR) $(1)/opt/etc/hotplug.d/usb/
+	$(INSTALL_BIN) ./files/20-hplip $(1)/opt/etc/hotplug.d/usb/
 
-	$(INSTALL_DIR) $(1)/usr/share/hplip/data/models/
-	$(CP) $(PKG_BUILD_DIR)/data/models/models.dat $(1)/usr/share/hplip/data/models/
+	$(INSTALL_DIR) $(1)/opt/etc/sane.d/dll.d/
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/sane.d/dll.conf $(1)/opt/etc/sane.d/dll.d/hplib
 
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/usb/
-	$(INSTALL_BIN) ./files/20-hplip $(1)/etc/hotplug.d/usb/
+	$(INSTALL_DIR) $(1)/opt/etc/udev/rules.d
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/udev/rules.d/*.rules $(1)/opt/etc/udev/rules.d
 
-	$(INSTALL_DIR) $(1)/etc/sane.d/dll.d/
-	$(INSTALL_DATA) ./files/hplib.conf $(1)/etc/sane.d/dll.d/hplib
+	$(INSTALL_DIR) $(1)/opt/var/{lib,log}/hp
 endef
 
 define Package/hplip/conffiles
-/etc/hp/hplip.conf
+/opt/etc/hp/hplip.conf
 endef
 
 $(eval $(call BuildPackage,hplip))
