--- a/config.c
+++ b/config.c
@@ -173,7 +173,7 @@
 
 static const char *config_def_script(void)
 {
-	return "/etc/vpnc/vpnc-script";
+	return "/opt/etc/vpnc/vpnc-script";
 }
 
 static const char *config_def_pid_file(void)
@@ -456,7 +456,7 @@
 {
 	char *realname;
 	
-	asprintf(&realname, "%s%s%s", index(name, '/') ? "" : "/etc/vpnc/", name, add_dot_conf ? ".conf" : "");
+	asprintf(&realname, "%s%s%s", index(name, '/') ? "" : "/opt/etc/vpnc/", name, add_dot_conf ? ".conf" : "");
 	return realname;
 }
 
@@ -675,8 +675,8 @@
 	}
 	
 	if (!got_conffile) {
-		read_config_file("/etc/vpnc/default.conf", config, 1);
-		read_config_file("/etc/vpnc.conf", config, 1);
+		read_config_file("/opt/etc/vpnc/default.conf", config, 1);
+		read_config_file("/opt/etc/vpnc/vpnc.conf", config, 1);
 	}
 	
 	if (!print_config) {
