--- ../../../openwrt_trunk/feeds/packages/utils/cryptsetup/Makefile.orig	2013-10-12 19:12:46.891363133 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/cryptsetup/Makefile	2013-10-12 20:08:41.995302374 +0400
@@ -19,13 +19,13 @@
 
 include $(INCLUDE_DIR)/package.mk
 
-TARGET_LDFLAGS+=-Wl,-rpath-link=$(STAGING_DIR)/usr/lib
+TARGET_LDFLAGS+=-Wl,-rpath-link=$(STAGING_DIR)/opt/lib
 
 define Package/cryptsetup/Default
   SECTION:=utils
   CATEGORY:=Utilities
   TITLE:=Cryptsetup
-  DEPENDS:=+libblkid +libuuid +libpopt +lvm2 +libdevmapper +@KERNEL_DIRECT_IO
+  DEPENDS:=+libblkid +libuuid +libpopt +libdevmapper
   URL:=http://code.google.com/p/cryptsetup/
 endef
 
@@ -63,10 +63,10 @@
 endif
 
 define Package/cryptsetup/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_BUILD_DIR)/src/.libs/cryptsetup $(1)/usr/sbin
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/.libs/libcryptsetup.so* $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(CP) $(PKG_BUILD_DIR)/src/.libs/cryptsetup $(1)/opt/sbin
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/.libs/libcryptsetup.so* $(1)/opt/lib
 endef
 
 Package/cryptsetup-openssl/install = $(Package/cryptsetup/install)
