--- ../../../backup/trunk_clean/trunk/package/libreadline/Makefile	2011-11-15 14:03:23.000000000 +0400
+++ ../../../trunk/package/libreadline/Makefile	2011-11-22 14:32:59.562517532 +0400
@@ -52,15 +52,15 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/readline $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.{a,so,so.5,so.5.2} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/readline $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{history,readline}.{a,so,so.5,so.5.2} $(1)/opt/lib/
 endef
 
 define Package/libreadline/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{history,readline}.{so,so.5,so.5.2} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{history,readline}.{so,so.5,so.5.2} $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libreadline))
