--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/keynote/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/keynote/Makefile
@@ -42,15 +42,15 @@
 	CFLAGS="$(TARGET_CFLAGS)"
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/keynote
-	$(CP) $(PKG_BUILD_DIR)/{assertion,header,keynote,signature}.h $(1)/usr/include/keynote/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libkeynote.a $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include/keynote
+	$(CP) $(PKG_BUILD_DIR)/{assertion,header,keynote,signature}.h $(1)/opt/include/keynote/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libkeynote.a $(1)/opt/lib/
 endef
 
 define Package/keynote/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/keynote $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/keynote $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,keynote))
