--- a/tests/CMakeLists.txt.orig	2011-11-27 18:46:59.000000000 +0400
+++ b/tests/CMakeLists.txt	2012-03-25 12:13:51.472794706 +0400
@@ -26,26 +26,3 @@
 endfunction(add_test_suite)
 
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-unused-function -Wno-unused-value")
-
-add_test_suite(aes)
-add_test_suite(arc4)
-add_test_suite(base64)
-add_test_suite(camellia)
-add_test_suite(cipher cipher.aes)
-add_test_suite(cipher cipher.camellia)
-add_test_suite(cipher cipher.des)
-add_test_suite(ctr_drbg)
-add_test_suite(debug)
-add_test_suite(des)
-add_test_suite(dhm)
-add_test_suite(error)
-add_test_suite(hmac_shax)
-add_test_suite(md)
-add_test_suite(mdx)
-add_test_suite(mpi)
-add_test_suite(pkcs1_v21)
-add_test_suite(shax)
-add_test_suite(rsa)
-add_test_suite(version)
-add_test_suite(xtea)
-add_test_suite(x509parse)
