--- ../../../downloads/openwrt_trunk-r35953/package/libs/polarssl/Makefile	2013-03-11 09:29:49.558073632 +0400
+++ ../../../openwrt_trunk/package/libs/polarssl/Makefile	2013-03-12 21:24:35.971719532 +0400
@@ -9,7 +9,7 @@
 
 PKG_NAME:=polarssl
 PKG_VERSION:=1.2.5
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-gpl.tgz
 PKG_SOURCE_URL:=https://polarssl.org/code/releases
@@ -61,20 +61,20 @@
 	-DUSE_SHARED_POLARSSL_LIBRARY:Bool=ON \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/polarssl $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpolarssl.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/polarssl $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpolarssl.so* $(1)/opt/lib/
 endef
 
 define Package/libpolarssl/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpolarssl.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libpolarssl.so* $(1)/opt/lib/
 endef
 
 define Package/polarssl-progs/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,libpolarssl))
