--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/isakmpd/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/isakmpd/Makefile
@@ -18,7 +18,6 @@
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION).orig
 
 include $(INCLUDE_DIR)/package.mk
-include $(INCLUDE_DIR)/kernel.mk
 
 define Package/isakmpd
   SECTION:=net
@@ -34,7 +33,7 @@
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		$(TARGET_CONFIGURE_OPTS) \
 		LINUX_DIR="$(LINUX_DIR)" \
-		EXTRA_CPPFLAGS="$(TARGET_CPPFLAGS) -I$(STAGING_DIR)/usr/include/openssl -I$(STAGING_DIR)/usr/include/keynote" \
+		EXTRA_CPPFLAGS="$(TARGET_CPPFLAGS) -I$(STAGING_DIR)/opt/include/openssl -I$(STAGING_DIR)/opt/include/keynote" \
 		EXTRA_LDFLAGS="$(TARGET_LDFLAGS)"
 
 	$(MAKE) -C $(PKG_BUILD_DIR) \
@@ -44,8 +43,8 @@
 endef
 
 define Package/isakmpd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/sbin/isakmpd $(1)/opt/sbin
 endef
 
 $(eval $(call BuildPackage,isakmpd))
