--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/mercurial/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mercurial/Makefile
@@ -41,10 +41,10 @@
 endef
 
 define Package/mercurial/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/bin $(1)/usr/
-	$(CP) $(PKG_INSTALL_DIR)/lib $(1)/usr/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/bin $(1)/opt/
+	$(CP) $(PKG_INSTALL_DIR)/lib $(1)/opt/
 endef
 
 $(eval $(call BuildPackage,mercurial))
