--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/libs/glib2/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/glib2/Makefile
@@ -60,42 +60,42 @@
 	ac_cv_func_posix_getgrgid_r=yes
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/include
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/glib-2.0 \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/glib-2.0 \
+		$(1)/opt/include/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/glib-2.0/include/*.h \
-		$(1)/usr/include/glib-2.0/
+		$(PKG_INSTALL_DIR)/opt/lib/glib-2.0/include/*.h \
+		$(1)/opt/include/glib-2.0/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/gio-unix-2.0 \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/gio-unix-2.0 \
+		$(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/glib-2.0 \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/glib-2.0 \
+		$(1)/opt/lib/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/*.{so*,a,la} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/*.{so*,a,la} \
+		$(1)/opt/lib/
 
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/*.pc \
-		$(1)/usr/lib/pkgconfig
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/*.pc \
+		$(1)/opt/lib/pkgconfig
 
 	$(INSTALL_DIR) $(2)/share/aclocal/
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/share/aclocal/*.m4  \
+		$(PKG_INSTALL_DIR)/opt/share/aclocal/*.m4  \
 		$(2)/share/aclocal/
 endef
 
 define Package/glib2/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/*.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/*.so* \
+		$(1)/opt/lib/
 endef
 
 $(eval $(call HostBuild))
