--- ../../../backup/trunk_clean/package/base-files/Makefile	2012-01-13 12:13:57.000000000 +0400
+++ ../../../trunk/package/base-files/Makefile	2012-01-17 09:42:21.789416949 +0400
@@ -46,19 +46,6 @@
 endef
 
 define Package/base-files/conffiles
-/etc/hosts
-/etc/inittab
-/etc/group
-/etc/passwd
-/etc/shadow
-/etc/profile
-/etc/shells
-/etc/sysctl.conf
-/etc/rc.local
-/etc/sysupgrade.conf
-/etc/config/
-/etc/dropbear/
-/etc/crontabs/
 $(call $(TARGET)/conffiles)
 endef
 
@@ -170,7 +157,7 @@
   DEPENDS:=+libgcc
 endif
   URL:=$(LIBC_URL)
-  PKG_FLAGS:=hold essential
+#  PKG_FLAGS:=hold essential
 endef
 
 
@@ -493,8 +480,8 @@
 ifeq ($(CONFIG_EXTERNAL_TOOLCHAIN),)
 
   define Package/libgcc/install
-	$(INSTALL_DIR) $(1)/lib
-	$(CP) $(TOOLCHAIN_DIR)/lib/libgcc_s.so.* $(1)/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(TOOLCHAIN_DIR)/lib/libgcc_s.so.* $(1)/opt/lib/
   endef
 
   define Package/libgfortran/install
@@ -508,8 +495,8 @@
   endef
 
   define Package/libstdcpp/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(TOOLCHAIN_DIR)/lib/libstdc++.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(TOOLCHAIN_DIR)/lib/libstdc++.so.* $(1)/opt/lib/
   endef
 
   use_libutil=$(if $(CONFIG_USE_GLIBC)$(CONFIG_EGLIBC_OPTION_EGLIBC_UTMP),libutil)
@@ -567,35 +554,35 @@
   endef
 
   define Package/libc/install_lib
-	$(CP) $(filter-out %/libdl_pic.a %/libpthread_pic.a %/libresolv_pic.a,$(wildcard $(TOOLCHAIN_DIR)/lib/lib*.a)) $(1)/lib/
-	$(if $(wildcard $(TOOLCHAIN_DIR)/lib/libc_so.a),$(CP) $(TOOLCHAIN_DIR)/lib/libc_so.a $(1)/lib/libc_pic.a)
+	$(CP) $(filter-out %/libdl_pic.a %/libpthread_pic.a %/libresolv_pic.a,$(wildcard $(TOOLCHAIN_DIR)/lib/lib*.a)) $(1)/opt/lib/
+	$(if $(wildcard $(TOOLCHAIN_DIR)/lib/libc_so.a),$(CP) $(TOOLCHAIN_DIR)/lib/libc_so.a $(1)/opt/lib/libc_pic.a)
 	$(if $(LIBGCC_MAP), \
-		$(CP) $(LIBGCC_A) $(1)/lib/libgcc_s_pic.a; \
-		$(CP) $(LIBGCC_MAP) $(1)/lib/libgcc_s_pic.map \
+		$(CP) $(LIBGCC_A) $(1)/opt/lib/libgcc_s_pic.a; \
+		$(CP) $(LIBGCC_MAP) $(1)/opt/lib/libgcc_s_pic.map \
 	)
   endef
 
   define Package/libpthread/install
-	$(INSTALL_DIR) $(1)/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
 		$(TOOLCHAIN_DIR)/lib/libpthread.so.* \
 		$(if $(BUILD_LIBGCC),\
 			$(PKG_BUILD_DIR)/libpthread-$(LIBC_SO_VERSION).so, \
 			$(TOOLCHAIN_DIR)/lib/libpthread-$(LIBC_SO_VERSION).so \
 		) \
-		$(1)/lib/
+		$(1)/opt/lib/
   endef
 
   define Package/libpthread/install_lib
-	$(if $(wildcard $(TOOLCHAIN_DIR)/lib/libpthread_so.a),$(CP) $(TOOLCHAIN_DIR)/lib/libpthread_so.a $(1)/lib/libpthread_pic.a)
+	$(if $(wildcard $(TOOLCHAIN_DIR)/lib/libpthread_so.a),$(CP) $(TOOLCHAIN_DIR)/lib/libpthread_so.a $(1)/opt/lib/libpthread_pic.a)
   endef
 
   define Package/librt/install
-	$(INSTALL_DIR) $(1)/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
 		$(TOOLCHAIN_DIR)/lib/librt.so.* \
 		$(TOOLCHAIN_DIR)/lib/librt-$(LIBC_SO_VERSION).so \
-		$(1)/lib/
+		$(1)/opt/lib/
   endef
 
   define Package/ldd/install
@@ -611,12 +598,14 @@
 else
 
   define Package/libgcc/install
-	for file in $(call qstrip,$(CONFIG_LIBGCC_FILE_SPEC)); do \
-		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBGCC_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
-	done ; \
-	exit 0
+#	for file in $(call qstrip,$(CONFIG_LIBGCC_FILE_SPEC)); do \
+#		dir=`dirname $$$$file` ; \
+#		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+#		$(CP) $(call qstrip,$(CONFIG_LIBGCC_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
+#	done ; \
+#	exit 0
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.libgcc-placeholder
   endef
 
   define Package/libgfortran/install
@@ -639,44 +628,50 @@
   define Package/libstdcpp/install
 	for file in $(call qstrip,$(CONFIG_LIBSTDCPP_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBSTDCPP_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+		$(CP) $(call qstrip,$(CONFIG_LIBSTDCPP_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
 	done ; \
 	exit 0
   endef
 
   define Package/libc/install
-	for file in $(call qstrip,$(CONFIG_LIBC_FILE_SPEC)); do \
-		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBC_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
-	done ; \
-	exit 0
+#	for file in $(call qstrip,$(CONFIG_LIBC_FILE_SPEC)); do \
+#		dir=`dirname $$$$file` ; \
+#		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+#		$(CP) $(call qstrip,$(CONFIG_LIBC_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
+#	done ; \
+#	exit 0
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.libc-placeholder
   endef
 
   define Package/libpthread/install
-	for file in $(call qstrip,$(CONFIG_LIBPTHREAD_FILE_SPEC)); do \
-		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBPTHREAD_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
-	done ; \
-	exit 0
+#	for file in $(call qstrip,$(CONFIG_LIBPTHREAD_FILE_SPEC)); do \
+#		dir=`dirname $$$$file` ; \
+#		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+#		$(CP) $(call qstrip,$(CONFIG_LIBPTHREAD_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
+#	done ; \
+#	exit 0
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.libpthread-placeholder
   endef
 
   define Package/librt/install
-	for file in $(call qstrip,$(CONFIG_LIBRT_FILE_SPEC)); do \
-		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LIBRT_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
-	done ; \
-	exit 0
+#	for file in $(call qstrip,$(CONFIG_LIBRT_FILE_SPEC)); do \
+#		dir=`dirname $$$$file` ; \
+#		$(INSTALL_DIR) $(1)/opt/$$$$dir ; \
+#		$(CP) $(call qstrip,$(CONFIG_LIBRT_ROOT_DIR))/$$$$file $(1)/opt/$$$$dir/ ; \
+#	done ; \
+#	exit 0
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.librt-placeholder
   endef
 
   define Package/ldd/install
 	for file in $(call qstrip,$(CONFIG_LDD_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LDD_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/bin ; \
+		$(CP) $(call qstrip,$(CONFIG_LDD_ROOT_DIR))/$$$$file $(1)/opt/bin/ ; \
 	done ; \
 	exit 0
   endef
@@ -684,8 +679,8 @@
   define Package/ldconfig/install
 	for file in $(call qstrip,$(CONFIG_LDCONFIG_FILE_SPEC)); do \
 		dir=`dirname $$$$file` ; \
-		$(INSTALL_DIR) $(1)/$$$$dir ; \
-		$(CP) $(call qstrip,$(CONFIG_LDCONFIG_ROOT_DIR))/$$$$file $(1)/$$$$dir/ ; \
+		$(INSTALL_DIR) $(1)/opt/bin ; \
+		$(CP) $(call qstrip,$(CONFIG_LDCONFIG_ROOT_DIR))/$$$$file $(1)/opt/bin/ ; \
 	done ; \
 	exit 0
   endef
