--- ../../../downloads/openwrt_trunk-r37769/feeds/packages/libs/libcrypto++/Makefile	2013-08-15 09:37:02.721572162 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libcrypto++/Makefile	2013-08-20 16:01:53.405281176 +0400
@@ -17,7 +17,7 @@
 
 PKG_FIXUP:=autoreconf
 PKG_REMOVE_FILES:=aclocal.m4 GNUmakefile
-PKG_INSTALL:=1
+PKG_INSTALL:=1a
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -42,16 +42,19 @@
 TARGET_LDFLAGS += \
 	-lpthread
 
+TARGET_CFLAGS += \
+	-O1
+
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/crypto++
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/crypto--/* $(1)/usr/include/crypto++/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcrypto++.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include/crypto++
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/crypto--/* $(1)/opt/include/crypto++/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcrypto++.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libcryptoxx/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcrypto++.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcrypto++.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libcryptoxx))
