--- ../../../backup/trunk_clean/trunk/feeds/packages/libs/aalib/Makefile	2011-11-15 14:04:00.000000000 +0400
+++ ../../../trunk/feeds/packages/libs/aalib/Makefile	2011-11-22 14:51:54.088254599 +0400
@@ -35,26 +35,26 @@
 		--without-x \
 		--without-x11-driver \
 		--without-slang-driver \
-		--with-ncurses=$(STAGING_DIR)/usr \
+		--with-ncurses=$(STAGING_DIR)/opt \
 	)
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/include
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/aalib.h \
-		$(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
+		$(PKG_INSTALL_DIR)/opt/include/aalib.h \
+		$(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libaa.{a,so*} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libaa.{a,so*} \
+		$(1)/opt/lib/
 endef
 
 define Package/libaa/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libaa.so.* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libaa.so.* \
+		$(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libaa))
