--- ../../../downloads/openwrt_trunk-r44486/package/network/utils/ifenslave/Makefile	2015-02-18 12:16:58.312306319 +0300
+++ ../../../openwrt_trunk/package/network/utils/ifenslave/Makefile	2015-02-19 14:48:31.642566250 +0300
@@ -6,9 +6,9 @@
 #
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=ifenslave
+PKG_VERSION:=2.6.22
 PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0
 
@@ -17,9 +17,7 @@
 define Package/ifenslave
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-bonding @(LINUX_3_8||LINUX_3_9||LINUX_3_10)
   TITLE:=A utility to manipulate kernel ethernet bonding devices
-  VERSION:=$(LINUX_VERSION)-$(PKG_RELEASE)
   URL:=http://www.kernel.org
 endef
 
@@ -31,13 +29,13 @@
 endef
 
 define Build/Compile
-	$(TARGET_CC) $(TARGET_CFLAGS) -o $(PKG_BUILD_DIR)/ifenslave \
+	$(TARGET_CC) $(TARGET_CFLAGS) $(TARGET_LDFLAGS) -o $(PKG_BUILD_DIR)/ifenslave \
 		$(LINUX_DIR)/Documentation/networking/ifenslave.c
 endef
 
 define Package/ifenslave/install
-	$(INSTALL_DIR) $(1)/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ifenslave $(1)/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ifenslave $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,ifenslave))
