--- ../../../openwrt_trunk/feeds/packages/libs/mysql/Makefile.prev	2013-06-01 15:48:47.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/mysql/Makefile	2013-06-01 15:51:14.901070448 +0400
@@ -13 +13 @@
-PKG_RELEASE:=1
+PKG_RELEASE:=1b
@@ -172,0 +173,3 @@
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/mysql_upgrade $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/resolveip $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/mysqlcheck $(1)/opt/bin/
