diff -urx .svn ../../../downloads/openwrt_trunk-r42803/package/system/opkg/files/opkg.conf ../../../openwrt_trunk/package/system/opkg/files/opkg.conf
--- ../../../downloads/openwrt_trunk-r42803/package/system/opkg/files/opkg.conf	2014-11-07 23:35:24.895177566 +0300
+++ ../../../openwrt_trunk/package/system/opkg/files/opkg.conf	2014-12-02 08:26:28.785287639 +0300
@@ -1,4 +1,5 @@
-dest root /
-dest ram /tmp
-lists_dir ext /var/opkg-lists
-option overlay_root /overlay
+src/gz packages http://qnapware.zyxmon.org/binaries
+dest root /Apps
+dest ram /Apps/opt/tmp
+lists_dir ext /Apps/opt/var/opkg-lists
+option tmp_dir /Apps/opt/tmp
diff -urx .svn ../../../downloads/openwrt_trunk-r42803/package/system/opkg/Makefile ../../../openwrt_trunk/package/system/opkg/Makefile
--- ../../../downloads/openwrt_trunk-r42803/package/system/opkg/Makefile	2014-11-07 23:35:24.895177566 +0300
+++ ../../../openwrt_trunk/package/system/opkg/Makefile	2014-12-02 20:27:49.341144571 +0300
@@ -5,7 +5,6 @@
 # See /LICENSE for more information.
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 include $(INCLUDE_DIR)/version.mk
 include $(INCLUDE_DIR)/feeds.mk
 
@@ -87,8 +86,9 @@
 CONFIGURE_ARGS += \
 	--disable-curl \
 	--disable-gpg \
-	--with-opkgetcdir=/etc \
-	--with-opkglockfile=/var/lock/opkg.lock
+	--with-opkgetcdir=/Apps/opt/etc \
+	--with-opkglibdir=/Apps/opt/lib \
+	--with-opkglockfile=/Apps/opt/var/lock/opkg.lock
 
 ifeq ($(BUILD_VARIANT),smime)
 	CONFIGURE_ARGS += --enable-openssl --enable-sha256
@@ -101,37 +101,24 @@
 		LDFLAGS="-Wl,--gc-sections" \
 
 define Package/opkg/Default/install
-	$(INSTALL_DIR) $(1)/usr/lib/opkg
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/opkg$(2).conf $(1)/etc/opkg.conf
-  ifeq ($(CONFIG_PER_FEED_REPO),)
-	echo "src/gz %n %U" >> $(1)/etc/opkg.conf
-  else
-	for d in base $(FEEDS_ENABLED); do \
-		echo "src/gz %n_$$$$d %U/$$$$d" >> $(1)/etc/opkg.conf; \
-	done
-    ifneq ($(CONFIG_PER_FEED_REPO_ADD_DISABLED),)
-	for d in $(FEEDS_DISABLED); do \
-		echo "# src/gz %n_$$$$d %U/$$$$d" >> $(1)/etc/opkg.conf; \
-	done
-    endif
-  endif
-	$(VERSION_SED) $(1)/etc/opkg.conf
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/opkg-cl $(1)/bin/opkg
+	$(INSTALL_DIR) $(1)/opt/lib/opkg
+	$(INSTALL_DIR) $(1)/opt/{bin,etc,tmp}
+	$(INSTALL_DIR) $(1)/opt/var/{lock,log,run}
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/bin/opkg-cl $(1)/opt/bin/opkg
+	$(INSTALL_DATA) ./files/opkg.conf $(1)/opt/etc
 endef
 
 Package/opkg/install = $(call Package/opkg/Default/install,$(1),)
 
 define Package/opkg-smime/install
 	$(call Package/opkg/Default/install,$(1),-smime)
-	$(INSTALL_DIR) $(1)/etc/ssl/certs
-	$(if $(CONFIG_OPKGSMIME_CERT),$(INSTALL_DATA) $(call qstrip,$(CONFIG_OPKGSMIME_CERT)) $(1)/etc/ssl/certs/opkg.pem,)
+	$(INSTALL_DIR) $(1)/opt/etc/ssl/certs
+	$(if $(CONFIG_OPKGSMIME_CERT),$(INSTALL_DATA) $(call qstrip,$(CONFIG_OPKGSMIME_CERT)) $(1)/opt/etc/ssl/certs/opkg.pem,)
 endef
 
 define Build/InstallDev
-	mkdir -p $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libopkg $(1)/usr/include/
+	mkdir -p $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/include/libopkg $(1)/opt/include/
 endef
 
 
