--- ../../../downloads/openwrt_trunk-r40859/package/system/opkg/Makefile	2014-05-27 14:18:46.857780074 +0400
+++ ../../../openwrt_trunk/package/system/opkg/Makefile	2014-05-27 20:52:47.629510332 +0400
@@ -5,7 +5,6 @@
 # See /LICENSE for more information.
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 include $(INCLUDE_DIR)/version.mk
 
 PKG_NAME:=opkg
@@ -50,8 +49,19 @@
   opkg knows how to install both .ipk and .deb packages.
 endef
 
+define Package/opkg-opt
+  $(call Package/opkg/Default)
+  DEPENDS:=+uclibc-opt
+  VARIANT:=unsigned
+endef
+
+define Package/opkg-opt/description
+  $(call Package/opkg/Default/description)
+endef
+
 define Package/opkg
   $(call Package/opkg/Default)
+  DEPENDS:=+opkg-opt
   VARIANT:=unsigned
 endef
 
@@ -59,9 +69,9 @@
   $(call Package/opkg/Default/description)
 endef
 
-define Package/opkg/conffiles
-/etc/opkg.conf
-endef
+#define Package/opkg-opt/conffiles
+#/opt/etc/opkg.conf
+#endef
 
 
 define Package/opkg-smime
@@ -86,8 +96,9 @@
 CONFIGURE_ARGS += \
 	--disable-curl \
 	--disable-gpg \
-	--with-opkgetcdir=/etc \
-	--with-opkglockfile=/var/lock/opkg.lock
+	--with-opkgetcdir=/opt/etc \
+	--with-opkglibdir=/opt/lib \
+	--with-opkglockfile=/opt/var/lock/opkg.lock
 
 ifeq ($(BUILD_VARIANT),smime)
 	CONFIGURE_ARGS += --enable-openssl --enable-sha256
@@ -97,28 +108,32 @@
 		CC="$(TARGET_CC)" \
 		DESTDIR="$(PKG_INSTALL_DIR)" \
 		HOST_CPU="$(PKGARCH)" \
-		LDFLAGS="-Wl,--gc-sections" \
+		LDFLAGS="-Wl,--gc-sections --static" \
 
 define Package/opkg/Default/install
-	$(INSTALL_DIR) $(1)/usr/lib/opkg
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/opkg$(2).conf $(1)/etc/opkg.conf
-	$(VERSION_SED) $(1)/etc/opkg.conf
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/opkg-cl $(1)/bin/opkg
+	$(INSTALL_DIR) $(1)/opt/lib/opkg
+	$(INSTALL_DIR) $(1)/opt/{bin,etc,tmp}
+	$(INSTALL_DIR) $(1)/opt/var/{lock,log,run}
+#	$(INSTALL_DATA) ./files/opkg.conf $(1)/opt/etc/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/opkg-cl $(1)/opt/bin/opkg
 endef
 
-Package/opkg/install = $(call Package/opkg/Default/install,$(1),)
+Package/opkg-opt/install = $(call Package/opkg/Default/install,$(1),)
 
 define Package/opkg-smime/install
 	$(call Package/opkg/Default/install,$(1),-smime)
-	$(INSTALL_DIR) $(1)/etc/ssl/certs
-	$(if $(CONFIG_OPKGSMIME_CERT),$(INSTALL_DATA) $(call qstrip,$(CONFIG_OPKGSMIME_CERT)) $(1)/etc/ssl/certs/opkg.pem,)
+	$(INSTALL_DIR) $(1)/opt/etc/ssl/certs
+	$(if $(CONFIG_OPKGSMIME_CERT),$(INSTALL_DATA) $(call qstrip,$(CONFIG_OPKGSMIME_CERT)) $(1)/opt/etc/ssl/certs/opkg.pem,)
+endef
+
+define Package/opkg/install
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.opkg-placeholder
 endef
 
 define Build/InstallDev
-	mkdir -p $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libopkg $(1)/usr/include/
+	mkdir -p $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libopkg $(1)/opt/include/
 endef
 
 
@@ -137,5 +152,6 @@
 endef
 
 $(eval $(call BuildPackage,opkg))
+$(eval $(call BuildPackage,opkg-opt))
 $(eval $(call BuildPackage,opkg-smime))
 $(eval $(call HostBuild))
