--- ../../../downloads/openwrt_trunk-r42803/package/system/opkg/Makefile
+++ ../../../openwrt_trunk/package/system/opkg/Makefile
@@ -5,7 +5,6 @@
 # See /LICENSE for more information.
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 include $(INCLUDE_DIR)/version.mk
 include $(INCLUDE_DIR)/feeds.mk
 
@@ -51,17 +50,24 @@
   opkg knows how to install both .ipk and .deb packages.
 endef
 
-define Package/opkg
+define Package/opkg-opt
   $(call Package/opkg/Default)
+  DEPENDS:=+uclibc-opt
   VARIANT:=unsigned
 endef
 
-define Package/opkg/description
+define Package/opkg-opt/description
   $(call Package/opkg/Default/description)
 endef
 
-define Package/opkg/conffiles
-/etc/opkg.conf
+define Package/opkg
+  $(call Package/opkg/Default)
+  DEPENDS:=+opkg-opt
+  VARIANT:=unsigned
+endef
+
+define Package/opkg/description
+  $(call Package/opkg/Default/description)
 endef
 
 
@@ -78,8 +84,6 @@
   This package allows the Package index to be verified with S/MIME.
 endef
 
-Package/opkg-smime/conffiles = $(Package/opkg/conffiles)
-
 TARGET_CFLAGS += $(if $(CONFIG_GCC_VERSION_4_3)$(CONFIG_GCC_VERSION_4_4),-Wno-array-bounds)
 TARGET_CFLAGS += -ffunction-sections -fdata-sections
 EXTRA_CFLAGS += $(TARGET_CPPFLAGS)
@@ -87,8 +91,9 @@
 CONFIGURE_ARGS += \
 	--disable-curl \
 	--disable-gpg \
-	--with-opkgetcdir=/etc \
-	--with-opkglockfile=/var/lock/opkg.lock
+	--with-opkgetcdir=/opt/etc \
+	--with-opkglibdir=/opt/lib \
+	--with-opkglockfile=/opt/var/lock/opkg.lock
 
 ifeq ($(BUILD_VARIANT),smime)
 	CONFIGURE_ARGS += --enable-openssl --enable-sha256
@@ -98,40 +103,31 @@
 		CC="$(TARGET_CC)" \
 		DESTDIR="$(PKG_INSTALL_DIR)" \
 		HOST_CPU="$(PKGARCH)" \
-		LDFLAGS="-Wl,--gc-sections" \
+		LDFLAGS="-Wl,--gc-sections --static" \
 
 define Package/opkg/Default/install
-	$(INSTALL_DIR) $(1)/usr/lib/opkg
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/opkg$(2).conf $(1)/etc/opkg.conf
-  ifeq ($(CONFIG_PER_FEED_REPO),)
-	echo "src/gz %n %U" >> $(1)/etc/opkg.conf
-  else
-	for d in base $(FEEDS_ENABLED); do \
-		echo "src/gz %n_$$$$d %U/$$$$d" >> $(1)/etc/opkg.conf; \
-	done
-    ifneq ($(CONFIG_PER_FEED_REPO_ADD_DISABLED),)
-	for d in $(FEEDS_DISABLED); do \
-		echo "# src/gz %n_$$$$d %U/$$$$d" >> $(1)/etc/opkg.conf; \
-	done
-    endif
-  endif
-	$(VERSION_SED) $(1)/etc/opkg.conf
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/opkg-cl $(1)/bin/opkg
+	$(INSTALL_DIR) $(1)/opt/lib/opkg
+	$(INSTALL_DIR) $(1)/opt/{bin,etc,tmp}
+	$(INSTALL_DIR) $(1)/opt/var/{lock,log,run}
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/opkg-cl $(1)/opt/bin/opkg
 endef
 
-Package/opkg/install = $(call Package/opkg/Default/install,$(1),)
+Package/opkg-opt/install = $(call Package/opkg/Default/install,$(1),)
 
 define Package/opkg-smime/install
 	$(call Package/opkg/Default/install,$(1),-smime)
-	$(INSTALL_DIR) $(1)/etc/ssl/certs
-	$(if $(CONFIG_OPKGSMIME_CERT),$(INSTALL_DATA) $(call qstrip,$(CONFIG_OPKGSMIME_CERT)) $(1)/etc/ssl/certs/opkg.pem,)
+	$(INSTALL_DIR) $(1)/opt/etc/ssl/certs
+	$(if $(CONFIG_OPKGSMIME_CERT),$(INSTALL_DATA) $(call qstrip,$(CONFIG_OPKGSMIME_CERT)) $(1)/opt/etc/ssl/certs/opkg.pem,)
+endef
+
+define Package/opkg/install
+	$(INSTALL_DIR) $(1)/opt/tmp
+	touch $(1)/opt/tmp/.opkg-placeholder
 endef
 
 define Build/InstallDev
-	mkdir -p $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libopkg $(1)/usr/include/
+	mkdir -p $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libopkg $(1)/opt/include/
 endef
 
 
@@ -150,5 +146,6 @@
 endef
 
 $(eval $(call BuildPackage,opkg))
+$(eval $(call BuildPackage,opkg-opt))
 $(eval $(call BuildPackage,opkg-smime))
 $(eval $(call HostBuild))
