diff -urx .svn ../../../downloads/openwrt_trunk-r46940/package/system/opkg/files/opkg-key ../../../openwrt_trunk/package/system/opkg/files/opkg-key
--- ../../../downloads/openwrt_trunk-r46940/package/system/opkg/files/opkg-key	2015-09-15 12:24:40.870451663 +0300
+++ ../../../openwrt_trunk/package/system/opkg/files/opkg-key	2015-09-15 12:28:17.114455262 +0300
@@ -19,7 +19,7 @@
 	(
 		zcat "$msgfile" 2>/dev/null ||
 		cat "$msgfile" 2>/dev/null
-	) | usign -V -P /etc/opkg/keys -q -x "$sigfile" -m -
+	) | usign -V -P /opt/etc/opkg/keys -q -x "$sigfile" -m -
 }
 
 opkg_key_add() {
@@ -27,8 +27,8 @@
 	[ -n "$key" ] || usage
 	[ -f "$key" ] || echo "Cannot open file $1"
 	local fingerprint="$(usign -F -p "$key")"
-	mkdir -p "/etc/opkg/keys"
-	cp "$key" "/etc/opkg/keys/$fingerprint"
+	mkdir -p "/opt/etc/opkg/keys"
+	cp "$key" "/opt/etc/opkg/keys/$fingerprint"
 }
 
 opkg_key_remove() {
@@ -36,7 +36,7 @@
 	[ -n "$key" ] || usage
 	[ -f "$key" ] || echo "Cannot open file $1"
 	local fingerprint="$(usign -F -p "$key")"
-	rm -f "/etc/opkg/keys/$fingerprint"
+	rm -f "/opt/etc/opkg/keys/$fingerprint"
 }
 
 case "$1" in
diff -urx .svn ../../../downloads/openwrt_trunk-r46940/package/system/opkg/Makefile ../../../openwrt_trunk/package/system/opkg/Makefile
--- ../../../downloads/openwrt_trunk-r46940/package/system/opkg/Makefile	2015-09-15 12:24:40.870451663 +0300
+++ ../../../openwrt_trunk/package/system/opkg/Makefile	2015-09-21 10:47:19.814982664 +0300
@@ -5,7 +5,6 @@
 # See /LICENSE for more information.
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 include $(INCLUDE_DIR)/version.mk
 include $(INCLUDE_DIR)/feeds.mk
 
@@ -40,6 +39,7 @@
 define Package/opkg/Default
   SECTION:=base
   CATEGORY:=Base system
+  DEPENDS:=+uclibc-opt
   TITLE:=opkg package manager
   URL:=http://wiki.openmoko.org/wiki/Opkg
 endef
@@ -63,13 +63,6 @@
   $(call Package/opkg/Default/description)
 endef
 
-define Package/opkg/conffiles
-/etc/opkg.conf
-/etc/opkg/keys/
-/etc/opkg/customfeeds.conf
-endef
-
-
 define Package/opkg-smime
   $(call Package/opkg/Default)
   TITLE+= (with S/MIME signature support)
@@ -83,18 +76,17 @@
   This package allows the Package index to be verified with S/MIME.
 endef
 
-Package/opkg-smime/conffiles = $(Package/opkg/conffiles)
-
 TARGET_CFLAGS += $(if $(CONFIG_GCC_VERSION_4_3)$(CONFIG_GCC_VERSION_4_4),-Wno-array-bounds)
-TARGET_CFLAGS += -ffunction-sections -fdata-sections
+TARGET_CFLAGS += -ffunction-sections -fdata-sections -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
 EXTRA_CFLAGS += $(TARGET_CPPFLAGS)
 
 CONFIGURE_ARGS += \
 	--disable-curl \
 	--disable-gpg \
 	--enable-sha256 \
-	--with-opkgetcdir=/etc \
-	--with-opkglockfile=/var/lock/opkg.lock
+	--with-opkgetcdir=/opt/etc \
+	--with-opkglibdir=/opt/lib \
+	--with-opkglockfile=/opt/var/lock/opkg.lock
 
 ifeq ($(BUILD_VARIANT),smime)
 	CONFIGURE_ARGS += --enable-openssl --disable-usign
@@ -108,39 +100,33 @@
 		CC="$(TARGET_CC)" \
 		DESTDIR="$(PKG_INSTALL_DIR)" \
 		HOST_CPU="$(PKGARCH)" \
-		LDFLAGS="-Wl,--gc-sections" \
+		LDFLAGS="-Wl,--gc-sections --static" \
 
 define Package/opkg/Default/install
-	$(INSTALL_DIR) $(1)/usr/lib/opkg
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_DIR) $(1)/etc/opkg
-	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(INSTALL_DATA) ./files/customfeeds.conf $(1)/etc/opkg/customfeeds.conf
-	$(INSTALL_DATA) ./files/opkg$(2).conf $(1)/etc/opkg.conf
-	$(call FeedSourcesAppend,$(1)/etc/opkg/distfeeds.conf)
-	$(VERSION_SED) $(1)/etc/opkg/distfeeds.conf
-	$(INSTALL_BIN) ./files/20_migrate-feeds $(1)/etc/uci-defaults/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/opkg-cl $(1)/bin/opkg
+	$(INSTALL_DIR) $(1)/opt/lib/opkg
+	$(INSTALL_DIR) $(1)/bin/{bin,etc,tmp}
+	$(INSTALL_DIR) $(1)/opt/var/{lock,log,run}
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/opkg-cl $(1)/opt/bin/opkg
 endef
 
 define Package/opkg/install
 	$(call Package/opkg/Default/install,$(1),)
   ifneq ($(CONFIG_SIGNED_PACKAGES),)
-	echo "option check_signature 1" >> $(1)/etc/opkg.conf
+	echo "option check_signature 1" >> $(1)/opt/etc/opkg.conf
   endif
-	mkdir $(1)/usr/sbin
-	$(INSTALL_BIN) ./files/opkg-key $(1)/usr/sbin/
+	mkdir $(1)/opt/sbin
+	$(INSTALL_BIN) ./files/opkg-key $(1)/opt/sbin/
 endef
 
 define Package/opkg-smime/install
 	$(call Package/opkg/Default/install,$(1),-smime)
-	$(INSTALL_DIR) $(1)/etc/ssl/certs
-	$(if $(CONFIG_OPKGSMIME_CERT),$(INSTALL_DATA) $(call qstrip,$(CONFIG_OPKGSMIME_CERT)) $(1)/etc/ssl/certs/opkg.pem,)
+	$(INSTALL_DIR) $(1)/opt/etc/ssl/certs
+	$(if $(CONFIG_OPKGSMIME_CERT),$(INSTALL_DATA) $(call qstrip,$(CONFIG_OPKGSMIME_CERT)) $(1)/opt/etc/ssl/certs/opkg.pem,)
 endef
 
 define Build/InstallDev
-	mkdir -p $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libopkg $(1)/usr/include/
+	mkdir -p $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libopkg $(1)/opt/include/
 endef
 
 
diff -urx .svn ../../../downloads/openwrt_trunk-r46940/package/system/opkg/patches/200-usign_support.patch ../../../openwrt_trunk/package/system/opkg/patches/200-usign_support.patch
--- ../../../downloads/openwrt_trunk-r46940/package/system/opkg/patches/200-usign_support.patch	2015-09-15 12:24:40.870451663 +0300
+++ ../../../openwrt_trunk/package/system/opkg/patches/200-usign_support.patch	2015-09-15 12:28:17.114455262 +0300
@@ -76,7 +76,7 @@
 +		return -1;
 +
 +	if (!pid) {
-+		execl("/usr/sbin/opkg-key", "opkg-key", "verify", sig_file, text_file, NULL);
++		execl("/opt/sbin/opkg-key", "opkg-key", "verify", sig_file, text_file, NULL);
 +		exit(255);
 +	}
 +
