--- ../../../downloads/openwrt_trunk-r40859/feeds/packages/multimedia/shairport/Makefile	2014-05-26 13:26:44.211410832 +0400
+++ ../../../openwrt_trunk/feeds/packages/multimedia/shairport/Makefile	2014-05-26 16:22:23.375217331 +0400
@@ -46,12 +46,8 @@
 	LDFLAGS="$(TARGET_LDFLAGS) $(LIBS)"
 
 define Package/shairport/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/shairport $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) files/shairport.init $(1)/etc/init.d/shairport
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) files/shairport.config $(1)/etc/config/shairport
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/shairport $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,shairport))
