--- ../../../openwrt_trunk.orig/feeds/oldpackages/lang/python-crypto/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/python-crypto/Makefile
@@ -33,7 +33,7 @@
 endef
 
 define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix=/usr --root=$(PKG_INSTALL_DIR))
+	$(call Build/Compile/PyMod,,install --prefix=/opt --root=$(PKG_INSTALL_DIR))
 endef
 
 define Package/python-crypto/install
