--- ../../../downloads/openwrt_trunk-r45513/feeds/oldpackages/net/pyload/Makefile	2015-04-21 10:37:38.304588031 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/net/pyload/Makefile	2015-04-21 14:24:29.584340087 +0300
@@ -9,7 +9,7 @@
 
 PKG_NAME:=pyload
 PKG_VERSION:=0.4.9
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_SOURCE:=$(PKG_NAME)-src-v$(PKG_VERSION).zip
 PKG_SOURCE_URL:=http://download.pyload.org/
@@ -25,7 +25,7 @@
   SECTION:=net
   CATEGORY:=Network
   DEPENDS:=+python +pyopenssl +python-curl +python-crypto \
-	+python-expat +python-imaging-library +python-sqlite3 +js \
+	+python-imglib +python-sqlite3 +duktape \
 	+tesseract +unrar
   TITLE:=A fast, lightweight and full featured download manager
   URL:=http://pyload.org
@@ -38,12 +38,12 @@
 endef
 
 define Package/pyload/install
-	$(INSTALL_DIR) $(1)/usr/share/python
-	$(CP) $(PKG_BUILD_DIR)/pyload $(1)/usr/share/python/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/pyload.init $(1)/etc/init.d/pyload
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(LN) ../share/python/pyload/pyLoadCore.py $(1)/usr/bin/pyload
+	$(INSTALL_DIR) $(1)/opt/share/python
+	$(CP) $(PKG_BUILD_DIR)/pyload $(1)/opt/share/python/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(LN) ../share/python/pyload/pyLoadCore.py $(1)/opt/bin/pyload
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S51pyload $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,pyload))
