--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/pyload/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/pyload/Makefile
@@ -38,12 +38,12 @@
 endef
 
 define Package/pyload/install
-	$(INSTALL_DIR) $(1)/usr/share/python
-	$(CP) $(PKG_BUILD_DIR)/pyload $(1)/usr/share/python/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/pyload.init $(1)/etc/init.d/pyload
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(LN) ../share/python/pyload/pyLoadCore.py $(1)/usr/bin/pyload
+	$(INSTALL_DIR) $(1)/opt/share/python
+	$(CP) $(PKG_BUILD_DIR)/pyload $(1)/opt/share/python/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(LN) ../share/python/pyload/pyLoadCore.py $(1)/opt/bin/pyload
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S51pyload $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,pyload))
