--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/utils/avrdude/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/avrdude/Makefile
@@ -48,14 +48,14 @@
 endef
 
 define Package/avrdude/conffiles
-/etc/avrdude.conf
+/opt/etc/avrdude.conf
 endef
 
 define Package/avrdude/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/avrdude.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/avrdude $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/avrdude.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/avrdude $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,avrdude))
