--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/lang/perl-www-curl/Makefile	2014-10-06 15:46:23.733250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/lang/perl-www-curl/Makefile	2014-10-07 11:59:34.011924243 +0400
@@ -34,7 +34,7 @@
 endef
 
 define Build/Configure
-	$(call perlmod/Configure,$(STAGING_DIR)/usr/include,)
+	$(call perlmod/Configure,$(STAGING_DIR)/opt/include,)
 endef
 
 define Build/Compile
