--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/utils/lsof/Makefile	2014-04-09 16:17:24.234286480 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/lsof/Makefile	2014-04-11 16:37:47.979115865 +0400
@@ -32,9 +32,9 @@
 
 define Build/Configure
 	cd $(PKG_BUILD_DIR); \
-		LINUX_CLIB="-DGLIBCV=2" \
+		LINUX_CLIB="-DGLIBCV=2 -DHASIPv6" \
 		LSOF_CC="$(TARGET_CC)" \
-		LSOF_INCLUDE="-I$(STAGING_DIR)/usr/include" \
+		LSOF_INCLUDE="-I$(STAGING_DIR)/opt/include" \
 		LSOF_VSTR="$(LINUX_VERSION)" \
 		LSOF_CFGF="$(TARGET_CFLAGS)" \
 		LSOF_CFGL="$(TARGET_LDFLAGS)" \
@@ -48,8 +48,8 @@
 endef
 
 define Package/lsof/install	
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/lsof $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/lsof $(1)/opt/bin
 endef
 
 $(eval $(call BuildPackage,lsof))
