--- ../../../openwrt_trunk.orig/feeds/packages/net/asterisk-chan-sccp-b/Makefile	2011-12-05 10:46:38.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/asterisk-chan-sccp-b/Makefile	2011-12-20 15:49:13.085181701 +0400
@@ -36,7 +36,7 @@
 endef
 
 CONFIGURE_ARGS += \
-	--with-asterisk=$(STAGING_DIR)/usr/include/asterisk-1.8
+	--with-asterisk=$(STAGING_DIR)/opt/include/asterisk-1.8
 
 define Build/Compile
 	$(MAKE) -C "$(PKG_BUILD_DIR)" \
@@ -46,14 +46,14 @@
 endef
 
 define Package/asterisk18-chan-sccp-b/conffiles
-/etc/asterisk/sccp.conf
+/opt/etc/asterisk/sccp.conf
 endef
 
 define Package/asterisk18-chan-sccp-b/install
-	$(INSTALL_DIR) $(1)/etc/asterisk
-	$(INSTALL_BIN)  ./files/sccp.openwrt.conf $(1)/etc/asterisk/sccp.conf
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/modules
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/asterisk/modules/chan_sccp.so $(1)/usr/lib/asterisk/modules/
+	$(INSTALL_DIR) $(1)/opt/etc/asterisk
+	$(INSTALL_BIN)  ./files/sccp.openwrt.conf $(1)/opt/etc/asterisk/sccp.conf
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/modules
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/asterisk/modules/chan_sccp.so $(1)/opt/lib/asterisk/modules/
 endef
 
 $(eval $(call BuildPackage,asterisk18-chan-sccp-b))
