--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/net-tools/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/net-tools/Makefile
@@ -223,87 +223,88 @@
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 	CC="$(TARGET_CC)" \
 	AR="$(TARGET_CROSS)ar" \
+	LOPTS="$(TARGET_LDFLAGS)" \
 	COPTS="$(TARGET_CFLAGS)"
 endef
 
 define Package/net-tools-arp/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/arp $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/arp $(1)/opt/bin/
 endef
 
 define Package/net-tools-dnsdomainname/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	ln -sf hostname $(1)/usr/bin/dnsdomainname
+	$(INSTALL_DIR) $(1)/opt/bin
+	ln -sf hostname $(1)/opt/bin/dnsdomainname
 endef
 
 define Package/net-tools-domainname/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	ln -sf hostname $(1)/usr/bin/domainname
+	$(INSTALL_DIR) $(1)/opt/bin
+	ln -sf hostname $(1)/opt/bin/domainname
 endef
 
 define Package/net-tools-hostname/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/hostname $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/hostname $(1)/opt/bin/
 endef
 
 define Package/net-tools-ifconfig/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ifconfig $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ifconfig $(1)/opt/sbin/
 endef
 
 define Package/net-tools-ipmaddr/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ipmaddr $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ipmaddr $(1)/opt/bin/
 endef
 
 define Package/net-tools-iptunnel/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/iptunnel $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/iptunnel $(1)/opt/bin/
 endef
 
 define Package/net-tools-mii-tool/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mii-tool $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mii-tool $(1)/opt/bin/
 endef
 
 define Package/net-tools-nameif/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/nameif $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/nameif $(1)/opt/bin/
 endef
 
 define Package/net-tools-netstat/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/netstat $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/netstat $(1)/opt/bin/
 endef
 
 define Package/net-tools-nisdomainname/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	ln -sf hostname $(1)/usr/bin/nisdomainname
+	$(INSTALL_DIR) $(1)/opt/bin
+	ln -sf hostname $(1)/opt/bin/nisdomainname
 endef
 
 define Package/net-tools-plipconfig/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/plipconfig $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/plipconfig $(1)/opt/bin/
 endef
 
 define Package/net-tools-rarp/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rarp $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rarp $(1)/opt/bin/
 endef
 
 define Package/net-tools-route/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/route $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/route $(1)/opt/bin/
 endef
 
 define Package/net-tools-slattach/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/slattach $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/slattach $(1)/opt/bin/
 endef
 
 define Package/net-tools-ypdomainname/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	ln -sf hostname $(1)/usr/bin/ypdomainname
+	$(INSTALL_DIR) $(1)/opt/bin
+	ln -sf hostname $(1)/opt/bin/ypdomainname
 endef
 
 $(eval $(call BuildPackage,net-tools-arp))
