--- ../../../downloads/openwrt_trunk-r37769/feeds/oldpackages/mail/fetchmail/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/mail/fetchmail/Makefile
@@ -62,7 +62,7 @@
 
 ifeq ($(BUILD_VARIANT),ssl)
 	CONFIGURE_ARGS += \
-		--with-ssl="$(STAGING_DIR)/usr"
+		--with-ssl="$(STAGING_DIR)/opt"
 endif
 
 ifeq ($(BUILD_VARIANT),nossl)
@@ -71,11 +71,11 @@
 endif
 
 CONFIGURE_VARS += \
-	ac_cv_path_procmail=/usr/sbin/procmail
+	ac_cv_path_procmail=/opt/sbin/procmail
 
 define Package/fetchmail/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/fetchmail $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/fetchmail $(1)/opt/bin/
 endef
 
 Package/fetchmail-nossl/install = $(Package/fetchmail/install)
