Index: inotify.c
===================================================================
RCS file: /cvsroot/minidlna/minidlna/inotify.c,v
retrieving revision 1.49
diff -u -b -w -r1.49 inotify.c
--- a/inotify.c	31 Jan 2012 01:17:00 -0000	1.49
+++ b/inotify.c	27 Mar 2012 10:11:06 -0000
@@ -349,9 +350,11 @@
 	else if( ts < st.st_mtime )
 	{
 		if( ts > 0 )
+		{
 			DPRINTF(E_DEBUG, L_INOTIFY, "%s is newer than the last db entry.\n", path);
 		inotify_remove_file(path);
 	}
+	}
 
 	/* Find the parentID.  If it's not found, create all necessary parents. */
 	len = strlen(path)+1;
