diff -urx .svn ../../../openwrt_trunk.orig/feeds/packages/multimedia/minidlna/Makefile ../../../openwrt_trunk/feeds/packages/multimedia/minidlna/Makefile
--- ../../../openwrt_trunk.orig/feeds/packages/multimedia/minidlna/Makefile	2012-06-11 17:16:08.052541580 +0400
+++ ../../../openwrt_trunk/feeds/packages/multimedia/minidlna/Makefile	2012-06-11 17:16:54.983951571 +0400
@@ -18,7 +18,6 @@
 PKG_BUILD_PARALLEL:=0
 PKG_BUILD_DEPENDS:=util-linux
 
-include $(INCLUDE_DIR)/kernel.mk
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
 
@@ -28,7 +27,7 @@
   TITLE:=UPnP A/V & DLNA Media Server
   URL:=http://minidlna.sourceforge.net/
   DEPENDS:= +libpthread +libexif +libjpeg +libsqlite3 +libffmpeg \
-  	+libid3tag +libflac +libvorbis +libuuid \
+  	+libid3tag +libflac +libvorbis +libuuid +libbz2 \
   	$(ICONV_DEPENDS) $(INTL_DEPENDS)
 endef
 
@@ -38,18 +37,18 @@
 endef
 
 define Package/minidlna/conffiles
-/etc/minidlna.conf
+/opt/etc/minidlna.conf
 endef
 
 TARGET_CPPFLAGS += \
-	-I$(STAGING_DIR)/usr/include \
-	-I$(STAGING_DIR)/usr/include/FLAC \
-	-I$(STAGING_DIR)/usr/include/libavcodec \
-	-I$(STAGING_DIR)/usr/include/libavformat \
-	-I$(STAGING_DIR)/usr/include/libavutil \
-	-I$(STAGING_DIR)/usr/include/libexif \
-	-I$(STAGING_DIR)/usr/include/uuid \
-	-I$(STAGING_DIR)/usr/include/vorbis \
+	-I$(STAGING_DIR)/opt/include \
+	-I$(STAGING_DIR)/opt/include/FLAC \
+	-I$(STAGING_DIR)/opt/include/libavcodec \
+	-I$(STAGING_DIR)/opt/include/libavformat \
+	-I$(STAGING_DIR)/opt/include/libavutil \
+	-I$(STAGING_DIR)/opt/include/libexif \
+	-I$(STAGING_DIR)/opt/include/uuid \
+	-I$(STAGING_DIR)/opt/include/vorbis \
 	-I$(ICONV_PREFIX)/include \
 	-I$(INTL_PREFIX)/include \
 	-D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
@@ -57,31 +56,31 @@
 TARGET_LDFLAGS += \
 	-L$(ICONV_PREFIX)/lib \
 	-L$(INTL_PREFIX)/lib \
-	-Wl,-rpath-link=$(STAGING_DIR)/usr/lib \
+	-Wl,-rpath-link=$(STAGING_DIR)/opt/lib \
 
 MAKE_FLAGS +=\
 	CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS)" \
 	LDFLAGS="$(TARGET_LDFLAGS)" \
-	ICONV_LIBS="-liconv" \
+	ICONV_LIBS="-liconv -lintl" \
 
 MAKE_VARS +=\
-	PREFIX="$(STAGING_DIR)/usr" \
+	PREFIX="$(STAGING_DIR)/opt" \
 	ICONV_PREFIX="$(ICONV_PREFIX)" \
 	INTL_PREFIX="$(INTL_PREFIX)" \
 	OS_NAME="OpenWrt Linux" \
 	OS_VERSION="$(LINUX_VERSION)" \
 	OS_URL="http://openwrt.org/" \
-	DB_PATH="/var/run/minidlna" \
-	LOG_PATH="/var/log" \
+	DB_PATH="/opt/var/run/minidlna" \
+	LOG_PATH="/opt/var/log" \
 
 
 define Package/minidlna/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/minidlna $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/minidlna.init $(1)/etc/init.d/minidlna
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/minidlna.config $(1)/etc/config/minidlna
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/minidlna $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/minidlna.conf $(1)/opt/etc
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S90minidlna $(1)/opt/etc/init.d
 endef
 
 define Package/minidlna/conffiles
diff -urx .svn ../../../openwrt_trunk.orig/feeds/packages/multimedia/minidlna/patches/010-genconfig-checks.patch ../../../openwrt_trunk/feeds/packages/multimedia/minidlna/patches/010-genconfig-checks.patch
--- ../../../openwrt_trunk.orig/feeds/packages/multimedia/minidlna/patches/010-genconfig-checks.patch	2012-06-11 17:16:08.052541580 +0400
+++ ../../../openwrt_trunk/feeds/packages/multimedia/minidlna/patches/010-genconfig-checks.patch	2012-06-11 17:16:54.987951521 +0400
@@ -4,13 +4,13 @@
  CONFIGFILE="config.h"
  CONFIGMACRO="__CONFIG_H__"
  
-+PREFIX="${PREFIX:-/usr}"
++PREFIX="${PREFIX:-/opt}"
 +ICONV_PREFIX="${ICONV_PREFIX:-$PREFIX}"
 +INTL_PREFIX="${INTL_PREFIX:-$PREFIX}"
 +
  # Database path
 -DB_PATH="/tmp/minidlna"
-+DB_PATH="${DB_PATH:-/tmp/minidlna}"
++DB_PATH="${DB_PATH:-/opt/tmp/minidlna}"
  # Log path
 -LOG_PATH="${DB_PATH}"
 +LOG_PATH="${LOG_PATH:-$DB_PATH}"
diff -urx .svn ../../../openwrt_trunk.orig/feeds/packages/multimedia/minidlna/patches/020-makefile-tweaks.patch ../../../openwrt_trunk/feeds/packages/multimedia/minidlna/patches/020-makefile-tweaks.patch
--- ../../../openwrt_trunk.orig/feeds/packages/multimedia/minidlna/patches/020-makefile-tweaks.patch	2012-06-11 17:16:08.052541580 +0400
+++ ../../../openwrt_trunk/feeds/packages/multimedia/minidlna/patches/020-makefile-tweaks.patch	2012-06-12 11:11:55.460370608 +0400
@@ -1,18 +1,19 @@
---- a/Makefile
-+++ b/Makefile
+--- ./Makefile.orig	2012-01-18 02:49:01.000000000 +0400
++++ ./Makefile	2012-06-12 11:11:06.680616425 +0400
 @@ -10,19 +10,23 @@
  # or :
  # $ make install
  #
-+PREFIX ?= /usr
++PREFIX ?= /opt
 +ICONV_PREFIX ?= $(PREFIX)
 +INTL_PREFIX ?= $(PREFIX)
  #CFLAGS = -Wall -O -D_GNU_SOURCE -g -DDEBUG
  #CFLAGS = -Wall -g -Os -D_GNU_SOURCE
- CFLAGS = -Wall -g -O3 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
+-CFLAGS = -Wall -g -O3 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
 -	 -I/usr/include/ffmpeg \
 -	 -I/usr/include/libavutil -I/usr/include/libavcodec -I/usr/include/libavformat \
 -	 -I/usr/include/ffmpeg/libavutil -I/usr/include/ffmpeg/libavcodec -I/usr/include/ffmpeg/libavformat
++CFLAGS = -Wall -g -O2 -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 \
 +	 -I$(PREFIX)/include/ffmpeg \
 +	 -I$(PREFIX)/include/libavutil -I$(PREFIX)/include/libavcodec -I$(PREFIX)/include/libavformat \
 +	 -I$(PREFIX)/include/ffmpeg/libavutil -I$(PREFIX)/include/ffmpeg/libavcodec -I$(PREFIX)/include/ffmpeg/libavformat
@@ -28,7 +29,7 @@
  SBININSTALLDIR = $(INSTALLPREFIX)/sbin
  ETCINSTALLDIR = $(DESTDIR)/etc
  
-@@ -37,7 +41,7 @@ BASEOBJS = minidlna.o upnphttp.o upnpdes
+@@ -37,7 +41,7 @@
  
  ALLOBJS = $(BASEOBJS) $(LNXOBJS)
  
@@ -37,7 +38,7 @@
  #STATIC_LINKING: LIBS = -lvorbis -logg -lm -lsqlite3 -lpthread -lexif -ljpeg -lFLAC -lm -lid3tag -lz -lavformat -lavutil -lavcodec -lm
  
  TESTUPNPDESCGENOBJS = testupnpdescgen.o upnpdescgen.o
-@@ -64,7 +68,7 @@ install-conf:
+@@ -64,7 +68,7 @@
  	$(INSTALL) -d $(ETCINSTALLDIR)
  	$(INSTALL) --mode=0644 minidlna.conf $(ETCINSTALLDIR)
  
