--- ../../../downloads/openwrt_trunk-r43718/feeds/oldpackages/lang/python-json/Makefile	2014-11-07 23:35:24.502715101 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/python-json/Makefile	2014-11-28 11:24:32.180057687 +0300
@@ -36,7 +36,7 @@
 	$(INSTALL_DIR) $(PKG_INSTALL_DIR)$(PYTHON_PKG_DIR)
 	$(INSTALL_DATA) \
 		$(PKG_BUILD_DIR)/json.py \
-		$(PKG_INSTALL_DIR)$(PYTHON_PKG_DIR)
+		$(PKG_INSTALL_DIR)/Apps$(PYTHON_PKG_DIR)
 endef
 
 define Package/python-json/install
