--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/spawn-fcgi/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/spawn-fcgi/Makefile
@@ -32,8 +32,8 @@
 endef
 
 define Package/spawn-fcgi/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/spawn-fcgi $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/spawn-fcgi $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,spawn-fcgi))
