--- ../../../downloads/openwrt_trunk-r42803/feeds/oldpackages/libs/clearsilver/Makefile	2014-11-07 23:35:24.439626265 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/clearsilver/Makefile	2014-11-26 17:44:25.400388894 +0300
@@ -32,7 +32,7 @@
   TITLE:=ClearSilver template system
   URL:=http://www.clearsilver.net/
   MAINTAINER:=Raphaël HUCK <rhk@cksum.org>
-  DEPENDS:=+CLEARSILVER_ENABLE_COMPRESSION:zlib +CLEARSILVER_ENABLE_GETTEXT:libintl
+  DEPENDS:=+CLEARSILVER_ENABLE_COMPRESSION:zlib +CLEARSILVER_ENABLE_GETTEXT:$(INTL_DEPENDS)
 endef
 
 define Package/clearsilver/config
@@ -57,7 +57,7 @@
 	--disable-ruby \
 	--disable-java \
 	--disable-csharp \
-	--prefix=/usr
+	--prefix=/Apps/opt
 
 ifeq ($(SDK)$(CONFIG_CLEARSILVER_ENABLE_COMPRESSION),y)
 	CONFIGURE_ARGS+= \
@@ -92,11 +92,11 @@
 	OSTYPE=Linux
 
 define Build/InstallDev
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/* $(1)/
 endef
 
 define Package/clearsilver/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 endef
 
 $(eval $(call BuildPackage,clearsilver))
