--- ../../../downloads/openwrt_trunk-r44486/feeds/oldpackages/libs/clearsilver/Makefile	2015-02-18 12:16:57.672306331 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/clearsilver/Makefile	2015-02-18 16:52:26.480009341 +0300
@@ -32,7 +32,7 @@
   TITLE:=ClearSilver template system
   URL:=http://www.clearsilver.net/
   MAINTAINER:=Raphaël HUCK <rhk@cksum.org>
-  DEPENDS:=+CLEARSILVER_ENABLE_COMPRESSION:zlib +CLEARSILVER_ENABLE_GETTEXT:libintl
+  DEPENDS:=+CLEARSILVER_ENABLE_COMPRESSION:zlib +CLEARSILVER_ENABLE_GETTEXT:$(INTL_DEPENDS)
 endef
 
 define Package/clearsilver/config
@@ -57,7 +57,7 @@
 	--disable-ruby \
 	--disable-java \
 	--disable-csharp \
-	--prefix=/usr
+	--prefix=/opt
 
 ifeq ($(SDK)$(CONFIG_CLEARSILVER_ENABLE_COMPRESSION),y)
 	CONFIGURE_ARGS+= \
@@ -96,7 +96,7 @@
 endef
 
 define Package/clearsilver/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 endef
 
 $(eval $(call BuildPackage,clearsilver))
