--- ../../../downloads/openwrt_trunk-r43593/feeds/packages/utils/sispmctl/Makefile	2014-12-10 12:26:33.030951680 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/sispmctl/Makefile	2014-12-11 12:04:08.001400889 +0300
@@ -49,8 +49,8 @@
 	--disable-dependency-tracking
 
 define Package/sispmctl/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME) $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,sispmctl))
