--- ../../../openwrt_trunk.orig/feeds/packages/net/iperf/Makefile	2012-07-10 18:59:23.033221824 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/iperf/Makefile	2012-07-10 19:00:03.448713732 +0400
@@ -75,8 +75,8 @@
 endif
 
 define Package/iperf/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/iperf $(1)/usr/bin/iperf
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/iperf $(1)/opt/bin/iperf
 endef
 Package/iperf-mt/install = $(Package/iperf/install)
 
