--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/dbus-glib/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/dbus-glib/Makefile
@@ -49,25 +49,25 @@
 	LDFLAGS="$$$$LDFLAGS -L$(INTL_PREFIX)/lib" \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/dbus-1.0/dbus
+	$(INSTALL_DIR) $(1)/opt/include/dbus-1.0/dbus
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/include/dbus-1.0/dbus/*.h \
-		$(1)/usr/include/dbus-1.0/dbus/
+		$(PKG_INSTALL_DIR)/opt/include/dbus-1.0/dbus/*.h \
+		$(1)/opt/include/dbus-1.0/dbus/
 
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libdbus-glib-1.{la,a,so*} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libdbus-glib-1.{la,a,so*} \
+		$(1)/opt/lib/
 
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/dbus-glib-1.pc \
-		$(1)/usr/lib/pkgconfig/
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/dbus-glib-1.pc \
+		$(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libdbus-glib/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libdbus-glib-1.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libdbus-glib-1.so.* $(1)/opt/lib/
 endef
 
 
