--- ../../../downloads/openwrt_trunk-r43593/package/libs/libusb-compat/Makefile	2014-12-10 12:26:33.266951674 +0300
+++ ../../../openwrt_trunk/package/libs/libusb-compat/Makefile	2014-12-12 11:15:27.647872884 +0300
@@ -42,12 +42,12 @@
 	$(INSTALL_DIR) $(1)
 	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
 	$(INSTALL_DIR) $(2)/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/libusb-config $(2)/bin/
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/libusb-config $(2)/bin/
 endef
 
 define Package/libusb-compat/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libusb-0.1.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libusb-0.1.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libusb-compat))
