--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/package/libs/libusb-compat/Makefile	2014-04-09 16:17:25.278286461 +0400
+++ ../../../openwrt_trunk/package/libs/libusb-compat/Makefile	2014-04-15 13:15:33.009042082 +0400
@@ -39,12 +39,12 @@
 
 define Build/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/libusb-config $(STAGING_DIR)/host/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/libusb-config $(STAGING_DIR)/host/bin
 endef
 
 define Package/libusb-compat/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libusb-0.1.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libusb-0.1.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libusb-compat))
