--- ../../../downloads/openwrt_trunk-r42803/package/libs/libusb-compat//Makefile	2014-11-07 23:35:24.734654965 +0300
+++ ../../../openwrt_trunk/package/libs/libusb-compat//Makefile	2014-11-26 14:55:07.232797784 +0300
@@ -40,13 +40,13 @@
 define Build/InstallDev
 	$(INSTALL_DIR) $(STAGING_DIR)/host/bin
 
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/libusb-config $(STAGING_DIR)/host/bin
+	$(CP) $(PKG_INSTALL_DIR)/Apps/* $(1)/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/libusb-config $(STAGING_DIR)/host/bin
 endef
 
 define Package/libusb-compat/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libusb-0.1.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libusb-0.1.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libusb-compat))
