diff -urx .svn ../../../downloads/openwrt_trunk-r45513/feeds/packages/lang/ruby/Makefile ../../../openwrt_trunk/feeds/packages/lang/ruby/Makefile
--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/lang/ruby/Makefile	2015-04-21 10:37:38.184588033 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/ruby/Makefile	2015-04-22 09:57:02.515054286 +0300
@@ -575,7 +575,7 @@
 define Package/ruby-psych
 $(call Package/ruby/Default)
   TITLE+=YAML parser and emitter
-  DEPENDS:=ruby +ruby-bigdecimal +ruby-datetime +ruby-misc +ruby-enc
+  DEPENDS:=ruby +ruby-bigdecimal +ruby-datetime +ruby-misc +ruby-enc +libyaml
 endef
 
 define Package/ruby-psych/description
@@ -833,24 +833,25 @@
 
 MAKE_FLAGS += \
 	DESTDIR="$(PKG_INSTALL_DIR)" \
+	EXTLDFLAGS="$(TARGET_LDFLAGS)" \
 	SHELL="/bin/bash"
 
 define Package/ruby/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/vendor_ruby/$(PKG_LIBVER)
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/site_ruby/$(PKG_LIBVER)
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ruby $(1)/usr/lib/ruby/ruby$(PKG_LIBVER)-bin
-	$(INSTALL_BIN) ./files/ruby $(1)/usr/bin/ruby
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/vendor_ruby/$(PKG_LIBVER)/* $(1)/usr/lib/ruby/vendor_ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/site_ruby/$(PKG_LIBVER)/* $(1)/usr/lib/ruby/site_ruby/$(PKG_LIBVER)/
-	sed -i -e "s%@RUBY_LIBPATH@%/usr/lib/ruby/$(PKG_LIBVER)%" $(1)/usr/bin/ruby
-	sed -i -e "s%@RUBY_BINPATH@%/usr/lib/ruby/ruby$(PKG_LIBVER)-bin%" $(1)/usr/bin/ruby
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/vendor_ruby/$(PKG_LIBVER)
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/site_ruby/$(PKG_LIBVER)
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ruby $(1)/opt/lib/ruby/ruby$(PKG_LIBVER)-bin
+	$(INSTALL_BIN) ./files/ruby $(1)/opt/bin/ruby
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/vendor_ruby/$(PKG_LIBVER)/* $(1)/opt/lib/ruby/vendor_ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/site_ruby/$(PKG_LIBVER)/* $(1)/opt/lib/ruby/site_ruby/$(PKG_LIBVER)/
+	sed -i -e "s%@RUBY_LIBPATH@%/opt/lib/ruby/$(PKG_LIBVER)%" $(1)/opt/bin/ruby
+	sed -i -e "s%@RUBY_BINPATH@%/opt/lib/ruby/ruby$(PKG_LIBVER)-bin%" $(1)/opt/bin/ruby
 endef
 
 define Package/libruby/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libruby.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libruby.so.* $(1)/opt/lib/
 endef
 
 define Package/ruby-stdlib/install
@@ -859,217 +860,217 @@
 
 define Package/ruby-bigdecimal/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/bigdecimal.so \
-		usr/lib/ruby/$(PKG_LIBVER)/bigdecimal/ \
-		usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/bigdecimal-*.gemspec \
+		opt/lib/ruby/$(PKG_LIBVER)/*/bigdecimal.so \
+		opt/lib/ruby/$(PKG_LIBVER)/bigdecimal/ \
+		opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/bigdecimal-*.gemspec \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-cgi/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/cgi $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/cgi.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/cgi $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/cgi.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-csv/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/csv.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/csv.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-datetime/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/time.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/date.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/date/ \
-		usr/lib/ruby/$(PKG_LIBVER)/*/date_core.so \
+		opt/lib/ruby/$(PKG_LIBVER)/time.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/date.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/date/ \
+		opt/lib/ruby/$(PKG_LIBVER)/*/date_core.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-dbm/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/dbm.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/dbm.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-debuglib/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/profile.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/profiler.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/debug.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/tracer.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/benchmark.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/objspace.so \
+		opt/lib/ruby/$(PKG_LIBVER)/profile.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/profiler.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/debug.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/tracer.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/benchmark.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/objspace.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-digest/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/digest \
-		usr/lib/ruby/$(PKG_LIBVER)/digest.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/digest.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/digest/* \
+		opt/lib/ruby/$(PKG_LIBVER)/digest \
+		opt/lib/ruby/$(PKG_LIBVER)/digest.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/digest.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/digest/* \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-drb/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/drb.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/drb \
+		opt/lib/ruby/$(PKG_LIBVER)/drb.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/drb \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-enc/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/enc/encdb.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/enc/iso_8859_1.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/enc/utf_* \
-		usr/lib/ruby/$(PKG_LIBVER)/*/enc/euc_jp.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/enc/encdb.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/enc/iso_8859_1.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/enc/utf_* \
+		opt/lib/ruby/$(PKG_LIBVER)/*/enc/euc_jp.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-enc-extra/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) \
-		--exclude=usr/lib/ruby/$(PKG_LIBVER)/*/enc/encdb.so \
-		--exclude=usr/lib/ruby/$(PKG_LIBVER)/*/enc/iso_8859_1.so \
-		--exclude=usr/lib/ruby/$(PKG_LIBVER)/*/enc/utf_* \
-		--exclude=usr/lib/ruby/$(PKG_LIBVER)/*/enc/euc_jp.so \
+		--exclude=opt/lib/ruby/$(PKG_LIBVER)/*/enc/encdb.so \
+		--exclude=opt/lib/ruby/$(PKG_LIBVER)/*/enc/iso_8859_1.so \
+		--exclude=opt/lib/ruby/$(PKG_LIBVER)/*/enc/utf_* \
+		--exclude=opt/lib/ruby/$(PKG_LIBVER)/*/enc/euc_jp.so \
 		-cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/enc \
+		opt/lib/ruby/$(PKG_LIBVER)/*/enc \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-erb/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/erb $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/erb.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/erb $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/erb.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-fiddle/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/fiddle.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/fiddle/ \
-		usr/lib/ruby/$(PKG_LIBVER)/*/fiddle.so \
+		opt/lib/ruby/$(PKG_LIBVER)/fiddle.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/fiddle/ \
+		opt/lib/ruby/$(PKG_LIBVER)/*/fiddle.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-filelib/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/tmpdir.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/tempfile.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/pathname.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/pathname.so \
-		usr/lib/ruby/$(PKG_LIBVER)/find.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/fileutils.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/tmpdir.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/tempfile.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/pathname.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/pathname.so \
+		opt/lib/ruby/$(PKG_LIBVER)/find.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/fileutils.rb \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-gdbm/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/gdbm.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/gdbm.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-gems/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/gem $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/ubygems.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rubygems.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/gem $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/ubygems.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rubygems.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 	# Remove tests (avoids extra deps)
-	$(RM) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rubygems/test_case.rb
-	$(RM) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rubygems/package/tar_test_case.rb
-	$(RM) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rubygems/installer_test_case.rb
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rubygems $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/doc
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/cache
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/extensions
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/build_info
+	$(RM) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rubygems/test_case.rb
+	$(RM) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rubygems/package/tar_test_case.rb
+	$(RM) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rubygems/installer_test_case.rb
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rubygems $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/doc
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/cache
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/extensions
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/build_info
 endef
 
 define Package/ruby-io-console/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/io/console.so \
-		usr/lib/ruby/$(PKG_LIBVER)/io/console/ \
-		usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/io-console-*.gemspec \
+		opt/lib/ruby/$(PKG_LIBVER)/*/io/console.so \
+		opt/lib/ruby/$(PKG_LIBVER)/io/console/ \
+		opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/io-console-*.gemspec \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-irb/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/irb $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/irb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/irb.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/irb $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/irb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/irb.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-json/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/json.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/json \
-		usr/lib/ruby/$(PKG_LIBVER)/*/json \
-		usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/json-*.gemspec \
+		opt/lib/ruby/$(PKG_LIBVER)/json.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/json \
+		opt/lib/ruby/$(PKG_LIBVER)/*/json \
+		opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/json-*.gemspec \
 	) | ( cd $(1); $(TAR) -xf - )
 	rm -rf \
-		$(1)/usr/lib/ruby/$(PKG_LIBVER)/psych/json
+		$(1)/opt/lib/ruby/$(PKG_LIBVER)/psych/json
 endef
 
 define Package/ruby-logger/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/logger.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/syslog/logger.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/syslog.so \
+		opt/lib/ruby/$(PKG_LIBVER)/logger.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/syslog/logger.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/syslog.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-math/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/prime.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/mathn.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/cmath.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/mathn \
-		usr/lib/ruby/$(PKG_LIBVER)/matrix.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/matrix \
+		opt/lib/ruby/$(PKG_LIBVER)/prime.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/mathn.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/cmath.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/mathn \
+		opt/lib/ruby/$(PKG_LIBVER)/matrix.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/matrix \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-minitest/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/minitest-*.gemspec $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/minitest-*.gemspec $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/
 	# Remove tests (avoids extra deps)
-	$(RM) -rf $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/minitest-*/test
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/minitest-* $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/
+	$(RM) -rf $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/minitest-*/test
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/minitest-* $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/
 endef
 
 define Package/ruby-misc/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/English.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/abbrev.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/base64.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/delegate.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/e2mmap.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/expect.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/getoptlong.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/open3.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/ostruct.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/scanf.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/securerandom.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/set.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/shellwords.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/tsort.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/weakref.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/continuation.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/coverage.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/etc.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/fcntl.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/fiber.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/pty.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/stringio.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/strscan.so \
+		opt/lib/ruby/$(PKG_LIBVER)/English.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/abbrev.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/base64.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/delegate.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/e2mmap.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/expect.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/getoptlong.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/open3.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/ostruct.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/scanf.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/securerandom.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/set.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/shellwords.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/tsort.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/weakref.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/continuation.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/coverage.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/etc.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/fcntl.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/fiber.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/pty.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/stringio.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/strscan.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
@@ -1077,242 +1078,242 @@
 
 define Package/ruby-mkmf/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/mkmf.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/un.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/mkmf.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/un.rb \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-multithread/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/monitor.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/timeout.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/thwait.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/mutex_m.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/sync.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/thread.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/io/wait.so \
-		usr/lib/ruby/$(PKG_LIBVER)/*/io/nonblock.so \
+		opt/lib/ruby/$(PKG_LIBVER)/monitor.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/timeout.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/thwait.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/mutex_m.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/sync.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/thread.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/io/wait.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/io/nonblock.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-net/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/open-uri.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/net/* \
+		opt/lib/ruby/$(PKG_LIBVER)/open-uri.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/net/* \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-nkf/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/kconv.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/nkf.so \
+		opt/lib/ruby/$(PKG_LIBVER)/kconv.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/nkf.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-openssl/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/openssl \
-		usr/lib/ruby/$(PKG_LIBVER)/openssl.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/openssl.so \
+		opt/lib/ruby/$(PKG_LIBVER)/openssl \
+		opt/lib/ruby/$(PKG_LIBVER)/openssl.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/openssl.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-optparse/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/optparse.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/optionparser.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/optparse \
+		opt/lib/ruby/$(PKG_LIBVER)/optparse.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/optionparser.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/optparse \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-patterns/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/observer.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/singleton.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/forwardable.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/observer.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/singleton.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/forwardable.rb \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-powerassert/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/power_assert-*.gemspec $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/power_assert-*.gemspec $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/
 	# Remove tests (avoids extra deps)
-	$(RM) -rf $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/power_assert-*/test
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/power_assert-* $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/
+	$(RM) -rf $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/power_assert-*/test
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/power_assert-* $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/
 endef
 
 define Package/ruby-prettyprint/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/pp.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/prettyprint.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/pp.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/prettyprint.rb \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-pstore/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/pstore.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/pstore.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-psych/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/psych \
-		usr/lib/ruby/$(PKG_LIBVER)/psych.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/psych.so \
-		usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/psych-*.gemspec \
+		opt/lib/ruby/$(PKG_LIBVER)/psych \
+		opt/lib/ruby/$(PKG_LIBVER)/psych.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/psych.so \
+		opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/psych-*.gemspec \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-racc/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/racc \
-		usr/lib/ruby/$(PKG_LIBVER)/*/racc/*.so \
+		opt/lib/ruby/$(PKG_LIBVER)/racc \
+		opt/lib/ruby/$(PKG_LIBVER)/*/racc/*.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-rake/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/rake $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rake.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/rake $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rake.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 	# Remove tests (avoids extra deps)
-	$(RM) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rake/runtest.rb
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rake $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/rake-*.gemspec \
-		$(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/rake-* $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/
+	$(RM) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rake/runtest.rb
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rake $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/rake-*.gemspec \
+		$(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/rake-* $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/
 endef
 
 define Package/ruby-rbconfig/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/rbconfig.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/rbconfig/* \
-		usr/lib/ruby/$(PKG_LIBVER)/*/rbconfig/*.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/rbconfig.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/rbconfig/* \
+		opt/lib/ruby/$(PKG_LIBVER)/*/rbconfig/*.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-rdoc/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/rdoc $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ri $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/rdoc $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ri $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
 	# Remove tests (avoids extra deps)
-	$(RM) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rdoc/test_case.rb
-	$(RM) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rdoc/markup/formatter_test_case.rb
-	$(RM) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rdoc/markup/text_formatter_test_case.rb
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rdoc.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rdoc $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/rdoc-*.gemspec \
-		$(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/rdoc-* \
-		$(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/
+	$(RM) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rdoc/test_case.rb
+	$(RM) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rdoc/markup/formatter_test_case.rb
+	$(RM) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rdoc/markup/text_formatter_test_case.rb
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rdoc.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rdoc $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/rdoc-*.gemspec \
+		$(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/default/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/rdoc-* \
+		$(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/
 endef
 
 define Package/ruby-readline/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/readline.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/readline.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-rexml/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rexml $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rexml $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-rinda/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/rinda \
+		opt/lib/ruby/$(PKG_LIBVER)/rinda \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-ripper/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/ripper.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/ripper \
-		usr/lib/ruby/$(PKG_LIBVER)/*/ripper.so \
+		opt/lib/ruby/$(PKG_LIBVER)/ripper.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/ripper \
+		opt/lib/ruby/$(PKG_LIBVER)/*/ripper.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-rss/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rss $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/rss.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rss $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/rss.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-sdbm/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/sdbm.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/sdbm.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-shell/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/shell.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/shell \
+		opt/lib/ruby/$(PKG_LIBVER)/shell.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/shell \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-socket/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/gserver.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/ipaddr.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/resolv-replace.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/resolv.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/socket.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/*/socket.so \
+		opt/lib/ruby/$(PKG_LIBVER)/gserver.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/ipaddr.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/resolv-replace.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/resolv.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/socket.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/*/socket.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-testunit/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/test-unit-*.gemspec $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/specifications/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/test-unit-*.gemspec $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/specifications/
 	# Remove tests (avoids extra deps)
-	$(RM) -rf $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/test-unit-*/test
-	$(RM) -rf $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/test-unit-*/sample
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/test-unit-* $(1)/usr/lib/ruby/gems/$(PKG_LIBVER)/gems/
+	$(RM) -rf $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/test-unit-*/test
+	$(RM) -rf $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/test-unit-*/sample
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/test-unit-* $(1)/opt/lib/ruby/gems/$(PKG_LIBVER)/gems/
 endef
 
 define Package/ruby-unicodenormalize/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/unicode_normalize.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/unicode_normalize \
+		opt/lib/ruby/$(PKG_LIBVER)/unicode_normalize.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/unicode_normalize \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-uri/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/uri.rb \
-		usr/lib/ruby/$(PKG_LIBVER)/uri \
+		opt/lib/ruby/$(PKG_LIBVER)/uri.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/uri \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-webrick/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/webrick $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/webrick.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/webrick $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/webrick.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-xmlrpc/install
-	$(INSTALL_DIR) $(1)/usr/lib/ruby/$(PKG_LIBVER)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/xmlrpc $(1)/usr/lib/ruby/$(PKG_LIBVER)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ruby/$(PKG_LIBVER)/xmlrpc.rb $(1)/usr/lib/ruby/$(PKG_LIBVER)/
+	$(INSTALL_DIR) $(1)/opt/lib/ruby/$(PKG_LIBVER)
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/xmlrpc $(1)/opt/lib/ruby/$(PKG_LIBVER)/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ruby/$(PKG_LIBVER)/xmlrpc.rb $(1)/opt/lib/ruby/$(PKG_LIBVER)/
 endef
 
 define Package/ruby-yaml/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/yaml \
-		usr/lib/ruby/$(PKG_LIBVER)/yaml.rb \
+		opt/lib/ruby/$(PKG_LIBVER)/yaml \
+		opt/lib/ruby/$(PKG_LIBVER)/yaml.rb \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
 define Package/ruby-zlib/install
 	( cd $(PKG_INSTALL_DIR); $(TAR) -cf - \
-		usr/lib/ruby/$(PKG_LIBVER)/*/zlib.so \
+		opt/lib/ruby/$(PKG_LIBVER)/*/zlib.so \
 	) | ( cd $(1); $(TAR) -xf - )
 endef
 
diff -urx .svn ../../../downloads/openwrt_trunk-r45513/feeds/packages/lang/ruby/ruby_find_pkgsdeps ../../../openwrt_trunk/feeds/packages/lang/ruby/ruby_find_pkgsdeps
--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/lang/ruby/ruby_find_pkgsdeps	2015-04-21 10:37:38.184588033 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/ruby/ruby_find_pkgsdeps	2015-04-21 10:37:45.224587905 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby -Eutf-8
+#!/opt/bin/ruby -Eutf-8
 # encoding: utf-8
 #
 # Find dependencies between ruby packages
diff -urx .svn ../../../downloads/openwrt_trunk-r45513/feeds/packages/lang/ruby/ruby_missingfiles ../../../openwrt_trunk/feeds/packages/lang/ruby/ruby_missingfiles
--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/lang/ruby/ruby_missingfiles	2015-04-21 10:37:38.184588033 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/ruby/ruby_missingfiles	2015-04-21 10:37:45.224587905 +0300
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/opt/bin/bash
 #
 
 function list_staging_files {
