--- ../../../downloads/openwrt_trunk-r37769/feeds/oldpackages/libs/libcrypto++/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libcrypto++/Makefile
@@ -9,7 +9,7 @@
 
 PKG_NAME:=libcrypto++
 PKG_VERSION:=5.6.0
-PKG_RELEASE:=3
+PKG_RELEASE:=3a
 
 PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).orig.tar.gz
 PKG_SOURCE_URL:=http://ftp.de.debian.org/debian/pool/main/libc/libcrypto++/
@@ -42,16 +42,19 @@
 TARGET_LDFLAGS += \
 	-lpthread
 
+TARGET_CFLAGS += \
+	-O1
+
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/crypto++
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/crypto--/* $(1)/usr/include/crypto++/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcrypto++.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include/crypto++
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/crypto--/* $(1)/opt/include/crypto++/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcrypto++.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libcryptoxx/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcrypto++.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcrypto++.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libcryptoxx))
