--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/opendchub/Makefile	2012-05-02 16:17:27.894369420 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/opendchub/Makefile	2012-05-31 21:25:32.868404839 +0400
@@ -37,8 +37,10 @@
 	--disable-perl \
 
 define Package/opendchub/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/opendchub $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/opendchub $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/var $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S61opendchub $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,opendchub))
