--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/wput/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/wput/Makefile
@@ -36,9 +36,12 @@
 	--disable-g-switch \
 	--without-ssl \
 
+MAKE_FLAGS += \
+	LIBS="$(TARGET_LDFLAGS)" \
+
 define Package/wput/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/wput $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/wput $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,wput))
