--- ../../../openwrt_trunk.orig/feeds/packages/net/wput/Makefile	2012-05-02 16:17:26.542386418 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/wput/Makefile	2012-05-27 21:56:01.791324606 +0400
@@ -36,9 +36,12 @@
 	--disable-g-switch \
 	--without-ssl \
 
+MAKE_FLAGS += \
+	LIBS="$(TARGET_LDFLAGS)" \
+
 define Package/wput/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/wput $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/wput $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,wput))
