--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/vtun/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/vtun/Makefile
@@ -20,7 +20,7 @@
 define Package/vtun
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-tun +VTUN_LZO:liblzo +VTUN_LZO:zlib +VTUN_SSL:libopenssl
+  DEPENDS:=+VTUN_LZO:liblzo +VTUN_LZO:zlib +VTUN_SSL:libopenssl
   TITLE:=VPN tunneling daemon
   URL:=http://vtun.sourceforge.net
   SUBMENU:=VPN
@@ -50,29 +50,29 @@
 endef
 
 define Package/vtun/conffiles
-/etc/vtund.conf
+/opt/etc/vtund.conf
 endef
 
 CONFIGURE_ARGS += \
-	$(if $(CONFIG_VTUN_LZO),--with-lzo-lib="$(STAGING_DIR)/usr/lib/",--disable-zlib) \
-	$(if $(CONFIG_VTUN_LZO),--with-lzo-headers="$(STAGING_DIR)/usr/include/lzo",--disable-lzo) \
-	$(if $(CONFIG_VTUN_SSL),--with-ssl-headers="$(STAGING_DIR)/usr/lib/",--disable-ssl) \
+	$(if $(CONFIG_VTUN_LZO),--with-lzo-lib="$(STAGING_DIR)/opt/lib/",--disable-zlib) \
+	$(if $(CONFIG_VTUN_LZO),--with-lzo-headers="$(STAGING_DIR)/opt/include/lzo",--disable-lzo) \
+	$(if $(CONFIG_VTUN_SSL),--with-ssl-headers="$(STAGING_DIR)/opt/lib/",--disable-ssl) \
 
 CONFIGURE_VARS += \
 	LIBS="$(TARGET_LDFLAGS)" \
-	$(if $(CONFIG_VTUN_LZO),LZO_HDR_DIR="$(STAGING_DIR)/usr/include/lzo") \
-	$(if $(CONFIG_VTUN_SSL),BLOWFISH_HDR_DIR="$(STAGING_DIR)/usr/include/openssl") \
-	$(if $(CONFIG_VTUN_SSL),SSL_HDR_DIR="$(STAGING_DIR)/usr/include/openssl") \
+	$(if $(CONFIG_VTUN_LZO),LZO_HDR_DIR="$(STAGING_DIR)/opt/include/lzo") \
+	$(if $(CONFIG_VTUN_SSL),BLOWFISH_HDR_DIR="$(STAGING_DIR)/opt/include/openssl") \
+	$(if $(CONFIG_VTUN_SSL),SSL_HDR_DIR="$(STAGING_DIR)/opt/include/openssl") \
 
 define Build/Compile
 	$(call Build/Compile/Default,vtund)
 endef
 
 define Package/vtun/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/vtund.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/vtund $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/vtund.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/vtund $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,vtun))
