diff -urx .svn ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/lighttpd/files/lighttpd.conf ../../../openwrt_trunk/feeds/packages/net/lighttpd/files/lighttpd.conf
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/lighttpd/files/lighttpd.conf	2014-10-15 10:43:09.827917919 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/lighttpd/files/lighttpd.conf	2014-10-16 09:00:17.362456611 +0400
@@ -1,12 +1,12 @@
 server.modules = (
 )
 
-server.document-root        = "/www"
+server.document-root        = "/opt/share/www"
 server.upload-dirs          = ( "/tmp" )
-server.errorlog             = "/var/log/lighttpd/error.log"
-server.pid-file             = "/var/run/lighttpd.pid"
-server.username             = "http"
-server.groupname            = "www-data"
+server.errorlog             = "/opt/var/log/lighttpd/error.log"
+server.pid-file             = "/opt/var/run/lighttpd.pid"
+#server.username             = "http"
+#server.groupname            = "www-data"
 
 index-file.names            = ( "index.php", "index.html",
                                 "index.htm", "default.htm",
@@ -23,10 +23,10 @@
 #server.network-backend      = "write"
 
 ### Use IPv6 if available
-#include_shell "/usr/share/lighttpd/use-ipv6.pl"
+#include_shell "/opt/share/lighttpd/use-ipv6.pl"
 
 #dir-listing.encoding        = "utf-8"
 #server.dir-listing          = "enable"
 
-include       "/etc/lighttpd/mime.conf"
-include_shell "cat /etc/lighttpd/conf.d/*.conf"
+include       "/opt/etc/lighttpd/mime.conf"
+include_shell "cat /opt/etc/lighttpd/conf.d/*.conf"
Only in ../../../openwrt_trunk/feeds/packages/net/lighttpd/files: S80lighttpd
diff -urx .svn ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/lighttpd/Makefile ../../../openwrt_trunk/feeds/packages/net/lighttpd/Makefile
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/lighttpd/Makefile	2014-10-15 10:43:09.827917919 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/lighttpd/Makefile	2014-10-15 10:43:18.371917761 +0400
@@ -50,8 +50,8 @@
 endef
 
 CONFIGURE_ARGS+= \
-	--libdir=/usr/lib/lighttpd \
-	--sysconfdir=/etc/lighttpd \
+	--libdir=/opt/lib/lighttpd \
+	--sysconfdir=/opt/etc/lighttpd \
 	--enable-shared \
 	--enable-static \
 	--disable-rpath \
@@ -60,7 +60,7 @@
 	--without-fam \
 	--without-gdbm \
 	--without-ldap \
-	--without-lua \
+	--with-lua \
 	--without-memcache \
 	--with-pcre \
 	--without-valgrind \
@@ -71,13 +71,13 @@
 
 ifneq ($(strip $(CONFIG_LIGHTTPD_SSL)),)
   CONFIGURE_ARGS+= \
-	--with-openssl="$(STAGING_DIR)/usr"
+	--with-openssl="$(STAGING_DIR)/opt"
 else
   CONFIGURE_ARGS+= \
 	--without-openssl
 endif
 
-ifneq ($(SDK)$(CONFIG_PACKAGE_lighttpd-mod-mysql-vhost),)
+ifneq ($(SDK)$(CONFIG_PACKAGE_lighttpd-mod-mysql_vhost),)
   CONFIGURE_ARGS+= \
 	--with-mysql
 else
@@ -106,22 +106,23 @@
 endef
 
 define Package/lighttpd/conffiles
-/etc/lighttpd/lighttpd.conf
+/opt/etc/lighttpd/lighttpd.conf
 endef
 
 define Package/lighttpd/install
-	$(INSTALL_DIR) $(1)/etc/lighttpd
-	$(INSTALL_DATA) ./files/lighttpd.conf $(1)/etc/lighttpd/
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/doc/config/conf.d/mime.conf $(1)/etc/lighttpd/
-	$(INSTALL_DIR) $(1)/etc/lighttpd/conf.d
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/lighttpd.init $(1)/etc/init.d/lighttpd
-	$(INSTALL_DIR) $(1)/usr/lib/lighttpd
+	$(INSTALL_DIR) $(1)/opt/etc/lighttpd
+	$(INSTALL_DATA) ./files/lighttpd.conf $(1)/opt/etc/lighttpd/
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/doc/config/conf.d/mime.conf $(1)/opt/etc/lighttpd/
+	$(INSTALL_DIR) $(1)/opt/etc/lighttpd/conf.d
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S80lighttpd $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/lib/lighttpd
 	for m in dirlisting indexfile staticfile; do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/lighttpd/mod_$$$${m}.so $(1)/usr/lib/lighttpd/ ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/lighttpd/mod_$$$${m}.so $(1)/opt/lib/lighttpd/ ; \
 	done
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/lighttpd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/lighttpd $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/share/www $(1)/opt/var/log/lighttpd
 endef
 
 define BuildPlugin
@@ -135,13 +136,13 @@
   endef
 
   define Package/lighttpd-mod-$(1)/install
-	$(INSTALL_DIR) $$(1)/usr/lib/lighttpd
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lighttpd/mod_$(1).so $$(1)/usr/lib/lighttpd
-	$(INSTALL_DIR) $$(1)/etc/lighttpd/conf.d
+	$(INSTALL_DIR) $$(1)/opt/lib/lighttpd
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lighttpd/mod_$(1).so $$(1)/opt/lib/lighttpd
+	$(INSTALL_DIR) $$(1)/opt/etc/lighttpd/conf.d
 	if [ -f $(PKG_BUILD_DIR)/doc/config/conf.d/$(1).conf ]; then \
-		$(CP) $(PKG_BUILD_DIR)/doc/config/conf.d/$(1).conf $$(1)/etc/lighttpd/conf.d/$(4)-$(1).conf ; \
+		$(CP) $(PKG_BUILD_DIR)/doc/config/conf.d/$(1).conf $$(1)/opt/etc/lighttpd/conf.d/$(4)-$(1).conf ; \
 	else \
-		echo 'server.modules += ( "mod_$(1)" )' > $$(1)/etc/lighttpd/conf.d/$(4)-$(1).conf ; \
+		echo 'server.modules += ( "mod_$(1)" )' > $$(1)/opt/etc/lighttpd/conf.d/$(4)-$(1).conf ; \
 	fi
   endef
 
@@ -154,7 +155,7 @@
 $(eval $(call BuildPlugin,alias,Directory alias,,10))
 $(eval $(call BuildPlugin,auth,Authentication,,05))
 $(eval $(call BuildPlugin,cgi,CGI,,10))
-$(eval $(call BuildPlugin,cml,Cache Meta Language,,10))
+$(eval $(call BuildPlugin,cml,Cache Meta Language,+liblua,10))
 $(eval $(call BuildPlugin,compress,Compress output,+PACKAGE_lighttpd-mod-compress:zlib,10))
 $(eval $(call BuildPlugin,evasive,Evasive,,10))
 $(eval $(call BuildPlugin,evhost,Exnhanced Virtual-Hosting,,10))
@@ -162,7 +163,7 @@
 $(eval $(call BuildPlugin,extforward,Extract client,,10))
 $(eval $(call BuildPlugin,fastcgi,FastCGI,,10))
 $(eval $(call BuildPlugin,flv_streaming,FLV streaming,,10))
-$(eval $(call BuildPlugin,magnet,Magnet,,10))
+$(eval $(call BuildPlugin,magnet,Magnet,+liblua,10))
 $(eval $(call BuildPlugin,mysql_vhost,Mysql virtual hosting,+PACKAGE_lighttpd-mod-mysql_vhost:libmysqlclient,10))
 $(eval $(call BuildPlugin,proxy,Proxy,,10))
 $(eval $(call BuildPlugin,redirect,URL redirection,+PACKAGE_lighttpd-mod-redirect:libpcre,10))
Only in ../../../openwrt_trunk/feeds/packages/net/lighttpd: patches
