--- ../../../openwrt_trunk/feeds/packages/lang/erlang/Makefile.orig
+++ ../../../openwrt_trunk/feeds/packages/lang/erlang/Makefile
@@ -268,50 +268,50 @@
 endef
 
 define Package/erlang/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	for f in epmd erl erlc escript run_erl; do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/bin/$$$$f $(1)/usr/bin/ ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/bin/$$$$f $(1)/opt/bin/ ; \
 	done
-	$(INSTALL_DIR) $(1)/usr/lib/erlang/bin
+	$(INSTALL_DIR) $(1)/opt/lib/erlang/bin
 	for f in erl erlc escript run_erl start start.boot start.script start_clean.boot start_erl start_sasl.boot to_erl; do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/bin/$$$$f $(1)/usr/lib/erlang/bin/ ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/bin/$$$$f $(1)/opt/lib/erlang/bin/ ; \
 	done
-	$(INSTALL_DIR) $(1)/usr/lib/erlang/lib
+	$(INSTALL_DIR) $(1)/opt/lib/erlang/lib
 	for m in erts kernel sasl stdlib; do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/lib/$$$$m-* $(1)/usr/lib/erlang/lib/ ; \
-		rm -rf $(1)/usr/lib/erlang/lib/$$$$m-*/examples ; \
-		rm -rf $(1)/usr/lib/erlang/lib/$$$$m-*/src ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/lib/$$$$m-* $(1)/opt/lib/erlang/lib/ ; \
+		rm -rf $(1)/opt/lib/erlang/lib/$$$$m-*/examples ; \
+		rm -rf $(1)/opt/lib/erlang/lib/$$$$m-*/src ; \
 	done
-	$(INSTALL_DIR) $(1)/usr/lib/erlang
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/erts-* $(1)/usr/lib/erlang/
-	rm -rf $(1)/usr/lib/erlang/erts-*/{doc,include,lib,man,src}
-	rm -rf $(1)/usr/lib/erlang/erts-*/bin/*.src
-	$(INSTALL_DIR) $(1)/usr/lib/erlang/releases
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/releases/* $(1)/usr/lib/erlang/releases/
-	$(SED) 's,%ERL_ROOT%,/usr/lib/erlang,g' \
-		$(1)/usr/lib/erlang/releases/RELEASES.src
-	mv -f $(1)/usr/lib/erlang/releases/RELEASES.src \
-		$(1)/usr/lib/erlang/releases/RELEASES
+	$(INSTALL_DIR) $(1)/opt/lib/erlang
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/erts-* $(1)/opt/lib/erlang/
+	rm -rf $(1)/opt/lib/erlang/erts-*/{doc,include,lib,man,src}
+	rm -rf $(1)/opt/lib/erlang/erts-*/bin/*.src
+	$(INSTALL_DIR) $(1)/opt/lib/erlang/releases
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/releases/* $(1)/opt/lib/erlang/releases/
+	$(SED) 's,%ERL_ROOT%,/opt/lib/erlang,g' \
+		$(1)/opt/lib/erlang/releases/RELEASES.src
+	mv -f $(1)/opt/lib/erlang/releases/RELEASES.src \
+		$(1)/opt/lib/erlang/releases/RELEASES
 	for f in bin/erl bin/start erts-*/bin/erl erts-*/bin/start; do \
-		$(SED) 's,^\(ROOTDIR\)=.*,\1=/usr/lib/erlang,g' \
-			$(1)/usr/lib/erlang/$$$$f ; \
+		$(SED) 's,^\(ROOTDIR\)=.*,\1=/opt/lib/erlang,g' \
+			$(1)/opt/lib/erlang/$$$$f ; \
 	done
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/erl_interface/obj/*/*.a $(1)/usr/lib
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/lib/erl_interface/include/*.h $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/erl_interface/obj/*/*.a $(1)/opt/lib
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_BUILD_DIR)/lib/erl_interface/include/*.h $(1)/opt/include
 endef
 
 define BuildModule
 
   define Package/erlang-$(1)/install
-	$(INSTALL_DIR) $$(1)/usr/lib/erlang/lib
+	$(INSTALL_DIR) $$(1)/opt/lib/erlang/lib
 	for m in $(2); do \
-		$(CP) $(PKG_INSTALL_DIR)/usr/lib/erlang/lib/$$$$$$$$m-* $$(1)/usr/lib/erlang/lib/ ; \
-		rm -rf $$(1)/usr/lib/erlang/lib/$$$$$$$$m-*/{examples,priv/obj,src} ; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/lib/erlang/lib/$$$$$$$$m-* $$(1)/opt/lib/erlang/lib/ ; \
+		rm -rf $$(1)/opt/lib/erlang/lib/$$$$$$$$m-*/{examples,priv/obj,src} ; \
 	done
   endef
 
