--- ../../../openwrt_trunk.orig/feeds/packages/libs/avahi/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/avahi/Makefile
@@ -112,9 +112,9 @@
 endef
 
 define Package/avahi-daemon/conffiles
-/etc/avahi/avahi-daemon.conf
-/etc/avahi/services/http.service
-/etc/avahi/services/ssh.service
+/opt/etc/avahi/avahi-daemon.conf
+/opt/etc/avahi/services/http.service
+/opt/etc/avahi/services/ssh.service
 endef
 
 define Package/avahi-dnsconfd
@@ -243,17 +243,17 @@
 	ac_cv_header_sys_capability_h=no \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libavahi-* $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/* $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libavahi-* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/* $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libavahi-dbus-support/install
-	$(INSTALL_DIR) $(1)/etc/dbus-1/system.d
-	$(CP) $(PKG_INSTALL_DIR)/etc/dbus-1/system.d/* $(1)/etc/dbus-1/system.d
+	$(INSTALL_DIR) $(1)/opt/etc/dbus-1/system.d
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/dbus-1/system.d/* $(1)/opt/etc/dbus-1/system.d
 endef
 
 define Package/libavahi/install
@@ -264,44 +264,44 @@
 	# "Other" means this: if the current build is "dbus", the other is "nodebus",
 	# and if the current build is "nodbus", the other is "dbus".
 	$(RM) -f $(PKG_ALT_DIR)/.built
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libavahi-{common,core}.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libavahi-{common,core}.so.* $(1)/opt/lib/
 endef
 
 define Package/libavahi-client/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libavahi-client.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libavahi-client.so.* $(1)/opt/lib/
 endef
 
 define Package/avahi-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 define Package/avahi-autoipd/install
-	$(INSTALL_DIR) $(1)/etc/avahi
-	$(CP) $(PKG_INSTALL_DIR)/etc/avahi/avahi-autoipd.action $(1)/etc/avahi/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/avahi-autoipd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/avahi/avahi-autoipd.action $(1)/opt/etc/avahi/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/avahi-autoipd $(1)/opt/sbin/
 endef
 
 define Package/avahi-daemon/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/avahi-daemon $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/avahi
-	$(INSTALL_DATA) ./files/avahi-daemon.conf $(1)/etc/avahi/
-	$(INSTALL_DIR) $(1)/etc/avahi/services
-	$(INSTALL_DATA) ./files/service-http $(1)/etc/avahi/services/http.service
-	$(INSTALL_DATA) ./files/service-ssh $(1)/etc/avahi/services/ssh.service
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/avahi-daemon.init $(1)/etc/init.d/avahi-daemon
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/avahi-daemon $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi
+	$(INSTALL_DATA) ./files/avahi-daemon.conf $(1)/opt/etc/avahi/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi/services
+	$(INSTALL_DATA) ./files/service-http $(1)/opt/etc/avahi/services/http.service
+	$(INSTALL_DATA) ./files/service-ssh $(1)/opt/etc/avahi/services/ssh.service
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S42avahi-daemon $(1)/opt/etc/init.d/
 endef
 
 define Package/avahi-dnsconfd/install
-	$(INSTALL_DIR) $(1)/etc/avahi
-	$(CP) $(PKG_INSTALL_DIR)/etc/avahi/avahi-dnsconfd.action $(1)/etc/avahi/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/avahi-dnsconfd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/avahi/avahi-dnsconfd.action $(1)/opt/etc/avahi/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/avahi-dnsconfd $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,libavahi-client))
