--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/libs/avahi/Makefile	2015-04-20 10:09:28.902135360 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/avahi/Makefile	2015-04-20 10:09:43.962135130 +0300
@@ -113,7 +113,7 @@
 Package/avahi-nodbus-daemon/description=$(Package/avahi-daemon/description)
 
 define Package/avahi-daemon/conffiles
-/etc/avahi/avahi-daemon.conf
+/opt/etc/avahi/avahi-daemon.conf
 endef
 
 Package/avahi-dbus-daemon/conffiles=$(Package/avahi-daemon/conffiles)
@@ -133,7 +133,7 @@
 endef
 
 define Package/avahi-daemon-service-http/conffiles
-/etc/avahi/services/http.service
+/opt/etc/avahi/services/http.service
 endef
 
 define Package/avahi-daemon-service-ssh
@@ -150,7 +150,7 @@
 endef
 
 define Package/avahi-daemon-service-ssh/conffiles
-/etc/avahi/services/ssh.service
+/opt/etc/avahi/services/ssh.service
 endef
 
 define Package/avahi-dnsconfd
@@ -313,82 +313,82 @@
 	ac_cv_header_sys_capability_h=no \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libavahi-* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libavahi-* $(1)/opt/lib/
 ifeq ($(CONFIG_PACKAGE_libavahi-compat-libdnssd)-$(BUILD_VARIANT),y-dbus)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libdns_sd* $(1)/usr/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libdns_sd* $(1)/opt/lib/
 endif
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/* $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/* $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libavahi/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libavahi-{common,core}.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libavahi-{common,core}.so.* $(1)/opt/lib/
 endef
 
 define Package/libavahi-dbus-support/install
 	$(call Package/libavahi/install,$(1))
-	$(INSTALL_DIR) $(1)/etc/dbus-1/system.d
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/etc/dbus-1/system.d/* $(1)/etc/dbus-1/system.d
+	$(INSTALL_DIR) $(1)/opt/etc/dbus-1/system.d
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/etc/dbus-1/system.d/* $(1)/opt/etc/dbus-1/system.d
 endef
 
 Package/libavahi-nodbus-support/install=$(Package/libavahi/install)
 
 define Package/libavahi-client/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libavahi-client.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libavahi-client.so.* $(1)/opt/lib/
 endef
 
 define Package/libavahi-compat-libdnssd/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libdns_sd.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libdns_sd.so.* $(1)/opt/lib/
 endef
 
 define Package/avahi-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 define Package/avahi-autoipd/install
-	$(INSTALL_DIR) $(1)/etc/avahi
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/etc/avahi/avahi-autoipd.action $(1)/etc/avahi/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/avahi-autoipd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/etc/avahi/avahi-autoipd.action $(1)/opt/etc/avahi/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/avahi-autoipd $(1)/opt/sbin/
 endef
 
 define Package/avahi-daemon/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/avahi-daemon $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/avahi
-	$(INSTALL_DATA) ./files/avahi-daemon.conf $(1)/etc/avahi/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/avahi-daemon $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi
+	$(INSTALL_DATA) ./files/avahi-daemon.conf $(1)/opt/etc/avahi/
 	# install empty service directory so that user knows where
 	# to place custom service files
-	$(INSTALL_DIR) $(1)/etc/avahi/services
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/avahi-daemon.init $(1)/etc/init.d/avahi-daemon
+	$(INSTALL_DIR) $(1)/opt/etc/avahi/services
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S42avahi-daemon $(1)/opt/etc/init.d/
 endef
 
 Package/avahi-dbus-daemon/install=$(Package/avahi-daemon/install)
 Package/avahi-nodbus-daemon/install=$(Package/avahi-daemon/install)
 
 define Package/avahi-daemon-service-http/install
-	$(INSTALL_DIR) $(1)/etc/avahi/services
-	$(INSTALL_DATA) ./files/service-http $(1)/etc/avahi/services/http.service
+	$(INSTALL_DIR) $(1)/opt/etc/avahi/services
+	$(INSTALL_DATA) ./files/service-http $(1)/opt/etc/avahi/services/http.service
 endef
 
 define Package/avahi-daemon-service-ssh/install
-	$(INSTALL_DIR) $(1)/etc/avahi/services
-	$(INSTALL_DATA) ./files/service-ssh $(1)/etc/avahi/services/ssh.service
+	$(INSTALL_DIR) $(1)/opt/etc/avahi/services
+	$(INSTALL_DATA) ./files/service-ssh $(1)/opt/etc/avahi/services/ssh.service
 endef
 
 define Package/avahi-dnsconfd/install
-	$(INSTALL_DIR) $(1)/etc/avahi
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/etc/avahi/avahi-dnsconfd.action $(1)/etc/avahi/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/avahi-dnsconfd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/etc/avahi/avahi-dnsconfd.action $(1)/opt/etc/avahi/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/avahi-dnsconfd $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,libavahi-client))
