--- a/src/ffmpeg_decoder_plugin.c.orig	2014-11-28 22:40:11.221975941 +0300
+++ b/src/ffmpeg_decoder_plugin.c	2014-11-28 22:41:59.578879133 +0300
@@ -47,6 +47,8 @@
 #undef G_LOG_DOMAIN
 #define G_LOG_DOMAIN "ffmpeg"
 
+#define MAX_AUDIO_FRAME_SIZE 192000
+
 static GLogLevelFlags
 level_ffmpeg_to_glib(int level)
 {
@@ -299,11 +301,11 @@
 #endif
 
 #if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(53,25,0)
-	uint8_t aligned_buffer[(AVCODEC_MAX_AUDIO_FRAME_SIZE * 3) / 2 + 16];
+	uint8_t aligned_buffer[(MAX_AUDIO_FRAME_SIZE * 3) / 2 + 16];
 	const size_t buffer_size = sizeof(aligned_buffer);
 #else
 	/* libavcodec < 0.8 needs an aligned buffer */
-	uint8_t audio_buf[(AVCODEC_MAX_AUDIO_FRAME_SIZE * 3) / 2 + 16];
+	uint8_t audio_buf[(MAX_AUDIO_FRAME_SIZE * 3) / 2 + 16];
 	size_t buffer_size = sizeof(audio_buf);
 	int16_t *aligned_buffer = align16(audio_buf, &buffer_size);
 #endif
