--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/sound/mpd/Makefile	2014-04-16 20:19:00.562912091 +0400
+++ ../../../openwrt_trunk/feeds/packages/sound/mpd/Makefile	2014-04-19 19:00:32.370281437 +0400
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mpd
-PKG_VERSION:=0.16.8
-PKG_RELEASE:=2
+PKG_VERSION:=0.17.6
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=http://www.musicpd.org/download/mpd/0.16/
-PKG_MD5SUM:=b412a9d0b3457bea1bb87904a8f80c04
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=http://www.musicpd.org/download/mpd/0.17/
+PKG_MD5SUM:=d0da6a6a1d9cf1e8710b6082f6ef7849
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
 PKG_BUILD_PARALLEL:=1
@@ -49,7 +49,7 @@
   DEPENDS+= \
 	+AUDIO_SUPPORT:alsa-lib \
 	+libaudiofile +BUILD_PATENTED:libfaad2 +libffmpeg +libid3tag \
-	+libmms +libogg +libshout +libsndfile +libvorbis
+	+libmms +libogg +libshout +libsndfile +libvorbis +libsoup
   PROVIDES:=mpd
   VARIANT:=full
 endef
@@ -61,8 +61,8 @@
 endef
 
 define Package/mpd-full/conffiles
-/etc/avahi/services/mpd.service
-/etc/mpd.conf
+/opt/etc/avahi/services/mpd.service
+/opt/etc/mpd.conf
 endef
 
 
@@ -81,7 +81,7 @@
 endef
 
 define Package/mpd-mini/conffiles
-/etc/mpd.conf
+/opt/etc/mpd.conf
 endef
 
 
@@ -131,13 +131,13 @@
 	--enable-un \
 
 CONFIGURE_VARS += \
-	FLAC_CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/usr/include/FLAC" \
+	FLAC_CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/opt/include/FLAC" \
 	FLAC_LIBS="$(TARGET_LDFLAGS) -lFLAC" \
 	$(if $(CONFIG_BUILD_PATENTED),MAD_CFLAGS="$(TARGET_CFLAGS)") \
 	$(if $(CONFIG_BUILD_PATENTED),MAD_LIBS="$(TARGET_LDFLAGS) -lmad") \
 
 TARGET_CFLAGS += -std=gnu99
-TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/usr/lib
+TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/opt/lib
 
 # use gcc instead of g++ to avoid unnecessary linking against libstdc++
 TARGET_CXX:=$(TARGET_CC)
@@ -160,7 +160,7 @@
 	--enable-sndfile \
 	--enable-vorbis \
 	--enable-vorbis-encoder \
-	--with-faad="$(STAGING_DIR)/usr" \
+	--with-faad="$(STAGING_DIR)/opt" \
 	--with-tremor=no \
 
 endif
@@ -185,22 +185,23 @@
 	--disable-vorbis \
 	--disable-vorbis-encoder \
 	--with-tremor=yes \
+	--disable-soup
 
 endif
 
 define Package/mpd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/mpd $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_BUILD_DIR)/doc/mpdconf.example $(1)/etc/mpd.conf
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/mpd.init $(1)/etc/init.d/mpd
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/mpd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_BUILD_DIR)/doc/mpdconf.example $(1)/opt/etc/mpd.conf
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S89mpd $(1)/opt/etc/init.d
 endef
 
 define Package/mpd-full/install
 $(call Package/mpd/install,$1)
-	$(INSTALL_DIR) $(1)/etc/avahi/services
-	$(INSTALL_DATA) ./files/mpd.service $(1)/etc/avahi/services/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi/services
+	$(INSTALL_DATA) ./files/mpd.service $(1)/opt/etc/avahi/services/
 endef
 
 define Package/mpd-mini/install
