--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/sound/mpd/Makefile
+++ ../../../openwrt_trunk/feeds/packages/sound/mpd/Makefile
@@ -63,8 +63,8 @@
 endef
 
 define Package/mpd-full/conffiles
-/etc/avahi/services/mpd.service
-/etc/mpd.conf
+/opt/etc/avahi/services/mpd.service
+/opt/etc/mpd.conf
 endef
 
 define Package/mpd-mini
@@ -82,7 +82,7 @@
 endef
 
 define Package/mpd-mini/conffiles
-/etc/mpd.conf
+/opt/etc/mpd.conf
 endef
 
 CONFIGURE_ARGS += \
@@ -92,7 +92,7 @@
 	--disable-gprof \
 	--disable-test \
 	--disable-werror \
-	\
+	--disable-signalfd \
 	--disable-ao \
 	--disable-bzip2 \
 	--enable-cue \
@@ -110,7 +110,6 @@
 	--disable-modplug \
 	--disable-mpc \
 	--disable-mpg123 \
-	--disable-mvp \
 	--disable-openal \
 	--disable-opus \
 	--disable-pulse \
@@ -124,7 +123,7 @@
 	--disable-wildmidi \
 	--disable-zzip \
 	--with-zeroconf=no \
-	\
+	--enable-epoll \
 	--enable-curl \
 	--enable-flac \
 	--enable-httpd-output \
@@ -134,12 +133,12 @@
 	--enable-un \
 
 CONFIGURE_VARS += \
-	FLAC_CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/usr/include/FLAC" \
+	FLAC_CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/opt/include/FLAC" \
 	FLAC_LIBS="$(TARGET_LDFLAGS) -lFLAC" \
 	$(if $(CONFIG_BUILD_PATENTED),MAD_CFLAGS="$(TARGET_CFLAGS)") \
 	$(if $(CONFIG_BUILD_PATENTED),MAD_LIBS="$(TARGET_LDFLAGS) -lmad") \
 
-TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/usr/lib $(if $(ICONV_FULL),-liconv)
+TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/opt/lib $(if $(ICONV_FULL),-liconv)
 
 ifeq ($(BUILD_VARIANT),full)
 
@@ -159,7 +158,7 @@
 	--enable-sndfile \
 	--enable-vorbis \
 	--disable-vorbis-encoder \
-	--with-faad="$(STAGING_DIR)/usr" \
+	--with-faad="$(STAGING_DIR)/opt" \
 	--with-tremor=yes \
 
 endif
@@ -174,9 +173,7 @@
 	--disable-fifo \
 	--disable-ffmpeg \
 	--disable-id3 \
-	--disable-lastfm \
 	--disable-mms \
-	--disable-oggflac \
 	--disable-pipe-output \
 	--disable-recorder-output \
 	--disable-shout \
@@ -188,18 +185,18 @@
 endif
 
 define Package/mpd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/mpd $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_BUILD_DIR)/doc/mpdconf.example $(1)/etc/mpd.conf
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/mpd.init $(1)/etc/init.d/mpd
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/mpd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_BUILD_DIR)/doc/mpdconf.example $(1)/opt/etc/mpd.conf
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S89mpd $(1)/opt/etc/init.d
 endef
 
 define Package/mpd-full/install
 $(call Package/mpd/install,$1)
-	$(INSTALL_DIR) $(1)/etc/avahi/services
-	$(INSTALL_DATA) ./files/mpd.service $(1)/etc/avahi/services/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi/services
+	$(INSTALL_DATA) ./files/mpd.service $(1)/opt/etc/avahi/services/
 endef
 
 define Package/mpd-mini/install
