--- ../../../downloads/openwrt_trunk-r44486/feeds/packages/sound/mpd/Makefile	2015-02-18 12:16:57.624306332 +0300
+++ ../../../openwrt_trunk/feeds/packages/sound/mpd/Makefile	2015-02-19 15:45:48.966502803 +0300
@@ -36,7 +36,7 @@
   TITLE:=Music Player Daemon
   URL:=http://www.musicpd.org/
   DEPENDS:= +glib2 +libcurl +libpthread +libmpdclient $(ICONV_DEPENDS) +libstdcpp \
-	    +libflac +BUILD_PATENTED:libmad +libvorbisidec +AUDIO_SUPPORT:alsa-lib
+	    +libflac +BUILD_PATENTED:libmad +libvorbisidec +alsa-lib
 endef
 
 define Package/mpd/Default/description
@@ -63,7 +63,7 @@
 endef
 
 define Package/mpd-full/conffiles
-/etc/mpd.conf
+/opt/etc/mpd.conf
 endef
 
 define Package/mpd-mini
@@ -81,7 +81,7 @@
 endef
 
 define Package/mpd-mini/conffiles
-/etc/mpd.conf
+/opt/etc/mpd.conf
 endef
 
 define Package/mpd-avahi-service
@@ -98,7 +98,7 @@
 endef
 
 define Package/mpd-avahi-service/conffiles
-/etc/avahi/services/mpd.service
+/opt/etc/avahi/services/mpd.service
 endef
 
 CONFIGURE_ARGS += \
@@ -108,7 +108,7 @@
 	--disable-gprof \
 	--disable-test \
 	--disable-werror \
-	\
+	--disable-signalfd \
 	--disable-ao \
 	--disable-bzip2 \
 	--enable-cue \
@@ -126,7 +126,6 @@
 	--disable-modplug \
 	--disable-mpc \
 	--disable-mpg123 \
-	--disable-mvp \
 	--disable-openal \
 	--disable-opus \
 	--disable-pulse \
@@ -140,28 +139,28 @@
 	--disable-wildmidi \
 	--disable-zzip \
 	--with-zeroconf=no \
-	\
+	--enable-epoll \
 	--enable-curl \
 	--enable-flac \
 	--enable-httpd-output \
 	$(call autoconf_bool,CONFIG_BUILD_PATENTED,mad) \
-	$(call autoconf_bool,CONFIG_AUDIO_SUPPORT,alsa) \
+	--enable-alsa \
 	--enable-tcp \
 	--enable-un \
 
 CONFIGURE_VARS += \
-	FLAC_CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/usr/include/FLAC" \
+	FLAC_CFLAGS="$(TARGET_CFLAGS) -I$(STAGING_DIR)/opt/include/FLAC" \
 	FLAC_LIBS="$(TARGET_LDFLAGS) -lFLAC" \
 	$(if $(CONFIG_BUILD_PATENTED),MAD_CFLAGS="$(TARGET_CFLAGS)") \
 	$(if $(CONFIG_BUILD_PATENTED),MAD_LIBS="$(TARGET_LDFLAGS) -lmad") \
 
-TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/usr/lib $(if $(ICONV_FULL),-liconv)
+TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/opt/lib $(if $(ICONV_FULL),-liconv)
 
 ifeq ($(BUILD_VARIANT),full)
 
   CONFIGURE_ARGS += \
 	$(call autoconf_bool,CONFIG_BUILD_PATENTED,aac) \
-	$(call autoconf_bool,CONFIG_AUDIO_SUPPORT,alsa) \
+	--enable-alsa \
 	--enable-audiofile \
 	--enable-fifo \
 	--enable-ffmpeg \
@@ -175,7 +174,7 @@
 	--enable-sndfile \
 	--enable-vorbis \
 	--disable-vorbis-encoder \
-	--with-faad="$(STAGING_DIR)/usr" \
+	--with-faad="$(STAGING_DIR)/opt" \
 	--with-tremor=yes \
 
 endif
@@ -190,9 +189,7 @@
 	--disable-fifo \
 	--disable-ffmpeg \
 	--disable-id3 \
-	--disable-lastfm \
 	--disable-mms \
-	--disable-oggflac \
 	--disable-pipe-output \
 	--disable-recorder-output \
 	--disable-shout \
@@ -204,12 +201,12 @@
 endif
 
 define Package/mpd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/mpd $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/doc/mpdconf.example $(1)/etc/mpd.conf
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/mpd.init $(1)/etc/init.d/mpd
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/mpd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/doc/mpdconf.example $(1)/opt/etc/mpd.conf
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S89mpd $(1)/opt/etc/init.d/
 endef
 
 define Package/mpd-full/install
@@ -221,8 +218,8 @@
 endef
 
 define Package/mpd-avahi-service/install
-	$(INSTALL_DIR) $(1)/etc/avahi/services
-	$(INSTALL_DATA) ./files/mpd.service $(1)/etc/avahi/services/
+	$(INSTALL_DIR) $(1)/opt/etc/avahi/services
+	$(INSTALL_DATA) ./files/mpd.service $(1)/opt/etc/avahi/services/
 endef
 
 $(eval $(call BuildPackage,mpd-full))
