--- ../../../openwrt_trunk.orig/feeds/packages/net/git/Makefile	2012-07-10 18:59:23.073221321 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/git/Makefile	2012-07-11 12:02:07.068478600 +0400
@@ -57,13 +57,13 @@
 		configure
 
 	( cd $(PKG_BUILD_DIR); \
-		./configure --prefix=/usr \
+		./configure --prefix=/opt \
 	);
 endef
 
 define Package/git/install
 	$(INSTALL_DIR) $(1)
-	$(RM) $(PKG_INSTALL_DIR)/usr/bin/git-cvsserver
+	$(RM) $(PKG_INSTALL_DIR)/opt/bin/git-cvsserver
 	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
 endef
 
