--- ../../../openwrt_trunk.orig/feeds/packages/net/git/Makefile	2012-01-20 19:21:04.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/git/Makefile	2012-01-21 19:50:22.351770679 +0400
@@ -57,14 +57,14 @@
 		configure
 
 	( cd $(PKG_BUILD_DIR); \
-		./configure --prefix=/usr \
+		./configure --prefix=/opt \
 	);
 endef
 
 define Package/git/install
 	$(INSTALL_DIR) $(1)
 	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
-	ln -s /usr/bin/git $(1)/usr/libexec/git-core/git
+	ln -s /opt/bin/git $(1)/opt/libexec/git-core/git
 endef
 
 $(eval $(call BuildPackage,git))
