--- ../../../downloads/openwrt_trunk-r45513/package/libs/librpc/Makefile	2015-04-20 10:09:29.142135354 +0300
+++ ../../../openwrt_trunk/package/libs/librpc/Makefile	2015-04-20 10:09:47.230135026 +0300
@@ -9,7 +9,6 @@
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
 PKG_SOURCE_VERSION:=308e9964bfb623773dc0dcc99ef9d18d1551d6ae
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
-CMAKE_INSTALL:=1
 PKG_USE_MIPS16:=0
 
 PKG_LICENSE:=LGPL-2.1
@@ -26,9 +25,25 @@
   TITLE:=uClibc RPC library
 endef
 
+define Build/Prepare
+	true
+endef
+
+define Build/Configure
+	true
+endef
+
+define Build/Compile
+	true
+endef
+
+define Build/Install
+	true
+endef
+
 define Package/librpc/install
-	$(INSTALL_DIR) $(1)/lib/
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/lib/librpc.so $(1)/lib/
+	$(INSTALL_DIR) $(1)/opt/tmp/
+	touch $(1)/opt/tmp/.librpc_placeholder
 endef
 
 $(eval $(call BuildPackage,librpc))
