--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/utils/xz/Makefile	2015-04-20 10:09:28.950135359 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/xz/Makefile	2015-04-20 14:41:28.733838763 +0300
@@ -61,8 +61,8 @@
   endef
 
   define Package/$(1)/install
-	$(INSTALL_DIR) $$(1)/usr/bin
-	$(CP) $(foreach f,$(1) $(3),$(PKG_INSTALL_DIR)/usr/bin/$(f)) $$(1)/usr/bin/
+	$(INSTALL_DIR) $$(1)/opt/bin
+	$(CP) $(foreach f,$(1) $(3),$(PKG_INSTALL_DIR)/opt/bin/$(f)) $$(1)/opt/bin/
   endef
 
   $$(eval $$(call BuildPackage,$(1)))
@@ -80,23 +80,23 @@
 	-std=c99 \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/include
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/lzma{,.h} \
-		$(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
+		$(PKG_INSTALL_DIR)/opt/include/lzma{,.h} \
+		$(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/liblzma.{a,so*} \
-		$(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+		$(PKG_INSTALL_DIR)/opt/lib/liblzma.{a,so*} \
+		$(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/liblzma.pc \
-		$(1)/usr/lib/pkgconfig/
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/liblzma.pc \
+		$(1)/opt/lib/pkgconfig/
 endef
 
 define Package/liblzma/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblzma.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/liblzma.so.* $(1)/opt/lib/
 endef
 
 
