--- a/lib/strutil/strutil.c
+++ b/lib/strutil/strutil.c
@@ -324,7 +324,7 @@ str_translate_char (GIConv conv, const c
 const char *
 str_detect_termencoding (void)
 {
-    return (nl_langinfo (CODESET));
+    return "UTF-8";
 }
 
 static int
--- a/lib/tty/tty-slang.c
+++ b/lib/tty/tty-slang.c
@@ -274,7 +274,7 @@ tty_init (gboolean mouse_enable, gboolea
 {
     SLtt_Ignore_Beep = 1;
 
-    SLutf8_enable (-1);         /* has to be called first before any of the other functions. */
+    SLutf8_enable (1);         /* has to be called first before any of the other functions. */
     SLtt_get_terminfo ();
     /*
      * If the terminal in not in terminfo but begins with a well-known
