--- ../../../openwrt_trunk/feeds/oldpackages/net/usbip/Makefile.orig	2012-08-06 14:54:12.000000000 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/usbip/Makefile	2012-08-06 14:56:30.218680372 +0400
@@ -25,7 +25,6 @@
 define Package/usbip/Common
   TITLE:=USB-over-IP
   URL:=http://usbip.sourceforge.net/
-  DEPENDS:=@USB_SUPPORT
   MAINTAINER:=Nuno Goncalves <nunojpg@gmail.com>
 endef
 
@@ -38,53 +37,57 @@
 define Package/usbip
 $(call Package/usbip/Default)
   TITLE+= (common)
-  DEPENDS+= +libsysfs +libwrap +kmod-usbip
+  DEPENDS+= +libsysfs +libwrap
 endef
 
 define Package/usbip-client
 $(call Package/usbip/Default)
   TITLE+= (client)
-  DEPENDS+= usbip +glib2 +kmod-usbip-client
+  DEPENDS+= usbip +glib2
 endef
 
 define Package/usbip-server
 $(call Package/usbip/Default)
   TITLE+= (server)
-  DEPENDS+= usbip +glib2 +kmod-usbip-server
+  DEPENDS+= usbip +glib2
 endef
 
 CONFIGURE_PATH:=./src
 MAKE_PATH:=./src
 LIBTOOL_PATHS:=./src
 
-CFLAGS+="$(TARGET_CFLAGS) -I$(STAGING_DIR)/usr/include"
+CFLAGS+="$(TARGET_CFLAGS) -I$(STAGING_DIR)/opt/include"
 
 define Package/usbip/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libusbip.so.* \
-		$(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/share/usbip
+		$(PKG_INSTALL_DIR)/opt/lib/libusbip.so.* \
+		$(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/share/usbip
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/share/usbip/usb.ids \
-		$(1)/usr/share/usbip/
+		$(PKG_INSTALL_DIR)/opt/share/usbip/usb.ids \
+		$(1)/opt/share/usbip/
 endef
 
 define Package/usbip-client/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/usbip \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/usbip \
+		$(1)/opt/bin/
 endef
 
 define Package/usbip-server/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/usbipd \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/usbipd \
+		$(1)/opt/bin/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/bind_driver \
-		$(1)/usr/bin/usbip_bind_driver
+		$(PKG_INSTALL_DIR)/opt/bin/bind_driver \
+		$(1)/opt/bin/usbip_bind_driver
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(CP) \
+		./files/S74usbipd \
+		$(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,usbip))
