--- a/tools/i2cdetect.c
+++ b/tools/i2cdetect.c
@@ -37,6 +37,10 @@
 #define MODE_READ	2
 #define MODE_FUNC	3
 
+#ifndef I2C_FUNC_SMBUS_PEC
+#define I2C_FUNC_SMBUS_PEC I2C_FUNC_SMBUS_HWPEC_CALC
+#endif
+
 static void help(void)
 {
 	fprintf(stderr,
--- a/tools/i2cget.c
+++ b/tools/i2cget.c
@@ -36,6 +36,10 @@
 #include "util.h"
 #include "../version.h"
 
+#ifndef I2C_FUNC_SMBUS_PEC
+#define I2C_FUNC_SMBUS_PEC I2C_FUNC_SMBUS_HWPEC_CALC
+#endif
+
 static void help(void) __attribute__ ((noreturn));
 
 static void help(void)
--- a/tools/i2cdump.c
+++ b/tools/i2cdump.c
@@ -33,6 +33,10 @@
 #include "util.h"
 #include "../version.h"
 
+#ifndef I2C_FUNC_SMBUS_PEC
+#define I2C_FUNC_SMBUS_PEC I2C_FUNC_SMBUS_HWPEC_CALC
+#endif
+
 static void help(void)
 {
 	fprintf(stderr,
--- a/tools/i2cset.c
+++ b/tools/i2cset.c
@@ -33,6 +33,10 @@
 #include "util.h"
 #include "../version.h"
 
+#ifndef I2C_FUNC_SMBUS_PEC
+#define I2C_FUNC_SMBUS_PEC I2C_FUNC_SMBUS_HWPEC_CALC
+#endif
+
 static void help(void) __attribute__ ((noreturn));
 
 static void help(void)
