--- ../../../openwrt_trunk.orig/feeds/oldpackages/lang/php-pear-xmlrpc/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/php-pear-xmlrpc/Makefile
@@ -36,11 +36,11 @@
 endef
 
 define Package/php-pear-xmlrpc/install
-	$(INSTALL_DIR) $(1)/usr/lib/php/XML
+	$(INSTALL_DIR) $(1)/opt/lib/php/XML
 	$(CP)	$(PKG_BUILD_DIR)/Dump.php \
 		$(PKG_BUILD_DIR)/RPC.php \
 		$(PKG_BUILD_DIR)/Server.php \
-		$(1)/usr/lib/php/XML/
+		$(1)/opt/lib/php/XML/
 endef
 
 $(eval $(call BuildPackage,php-pear-xmlrpc))
