--- ../../../downloads/openwrt_trunk-r40859/feeds/telephony/net/asterisk-11.x-chan-dongle/Makefile
+++ ../../../openwrt_trunk/feeds/telephony/net/asterisk-11.x-chan-dongle/Makefile
@@ -29,7 +29,7 @@
   SECTION:=net
   CATEGORY:=Network
   URL:=https://code.google.com/p/asterisk-chan-dongle/
-  DEPENDS:= asterisk11 +libiconv-full +kmod-usb-acm +kmod-usb-serial +kmod-usb-serial-option +libusb-1.0 +usb-modeswitch
+  DEPENDS:= asterisk11 +libiconv-full +libusb-1.0
   TITLE:=Huawei UMTS 3G dongle support
   MAINTAINER:=Jiri Slachta <slachta@cesnet.cz>
 endef
@@ -41,9 +41,9 @@
 MAKE_ARGS:= \
 	CC="$(TARGET_CC)" \
 	LD="$(TARGET_CC)" \
-	CFLAGS="$(TARGET_CFLAGS) -DASTERISK_VERSION_NUM=110000 -DLOW_MEMORY -D_XOPEN_SOURCE=600 $(TARGET_CPPFLAGS) -I$(STAGING_DIR)/usr/lib/libiconv-full/include -I$(STAGING_DIR)/usr/include/asterisk-11/include -DHAVE_CONFIG_H -I. -fPIC" \
-	LDFLAGS="$(TARGET_LDFLAGS) -L$(STAGING_DIR)/usr/lib/libiconv-full/lib -liconv" \
-	DESTDIR="$(PKG_INSTALL_DIR)/usr/lib/asterisk/modules"
+	CFLAGS="$(TARGET_CFLAGS) -DASTERISK_VERSION_NUM=110000 -DLOW_MEMORY -D_XOPEN_SOURCE=600 $(TARGET_CPPFLAGS) -I$(STAGING_DIR)/opt/lib/libiconv-full/include -I$(STAGING_DIR)/opt/include/asterisk-11/include -DHAVE_CONFIG_H -I. -fPIC" \
+	LDFLAGS="$(TARGET_LDFLAGS) -L$(STAGING_DIR)/opt/lib/libiconv-full/lib -liconv" \
+	DESTDIR="$(PKG_INSTALL_DIR)/opt/lib/asterisk/modules"
 
 CONFIGURE_VARS += \
 	ac_cv_type_size_t=yes \
@@ -51,25 +51,25 @@
 
 define Build/Configure
 	$(call Build/Configure/Default, \
-	    --with-asterisk=$(STAGING_DIR)/usr/include/asterisk-11/include \
+	    --with-asterisk=$(STAGING_DIR)/opt/include/asterisk-11/include \
 	    $(MAKE_ARGS) \
 	)
 endef
 
 define Build/Compile
-	mkdir -p $(PKG_INSTALL_DIR)/usr/lib/asterisk/modules
+	mkdir -p $(PKG_INSTALL_DIR)/opt/lib/asterisk/modules
 	$(MAKE) -C "$(PKG_BUILD_DIR)" $(MAKE_ARGS) all install
 endef
 
 define Package/asterisk11-chan-dongle/conffiles
-/etc/asterisk/dongle.conf
+/opt/etc/asterisk/dongle.conf
 endef
 
 define Package/asterisk11-chan-dongle/install
-	$(INSTALL_DIR) $(1)/etc/asterisk
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/etc/dongle.conf $(1)/etc/asterisk/
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/modules
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/asterisk/modules/chan_dongle.so $(1)/usr/lib/asterisk/modules/
+	$(INSTALL_DIR) $(1)/opt/etc/asterisk
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/etc/dongle.conf $(1)/opt/etc/asterisk/
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/modules
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/asterisk/modules/chan_dongle.so $(1)/opt/lib/asterisk/modules/
 endef
 
 $(eval $(call BuildPackage,asterisk11-chan-dongle))
