--- ../../../downloads/toolchain-src-git/rules.mk	2015-07-20 16:49:47.108734641 +0300
+++ ../../../toolchain-src-git/rules.mk	2015-07-20 16:51:56.124736788 +0300
@@ -47,7 +47,7 @@
 BUILD_DIR_TOOLCHAIN:=$(BUILD_DIR_BASE)/toolchain-$(ARCH)_gcc$(GCCV)
 STAGING_DIR:=$(TOPDIR)/staging_dir/$(ARCH)
 STAGING_DIR_HOST:=$(TOPDIR)/staging_dir/host
-TOOLCHAIN_DIR:=/opt/brcm/hndtools-$(ARCH)-uclibc-$(GCCV)-K26
+TOOLCHAIN_DIR:=/opt/entware/toolchain-mipselsf
 #TOOLCHAIN_DIR:=$(TOPDIR)/staging_dir/toolchain-$(ARCH)_gcc$(GCCV)
 PACKAGE_DIR:=$(BIN_DIR)/packages/$(ARCH)
 STAMP_DIR:=$(BUILD_DIR)/stamp
