--- ../../../toolchain-1.9.2.7-rtn/src/rules.mk.orig
+++ ../../../toolchain-1.9.2.7-rtn/src/rules.mk
@@ -50,7 +50,7 @@
 BUILD_DIR_TOOLCHAIN:=$(BUILD_DIR_BASE)/toolchain-$(ARCH)_gcc$(GCCV)
 STAGING_DIR:=$(TOPDIR)/staging_dir/$(ARCH)
 STAGING_DIR_HOST:=$(TOPDIR)/staging_dir/host
-TOOLCHAIN_DIR:=/opt/brcm/hndtools-$(ARCH)-uclibc-$(GCCV)$(TGT_SUFFIX)
+TOOLCHAIN_DIR:=/opt/entware/toolchain-mipselsf
 #TOOLCHAIN_DIR:=$(TOPDIR)/staging_dir/toolchain-$(ARCH)_gcc$(GCCV)
 PACKAGE_DIR:=$(BIN_DIR)/packages/$(ARCH)
 STAMP_DIR:=$(BUILD_DIR)/stamp
