--- ../../../downloads/openwrt_trunk-r43718/feeds/telephony/net/asterisk-1.8.x-codec-g729/Makefile	2014-12-15 16:50:13.464093575 +0300
+++ ../../../openwrt_trunk/feeds/telephony/net/asterisk-1.8.x-codec-g729/Makefile	2014-12-18 12:04:03.011681582 +0300
@@ -34,19 +34,19 @@
   TITLE:=G.729 codec for Asterisk open source PBX
 endef
 
-define Package/asterisk11-codec-g729/description
+define Package/asterisk18-codec-g729/description
  Asterisk G.729 codec based on bcg729 implementation.
 endef
 
 MAKE_ARGS:= \
 	CC="$(TARGET_CC)" \
 	LD="$(TARGET_LD)" \
-	CFLAGS="$(TARGET_CFLAGS) -DASTERISK_VERSION_NUM=110000 -DLOW_MEMORY -D_XOPEN_SOURCE=600 $(TARGET_CPPFLAGS) -I$(STAGING_DIR)/usr/include/asterisk-1.8/include -DHAVE_CONFIG_H -I. -fPIC" \
+	CFLAGS="$(TARGET_CFLAGS) -DASTERISK_VERSION_NUM=110000 -DLOW_MEMORY -D_XOPEN_SOURCE=600 $(TARGET_CPPFLAGS) -I$(STAGING_DIR)/opt/include/asterisk-1.8/include -DHAVE_CONFIG_H -I. -fPIC" \
 	LDFLAGS="$(TARGET_LDFLAGS)" \
 	DESTDIR="$(PKG_INSTALL_DIR)"
 
 CONFIGURE_ARGS+=\
-	--with-asterisk-includes=$(STAGING_DIR)/usr/include/asterisk-11/include \
+	--with-asterisk-includes=$(STAGING_DIR)/opt/include/asterisk-1.8/include \
 	--with-asterisk18 \
 	--with-bcg729 \
 	--enable-shared \
@@ -60,8 +60,8 @@
 endef
 
 define Package/asterisk18-codec-g729/install
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/modules
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/asterisk/modules/codec_g729.so $(1)/usr/lib/asterisk/modules/
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/modules
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/asterisk/modules/codec_g729.so $(1)/opt/lib/asterisk/modules/
 endef
 
 $(eval $(call BuildPackage,asterisk18-codec-g729))
