--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/net/udpxy/Makefile	2014-04-09 16:17:24.178286483 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/udpxy/Makefile	2014-04-11 15:42:14.775183823 +0400
@@ -39,15 +39,15 @@
 endef
 
 MAKE_FLAGS += \
-	INSTALLROOT="$(PKG_INSTALL_DIR)/usr" \
+	INSTALLROOT="$(PKG_INSTALL_DIR)/opt" \
 	ALL_CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS)"
 
 define Package/udpxy/install
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/udpxy.init $(1)/etc/init.d/udpxy
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/udpxy $(1)/usr/bin/
-	ln -sf udpxy $(1)/usr/bin/udpxrec
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S29udpxy $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/udpxy $(1)/opt/bin/
+	ln -sf udpxy $(1)/opt/bin/udpxrec
 endef
 
 $(eval $(call BuildPackage,udpxy))
