--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/udpxy/Makefile
+++ ../../../openwrt_trunk/feeds/packages/net/udpxy/Makefile
@@ -44,21 +44,16 @@
 	popcornhour/mediatomb/vlc).
 endef
 
-define Package/udpxy/conffiles
-/etc/config/udpxy
-endef
-
 MAKE_FLAGS += \
-	INSTALLROOT="$(PKG_INSTALL_DIR)/usr" \
+	INSTALLROOT="$(PKG_INSTALL_DIR)/opt" \
 	ALL_CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS)"
 
 define Package/udpxy/install
-	$(INSTALL_DIR) $(1)/etc/init.d $(1)/etc/config
-	$(INSTALL_CONF) ./files/udpxy.conf $(1)/etc/config/udpxy
-	$(INSTALL_BIN) ./files/udpxy.init $(1)/etc/init.d/udpxy
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/udpxy $(1)/usr/bin/
-	ln -sf udpxy $(1)/usr/bin/udpxrec
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S29udpxy $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/udpxy $(1)/opt/bin/
+	ln -sf udpxy $(1)/opt/bin/udpxrec
 endef
 
 $(eval $(call BuildPackage,udpxy))
