diff -urx .svn ../../../trunk_clean/feeds/packages/net/rsync/files/rsyncd.conf ../../../trunk/feeds/packages/net/rsync/files/rsyncd.conf
--- ../../../trunk_clean/feeds/packages/net/rsync/files/rsyncd.conf	2012-06-21 10:11:57.000000000 +0400
+++ ../../../trunk/feeds/packages/net/rsync/files/rsyncd.conf	2012-07-02 23:43:19.000000000 +0400
@@ -1,15 +1,16 @@
-# /etc/rsyncd.conf
+# /opt/etc/rsyncd.conf
 # Minimal configuration for rsync daemon
 
 # Next line required for init script
-pid file = /var/run/rsyncd.pid
+#pid file = /opt/var/run/rsyncd.pid
+log file = /opt/var/log/rsyncd.log
 use chroot = yes
 uid = nobody
-gid = nogroup
+gid = nobody
 read only = yes
 
 # Simple example to configure server
-#[openwrt-etc]
-#path = /etc
-#comment = OpenWrt Configuration Files
-#exclude = /init.d
+[test]
+path = /opt/etc
+comment = Test backup (/opt/etc)
+#exclude = /opt/etc/init.d
diff -urx .svn ../../../trunk_clean/feeds/packages/net/rsync/Makefile ../../../trunk/feeds/packages/net/rsync/Makefile
--- ../../../trunk_clean/feeds/packages/net/rsync/Makefile	2012-06-21 10:11:57.000000000 +0400
+++ ../../../trunk/feeds/packages/net/rsync/Makefile	2012-07-02 19:46:43.000000000 +0400
@@ -9,7 +9,7 @@
 
 PKG_NAME:=rsync
 PKG_VERSION:=3.0.9
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://rsync.samba.org/ftp/rsync/src
@@ -55,10 +55,12 @@
 	--disable-locale \
 	--disable-xattr-support \
 	--disable-acl-support \
+	--with-rsyncd-conf=/opt/etc/rsyncd.conf \
+	--with-nobody-group=nobody \
 
 define Package/rsync/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME) $(1)/opt/bin/
 endef
 
 define Package/rsyncd/description
@@ -68,14 +70,17 @@
 endef
 
 define Package/rsyncd/conffiles
-/etc/rsyncd.conf
+/opt/etc/rsyncd.conf
 endef
 
 define Package/rsyncd/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/rsyncd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/rsyncd.init $(1)/etc/init.d/rsyncd
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/rsyncd.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S50rsyncd $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/var/log
+	$(INSTALL_DIR) $(1)/opt/etc/xinetd.d
+	$(INSTALL_DATA) ./files/rsyncd $(1)/opt/etc/xinetd.d/
 endef
 
 $(eval $(call BuildPackage,rsync))
