--- a/mcabberrc.example
+++ b/mcabberrc.example
@@ -1,4 +1,4 @@
-# Sample ~/.mcabber/mcabberrc file
+# Sample /opt/share/mcabber/mcabberrc file
 
 # Note about this file syntax:
 # - Leading and trailing spaces are ignored.
@@ -102,7 +102,7 @@
 # This may be convenient to print help not into the status buffer, but into
 # currently opened buffer. For that set help_to_current option.
 #set lang = en
-#set help_dirs = "/usr/share/mcabber/help;~/share/mcabber/help"
+#set help_dirs = "/opt/share/mcabber/help;~/share/mcabber/help"
 #set help_to_current = 0
 
 # Aspell
@@ -148,7 +148,7 @@
 # (and status messages) to the log files (default: 0).
 # You can load (read) the messages history: set load_logs = 1
 # If you enable load_logs, you can use the 'max_history_age' setting below.
-# Default logging directory (logging_dir) is $HOME/.mcabber/histo/
+# Default logging directory (logging_dir) is /opt/share/mcabber/histo/
 # Defaults for logging, load_logs are 0 (disabled)
 # Note: the logging directory must exist if you enable logging, mcabber
 #       will not create it.
@@ -156,7 +156,7 @@
 # are used at startup time.
 #set logging = 1
 #set load_logs = 1
-#set logging_dir = ~/.mcabber/histo/
+#set logging_dir = /opt/share/mcabber/histo/
 #set logging_ignore_status = 1
 
 # Set log_muc_conf to 1 to enable MUC chatrooms logging (default = 0)
@@ -175,7 +175,7 @@
 # mcabber can store the list of unread messages in a state file,
 # so that the message flags are set back at next startup.
 # Note that 'logging' must be enabled for this feature to work.
-#set statefile = ~/.mcabber/mcabber.state
+#set statefile = /opt/share/mcabber/mcabber.state
 
 # You can specify a maximum number of data blocks per buffer (1 block contains
 # about 8kB).  The default is 0 (unlimited).  If set, this value must be > 2.
@@ -201,7 +201,7 @@
 # If mcabber is built with modules support, you can specify the path
 # to the directory where your modules reside. Though, default compiled-in
 # value should be appropriate.
-#set modules_dir = /usr/lib/mcabber/
+#set modules_dir = /opt/lib/mcabber/
 
 # Beep
 # Set beep_on_message to 1 if you want mcabber to beep when receiving
@@ -229,12 +229,12 @@
 #   $events_command UNREAD "N x y z"        (number of unread buddy buffers)
 #   (x=attention y=muc unread buffers z=muc unread buffers with attention sign)
 # See sample script in contrib/ directory.
-#set events_command = ~/.mcabber/eventcmd
+#set events_command = /opt/share/mcabber/eventcmd
 #
 #set events_ignore_active_window = 0
 #
 #set event_log_files = 0
-#set event_log_dir = ~/.mcabber/event_files
+#set event_log_dir = /opt/share/mcabber/event_files
 
 # If you set 'eventcmd_use_nickname' to 1, mcabber will pass the nickname
 # (if it is defined) to the event script instead of the JID (default: 0).
@@ -267,7 +267,7 @@
 # When FIFO  is configured, you can turn it off and on in real time with
 # the 'fifo_ignore' option (default: 0).  When set to 1, the FIFO input is
 # still read but it is discarded.
-#set fifo_name = ~/.mcabber/mcabber.fifo
+#set fifo_name = /opt/share/mcabber/mcabber.fifo
 #set fifo_hide_commands = 0
 #set fifo_ignore = 0
 #
@@ -291,7 +291,7 @@
 #  lvl 4: noisy debug logging (Loudmouth parser...)
 # Default is level 0, no trace logging
 #set tracelog_level = 1
-#set tracelog_file = ~/.mcabber/mcabber.log
+#set tracelog_file = /opt/share/mcabber/mcabber.log
 
 # Set the auto-away timeout, in seconds.  If set to a value >0,
 # mcabber will change your status to away if no real activity is detected
@@ -509,8 +509,8 @@
 #
 # OTR directory
 # You can specify the OTR directory with the otr_dir option
-# (default = ~/.mcabber/otr/).
-#set otr_dir = "~/.mcabber/otr/"
+# (default = /opt/share/mcabber/otr/).
+#set otr_dir = "/opt/share/mcabber/otr/"
 #
 # You can set OTR policies from the configuration file (use /help otrpolicy
 # for the details).
