--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/btpd/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/btpd/Makefile
@@ -40,19 +40,13 @@
 	the casual user as well.
 endef
 
-define Package/btpd/conffiles
-/etc/config/btpd
-endef
-
 CONFIGURE_ARGS += -C --with-evloop-method=poll
 
 define Package/btpd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/bt{cli,info,pd} $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/btpd.init $(1)/etc/init.d/btpd
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/btpd.config $(1)/etc/config/btpd
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/bt{cli,info,pd} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S93btpd $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,btpd))
