--- ../../../downloads/openwrt_trunk-r43708/feeds/packages/net/rsync/Makefile	2014-12-15 12:19:58.032388959 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/rsync/Makefile	2014-12-15 12:20:24.512388475 +0300
@@ -41,6 +41,8 @@
 	--with-included-popt=no \
 	--disable-debug \
 	--disable-locale \
+	--with-rsyncd-conf=/opt/etc/rsyncd.conf \
+	--with-nobody-group=nobody \
 
 ifeq ($(CONFIG_RSYNC_xattr),y)
 	CONFIGURE_ARGS+= --enable-xattr-support
@@ -80,8 +82,8 @@
 endef
 
 define Package/rsync/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME) $(1)/opt/bin/
 endef
 
 define Package/rsyncd/description
@@ -90,14 +92,17 @@
 endef
 
 define Package/rsyncd/conffiles
-/etc/rsyncd.conf
+/opt/etc/rsyncd.conf
 endef
 
 define Package/rsyncd/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/rsyncd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/rsyncd.init $(1)/etc/init.d/rsyncd
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/rsyncd.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S50rsyncd $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/var/log
+	$(INSTALL_DIR) $(1)/opt/etc/xinetd.d
+	$(INSTALL_DATA) ./files/rsyncd $(1)/opt/etc/xinetd.d/
 endef
 
 $(eval $(call BuildPackage,rsync))
