diff -urx .svn ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/net/rsync/files/rsyncd.conf ../../../openwrt_trunk/feeds/packages/net/rsync/files/rsyncd.conf
--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/net/rsync/files/rsyncd.conf	2014-04-09 16:17:24.142286482 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/rsync/files/rsyncd.conf	2014-04-09 16:53:33.518246966 +0400
@@ -1,15 +1,16 @@
-# /etc/rsyncd.conf
+# /opt/etc/rsyncd.conf
 # Minimal configuration for rsync daemon
 
 # Next line required for init script
-pid file = /var/run/rsyncd.pid
+#pid file = /opt/var/run/rsyncd.pid
+log file = /opt/var/log/rsyncd.log
 use chroot = yes
 uid = nobody
-gid = nogroup
+gid = nobody
 read only = yes
 
 # Simple example to configure server
-#[openwrt-etc]
-#path = /etc
-#comment = OpenWrt Configuration Files
-#exclude = /init.d
+[test]
+path = /opt/etc
+comment = Test backup (/opt/etc)
+#exclude = /opt/etc/init.d
diff -urx .svn ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/net/rsync/Makefile ../../../openwrt_trunk/feeds/packages/net/rsync/Makefile
--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/net/rsync/Makefile	2014-04-09 16:17:24.142286482 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/rsync/Makefile	2014-04-09 16:53:33.518246966 +0400
@@ -55,10 +55,12 @@
 	--disable-locale \
 	--disable-xattr-support \
 	--disable-acl-support \
+	--with-rsyncd-conf=/opt/etc/rsyncd.conf \
+	--with-nobody-group=nobody \
 
 define Package/rsync/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME) $(1)/opt/bin/
 endef
 
 define Package/rsyncd/description
@@ -67,14 +69,17 @@
 endef
 
 define Package/rsyncd/conffiles
-/etc/rsyncd.conf
+/opt/etc/rsyncd.conf
 endef
 
 define Package/rsyncd/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/rsyncd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/rsyncd.init $(1)/etc/init.d/rsyncd
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/rsyncd.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S50rsyncd $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/var/log
+	$(INSTALL_DIR) $(1)/opt/etc/xinetd.d
+	$(INSTALL_DATA) ./files/rsyncd $(1)/opt/etc/xinetd.d/
 endef
 
 $(eval $(call BuildPackage,rsync))
