--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/rsync/Makefile
+++ ../../../openwrt_trunk/feeds/packages/net/rsync/Makefile
@@ -42,6 +42,8 @@
 	--with-included-popt=no \
 	--disable-debug \
 	--disable-locale \
+	--with-rsyncd-conf=/opt/etc/rsyncd.conf \
+	--with-nobody-group=nobody \
 
 ifeq ($(CONFIG_RSYNC_xattr),y)
 	CONFIGURE_ARGS+= --enable-xattr-support
@@ -81,8 +83,8 @@
 endef
 
 define Package/rsync/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME) $(1)/opt/bin/
 endef
 
 define Package/rsyncd/description
@@ -91,14 +93,17 @@
 endef
 
 define Package/rsyncd/conffiles
-/etc/rsyncd.conf
+/opt/etc/rsyncd.conf
 endef
 
 define Package/rsyncd/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/rsyncd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/rsyncd.init $(1)/etc/init.d/rsyncd
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/rsyncd.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S50rsyncd $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/var/log
+	$(INSTALL_DIR) $(1)/opt/etc/xinetd.d
+	$(INSTALL_DATA) ./files/rsyncd $(1)/opt/etc/xinetd.d/
 endef
 
 $(eval $(call BuildPackage,rsync))
