--- ../../../openwrt_trunk.orig/feeds/packages/libs/libid3tag/Makefile	2011-12-05 10:46:45.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libid3tag/Makefile	2011-12-09 11:59:52.756786741 +0400
@@ -45,15 +45,15 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/id3tag.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3tag.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/id3tag.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3tag.so* $(1)/opt/lib/
 endef
 
 define Package/libid3tag/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libid3tag.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libid3tag.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libid3tag))
