--- ../../../downloads/openwrt_trunk-r44486/feeds/packages/utils/attr/Makefile	2015-02-18 12:16:57.636306330 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/attr/Makefile	2015-02-18 12:17:23.204305869 +0300
@@ -69,25 +69,25 @@
 CONFIGURE_ARGS += --enable-static --enable-shared
 
 define Package/attr/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 define Package/libattr/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc $(1)/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc $(1)/opt/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/*.so* $(1)/opt/lib/
 endef
 
 define Package/attr/conffiles
-/etc/xattr.conf
+/opt/etc/xattr.conf
 endef
 
 define Build/InstallDev
-	mkdir -p $(1)/usr/include
-	mkdir -p $(1)/usr/lib/pkgconfig
-	$(CP) -r $(PKG_INSTALL_DIR)/usr/{include,lib} $(1)/usr/
+	mkdir -p $(1)/opt/include
+	mkdir -p $(1)/opt/lib/pkgconfig
+	$(CP) -r $(PKG_INSTALL_DIR)/opt/{include,lib} $(1)/opt/
 endef
 
 $(eval $(call BuildPackage,attr))
