--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/mcabber/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mcabber/Makefile
@@ -18,7 +18,7 @@
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
 
 include $(INCLUDE_DIR)/package.mk
-#include $(INCLUDE_DIR)/nls.mk
+include $(INCLUDE_DIR)/nls.mk
 
 define Package/mcabber
   SUBMENU:=Instant Messaging
@@ -38,20 +38,20 @@
 endef
 
 CONFIGURE_ARGS += \
-	--with-libidn=$(STAGING_DIR)/usr/lib \
-	--with-gpgme-prefix=$(STAGING_DIR)/usr \
+	--with-libidn=$(STAGING_DIR)/opt/lib \
+	--with-gpgme-prefix=$(STAGING_DIR)/opt \
 	--enable-otr
 
 define Package/mcabber/install
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mcabber/mcabber $(1)/bin
-	$(INSTALL_DIR) $(1)/usr/lib/mcabber
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/beep/.libs/libbeep.so $(1)/usr/lib/mcabber
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/xttitle/.libs/libxttitle.so $(1)/usr/lib/mcabber
-	$(INSTALL_DIR) $(1)/usr/share/mcabber/ $(1)/usr/share/mcabber/help/en
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/mcabberrc.example $(1)/usr/share/mcabber/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/contrib/events/eventcmd $(1)/usr/share/mcabber/eventcmd.example
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/doc/help/en/*.txt $(1)/usr/share/mcabber/help/en/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mcabber/mcabber $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/lib/mcabber
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/beep/.libs/libbeep.so $(1)/opt/lib/mcabber
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/modules/xttitle/.libs/libxttitle.so $(1)/opt/lib/mcabber
+	$(INSTALL_DIR) $(1)/opt/share/mcabber/{histo,event_files} $(1)/opt/share/mcabber/help/en
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/mcabberrc.example $(1)/opt/share/mcabber/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/contrib/events/eventcmd $(1)/opt/share/mcabber/eventcmd.example
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/doc/help/en/*.txt $(1)/opt/share/mcabber/help/en/
 endef
 
 $(eval $(call BuildPackage,mcabber))
