--- ../../../downloads/openwrt_trunk-r40423/feeds/packages/net/iodine/Makefile
+++ ../../../openwrt_trunk/feeds/packages/net/iodine/Makefile
@@ -25,7 +25,7 @@
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=Firewall Tunnel
-  DEPENDS:= +kmod-tun +zlib
+  DEPENDS:=+zlib
   TITLE:=IP over DNS tunneling
   URL:=http://code.kryo.se/iodine/
 endef
@@ -52,21 +52,15 @@
 endef
 
 define Package/iodine/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/iodine $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/iodine $(1)/opt/sbin
 endef
 
 define Package/iodined/install
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/iodined.init $(1)/etc/init.d/iodined
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/iodined.config $(1)/etc/config/iodined
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/iodined $(1)/usr/sbin
-endef
-
-define Package/iodined/conffiles
-/etc/config/iodined
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S19iodined $(1)/opt/etc/init.d
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/iodined $(1)/opt/sbin
 endef
 
 $(eval $(call BuildPackage,iodine))
