--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r38800/feeds/packages/utils/nano/Makefile	2013-11-14 10:01:15.911186156 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/nano/Makefile	2013-11-14 10:01:35.287185805 +0400
@@ -9,7 +9,7 @@
 
 PKG_NAME:=nano
 PKG_VERSION:=2.2.6
-PKG_RELEASE:=1
+PKG_RELEASE:=1b
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.nano-editor.org/dist/v2.2
@@ -36,17 +36,27 @@
 endef
 
 CONFIGURE_ARGS += \
-	--enable-tiny \
 	--disable-glibtest \
 	--disable-utf8 \
 	--without-slang \
-
-CONFIGURE_VARS += \
-	ac_cv_header_regex_h=no \
+	--enable-wrapping \
+	--disable-help \
+	--disable-justify \
+	--disable-mouse \
+	--disable-operatingdir \
+	--disable-speller \
+	--disable-tabcomp \
+	--disable-browser \
+	--enable-color \
 
 define Package/nano/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME) $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share/nano
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/nano/*.nanorc $(1)/opt/share/nano/
+	$(SED) 's/\\>/\\b/g' -e 's/\\</\\b/g' $(1)/opt/share/nano/*.nanorc
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_BUILD_DIR)/doc/nanorc.sample $(1)/opt/etc/nanorc
 endef
 
 $(eval $(call BuildPackage,nano))
