--- ../../../downloads/openwrt_trunk-r34120/feeds/packages/utils/nano/Makefile	2012-11-09 11:32:20.661677175 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/nano/Makefile	2012-11-30 08:50:40.568801827 +0400
@@ -9,7 +9,7 @@
 
 PKG_NAME:=nano
 PKG_VERSION:=2.2.6
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.nano-editor.org/dist/v2.2
@@ -36,17 +36,25 @@
 endef
 
 CONFIGURE_ARGS += \
-	--enable-tiny \
 	--disable-glibtest \
 	--disable-utf8 \
 	--without-slang \
+	--enable-wrapping \
+	--disable-help \
+	--disable-justify \
+	--disable-mouse \
+	--disable-operatingdir \
+	--disable-speller \
+	--disable-tabcomp \
+	--disable-browser \
+	--disable-color \
 
 CONFIGURE_VARS += \
 	ac_cv_header_regex_h=no \
 
 define Package/nano/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME) $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,nano))
