--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/aria2/Makefile	2014-11-07 23:35:24.524646726 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/aria2/Makefile	2014-11-28 16:28:22.764979981 +0300
@@ -7,12 +7,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aria2
-PKG_VERSION:=1.18.7
+PKG_VERSION:=1.17.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=@SF/aria2
-PKG_MD5SUM:=36e92af92b4370817c577ed802546842
+PKG_MD5SUM:=86229ef8d289893934cb3af25c8fddf6
 PKG_INSTALL:=1
 
 PKG_MAINTAINER:=Imre Kaloz <kaloz@openwrt.org>
@@ -63,8 +63,11 @@
 	--with-libz
 
 define Package/aria2/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/aria2c $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/bin/aria2c $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/var/{log,run} $(1)/opt/etc/init.d
+	$(INSTALL_CONF) ./files/aria2.conf $(1)/opt/etc
+	$(INSTALL_BIN) ./files/S81aria2 $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,aria2))
