diff -urx .svn ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/files/base.config ../../../openwrt_trunk/feeds/packages/lang/perl/files/base.config
--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/files/base.config	2015-09-24 20:31:27.218958283 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/perl/files/base.config	2015-09-27 09:45:08.502449431 +0300
@@ -75,6 +75,7 @@
 d_aintl='undef'
 d_alarm='define'
 d_archlib='undef'
+d_asctime_r='undef'
 d_asctime64='undef'
 d_asinh='define'
 d_atanh='define'
@@ -116,9 +117,12 @@
 d_copysign='define'
 d_copysignl='undef'
 d_cplusplus='undef'
+d_crypt='define'
+d_crypt_r='undef'
 d_csh='undef'
 d_ctermid='define'
 d_ctermid_r='undef'
+d_ctime_r='undef'
 d_ctime64='undef'
 d_cuserid='define'
 d_dbl_dig='define'
@@ -133,6 +137,7 @@
 d_dlopen='define'
 d_dlsymun='undef'
 d_dosuid='undef'
+d_drand48_r='undef'
 d_drand48proto='define'
 d_dup2='define'
 d_eaccess='undef'
@@ -163,9 +168,12 @@
 d_fd_macros='define'
 d_fd_set='define'
 d_fdim='define'
+d_fds_bits='undef'
+d_fegetround='undef'
 d_fgetpos='define'
 d_finite='define'
 d_finitel='undef'
+d_isfinitel='undef'
 d_flexfnam='define'
 d_flock='define'
 d_flockproto='define'
@@ -178,7 +186,7 @@
 d_fp_classl='undef'
 d_fpathconf='define'
 d_fpclass='undef'
-d_fpclassify='define'
+d_fpclassify='undef'
 d_fpclassl='undef'
 d_fpgetround='undef'
 d_fpos64_t='undef'
@@ -199,39 +207,61 @@
 d_getespwnam='undef'
 d_getfsstat='undef'
 d_getgrent='define'
+d_getgrent_r='undef'
+d_getgrgid_r='undef'
+d_getgrnam_r='undef'
 d_getgrps='define'
 d_gethbyaddr='define'
 d_gethbyname='define'
 d_gethent='define'
 d_gethname='define'
+d_gethostbyaddr_r='undef'
+d_gethostbyname_r='undef'
+d_gethostent_r='undef'
 d_gethostprotos='define'
 d_getitimer='define'
 d_getlogin='define'
+d_getlogin_r='undef'
 d_getmnt='undef'
 d_getmntent='define'
 d_getnameinfo='define'
 d_getnbyaddr='define'
 d_getnbyname='define'
 d_getnent='define'
+d_getnetbyaddr_r='undef'
+d_getnetbyname_r='undef'
+d_getnetent_r='undef'
 d_getnetprotos='define'
 d_getpagsz='define'
 d_getpbyname='define'
 d_getpbynumber='define'
 d_getpent='define'
 d_getpgid='define'
+d_getpgrp2='undef'
 d_getpgrp='define'
 d_getpgrp2='undef'
 d_getppid='define'
 d_getprior='define'
+d_getprotobyname_r='undef'
+d_getprotobynumber_r='undef'
+d_getprotoent_r='undef'
 d_getprotoprotos='define'
 d_getprpwnam='undef'
 d_getpwent='define'
+d_getpwent_r='undef'
+d_getpwnam_r='undef'
+d_getpwuid_r='undef'
 d_getsbyname='define'
 d_getsbyport='define'
 d_getsent='define'
+d_getservbyname_r='undef'
+d_getservbyport_r='undef'
+d_getservent_r='undef'
 d_getservprotos='define'
 d_getspnam='define'
+d_getspnam_r='undef'
 d_gettimeod='define'
+d_gmtime_r='undef'
 d_gmtime64='undef'
 d_gnulibc='undef'
 d_grpasswd='define'
@@ -252,16 +282,18 @@
 d_ipv6_mreq_source='undef'
 d_isascii='define'
 d_isblank='define'
-d_isfinite='define'
+d_isfinite='undef'
 d_isfinitel='undef'
 d_isinf='define'
 d_isinfl='define'
 d_isless='undef'
 d_isnan='define'
 d_isnanl='undef'
-d_isnormal='define'
+d_isnormal='undef'
+d_j0='undef'
+d_j0l='undef'
 d_killpg='define'
-d_lc_monetary_2008='define'
+d_lc_monetary_2008='undef'
 d_lchown='define'
 d_ldbl_dig='define'
 d_ldexpl='define'
@@ -274,6 +306,8 @@
 d_llrintl='define'
 d_llround='define'
 d_llroundl='define'
+d_localtime_r='undef'
+d_localtime_r_needs_tzset='undef'
 d_localtime64='undef'
 d_locconv='define'
 d_lockf='define'
@@ -309,6 +343,7 @@
 d_mmap='define'
 d_modfl='undef'
 d_modfl_pow32_bug='undef'
+d_modflproto='undef'
 d_mprotect='define'
 d_msg='define'
 d_msg_ctrunc='define'
@@ -324,15 +359,18 @@
 d_msync='define'
 d_munmap='define'
 d_mymalloc='undef'
-d_nan='define'
+d_nan='undef'
 d_nanosleep='define'
 d_ndbm='undef'
 d_ndbm_h_uses_prototypes='undef'
 d_nearbyint='define'
 d_nextafter='define'
+d_nexttoward='undef'
 d_nice='define'
 d_nl_langinfo='define'
+d_nv_preserves_uv='define'
 d_nv_zero_is_allbits_zero='define'
+d_off64_t='undef'
 d_old_pthread_create_joinable='undef'
 d_oldpthreads='undef'
 d_oldsock='undef'
@@ -348,8 +386,10 @@
 d_prctl_set_name='define'
 d_procselfexe='define'
 d_pseudofork='undef'
+d_pthread_atfork='undef'
 d_pthread_attr_setscope='define'
-d_ptrdiff_t='define'
+d_pthread_yield='undef'
+d_ptrdiff_t='undef'
 d_pwage='undef'
 d_pwchange='undef'
 d_pwclass='undef'
@@ -360,8 +400,11 @@
 d_pwquota='undef'
 d_qgcvt='undef'
 d_quad='define'
+d_random_r='undef'
 d_re_comp='undef'
+d_readdir64_r='undef'
 d_readdir='define'
+d_readdir_r='undef'
 d_readlink='define'
 d_readv='define'
 d_recvmsg='define'
@@ -406,6 +449,7 @@
 d_setnetent_r='undef'
 d_setpent='define'
 d_setpgid='define'
+d_setpgrp2='undef'
 d_setpgrp='define'
 d_setpgrp2='undef'
 d_setprior='define'
@@ -447,7 +491,11 @@
 d_socks5_init='undef'
 d_sprintf_returns_strlen='define'
 d_sqrtl='undef'
-d_stat='define'
+d_srand48_r='undef'
+d_srandom_r='undef'
+d_sresgproto='undef'
+d_sresuproto='undef'
+d_stat='undef'
 d_statblks='define'
 d_statfs_f_flags='undef'
 d_statfs_s='define'
@@ -465,9 +513,10 @@
 d_strctcpy='define'
 d_strerrm='strerror(e)'
 d_strerror='define'
+d_strerror_r='undef'
 d_strftime='define'
-d_strlcat='define'
-d_strlcpy='define'
+d_strlcat='undef'
+d_strlcpy='undef'
 d_strtod='define'
 d_strtol='define'
 d_strtold='define'
@@ -495,10 +544,13 @@
 d_times='define'
 d_tm_tm_gmtoff='define'
 d_tm_tm_zone='define'
+d_tmpnam_r='undef'
 d_trunc='define'
 d_truncate='define'
 d_truncl='define'
+d_ttyname_r='undef'
 d_tzname='define'
+d_u32align='undef'
 d_ualarm='define'
 d_umask='define'
 d_uname='define'
@@ -690,21 +742,21 @@
 inc_version_list_init='0'
 incpath=''
 inews=''
-initialinstalllocation='/usr/bin'
-installbin='/usr/bin'
+initialinstalllocation='/Apps/opt/bin'
+installbin='/Apps/opt/bin'
 installhtml1dir=''
 installhtml3dir=''
 installman1dir=''
 installman3dir=''
-installprefix='/usr'
-installprefixexp='/usr'
-installscript='/usr/bin'
-installsitebin='/usr/bin'
+installprefix='/Apps/opt'
+installprefixexp='/Apps/opt'
+installscript='/Apps/opt/bin'
+installsitebin='/Apps/opt/bin'
 installsitehtml1dir=''
 installsitehtml3dir=''
 installsiteman1dir=''
 installsiteman3dir=''
-installsitescript='/usr/bin'
+installsitescript='/Apps/opt/bin'
 installstyle='lib/perl5'
 installusrbinperl='define'
 installvendorarch=''
@@ -727,9 +779,9 @@
 libc=''
 libdb_needs_pthread='N'
 libperl='libperl.so'
-libpth="$owrt:staging_dir/lib $owrt:staging_dir/usr/lib"
-libsdirs="$owrt:staging_dir/lib $owrt:staging_dir/usr/lib"
-libspath="$owrt:staging_dir/lib $owrt:staging_dir/usr/lib"
+libpth="$owrt:staging_dir/lib $owrt:staging_dir/opt/lib"
+libsdirs="$owrt:staging_dir/lib $owrt:staging_dir/opt/lib"
+libspath="$owrt:staging_dir/lib $owrt:staging_dir/opt/lib"
 libswanted_uselargefiles=''
 line=''
 lint=''
@@ -798,15 +850,17 @@
 perl5=''
 perl_patchlevel=''
 perl_static_inline='static __inline__'
-perlpath='/usr/bin/perl'
+perlpath='/Apps/opt/bin/perl'
 pg='pg'
 phostname='hostname'
 pidtype='pid_t'
 plibpth=''
 pmake=''
 pr=''
-prefix='/usr'
-prefixexp='/usr'
+prefix='/opt'
+prefixexp='/opt'
+
+
 procselfexe='"/proc/self/exe"'
 prototype='define'
 randbits='48'
@@ -843,8 +897,8 @@
 signal_t='void'
 sitearch=''
 sitearchexp=''
-sitebin='/usr/bin'
-sitebinexp='/usr/bin'
+sitebin='/Apps/opt/bin'
+sitebinexp='/Apps/opt/bin'
 sitehtml1dir=''
 sitehtml1direxp=''
 sitehtml3dir=''
@@ -855,10 +909,10 @@
 siteman1direxp=''
 siteman3dir=''
 siteman3direxp=''
-siteprefix='/usr'
-siteprefixexp='/usr'
-sitescript='/usr/bin'
-sitescriptexp='/usr/bin'
+siteprefix='/opt'
+siteprefixexp='/opt'
+sitescript='/Apps/opt/bin'
+sitescriptexp='/Apps/opt/bin'
 sizetype='size_t'
 sleep=''
 smail=''
@@ -873,7 +927,7 @@
 ssizetype='ssize_t'
 st_ino_sign='1'
 st_ino_size='8'
-startperl='#!/usr/bin/perl'
+startperl='#!/Apps/opt/bin/perl'
 startsh='#!/bin/sh'
 static_ext=' '
 stdchar='char'
@@ -942,7 +996,7 @@
 usevendorprefix='undef'
 useversionedarchname='undef'
 usevfork='false'
-usrinc="$owrt:staging_dir/include $owrt:staging_dir/usr/include"
+usrinc="$owrt:staging_dir/include $owrt:staging_dir/opt/include"
 uuname=''
 uvXUformat='"lX"'
 uvoformat='"lo"'
@@ -992,9 +1046,9 @@
 cppflags="$owrt:cflags"
 ccflags="-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 $owrt:cflags"
 ccflags_uselargefiles='-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64'
-ccdlflags="-fPIC -Wl,-rpath,$owrt:perllibpath/CORE"
+ccdlflags="-fPIC -Wl,-rpath,$owrt:perllibpath/CORE:/Apps/opt/lib"
 cccdlflags='-fPIC'
-ldflags=''
+ldflags="$owrt:ldflags"
 ldflags_uselargefiles=''
 lddlflags="-shared $owrt:ldflags"
 
diff -urx .svn ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/files/misc.config ../../../openwrt_trunk/feeds/packages/lang/perl/files/misc.config
--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/files/misc.config	2015-09-24 20:31:27.218958283 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/perl/files/misc.config	2015-09-27 09:27:43.466449431 +0300
@@ -1,6 +1,6 @@
 cf_email='naoir@gmx.net'
-osvers='3.18.19'
-gccversion='4.8.0'
+osvers='2.6.32'
+gccversion='4.8.3'
 myhostname='OpenWrt'
 cf_time='Thu Jan 1 12:00:00 CEST 2015'
 
diff -urx .svn ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/files/perl-run_tests.sh ../../../openwrt_trunk/feeds/packages/lang/perl/files/perl-run_tests.sh
--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/files/perl-run_tests.sh	2015-09-24 20:31:27.218958283 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/perl/files/perl-run_tests.sh	2015-09-27 09:28:01.266449431 +0300
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-PERL_TESTSDIR="/usr/share/perl/perl-tests"
-PERL_LIBDIR="/usr/lib/perl5/%%PERL_VERSION%%/"
+PERL_TESTSDIR="/Apps/opt/share/perl/perl-tests"
+PERL_LIBDIR="/Apps/opt/lib/perl5/%%PERL_VERSION%%/"
 PERL_DISABLEDTESTS="%%PERL_DISABLEDTESTS%%"
 
 no_run=""
diff -urx .svn ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/files/version.config ../../../openwrt_trunk/feeds/packages/lang/perl/files/version.config
--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/files/version.config	2015-09-24 20:31:27.218958283 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/perl/files/version.config	2015-09-27 09:28:22.758449431 +0300
@@ -14,7 +14,7 @@
 dynamic_ext='B Compress/Raw/Bzip2 Compress/Raw/Zlib Cwd DB_File Data/Dumper Devel/PPPort Devel/Peek Digest/MD5 Digest/SHA Encode Fcntl File/DosGlob File/Glob Filter/Util/Call GDBM_File Hash/Util Hash/Util/FieldHash I18N/Langinfo IO IPC/SysV List/Util MIME/Base64 Math/BigInt/FastCalc Opcode POSIX PerlIO/encoding PerlIO/mmap PerlIO/scalar PerlIO/via SDBM_File Socket Storable Sys/Hostname Sys/Syslog Tie/Hash/NamedCapture Time/HiRes Time/Piece Unicode/Collate XS/APItest XS/Typemap arybase attributes mro re threads threads/shared'
 
 # No need to change anything from here on
-owrt:perllibpath="/usr/lib/perl5/$PERL_REVISION.$PERL_VERSION"
+owrt:perllibpath="/Apps/opt/lib/perl5/$PERL_REVISION.$PERL_VERSION"
 
 revision="$PERL_REVISION"
 patchlevel="$PERL_VERSION"
diff -urx .svn ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/Makefile ../../../openwrt_trunk/feeds/packages/lang/perl/Makefile
--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/Makefile	2015-09-24 20:31:27.218958283 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/perl/Makefile	2015-09-27 09:31:56.606449431 +0300
@@ -96,8 +96,8 @@
 	                                -Dowrt:target_cross='$(TARGET_CROSS)' \
 	                                -Dowrt:cflags='$(TARGET_CFLAGS_PERL) $(TARGET_CPPFLAGS_PERL)' \
 	                                -Dowrt:ldflags='-rdynamic $(TARGET_LDFLAGS)' \
-	                                -Dowrt:libc=$(CONFIG_LIBC) \
-	                                -Dowrt:ipv6=$(if $($(CONFIG_IPV6)),define,undef) \
+	                                -Dowrt:libc='gclibc' \
+	                                -Dowrt:ipv6='define' \
 	                                -Dowrt:threads=$(if $(CONFIG_PERL_THREADS),yes,no) \
 	                                -Dowrt:staging_dir='$(STAGING_DIR)' \
 	                                -Dowrt:host_perl_prefix='$(HOST_PERL_PREFIX)' \
@@ -119,17 +119,17 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/lib/perl5/$(PERL_VERSION)
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/perl5/$(PERL_VERSION) $(1)/usr/lib/perl5/
+	$(INSTALL_DIR) $(1)/opt/lib/perl5/$(PERL_VERSION)
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/perl5/$(PERL_VERSION) $(1)/opt/lib/perl5/
 endef
 
 define Package/perl/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/perl$(PKG_VERSION) $(1)/usr/bin
-	ln -nsf perl$(PKG_VERSION) $(1)/usr/bin/perl
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/bin/perl$(PKG_VERSION) $(1)/opt/bin
+	ln -nsf perl$(PKG_VERSION) $(1)/opt/bin/perl
 	
-	$(INSTALL_DIR) $(1)/usr/lib/perl5/$(PERL_VERSION)/CORE
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/perl5/$(PERL_VERSION)/CORE/libperl.so $(1)/usr/lib/perl5/$(PERL_VERSION)/CORE/
+	$(INSTALL_DIR) $(1)/opt/lib/perl5/$(PERL_VERSION)/CORE
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/perl5/$(PERL_VERSION)/CORE/libperl.so $(1)/opt/lib/perl5/$(PERL_VERSION)/CORE/
 endef
 
 
@@ -154,9 +154,9 @@
 	$(INSTALL_DIR) $(1)/$(PERL_TESTSDIR)/Porting
 	$(INSTALL_DIR) $(1)/$(PERL_TESTSDIR)/regen
 	$(INSTALL_DIR) $(1)/$(PERL_TESTSDIR)/lib
-	$(INSTALL_DIR) $(1)/usr/lib/perl5/$(PERL_VERSION)/XS
-	$(INSTALL_DIR) $(1)/usr/lib/perl5/$(PERL_VERSION)/auto/XS
-	$(INSTALL_DIR) $(1)/usr/lib/perl5/$(PERL_VERSION)/unicore
+	$(INSTALL_DIR) $(1)/opt/lib/perl5/$(PERL_VERSION)/XS
+	$(INSTALL_DIR) $(1)/opt/lib/perl5/$(PERL_VERSION)/auto/XS
+	$(INSTALL_DIR) $(1)/opt/lib/perl5/$(PERL_VERSION)/unicore
 	
 	$(CP) $(PKG_BUILD_DIR)/t $(1)/$(PERL_TESTSDIR)
 	$(CP) $(PKG_BUILD_DIR)/Porting $(1)/$(PERL_TESTSDIR)
@@ -165,12 +165,12 @@
 	$(CP) $(PKG_BUILD_DIR)/TestInit.pm $(1)/$(PERL_TESTSDIR)
 	$(CP) $(PKG_BUILD_DIR)/vutil.c $(1)/$(PERL_TESTSDIR)
 	$(CP) $(PKG_BUILD_DIR)/vxs.inc $(1)/$(PERL_TESTSDIR)
-	$(CP) $(PKG_BUILD_DIR)/lib/XS $(1)/usr/lib/perl5/$(PERL_VERSION)/
-	$(CP) $(PKG_BUILD_DIR)/lib/auto/XS $(1)/usr/lib/perl5/$(PERL_VERSION)/auto
-	$(CP) $(PKG_BUILD_DIR)/lib/vmsish.pm $(1)/usr/lib/perl5/$(PERL_VERSION)/
+	$(CP) $(PKG_BUILD_DIR)/lib/XS $(1)/opt/lib/perl5/$(PERL_VERSION)/
+	$(CP) $(PKG_BUILD_DIR)/lib/auto/XS $(1)/opt/lib/perl5/$(PERL_VERSION)/auto
+	$(CP) $(PKG_BUILD_DIR)/lib/vmsish.pm $(1)/opt/lib/perl5/$(PERL_VERSION)/
 	$(CP) $(PKG_BUILD_DIR)/lib/vmsish.t $(1)/$(PERL_TESTSDIR)/lib
 	$(CP) $(PKG_BUILD_DIR)/lib/Internals.t $(1)/$(PERL_TESTSDIR)/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/unicore/TestProp.pl $(1)/usr/lib/perl5/$(PERL_VERSION)/unicore
+	$(CP) $(PKG_BUILD_DIR)/lib/unicore/TestProp.pl $(1)/opt/lib/perl5/$(PERL_VERSION)/unicore
 	$(CP) files/perl-run_tests.sh $(1)/$(PERL_TESTSDIR)/run_tests.sh
 	sed \
 		-e 's!%%PERL_DISABLEDTESTS%%!$(PERL_DISABLEDTESTS)!' \
diff -urx .svn ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/perlbase.mk ../../../openwrt_trunk/feeds/packages/lang/perl/perlbase.mk
--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/perlbase.mk	2015-09-24 20:31:27.218958283 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/perl/perlbase.mk	2015-09-27 09:35:12.778449431 +0300
@@ -42,10 +42,10 @@
 define Package/perlbase-archive/install
 $(call perlmod/Install,$(1),Archive,)
 $(call perlmod/InstallBaseTests,$(1),cpan/Archive-Tar/bin cpan/Archive-Tar/t)
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ptar $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ptardiff $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ptargrep $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/ptar $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/ptardiff $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/ptargrep $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,perlbase-archive))
@@ -1143,9 +1143,9 @@
 $(call perlmod/Install,$(1),Pod,Pod/Usage.pm)
 $(call perlmod/Install/NoStrip,$(1),Pod/Usage.pm,)
 $(call perlmod/InstallBaseTests,$(1),cpan/Pod-Checker/t cpan/Pod-Escapes/t cpan/Pod-Parser/lib cpan/Pod-Parser/scripts cpan/Pod-Parser/t cpan/Pod-Perldoc/t cpan/Pod-Simple/t cpan/Pod-Usage/scripts cpan/Pod-Usage/t cpan/podlators/t ext/Pod-Functions/Functions.pm ext/Pod-Functions/t ext/Pod-Html/t lib/Pod/t)
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/pod2man $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/pod2text $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/pod2man $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/pod2text $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,perlbase-pod))
@@ -1385,8 +1385,8 @@
 $(call perlmod/Install,$(1),Test Test.pm ok.pm,Test/Builder.pm Test/Harness/TAP.pod Test/More.pm Test/Tutorial.pod)
 $(call perlmod/Install/NoStrip,$(1),Test/Builder.pm Test/More.pm,)
 $(call perlmod/InstallBaseTests,$(1),cpan/Test-Harness/t cpan/Test-Simple/t cpan/Test/t)
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/prove $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/bin/prove $(1)/opt/bin
 endef
 
 $(eval $(call BuildPackage,perlbase-test))
diff -urx .svn ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/perlmod.mk ../../../openwrt_trunk/feeds/packages/lang/perl/perlmod.mk
--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/perl/perlmod.mk	2015-09-24 20:31:27.218958283 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/perl/perlmod.mk	2015-09-27 09:36:22.218449431 +0300
@@ -7,7 +7,7 @@
 HOST_PERL_PREFIX:=$(STAGING_DIR_HOST)/usr
 ifneq ($(CONFIG_USE_GLIBC),)
 	EXTRA_LIBS:=bsd
-	EXTRA_LIBDIRS:=$(STAGING_DIR)/lib
+	EXTRA_LIBDIRS:=$(STAGING_DIR)/opt/lib
 endif
 PERL_CMD:=$(STAGING_DIR_HOST)/usr/bin/perl$(PERL_VERSION).0
 
@@ -17,10 +17,10 @@
 endif
 
 # Module install prefix
-PERL_SITELIB:=/usr/lib/perl5/$(PERL_VERSION)
-PERL_TESTSDIR:=/usr/share/perl/perl-tests
-PERLBASE_TESTSDIR:=/usr/share/perl/perlbase-tests
-PERLMOD_TESTSDIR:=/usr/share/perl/perlmod-tests
+PERL_SITELIB:=/opt/lib/perl5/$(PERL_VERSION)
+PERL_TESTSDIR:=/opt/share/perl/perl-tests
+PERLBASE_TESTSDIR:=/opt/share/perl/perlbase-tests
+PERLMOD_TESTSDIR:=/opt/share/perl/perlmod-tests
 
 define perlmod/host/relink
 	rm -f $(1)/Makefile.aperl
@@ -57,18 +57,18 @@
 	(cd $(if $(3),$(3),$(PKG_BUILD_DIR)); \
 	PERL_MM_USE_DEFAULT=1 \
 	$(2) \
-	$(PERL_CMD) -MConfig -e '$$$${tied %Config::Config}{cpprun}="$(GNU_TARGET_NAME)-cpp -E"; do "Makefile.PL"' \
+	$(PERL_CMD) -MConfig -e '$$$${tied %Config::Config}{cpprun}="$(TARGET_CROSS)cpp -E"; do "Makefile.PL"' \
 		$(1) \
 		AR=ar \
-		CC=$(GNU_TARGET_NAME)-gcc \
+		CC=$(TARGET_CROSS)gcc \
 		CCFLAGS="$(MOD_CFLAGS_PERL)" \
 		CCCDLFLAGS=-fPIC \
 		CCDLFLAGS=-Wl,-E \
 		DLEXT=so \
 		DLSRC=dl_dlopen.xs \
 		EXE_EXT=" " \
-		FULL_AR=$(GNU_TARGET_NAME)-ar \
-		LD=$(GNU_TARGET_NAME)-gcc \
+		FULL_AR=$(TARGET_CROSS)ar \
+		LD=$(TARGET_CROSS)gcc \
 		LDDLFLAGS="-shared -rdynamic $(TARGET_LDFLAGS)"  \
 		LDFLAGS="$(EXTRA_LIBDIRS:%=-L%) $(EXTRA_LIBS:%=-l%) " \
 		LIBC=" " \
@@ -82,29 +82,29 @@
 		SO=so  \
 		VENDORARCHEXP=" " \
 		VENDORLIBEXP=" " \
-		SITEPREFIX=/usr \
+		SITEPREFIX=/opt \
 		INSTALLPRIVLIB=$(PERL_SITELIB) \
 		INSTALLSITELIB=$(PERL_SITELIB) \
 		INSTALLVENDORLIB=" " \
 		INSTALLARCHLIB=$(PERL_SITELIB) \
 		INSTALLSITEARCH=$(PERL_SITELIB) \
 		INSTALLVENDORARCH=" " \
-		INSTALLBIN=/usr/bin \
-		INSTALLSITEBIN=/usr/bin \
+		INSTALLBIN=/opt/bin \
+		INSTALLSITEBIN=/opt/bin \
 		INSTALLVENDORBIN=" " \
-		INSTALLSCRIPT=/usr/bin \
-		INSTALLSITESCRIPT=/usr/bin \
+		INSTALLSCRIPT=/opt/bin \
+		INSTALLSITESCRIPT=/opt/bin \
 		INSTALLVENDORSCRIPT=" " \
-		INSTALLMAN1DIR=/usr/man/man1 \
-		INSTALLSITEMAN1DIR=/usr/man/man1 \
+		INSTALLMAN1DIR=/opt/man/man1 \
+		INSTALLSITEMAN1DIR=/opt/man/man1 \
 		INSTALLVENDORMAN1DIR=" " \
-		INSTALLMAN3DIR=/usr/man/man3 \
-		INSTALLSITEMAN3DIR=/usr/man/man3 \
+		INSTALLMAN3DIR=/opt/man/man3 \
+		INSTALLSITEMAN3DIR=/opt/man/man3 \
 		INSTALLVENDORMAN3DIR=" " \
 		LINKTYPE=dynamic \
 		DESTDIR=$(PKG_INSTALL_DIR) \
 	);
-	sed 's!^PERL_INC = .*!PERL_INC = $(STAGING_DIR)/usr/lib/perl5/$(PERL_VERSION)/CORE/!' -i $(if $(3),$(3),$(PKG_BUILD_DIR))/Makefile
+	sed 's!^PERL_INC = .*!PERL_INC = $(STAGING_DIR)/opt/lib/perl5/$(PERL_VERSION)/CORE/!' -i $(if $(3),$(3),$(PKG_BUILD_DIR))/Makefile
 endef
 
 define perlmod/Compile
@@ -115,15 +115,28 @@
 		install
 endef
 
+
+# Dirty trick to check /Apps/opt or /opt ---- just temporary
+
 define perlmod/Install/NoStrip
 	$(INSTALL_DIR) $(strip $(1))$(PERL_SITELIB)
-	(cd $(PKG_INSTALL_DIR)$(PERL_SITELIB) && \
-	rsync --relative -rlHp --itemize-changes \
+	if [ -d $(PKG_INSTALL_DIR)/Apps ]; then \
+		(cd $(PKG_INSTALL_DIR)/Apps$(PERL_SITELIB) && \
+		rsync --relative -rlHp --itemize-changes \
+			--exclude=\*.pod \
+			--exclude=.packlist \
+			$(addprefix --exclude=/,$(strip $(3))) \
+			--prune-empty-dirs \
+			$(strip $(2)) $(strip $(1))$(PERL_SITELIB)) \
+	else \
+	    (cd $(PKG_INSTALL_DIR)$(PERL_SITELIB) && \
+	    rsync --relative -rlHp --itemize-changes \
 		--exclude=\*.pod \
 		--exclude=.packlist \
 		$(addprefix --exclude=/,$(strip $(3))) \
 		--prune-empty-dirs \
-		$(strip $(2)) $(strip $(1))$(PERL_SITELIB))
+		$(strip $(2)) $(strip $(1))$(PERL_SITELIB)) \
+	fi
 
 	chmod -R u+w $(strip $(1))$(PERL_SITELIB)
 endef
