--- ../../../openwrt_trunk.orig/feeds/packages/libs/libupnp/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/libupnp/Makefile
@@ -48,28 +48,28 @@
 endef
 
 define Package/libupnp-sample/description
-TVcontrolpoint & tvdevice sample applications run inside /etc/upnp-tvdevice/
+TVcontrolpoint & tvdevice sample applications run inside /opt/etc/upnp-tvdevice/
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/upnp $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{ixml,threadutil,upnp}.{a,so*,la} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libupnp.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/upnp $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{ixml,threadutil,upnp}.{a,so*,la} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/libupnp.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libupnp/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{ixml,threadutil,upnp}.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{ixml,threadutil,upnp}.so.* $(1)/opt/lib/
 endef
 
 define Package/libupnp-sample/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/upnp/sample/.libs/* $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/etc/upnp-tvdevice/web
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/upnp/sample/web/* $(1)/etc/upnp-tvdevice
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/upnp/sample/.libs/* $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/etc/upnp-tvdevice/web
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/upnp/sample/web/* $(1)/opt/etc/upnp-tvdevice
 endef
 
 $(eval $(call BuildPackage,libupnp))
