--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/utils/dialog/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/dialog/Makefile
@@ -8,19 +8,19 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dialog
-PKG_VERSION:=1.1-20120706
-PKG_RELEASE:=2
+PKG_VERSION:=1.2-20140112
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tgz
 PKG_SOURCE_URL:=ftp://invisible-island.net/dialog
-PKG_MD5SUM:=2e538305977178eb085a9859511c299d
+PKG_MD5SUM:=373dbe256bd3bcb37f5172aa77af9fa5
 
 include $(INCLUDE_DIR)/package.mk
 
 define Package/dialog
   SECTION:=utils
   CATEGORY:=Utilities
-  DEPENDS:=+libncurses
+  DEPENDS:=+libncursesw
   TITLE:=Dialog
   URL:=http://invisible-island.net/dialog/
 endef
@@ -29,9 +29,12 @@
  A script-interpreter which provides a set of curses widgets.
 endef
 
+CONFIGURE_ARGS+= \
+	--with-ncursesw \
+
 define Package/dialog/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dialog $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dialog $(1)/opt/bin
 endef
 
 $(eval $(call BuildPackage,dialog))
