--- ../../../downloads/openwrt_trunk-r40859/feeds/oldpackages/lang/eggdrop/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/eggdrop/Makefile
@@ -9,7 +9,7 @@
 
 PKG_NAME:=eggdrop
 PKG_VERSION:=1.6.17
-PKG_RELEASE:=1
+PKG_RELEASE:=1b
 
 PKG_SOURCE:=eggdrop$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=ftp://ftp.eggheads.org/pub/eggdrop/source/1.6
@@ -34,10 +34,10 @@
 endef
 
 CONFIGURE_ARGS+=\
-	--prefix="$(PKG_INSTALL_DIR)/eggdrop" \
-	--enable-threads \
-	--with-tcllib="$(STAGING_DIR)/usr/lib/libtcl8.5.so" \
-	--with-tclinc="$(STAGING_DIR)/usr/include/tcl.h"
+	--prefix="$(PKG_INSTALL_DIR)/opt/eggdrop" \
+	--disable-tcl-threads \
+	--with-tcllib="$(STAGING_DIR)/opt/lib/libtcl8.5.so" \
+	--with-tclinc="$(STAGING_DIR)/opt/include/tcl.h"
 
 define Build/Compile
 	$(MAKE) config -C $(PKG_BUILD_DIR)
@@ -46,12 +46,12 @@
 
 define Build/Install
 	$(call Build/Install/Default)
-	rm -Rvf $(PKG_INSTALL_DIR)/eggdrop/doc
+	rm -Rvf $(PKG_INSTALL_DIR)/opt/eggdrop/doc
 endef
 
 define Package/eggdrop/install
-	$(INSTALL_DIR) $(1)/eggdrop
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/eggdrop
+	$(INSTALL_DIR) $(1)/opt/eggdrop
+	$(CP) $(PKG_INSTALL_DIR)/opt/eggdrop/* $(1)/opt/eggdrop
 endef
 
 $(eval $(call BuildPackage,eggdrop))
