--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/libs/argp-standalone/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/argp-standalone/Makefile
@@ -37,12 +37,12 @@
 
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/include
 	$(CP)   $(PKG_BUILD_DIR)/argp.h \
-		$(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
+		$(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP)   $(PKG_BUILD_DIR)/libargp.a \
-		$(1)/usr/lib/
+		$(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,argp-standalone))
