--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/libs/libv4l/Makefile	2014-11-07 23:35:24.516548059 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/libv4l/Makefile	2014-11-26 17:05:14.218916634 +0300
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2009-2014 OpenWrt.org
+# Copyright (C) 2009-2012 OpenWrt.org
 # Copyright (C) 2009 David Cooper <dave@kupesoft.com>
 #
 # This is free software, licensed under the GNU General Public License v2.
@@ -9,23 +9,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=v4l-utils
-PKG_VERSION:=1.2.1
+PKG_VERSION:=0.8.6
 PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=http://www.linuxtv.org/downloads/v4l-utils
-PKG_MD5SUM:=4cc0fb4ded302ea9e89e5e1b56a7252b
-
-PKG_MAINTAINER:=Ted Hess <thess@kitschensync.net>
-
-PKG_LICENSE:=GPL-2.0 LGPL-2.1
-PKG_LICENSE_FILE:=COPYING COPYING.libv4l
+PKG_MD5SUM:=80062780ae90e5955473b09b31e9828a
 
 PKG_BUILD_DEPENDS:=argp-standalone
 
 include $(INCLUDE_DIR)/uclibc++.mk
 include $(INCLUDE_DIR)/package.mk
-include $(INCLUDE_DIR)/nls.mk
 
 define Package/libv4l/Default
   TITLE:=Video 4 Linux
@@ -52,7 +46,7 @@
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE+= wrapper libraries
-  DEPENDS := +libpthread +librt $(ICONV_DEPENDS)
+  DEPENDS := +libpthread +librt
 endef
 
 define Package/libv4l/description
@@ -64,7 +58,7 @@
   SECTION:=utils
   CATEGORY:=Utilities
   TITLE+= utilities
-  DEPENDS := +libv4l $(CXX_DEPENDS) $(ICONV_DEPENDS)
+  DEPENDS := +libv4l $(CXX_DEPENDS)
 endef
 
 define Package/v4l-utils/description
@@ -73,48 +67,46 @@
 endef
 
 TARGET_CFLAGS += $(FPIC)
-TARGET_LDFLAGS += -largp
-
-CONFIGURE_ARGS+= \
-	--disable-libdvbv5 \
-	--disable-qv4l2 \
-	--without-jpeg \
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
-		DESTDIR="$(PKG_INSTALL_DIR)" \
+		DESTDIR="$(PKG_INSTALL_DIR)" PREFIX="/Apps/opt" \
+		DISABLE_LIBJPEG="1" \
+		$(CONFIGURE_VARS) \
 		all install
 	$(MAKE) -C $(PKG_BUILD_DIR) \
-		DESTDIR="$(PKG_INSTALL_DIR)" \
+		DESTDIR="$(PKG_INSTALL_DIR)" PREFIX="/Apps/opt" \
+		DISABLE_LIBJPEG="1" \
+		$(CONFIGURE_VARS) \
 		LINKTYPE="static" \
 		all install
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/*.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libv4l{1,2,convert}.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libv4l{1,2,convert}.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/include/*.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libv4l{1,2,convert}.{a,so*} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/pkgconfig/libv4l{1,2,convert}.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libv4l/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libv4l{1,2,convert}.so.* $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/libv4l
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libv4l/v4l{1compat,2convert}.so $(1)/usr/lib/libv4l/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libv4l{1,2,convert}.so.* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/libv4l
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libv4l/v4l{1compat,2convert}.so $(1)/opt/lib/libv4l/
 endef
 
 define Package/v4l-utils/install
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/rc_maps.cfg $(1)/etc/
-	$(CP) $(PKG_INSTALL_DIR)/etc/rc_keymaps $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/{cx18,ivtv}-ctl $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/decode_tm6000 $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ir-keytable $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/v4l2-{compliance,ctl,sysfs-path} $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_INSTALL_DIR)/etc/rc_maps.cfg $(1)/opt/etc/
+	$(CP) $(PKG_INSTALL_DIR)/etc/rc_keymaps $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/{cx18,ivtv}-ctl $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/decode_tm6000 $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/ir-keytable $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/v4l2-{compliance,ctl,sysfs-path} $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,libv4l))
