--- ../../../openwrt_trunk.orig/feeds/oldpackages/sound/streamripper/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/sound/streamripper/Makefile
@@ -24,7 +24,7 @@
 define Package/streamripper
   SECTION:=multimedia
   CATEGORY:=Multimedia
-  DEPENDS:=+libmad +libfaad2 +libpthread
+  DEPENDS:=+libmad +libfaad2 +libpthread +librt
   TITLE:=streamripper
   URL:=http://streamripper.sourceforge.net/
 endef
@@ -45,11 +45,11 @@
 	--with-gnu-ld \
 
 define Package/streamripper/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/streamripper $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/agrep $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libtre.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/streamripper $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/agrep $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libtre.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,streamripper))
