--- ../../../downloads/openwrt_trunk-r45513/feeds/packages/libs/libsoxr/Makefile	2015-04-23 10:21:55.509457455 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/libsoxr/Makefile	2015-05-22 15:11:54.044515280 +0300
@@ -47,15 +47,15 @@
 CMAKE_OPTIONS+= -DHAVE_WORDS_BIGENDIAN_EXITCODE=$(if $(CONFIG_BIG_ENDIAN),0,1)
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/src/soxr.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/src/*.a $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_BUILD_DIR)/src/soxr.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/src/*.a $(1)/opt/lib/
 endef
 
 define Package/libsoxr/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/src/*.a $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/src/*.a $(1)/opt/lib/
 endef
 
 
