--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/multimedia/imagemagick/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/multimedia/imagemagick/Makefile
@@ -96,7 +96,9 @@
 	--without-xml \
 	--without-x \
 	--without-magick-plus-plus \
-	--without-lzma
+	--without-lzma \
+	--without-lcms2 \
+	--without-jbig \
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
@@ -105,47 +107,47 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/magick
-	$(INSTALL_DIR) $(1)/usr/include/wand
+	$(INSTALL_DIR) $(1)/opt/include/magick
+	$(INSTALL_DIR) $(1)/opt/include/wand
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/* \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/* \
+		$(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/*.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/*.so* \
+		$(1)/opt/lib/
 
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig/
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/* \
-		$(1)/usr/lib/pkgconfig/
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/* \
+		$(1)/opt/lib/pkgconfig/
 endef
 
 define Package/imagemagick/install
-	$(INSTALL_DIR) $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/*.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/*.so* \
+		$(1)/opt/lib/
 endef
 
 define Package/imagemagick-tools/install
-	$(INSTALL_DIR) $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/mogrify \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/mogrify \
+		$(1)/opt/bin/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/bin/convert \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/convert \
+		$(1)/opt/bin/
 endef
 
 define BuildPlugin
   define Package/imagemagick-$(1)/install
-	$(INSTALL_DIR) $$(1)/usr/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders/$(1).so \
-		$$(1)/usr/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders/$(1).la \
-		$$(1)/usr/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders
+	$(INSTALL_DIR) $$(1)/opt/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders/$(1).so \
+		$$(1)/opt/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders/$(1).la \
+		$$(1)/opt/lib/ImageMagick-$(PKG_VERSION)/modules-Q16/coders
   endef
 
   $$(eval $$(call BuildPackage,imagemagick-$(1)))
