--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/libdnet/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libdnet/Makefile
@@ -94,7 +94,7 @@
 	$(if $(CONFIG_PACKAGE_pydnet), \
 		$(call Build/Compile/PyMod, \
 			python/, \
-			install --no-compile --prefix $(PKG_INSTALL_DIR)/usr \
+			install --no-compile --prefix $(PKG_INSTALL_DIR)/opt \
 		) \
 	)
 endef
@@ -102,41 +102,41 @@
 define Build/InstallDev
 	$(INSTALL_DIR) $(2)/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/dnet-config \
+		$(PKG_INSTALL_DIR)/opt/bin/dnet-config \
 		$(2)/bin/
 	$(SED) \
-		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/usr,g' \
+		's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/opt,g' \
 		$(2)/bin/dnet-config
 
-	$(INSTALL_DIR) $(1)/usr/include
+	$(INSTALL_DIR) $(1)/opt/include
 	$(INSTALL_DATA)\
-		$(PKG_INSTALL_DIR)/usr/include/dnet.h \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/dnet.h \
+		$(1)/opt/include/
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/dnet \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/dnet \
+		$(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libdnet.{la,a,so*} \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libdnet.{la,a,so*} \
+		$(1)/opt/lib/
 endef
 
 define Package/libdnet/install
-	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libdnet.so.* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libdnet.so.* \
+		$(1)/opt/lib/
 
-	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/sbin/dnet \
-		$(1)/usr/sbin/
+		$(PKG_INSTALL_DIR)/opt/sbin/dnet \
+		$(1)/opt/sbin/
 endef
 
 define Package/pydnet/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/python* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/python* $(1)/opt/lib/
 endef
 
 
