--- ../../../openwrt_trunk/package/libs/libconfig/Makefile.orig	2013-08-09 12:56:05.000000000 +0400
+++ ../../../openwrt_trunk/package/libs/libconfig/Makefile	2013-08-14 15:39:00.314747429 +0400
@@ -51,8 +51,8 @@
 endef
 
 define Package/libconfig/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libconfig.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libconfig.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libconfig))
