--- ../../../downloads/toolchain-src-git/scripts/download.pl
+++ ../../../toolchain-src-git/scripts/download.pl
@@ -168,6 +168,10 @@
 	}
 }
 
+push @mirrors, 'http://x.vm0.ru/wl500g-repo/sources-mirror';
+push @mirrors, 'http://entware.wl500g.info/sources';
+push @mirrors, 'http://entware.dyndns.info/sources';
+
 while (!$ok) {
 	my $mirror = shift @mirrors;
 	$mirror or die "No more mirrors to try - giving up.\n";
