diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/files/mini_httpd.conf ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/files/mini_httpd.conf
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/files/mini_httpd.conf	2012-05-02 16:17:23.890419757 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/files/mini_httpd.conf	2012-05-02 16:22:04.553626438 +0400
@@ -1,5 +1,5 @@
 cgipat=cgi-bin/**|**.sh
-dir=/www
+dir=/opt/share/www
 nochroot
-pidfile=/var/run/mini_httpd.pid
+pidfile=/opt/var/run/mini_httpd.pid
 user=root
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/files/mini_httpd-ssl.conf ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/files/mini_httpd-ssl.conf
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/files/mini_httpd-ssl.conf	2012-05-02 16:17:23.890419757 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/files/mini_httpd-ssl.conf	2012-05-02 16:22:04.553626438 +0400
@@ -1,7 +1,7 @@
 cgipat=cgi-bin/**|**.sh
-dir=/www
+dir=/opt/share/www
 nochroot
-pidfile=/var/run/mini_httpd.pid
+pidfile=/opt/var/run/mini_httpd.pid
 user=root
 ssl
-certfile=/etc/mini_httpd.pem
+certfile=/opt/etc/mini_httpd.pem
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/Makefile ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/Makefile
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/Makefile	2012-05-02 16:17:23.886419808 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/Makefile	2012-05-31 17:25:01.582780509 +0400
@@ -28,8 +28,8 @@
 endef
 
 define Package/mini-httpd/Default/conffiles
-/etc/mini_httpd.conf
-/etc/default/mini_httpd
+/opt/etc/mini_httpd.conf
+/opt/etc/default/mini_httpd
 endef
 
 define Package/mini-httpd/Default/description
@@ -70,7 +70,7 @@
 
 define Package/mini-httpd-matrixssl/conffiles
 $(call Package/mini-httpd/Default/conffiles)
-/etc/mini_httpd.pem
+/opt/etc/mini_httpd.pem
 endef
 
 define Package/mini-httpd-matrixssl/description
@@ -86,7 +86,7 @@
 
 define Package/mini-httpd-openssl/conffiles
 $(call Package/mini-httpd/Default/conffiles)
-/etc/mini_httpd.pem
+/opt/etc/mini_httpd.pem
 endef
 
 define Package/mini-httpd-openssl/description
@@ -112,6 +112,7 @@
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		$(TARGET_CONFIGURE_OPTS) \
 		OFLAGS="$(TARGET_CFLAGS)" \
+		LDFLAGS="$(TARGET_LDFLAGS)" \
 		$(3) \
 		all
 	( cd $(PKG_BUILD_DIR); mv -f mini_httpd mini_httpd-$(2) )
@@ -120,19 +121,17 @@
 $(STAMP_BUILT): $(STAMP_BUILT)-$(2)
 
 define Package/$(1)/install
-	$(INSTALL_DIR) $$(1)/etc
+	$(INSTALL_DIR) $$(1)/opt/etc
 ifeq ($(2),without-ssl)
-	$(INSTALL_DATA) ./files/mini_httpd.conf $$(1)/etc/mini_httpd.conf
+	$(INSTALL_DATA) ./files/mini_httpd.conf $$(1)/opt/etc/mini_httpd.conf
 else
-	$(INSTALL_DATA) ./files/mini_httpd-ssl.conf $$(1)/etc/mini_httpd.conf
-	$(INSTALL_CONF) ./files/mini_httpd.pem $$(1)/etc/mini_httpd.pem
+	$(INSTALL_DATA) ./files/mini_httpd-ssl.conf $$(1)/opt/etc/mini_httpd.conf
+	$(INSTALL_CONF) ./files/mini_httpd.pem $$(1)/opt/etc/mini_httpd.pem
 endif
-	$(INSTALL_DIR) $$(1)/etc/default
-	$(INSTALL_DATA) ./files/mini_httpd.default $$(1)/etc/default/mini_httpd
-	$(INSTALL_DIR) $$(1)/etc/init.d
-	$(INSTALL_BIN) ./files/mini_httpd.init $$(1)/etc/init.d/mini_httpd
-	$(INSTALL_DIR) $$(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mini_httpd-$(2) $$(1)/usr/sbin/mini_httpd
+	$(INSTALL_DIR) $$(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S80mini_httpd $$(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $$(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mini_httpd-$(2) $$(1)/opt/sbin/mini_httpd
 endef
 
 endef
@@ -179,8 +178,8 @@
 $(eval $(Build/with-openssl))
 
 define Package/mini-httpd-htpasswd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/htpasswd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/htpasswd $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,mini-httpd))
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/patches/001-cgi_path.patch ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/patches/001-cgi_path.patch
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/patches/001-cgi_path.patch	2012-05-02 16:17:23.886419808 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/patches/001-cgi_path.patch	2012-05-02 16:22:04.557628498 +0400
@@ -6,11 +6,11 @@
  #endif /* CGI_NICE */
  #ifndef CGI_PATH
 -#define CGI_PATH "/usr/local/bin:/usr/ucb:/bin:/usr/bin"
-+#define CGI_PATH "/usr/bin:/bin:/usr/sbin:/sbin"
++#define CGI_PATH "/opt/bin:/bin:/usr/bin:/opt/sbin:/sbin:/usr/sbin"
  #endif /* CGI_PATH */
  #ifndef CGI_LD_LIBRARY_PATH
 -#define CGI_LD_LIBRARY_PATH "/usr/local/lib:/usr/lib"
-+#define CGI_LD_LIBRARY_PATH "/usr/lib:/lib"
++#define CGI_LD_LIBRARY_PATH "/opt/lib:/lib"
  #endif /* CGI_LD_LIBRARY_PATH */
  #ifndef AUTH_FILE
  #define AUTH_FILE ".htpasswd"
diff -urx .svn ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/patches/500-matrixssl.patch ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/patches/500-matrixssl.patch
--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/mini_httpd/patches/500-matrixssl.patch	2012-05-02 16:17:23.886419808 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mini_httpd/patches/500-matrixssl.patch	2012-05-02 16:22:04.561630501 +0400
@@ -25,8 +25,8 @@
 -MANDIR =	/usr/local/man
 +DESTDIR =
 +
-+BINDIR =	$(DESTDIR)/usr/sbin
-+MANDIR =	$(DESTDIR)/usr/share/man
++BINDIR =	$(DESTDIR)/opt/sbin
++MANDIR =	$(DESTDIR)/opt/share/man
  CC =		gcc
  CDEFS =		${SSL_DEFS} ${SSL_INC}
 -CFLAGS =	-O ${CDEFS}
