--- ../../../downloads/openwrt_trunk-r35474/feeds/packages/multimedia/motion/Makefile	2013-02-05 10:42:11.470946402 +0400
+++ ../../../openwrt_trunk/feeds/packages/multimedia/motion/Makefile	2013-02-06 17:04:05.360956990 +0400
@@ -23,13 +23,13 @@
 define Package/motion
   SECTION:=multimedia
   CATEGORY:=Multimedia
-  DEPENDS:=+libjpeg +libpthread
+  DEPENDS:=+libjpeg +libpthread
   TITLE:=webcam motion sensing and logging
   URL:=http://www.lavrsen.dk/twiki/bin/view/Motion/WebHome
 endef
 
 define Package/motion/conffiles
-/etc/motion.conf
+/opt/etc/motion.conf
 endef
 
 CONFIGURE_ARGS+= \
@@ -42,11 +42,11 @@
 	--without-sqlite3 \
 
 define Package/motion/install
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/motion-dist.conf $(1)/etc/motion.conf
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/motion $(1)/usr/bin/
-
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/motion-dist.conf $(1)/opt/etc/motion.conf
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/motion $(1)/opt/bin/
+	$(INSTALL_BIN) ./files/S99motion  $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,motion))
