--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/multimedia/motion/Makefile
+++ ../../../openwrt_trunk/feeds/packages/multimedia/motion/Makefile
@@ -30,7 +30,7 @@
 endef
 
 define Package/motion/conffiles
-/etc/motion.conf
+/opt/etc/motion.conf
 endef
 
 CONFIGURE_ARGS+= \
@@ -43,11 +43,11 @@
 	--without-sqlite3 \
 
 define Package/motion/install
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/motion-dist.conf $(1)/etc/motion.conf
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/motion $(1)/usr/bin/
-
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/motion-dist.conf $(1)/opt/etc/motion.conf
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/motion $(1)/opt/bin/
+	$(INSTALL_BIN) ./files/S99motion  $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,motion))
