--- ../../../downloads/openwrt_trunk-r42803/feeds/oldpackages/libs/leptonica/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/leptonica/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=leptonica
-PKG_VERSION:=1.68
+PKG_VERSION:=1.71
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://leptonica.googlecode.com/files/
-PKG_MD5SUM:=5cd7092f9ff2ca7e3f3e73bfcd556403
+PKG_SOURCE_URL:=http://leptonica.org/source
+PKG_MD5SUM:=790f34d682e6150c12c54bfe4a824f7f
 
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
@@ -29,18 +29,17 @@
 endef
 
 TARGET_CFLAGS += $(FPIC)
-TARGET_LDFLAGS += -Wl,-rpath-link=$(STAGING_DIR)/usr/lib
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/leptonica $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblept.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/leptonica $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/liblept.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libleptonica/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblept.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/liblept.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libleptonica))
