--- ../../../downloads/openwrt_trunk-r35474/feeds/packages/multimedia/tvheadend/Makefile	2013-02-05 10:42:11.474946402 +0400
+++ ../../../openwrt_trunk/feeds/packages/multimedia/tvheadend/Makefile	2013-02-13 22:13:49.369599054 +0400
@@ -8,12 +8,29 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tvheadend
-PKG_VERSION:=3.2
-PKG_RELEASE:=0.1
+#PKG_VERSION:=3.2
+#PKG_RELEASE:=0.1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://github.com/downloads/tvheadend/tvheadend/
-PKG_MD5SUM:=4f7faa75c4ea24942c5a12a77ebdcdf0
+#PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+#PKG_SOURCE_URL:=https://github.com/downloads/tvheadend/tvheadend/
+#PKG_MD5SUM:=4f7faa75c4ea24942c5a12a77ebdcdf0
+
+PKG_VERSION:=2.12.99
+PKG_RELEASE:=2
+
+PKG_REV:=d66746ca56b8937d2cf87ca60e51956647b731a5
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/tvheadend/tvheadend.git
+PKG_SOURCE_VERSION:=$(PKG_REV)
+
+EXTRA_CPPFLAGS += \
+	-std=gnu99
+
+CONFIGURE_VARS += \
+	PKG_CONFIG_LIBDIR="$(STAGING_DIR)/opt/lib/pkgconfig" \
+	PKG_CONFIG_PATH="$(STAGING_DIR)/opt/lib/pkgconfig" \
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -65,14 +82,12 @@
 	--enable-bundle
 
 define Package/tvheadend/install
-	$(INSTALL_DIR) $(1)/etc/tvheadend
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/tvheadend.init $(1)/etc/init.d/tvheadend
-	$(INSTALL_DIR) $(1)/lib/upgrade/keep.d
-	$(INSTALL_DATA) ./files/tvheadend.upgrade $(1)/lib/upgrade/keep.d/tvheadend
-	
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/build.linux/tvheadend $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/{init.d,tvheadend}
+	$(INSTALL_BIN) ./files/S71tvheadend $(1)/opt/etc/init.d/
+	$(INSTALL_CONF) ./files/tv_grab_wl500g_info.cfg $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/build.Linux/tvheadend $(1)/opt/bin/
+	$(INSTALL_BIN) ./files/tv_grab_wl500g_info $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,tvheadend))
