--- ./configure.in.orig	2013-10-16 13:45:14.683775769 +0400
+++ ./configure.in	2013-10-16 13:46:52.927777332 +0400
@@ -702,17 +702,7 @@
 AC_CHECK_FUNCS([daq_hup_apply] [daq_acquire_with_meta])
 
 AC_MSG_CHECKING([for daq address space ID])
-AC_RUN_IFELSE(
-[AC_LANG_PROGRAM(
-[[
-#include <daq.h>
-]],
-[[
-   DAQ_PktHdr_t hdr;
-   hdr.address_space_id = 0;
-]])],
-[have_daq_address_space_id="yes"],
-[have_daq_address_space_id="no"])
+have_daq_address_space_id="no"
 AC_MSG_RESULT($have_daq_address_space_id)
 if test "x$have_daq_address_space_id" = "xyes"; then
     AC_DEFINE([HAVE_DAQ_ADDRESS_SPACE_ID],[1],
