--- ../../../downloads/openwrt_trunk-r44497/feeds/packages/net/snort/Makefile	2015-02-20 12:39:13.429134248 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/snort/Makefile	2015-02-21 17:12:18.918157686 +0300
@@ -26,7 +26,7 @@
   SUBMENU:=Firewall
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libdaq +libdnet +libopenssl +libpcap +libpcre +libpthread +libuuid +zlib
+  DEPENDS:=+libdaq +libdnet +libopenssl +libpcap +libpcre +libpthread +libuuid +zlib +liblzma
   TITLE:=Lightweight Network Intrusion Detection System
   URL:=http://www.snort.org/
 endef
@@ -73,22 +73,22 @@
 endef
 
 CONFIGURE_ARGS += \
-	--prefix="/usr" \
+	--prefix="/opt" \
 	--enable-flexresp \
-	--with-dnet-includes="$(STAGING_DIR)/usr/include" \
-	--with-dnet-libraries="$(STAGING_DIR)/usr/lib" \
-	--with-libpcap-includes="$(STAGING_DIR)/usr/include" \
-	--with-libpcap-libraries="$(STAGING_DIR)/usr/lib" \
-	--with-libpcre-includes="$(STAGING_DIR)/usr/include" \
-	--with-libpcre-libraries="$(STAGING_DIR)/usr/lib" \
-	--with-daq-includes="$(STAGING_DIR)/usr/include" \
-	--with-daq-libraries="$(STAGING_DIR)/usr/lib" \
+	--with-dnet-includes="$(STAGING_DIR)/opt/include" \
+	--with-dnet-libraries="$(STAGING_DIR)/opt/lib" \
+	--with-libpcap-includes="$(STAGING_DIR)/opt/include" \
+	--with-libpcap-libraries="$(STAGING_DIR)/opt/lib" \
+	--with-libpcre-includes="$(STAGING_DIR)/opt/include" \
+	--with-libpcre-libraries="$(STAGING_DIR)/opt/lib" \
+	--with-daq-includes="$(STAGING_DIR)/opt/include" \
+	--with-daq-libraries="$(STAGING_DIR)/opt/lib" \
 	--disable-static-daq
 
 CONFIGURE_VARS += \
-	CPPFLAGS="$$$$CPPFLAGS -I$(STAGING_DIR)/usr/include/mysql" \
-	LDFLAGS="$$$$LDFLAGS -L$(STAGING_DIR)/usr/lib/mysql" \
-	PATH="$(STAGING_DIR)/usr/lib/libnet-1.0.x/bin:$$$$PATH"
+	CPPFLAGS="$$$$CPPFLAGS -I$(STAGING_DIR)/opt/include/mysql" \
+	LDFLAGS="$$$$LDFLAGS -L$(STAGING_DIR)/opt/lib/mysql" \
+	PATH="$(STAGING_DIR)/opt/lib/libnet-1.0.x/bin:$$$$PATH"
 
 MAKE_FLAGS += \
 	extra_incl=""
@@ -102,37 +102,37 @@
   CONFIGURE_VARS += \
 	mysql_has_reconnect=yes
   CONFIGURE_ARGS += \
-	--with-mysql="$(STAGING_DIR)/usr" \
+	--with-mysql="$(STAGING_DIR)/opt" \
 	--without-postgresql
 endif
 ifeq ($(BUILD_VARIANT),pgsql)
   CONFIGURE_ARGS += \
 	--without-mysql \
-	--with-postgresql="$(STAGING_DIR)/usr"
+	--with-postgresql="$(STAGING_DIR)/opt"
 endif
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(STAGING_DIR)/usr/include/snort/dynamic_preproc
+	$(INSTALL_DIR) $(STAGING_DIR)/opt/include/snort/dynamic_preproc
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/snort/dynamic_preproc/* \
-		$(STAGING_DIR)/usr/include/snort/dynamic_preproc/
-	$(INSTALL_DIR) $(STAGING_DIR)/usr/lib/snort/dynamic_preproc
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/snort/dynamic_preproc/* \
-		$(STAGING_DIR)/usr/lib/snort/dynamic_preproc/
-	$(INSTALL_DIR) $(STAGING_DIR)/usr/lib/snort_dynamicengine
-	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/snort_dynamicengine/* \
-		$(STAGING_DIR)/usr/lib/snort_dynamicengine/
-	$(INSTALL_DIR) $(STAGING_DIR)/usr/lib/snort_dynamicpreprocessor
+		$(PKG_INSTALL_DIR)/opt/include/snort/dynamic_preproc/* \
+		$(STAGING_DIR)/opt/include/snort/dynamic_preproc/
+	$(INSTALL_DIR) $(STAGING_DIR)/opt/lib/snort/dynamic_preproc
+	$(CP) \
+		$(PKG_INSTALL_DIR)/opt/lib/snort/dynamic_preproc/* \
+		$(STAGING_DIR)/opt/lib/snort/dynamic_preproc/
+	$(INSTALL_DIR) $(STAGING_DIR)/opt/lib/snort_dynamicengine
+	$(CP) \
+		$(PKG_INSTALL_DIR)/opt/lib/snort_dynamicengine/* \
+		$(STAGING_DIR)/opt/lib/snort_dynamicengine/
+	$(INSTALL_DIR) $(STAGING_DIR)/opt/lib/snort_dynamicpreprocessor
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/snort_dynamicpreprocessor/* \
-		$(STAGING_DIR)/usr/lib/snort_dynamicpreprocessor/
+		$(PKG_INSTALL_DIR)/opt/lib/snort_dynamicpreprocessor/* \
+		$(STAGING_DIR)/opt/lib/snort_dynamicpreprocessor/
 endef
 
 define Package/snort/conffiles
-/etc/config/snort
-/etc/snort/snort.conf
+/opt/etc/config/snort
+/opt/etc/snort/snort.conf
 endef
 
 Package/snort-mysql/conffiles = $(Package/snort/conffiles)
@@ -142,59 +142,49 @@
 Package/snort-pgsql/install = $(Package/snort/install)
 
 define Package/snort/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/snort  \
-		$(1)/usr/bin/snort
+		$(PKG_INSTALL_DIR)/opt/bin/snort  \
+		$(1)/opt/bin/snort
 
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/u2{boat,spewfoo} \
-		$(1)/usr/bin
+		$(PKG_INSTALL_DIR)/opt/bin/u2{boat,spewfoo} \
+		$(1)/opt/bin
 
-	$(INSTALL_DIR) $(1)/usr/lib/snort_dynamicengine
+	$(INSTALL_DIR) $(1)/opt/lib/snort_dynamicengine
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/snort_dynamicengine/libsf_engine.so* \
-		$(1)/usr/lib/snort_dynamicengine/
+		$(PKG_INSTALL_DIR)/opt/lib/snort_dynamicengine/libsf_engine.so* \
+		$(1)/opt/lib/snort_dynamicengine/
 
-	$(INSTALL_DIR) $(1)/usr/lib/snort_dynamicpreprocessor
+	$(INSTALL_DIR) $(1)/opt/lib/snort_dynamicpreprocessor
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/snort_dynamicpreprocessor/libsf*.so* \
-		$(1)/usr/lib/snort_dynamicpreprocessor/
+		$(PKG_INSTALL_DIR)/opt/lib/snort_dynamicpreprocessor/libsf*.so* \
+		$(1)/opt/lib/snort_dynamicpreprocessor/
 
-	$(INSTALL_DIR) $(1)/etc/snort
+	$(INSTALL_DIR) $(1)/opt/etc/snort
 	$(INSTALL_CONF) \
 		$(PKG_BUILD_DIR)/etc/snort.conf \
-		$(1)/etc/snort/
+		$(1)/opt/etc/snort/
 	$(INSTALL_CONF) \
 		$(PKG_BUILD_DIR)/etc/attribute_table.dtd \
-		$(1)/etc/snort/
+		$(1)/opt/etc/snort/
 	$(INSTALL_CONF) \
 		$(PKG_BUILD_DIR)/etc/classification.config \
-		$(1)/etc/snort/
+		$(1)/opt/etc/snort/
 	$(INSTALL_CONF) \
 		$(PKG_BUILD_DIR)/etc/gen-msg.map \
-		$(1)/etc/snort/
+		$(1)/opt/etc/snort/
 	$(INSTALL_CONF) \
 			$(PKG_BUILD_DIR)/etc/reference.config \
-			$(1)/etc/snort/
+			$(1)/opt/etc/snort/
 	$(INSTALL_CONF) \
 		$(PKG_BUILD_DIR)/etc/unicode.map \
-		$(1)/etc/snort/
+		$(1)/opt/etc/snort/
 
-	$(INSTALL_DIR) $(1)/etc/snort/preproc_rules
+	$(INSTALL_DIR) $(1)/opt/etc/snort/preproc_rules
 	$(INSTALL_CONF) \
 		$(PKG_BUILD_DIR)/preproc_rules/*.rules \
-		$(1)/etc/snort/preproc_rules/
-
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) \
-		./files/snort.init \
-		$(1)/etc/init.d/snort
-
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) \
-		./files/snort.config \
-		$(1)/etc/config/snort
+		$(1)/opt/etc/snort/preproc_rules/
 endef
 
 $(eval $(call BuildPackage,snort))
