--- ../../../openwrt_trunk.orig/package/ncurses/Makefile	2012-06-01 10:50:57.147997037 +0400
+++ ../../../openwrt_trunk/package/ncurses/Makefile	2012-06-01 10:51:34.491527565 +0400
@@ -61,12 +61,12 @@
 	--disable-home-terminfo \
 	--with-normal \
 	--with-shared \
-	--with-terminfo-dirs=/usr/share/terminfo \
-	--with-default-terminfo-dir=/usr/share/terminfo
+	--with-terminfo-dirs=/opt/share/terminfo \
+	--with-default-terminfo-dir=/opt/share/terminfo
 
 ifeq ($(HOST_OS),FreeBSD)
 	CONFIGURE_ARGS +=
-		--with-terminfo=/usr/share/terminfo.db 
+		--with-terminfo=/opt/share/terminfo.db 
 endif
 
 ifeq ($(BUILD_VARIANT),libncursesw)
@@ -91,58 +91,58 @@
 define Package/terminfo/install
 	echo ""
 ifneq ($(HOST_OS),FreeBSD)
-	$(INSTALL_DIR) $(1)/usr/share/terminfo
-	(cd $(PKG_INSTALL_DIR)/usr/share/terminfo; \
+	$(INSTALL_DIR) $(1)/opt/share/terminfo
+	(cd $(PKG_INSTALL_DIR)/opt/share/terminfo; \
 		for dir in ??; do \
 			[ -d "$$$$dir" ] || continue; \
 			mv $$$$dir $$$$(echo -ne "\x$$$$dir"); \
 		done \
 	)
 	for file in a/ansi d/dumb l/linux r/rxvt r/rxvt-unicode s/screen v/vt100 v/vt102 x/xterm x/xterm-color; do \
-		$(INSTALL_DIR) $(1)/usr/share/terminfo/`dirname $$$$file`; \
-		$(CP) $(PKG_INSTALL_DIR)/usr/share/terminfo/$$$$file \
-			$(1)/usr/share/terminfo/$$$$file; \
+		$(INSTALL_DIR) $(1)/opt/share/terminfo/`dirname $$$$file`; \
+		$(CP) $(PKG_INSTALL_DIR)/opt/share/terminfo/$$$$file \
+			$(1)/opt/share/terminfo/$$$$file; \
 	done
 endif
 endef
 
 define Package/libncurses/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{curses,ncurses,panel,menu,form}.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{curses,ncurses,panel,menu,form}.so* $(1)/opt/lib/
 endef
 
 define Package/libncursesw/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{ncurses,panel,menu,form}w.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{ncurses,panel,menu,form}w.so* $(1)/opt/lib/
 endef
 
 ifeq ($(BUILD_VARIANT),libncursesw)
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/ncursesw/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/*.h $(1)/usr/include/ncursesw/
+	$(INSTALL_DIR) $(1)/opt/include/ncursesw/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/*.h $(1)/opt/include/ncursesw/
 
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{ncurses,panel,menu,form}w.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{ncurses,panel,menu,form}w.{a,so*} $(1)/opt/lib/
 
-	$(INSTALL_DIR) $(1)/usr/bin $(2)/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ncursesw5-config $(2)/bin/
-	$(SED) 's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/usr,g' -e 's/$$$$INCS //g' \
+	$(INSTALL_DIR) $(1)/opt/bin $(2)/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/ncursesw5-config $(2)/bin/
+	$(SED) 's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/opt,g' -e 's/$$$$INCS //g' \
 		$(2)/bin/ncursesw5-config
-	ln -sf $(STAGING_DIR)/host/bin/ncursesw5-config $(1)/usr/bin/ncursesw5-config
+	ln -sf $(STAGING_DIR)/host/bin/ncursesw5-config $(1)/opt/bin/ncursesw5-config
 endef
 else
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/*.h $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/*.h $(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib{curses,ncurses,panel,menu,form}.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib{curses,ncurses,panel,menu,form}.{a,so*} $(1)/opt/lib/
 
-	$(INSTALL_DIR) $(1)/usr/bin $(2)/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ncurses5-config $(2)/bin/
-	$(SED) 's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/usr,g' \
+	$(INSTALL_DIR) $(1)/opt/bin $(2)/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/ncurses5-config $(2)/bin/
+	$(SED) 's,^\(prefix\|exec_prefix\)=.*,\1=$(STAGING_DIR)/opt,g' \
 		$(2)/bin/ncurses5-config
-	ln -sf $(STAGING_DIR)/host/bin/ncurses5-config $(1)/usr/bin/ncurses5-config
+	ln -sf $(STAGING_DIR)/host/bin/ncurses5-config $(1)/opt/bin/ncurses5-config
 endef
 endif
 
