--- ../../../trunk/package/ncurses/Makefile.release5	2012-07-03 18:36:34.000000000 +0400
+++ ../../../trunk/package/ncurses/Makefile	2012-07-03 23:26:04.000000000 +0400
@@ -9,7 +9,7 @@
 
 PKG_NAME:=ncurses
 PKG_VERSION:=5.7
-PKG_RELEASE:=5
+PKG_RELEASE:=6
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
@@ -46,6 +46,16 @@
   VARIANT:=libncursesw
 endef
 
+define Package/ncurses-bin
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=Terminal
+  DEPENDS:=+libncurses
+  TITLE:=Terminal-related programs
+  URL:=http://www.gnu.org/software/ncurses/
+  VARIANT:=libncurses
+endef
+
 TARGET_CFLAGS += $(FPIC)
 
 CONFIGURE_ARGS += \
@@ -56,7 +66,6 @@
 	--without-ada \
 	--without-debug \
 	--without-profile \
-	--without-progs \
 	--disable-big-core \
 	--disable-home-terminfo \
 	--with-normal \
@@ -79,13 +88,19 @@
 	BUILD_CC="$(HOSTCC)" \
 	HOSTCC="$(HOSTCC)" \
 	HOSTCCFLAGS="" \
-	libs
+	libs \
+	progs
 
 define Build/Install/Default
 	$(MAKE_VARS) \
 	$(MAKE) -C $(PKG_BUILD_DIR)/$(MAKE_PATH) \
 		$(MAKE_INSTALL_FLAGS) \
-		$(1) install.libs install.data;
+		$(1) install.libs install.progs install.data;
+endef
+
+define Package/ncurses-bin/install
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 define Package/terminfo/install
@@ -149,3 +164,4 @@
 $(eval $(call BuildPackage,terminfo))
 $(eval $(call BuildPackage,libncurses))
 $(eval $(call BuildPackage,libncursesw))
+$(eval $(call BuildPackage,ncurses-bin))
