--- ../../../openwrt_trunk.orig/package/opkg/Makefile	2012-07-21 23:10:04.294539830 +0400
+++ ../../../openwrt_trunk/package/opkg/Makefile	2012-07-27 12:46:27.904265467 +0400
@@ -5,7 +5,6 @@
 # See /LICENSE for more information.
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 include $(INCLUDE_DIR)/version.mk
 
 PKG_NAME:=opkg
@@ -27,6 +26,7 @@
 define Package/opkg
   SECTION:=base
   CATEGORY:=Base system
+  DEPENDS:=+uclibc-opt
   TITLE:=opkg package management system
   MAINTAINER:=Jo-Philipp Wich <xm@subsignal.org>
   URL:=http://wiki.openmoko.org/wiki/Opkg
@@ -43,40 +43,39 @@
 endef
 
 define Package/opkg/conffiles
-/etc/opkg.conf
+/opt/etc/opkg.conf
 endef
 
-TARGET_CFLAGS += $(if $(CONFIG_GCC_VERSION_4_3)$(CONFIG_GCC_VERSION_4_4),-Wno-array-bounds)
-TARGET_CFLAGS += -ffunction-sections -fdata-sections
 EXTRA_CFLAGS += $(TARGET_CPPFLAGS)
 
 CONFIGURE_ARGS += \
 	--disable-curl \
 	--disable-gpg \
-	--with-opkgetcdir=/etc \
-	--with-opkglockfile=/var/lock/opkg.lock
+	--with-opkgetcdir=/opt/etc \
+	--with-opkglibdir=/opt/lib \
+	--with-opkglockfile=/opt/var/lock/opkg.lock
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		CC="$(TARGET_CC)" \
 		DESTDIR="$(PKG_INSTALL_DIR)" \
 		HOST_CPU="$(PKGARCH)" \
-		LDFLAGS="-Wl,--gc-sections" \
 		all install
 endef
 
 define Package/opkg/install
-	$(INSTALL_DIR) $(1)/usr/lib/opkg
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/opkg.conf $(1)/etc/
-	$(VERSION_SED) $(1)/etc/opkg.conf
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/opkg-cl $(1)/bin/opkg
+	$(INSTALL_DIR) $(1)/opt/lib/opkg
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DIR) $(1)/opt/tmp
+	$(INSTALL_DIR) $(1)/opt/var/lock
+	$(INSTALL_DATA) ./files/opkg.conf $(1)/opt/etc/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/opkg-cl $(1)/opt/bin/opkg
 endef
 
 define Build/InstallDev
-	mkdir -p $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libopkg $(1)/usr/include/
+	mkdir -p $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libopkg $(1)/opt/include/
 endef
 
 
@@ -84,7 +83,7 @@
 	--disable-curl \
 	--disable-gpg \
 	--with-opkgetcdir=/etc \
-	--with-opkglockfile=/tmp/opkg.lock
+	--with-opkglockfile=/opkg.lock
 
 define Host/Compile
 	$(MAKE) -C $(HOST_BUILD_DIR) CC="$(HOSTCC)" all
