diff -urx .svn ../../../downloads/openwrt_trunk-r42803/package/network/services/samba36/Makefile ../../../openwrt_trunk/package/network/services/samba36/Makefile
--- ../../../downloads/openwrt_trunk-r42803/package/network/services/samba36/Makefile	2014-10-06 15:46:24.073250638 +0400
+++ ../../../openwrt_trunk/package/network/services/samba36/Makefile	2014-10-07 11:59:34.723924674 +0400
@@ -86,7 +86,7 @@
 	samba_cv_zlib_1_2_3=no
 
 CONFIGURE_ARGS += \
-	--exec-prefix=/usr \
+	--exec-prefix=/opt \
 	--prefix=/ \
 	--disable-avahi \
 	--disable-cups \
@@ -95,15 +95,15 @@
 	--disable-static \
 	--disable-swat \
 	--disable-shared-libs \
-	--with-codepagedir=/etc/samba \
-	--with-configdir=/etc/samba \
+	--with-codepagedir=/opt/etc/samba \
+	--with-configdir=/opt/etc/samba \
 	--with-included-iniparser \
 	--with-included-popt \
-	--with-lockdir=/var/lock \
-	--with-logfilebase=/var/log \
-	--with-nmbdsocketdir=/var/nmbd \
-	--with-piddir=/var/run \
-	--with-privatedir=/etc/samba \
+	--with-lockdir=/opt/var/lock \
+	--with-logfilebase=/opt/var/log \
+	--with-nmbdsocketdir=/opt/var/nmbd \
+	--with-piddir=/opt/var/run \
+	--with-privatedir=/opt/etc/samba \
 	--with-sendfile-support \
 	--without-acl-support \
 	--without-cluster-support \
@@ -124,26 +124,27 @@
 MAKE_FLAGS += DYNEXP= PICFLAG= MODULES=
 
 define Package/samba36-server/install
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/samba.config $(1)/etc/config/samba
-	$(INSTALL_DIR) $(1)/etc/samba
-	$(INSTALL_DATA) ./files/smb.conf.template $(1)/etc/samba
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/codepages/lowcase.dat $(1)/etc/samba
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/codepages/upcase.dat $(1)/etc/samba
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/codepages/valid.dat $(1)/etc/samba
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/samba.init $(1)/etc/init.d/samba
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_BIN)/samba_multicall $(1)/usr/sbin
-	ln -sf samba_multicall $(1)/usr/sbin/smbd
-	ln -sf samba_multicall $(1)/usr/sbin/nmbd
-	ln -sf samba_multicall $(1)/usr/sbin/smbpasswd
+	$(INSTALL_DIR) $(1)/opt/etc/samba
+	$(INSTALL_DATA) ./files/smb.conf.template $(1)/opt/etc/samba
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/codepages/lowcase.dat $(1)/opt/etc/samba
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/codepages/upcase.dat $(1)/opt/etc/samba
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/codepages/valid.dat $(1)/opt/etc/samba
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_DIR) $(1)/opt/var/nmbd
+	$(INSTALL_DIR) $(1)/opt/var/log
+	$(INSTALL_DIR) $(1)/opt/var/run
+	$(INSTALL_BIN) $(PKG_BUILD_BIN)/samba_multicall $(1)/opt/sbin
+	ln -sf samba_multicall $(1)/opt/sbin/smbd
+	ln -sf samba_multicall $(1)/opt/sbin/nmbd
+	ln -sf samba_multicall $(1)/opt/sbin/smbpasswd
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S08samba $(1)/opt/etc/init.d
 endef
 
 define Package/samba36-client/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_BIN)/smbclient $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_BIN)/nmblookup $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_BIN)/smbclient $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_BIN)/nmblookup $(1)/opt/sbin
 endef
 
 $(eval $(call BuildPackage,samba36-client))
diff -urx .svn ../../../downloads/openwrt_trunk-r42803/package/network/services/samba36/patches/111-owrt_smbpasswd.patch ../../../openwrt_trunk/package/network/services/samba36/patches/111-owrt_smbpasswd.patch
--- ../../../downloads/openwrt_trunk-r42803/package/network/services/samba36/patches/111-owrt_smbpasswd.patch	2014-10-06 15:46:24.073250638 +0400
+++ ../../../openwrt_trunk/package/network/services/samba36/patches/111-owrt_smbpasswd.patch	2014-10-07 11:59:34.727924674 +0400
@@ -236,15 +236,15 @@
 +		}
 +	}
 +
-+	fp = fopen("/etc/samba/smbpasswd", "r+");
++	fp = fopen("/opt/etc/samba/smbpasswd", "r+");
 +	if(!fp) {
-+		fprintf(stderr, "Failed to open /etc/samba/smbpasswd");
++		fprintf(stderr, "Failed to open /opt/etc/samba/smbpasswd");
 +		return 3;
 +	}
 +
 +	found = find_passwd_line(fp, user, NULL);
 +	if (!add && !found) {
-+		fprintf(stderr, "Could not find user '%s' in /etc/samba/smbpasswd\n", user);
++		fprintf(stderr, "Could not find user '%s' in /opt/etc/samba/smbpasswd\n", user);
 +		return 3;
 +	}
 +
