--- ../../../downloads/openwrt_trunk-r40423/feeds/oldpackages/net/ncftp/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/ncftp/Makefile
@@ -53,15 +53,15 @@
 endef
 
 define Package/ncftp/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ncftp $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ncftp $(1)/opt/bin/
 endef
 
 define Package/ncftp-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/ncftp{batch,get,ls,put,spooler} \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/ncftp{batch,get,ls,put,spooler} \
+		$(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,ncftp))
