--- ../../../downloads/openwrt_trunk-r35474/feeds/oldpackages/libs/st2205tool/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/st2205tool/Makefile
@@ -24,7 +24,7 @@
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=ST220xu powered photo-frames library
-  DEPENDS:=+libgd
+  DEPENDS:=+libgd +libjpeg
   URL:=http://picframe.spritesserver.nl/wiki/
 endef
 
@@ -42,18 +42,18 @@
 
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/libst2205/st2205.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libst2205/libst2205.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_BUILD_DIR)/libst2205/st2205.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libst2205/libst2205.{a,so*} $(1)/opt/lib/
 endef
 
 
 define Package/st2205tool/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/setpic/setpic $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libst2205/libst2205.so $(1)/usr/lib/libst2205.so.1
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/setpic/setpic $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libst2205/libst2205.so $(1)/opt/lib/libst2205.so.1
 endef
 
 $(eval $(call BuildPackage,st2205tool))
