--- a/src/contentscanners/icapscan.cpp.orig	2008-11-18 14:27:04.000000000 +0300
+++ b/src/contentscanners/icapscan.cpp	2014-12-19 14:30:42.014070098 +0300
@@ -36,6 +36,7 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <netdb.h>		// for gethostby
+#include <cstdio>
 
 
 // DEFINES
--- a/src/downloadmanagers/fancy.cpp.orig	2008-11-18 14:27:04.000000000 +0300
+++ b/src/downloadmanagers/fancy.cpp	2014-12-19 14:30:18.653816380 +0300
@@ -32,6 +32,7 @@
 #include <unistd.h>
 #include <iostream>
 #include <math.h>
+#include <cstdio>
 
 
 // GLOBALS
