--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/dansguardian/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/dansguardian/Makefile
@@ -30,7 +30,7 @@
 endef
 
 define Package/dansguardian/conffiles
-/etc/dansguardian/dansguardian.conf
+/opt/etc/dansguardian/dansguardian.conf
 endef
 
 CONFIGURE_VARS += \
@@ -49,14 +49,16 @@
 endef
 
 define Package/dansguardian/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/dansguardian $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/dansguardian $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/share/dansguardian
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/share/dansguardian/transparent1x1.gif $(1)/usr/share/dansguardian/
-	$(INSTALL_DIR) $(1)/usr/share/dansguardian/languages/ukenglish
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/dansguardian/languages/ukenglish/* $(1)/usr/share/dansguardian/languages/ukenglish/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/dansguardian $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/dansguardian $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/share/dansguardian
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/share/dansguardian/transparent1x1.gif $(1)/opt/share/dansguardian/
+	$(INSTALL_DIR) $(1)/opt/share/dansguardian/languages/ukenglish
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/dansguardian/languages/ukenglish/* $(1)/opt/share/dansguardian/languages/ukenglish/
+	$(INSTALL_DIR) $(1)/opt/var/run
+	$(INSTALL_BIN) ./files/S24dansguardian $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,dansguardian))
