diff -urx .svn ../../../openwrt_trunk.orig/feeds/packages/net/pptpd/files/options.pptpd ../../../openwrt_trunk/feeds/packages/net/pptpd/files/options.pptpd
--- ../../../openwrt_trunk.orig/feeds/packages/net/pptpd/files/options.pptpd	2012-05-02 16:17:23.886419808 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/pptpd/files/options.pptpd	2012-05-02 16:22:05.962312733 +0400
@@ -1,5 +1,5 @@
 #debug
-#logfile /tmp/pptp-server.log
+#logfile /opt/tmp/pptp-server.log
 172.16.1.1:
 auth
 name "pptp-server"
@@ -20,4 +20,4 @@
 refuse-pap
 #ms-dns 172.16.1.1
 #plugin radius.so
-#radius-config-file /etc/radius.conf
+#radius-config-file /opt/etc/radius.conf
diff -urx .svn ../../../openwrt_trunk.orig/feeds/packages/net/pptpd/files/pptpd.conf ../../../openwrt_trunk/feeds/packages/net/pptpd/files/pptpd.conf
--- ../../../openwrt_trunk.orig/feeds/packages/net/pptpd/files/pptpd.conf	2012-05-02 16:17:23.886419808 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/pptpd/files/pptpd.conf	2012-05-02 16:22:05.962312733 +0400
@@ -1,5 +1,5 @@
 #debug
-option /etc/ppp/options.pptpd
+option /opt/etc/ppp/options.pptpd
 speed 115200
 stimeout 10
 #localip & remoteip are not needed, ip management is done by pppd
diff -urx .svn ../../../openwrt_trunk.orig/feeds/packages/net/pptpd/Makefile ../../../openwrt_trunk/feeds/packages/net/pptpd/Makefile
--- ../../../openwrt_trunk.orig/feeds/packages/net/pptpd/Makefile	2012-05-02 16:17:23.886419808 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/pptpd/Makefile	2012-05-31 22:31:41.522096137 +0400
@@ -22,7 +22,7 @@
 define Package/pptpd
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+kmod-ppp +kmod-gre +ppp
+#  DEPENDS:=+kmod-ppp +kmod-gre +ppp
   TITLE:=PopTop pptp server
   URL:=http://www.poptop.org/
   SUBMENU:=VPN
@@ -39,25 +39,25 @@
 	INSTALL="install" \
 
 define Package/pptpd/conffiles
-/etc/pptpd.conf
-/etc/ppp/options.pptpd
+/opt/etc/pptpd.conf
+/opt/etc/ppp/options.pptpd
 endef
 
 define Package/pptpd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/sbin/bcrelay \
-		$(PKG_INSTALL_DIR)/usr/sbin/pptpctrl \
-		$(PKG_INSTALL_DIR)/usr/sbin/pptpd \
-		$(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib/pptpd
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pptpd/* $(1)/usr/lib/pptpd/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/pptpd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/pptpd.init $(1)/etc/init.d/pptpd
-	$(INSTALL_DIR) $(1)/etc/ppp
-	$(INSTALL_DATA) ./files/options.pptpd $(1)/etc/ppp/
+		$(PKG_INSTALL_DIR)/opt/sbin/bcrelay \
+		$(PKG_INSTALL_DIR)/opt/sbin/pptpctrl \
+		$(PKG_INSTALL_DIR)/opt/sbin/pptpd \
+		$(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib/pptpd
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pptpd/* $(1)/opt/lib/pptpd/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/pptpd.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/ppp
+	$(INSTALL_DATA) ./files/options.pptpd $(1)/opt/etc/ppp/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S20poptop $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,pptpd))
