--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/subversion/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/subversion/Makefile
@@ -71,16 +71,16 @@
 endef
 
 define Package/subversion-server/conffiles
-/etc/config/subversion
+/opt/etc/config/subversion
 endef
 
 TARGET_CFLAGS += $(FPIC)
 TARGET_CPPFLAGS += -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE
 
 CONFIGURE_ARGS += \
-	--with-apr="$(STAGING_DIR)/usr/bin/apr-1-config" \
-	--with-apr-util="$(STAGING_DIR)/usr/bin/apu-1-config" \
-	--with-neon="$(STAGING_DIR)/usr" \
+	--with-apr="$(STAGING_DIR)/opt/bin/apr-1-config" \
+	--with-apr-util="$(STAGING_DIR)/opt/bin/apu-1-config" \
+	--with-neon="$(STAGING_DIR)/opt" \
 	--disable-mod-activation \
 	--without-ruby-sitedir \
 	--without-swig \
@@ -95,7 +95,7 @@
 CONFIGURE_VARS += \
 	svn_lib_neon=yes \
 	LDFLAGS="$(TARGET_LDFLAGS) -lcrypt -lm \
-		-lz -lpthread -L$(TOOLCHAIN_DIR)/usr/lib \
+		-lz -lpthread -L$(TOOLCHAIN_DIR)/opt/lib \
 		-L$(TOOLCHAIN_DIR)/lib" \
 	CPPFLAGS="$(TARGET_CPPFLAGS)" \
 
@@ -106,22 +106,18 @@
 endef
 
 define Package/subversion-libs/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libsvn_*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libsvn_*.so.* $(1)/opt/lib/
 endef
 
 define Package/subversion-client/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/{svn,svnversion,svnsync} $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/{svn,svnversion,svnsync} $(1)/opt/bin/
 endef
 
 define Package/subversion-server/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/svn{look,admin,dumpfilter,serve} $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/subversion.config $(1)/etc/config/subversion
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/subversion.init $(1)/etc/init.d/subversion
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/svn{look,admin,dumpfilter,serve} $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,subversion-libs))
