--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/memtester/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/memtester/Makefile
@@ -29,8 +29,8 @@
 endef
 
 define Build/Configure
-	$(SED) 's,cc -O2,$(TARGET_CC) $(TARGET_CFLAGS),' $(PKG_BUILD_DIR)/conf-cc
-	$(SED) "s,cc,$(TARGET_CC)," $(PKG_BUILD_DIR)/conf-ld
+	$(SED) 's|cc|$(TARGET_CC) $(TARGET_CFLAGS)|' $(PKG_BUILD_DIR)/conf-cc
+	$(SED) 's|cc|$(TARGET_CC) $(TARGET_LDFLAGS)|' $(PKG_BUILD_DIR)/conf-ld
 endef
 
 define Build/Compile
@@ -38,8 +38,8 @@
 endef
 
 define Package/memtester/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/memtester $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/memtester $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,memtester))
