--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r38784/package/libs/cyassl/Makefile
+++ ../../../openwrt_trunk/package/libs/cyassl/Makefile
@@ -42,16 +42,16 @@
 	--disable-examples
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcyassl.{so*,la} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcyassl.{so*,la} $(1)/opt/lib/
 endef
 
 define Package/libcyassl/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcyassl.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcyassl.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libcyassl))
