--- ../../../trunk_clean/feeds/packages/net/cutter/Makefile	2012-05-02 16:17:28.226365246 +0400
+++ ../../../trunk/feeds/packages/net/cutter/Makefile	2012-05-28 09:38:06.430174790 +0400
@@ -31,12 +31,12 @@
 endef
 	
 define Build/Compile
-	$(TARGET_CC) $(PKG_BUILD_DIR)/cutter.c -o $(PKG_BUILD_DIR)/cutter
+	$(TARGET_CC) $(TARGET_CFLAGS) $(TARGET_LDFLAGS) $(PKG_BUILD_DIR)/cutter.c -o $(PKG_BUILD_DIR)/cutter
 endef
 	
 define Package/cutter/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/cutter $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/cutter $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,cutter))
