--- ../../../downloads/openwrt_trunk-q45513/feeds/oldpackages/lang/pyopenssl/Makefile	2015-04-29 22:11:28.813020396 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/pyopenssl/Makefile	2015-05-18 13:12:17.849932132 +0300
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=pyopenssl
-PKG_VERSION:=0.10
+PKG_VERSION:=0.13.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=pyOpenSSL-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://pypi.python.org/packages/source/p/pyOpenSSL/
-PKG_MD5SUM:=34db8056ec53ce80c7f5fc58bee9f093
+PKG_MD5SUM:=e27a3b76734c39ea03952ca94cc56715
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/pyOpenSSL-$(PKG_VERSION)
 PKG_BUILD_DEPENDS:=python
@@ -35,7 +35,7 @@
 endef
 
 define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix="$(PKG_INSTALL_DIR)/usr")
+	$(call Build/Compile/PyMod,,install --prefix=/opt --root=$(PKG_INSTALL_DIR))
 endef
 
 define Package/pyopenssl/install
