--- ../../../downloads/openwrt_trunk-q44497/feeds/oldpackages/lang/pyopenssl/Makefile	2015-02-25 12:52:24.716419944 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/pyopenssl/Makefile	2015-02-28 23:41:25.751902544 +0300
@@ -35,13 +35,13 @@
 endef
 
 define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix="$(PKG_INSTALL_DIR)/usr")
+	$(call Build/Compile/PyMod,,install --prefix="$(PKG_INSTALL_DIR)/Apps/opt")
 endef
 
 define Package/pyopenssl/install
 	$(INSTALL_DIR) $(1)$(PYTHON_PKG_DIR)
 	$(CP) \
-	    $(PKG_INSTALL_DIR)$(PYTHON_PKG_DIR)/* \
+	    $(PKG_INSTALL_DIR)/Apps$(PYTHON_PKG_DIR)/* \
 	    $(1)$(PYTHON_PKG_DIR)
 endef
 
