--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/cgilib/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/cgilib/Makefile
@@ -41,15 +41,15 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/cgi.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcgi.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/cgi.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcgi.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/cgilib/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcgi.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcgi.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,cgilib))
