--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/utils/at/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/at/Makefile
@@ -35,22 +35,33 @@
 EXTRA_CFLAGS:=-DNEED_YYWRAP -I$(PKG_BUILD_DIR) \
 	$(TARGET_LDFLAGS)
 
-CONFIGURE_ARGS+=--prefix=/usr \
+CONFIGURE_ARGS+=--prefix=/opt \
 		--with-daemon_username=nobody \
 		--with-daemon_groupname=nogroup \
-		--with-jobdir=/var/spool/cron/atjobs \
-		--with-atspool=/var/spool/cron/atspool
+		--with-jobdir=/opt/var/spool/cron/atjobs \
+		--with-atspool=/opt/var/spool/cron/atspool \
+		--with-loadavg_mx=0.5
 
 CONFIGURE_VARS += \
 	ac_cv_header_security_pam_appl_h=no
 
 define Package/at/install
-	$(INSTALL_DIR) $(1)/usr/bin $(1)/usr/sbin $(1)/etc/init.d/
-	$(INSTALL_BIN) ./files/atd.init $(1)/etc/init.d/atd
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/at $(1)/usr/bin
-	ln -sf at $(1)/usr/bin/atq
-	ln -sf at $(1)/usr/bin/atrm
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/atd $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/bin $(1)/opt/sbin $(1)/opt/etc/init.d/
+	$(INSTALL_BIN) ./files/S50atd $(1)/opt/etc/init.d
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/at $(1)/opt/bin
+	ln -sf at $(1)/opt/bin/atq
+	ln -sf at $(1)/opt/bin/atrm
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/atd $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/batch $(1)/opt/bin
+endef
+
+define Package/at/postinst
+#!/bin/sh
+mkdir -m 0755 -p /opt/var/spool/cron/atjobs
+touch /opt/var/spool/cron/atjobs/.SEQ
+chown -R nobody:nobody /opt/var/spool/cron/atjobs
+mkdir -m 0755 -p /opt/var/spool/cron/atspool
+chown -R nobody:nobody /opt/var/spool/cron/atspool
 endef
 
 $(eval $(call BuildPackage,at))
