--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/unzip/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/unzip/Makefile
@@ -49,7 +49,7 @@
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) -f unix/Makefile unzips \
 		$(TARGET_CONFIGURE_OPTS) \
-		prefix="$(PKG_INSTALL_DIR)/usr" \
+		prefix="$(PKG_INSTALL_DIR)/opt" \
 		CFLAGS="$(TARGET_CFLAGS)" \
 		CF="$(TARGET_CFLAGS) -O $(TARGET_CPPFLAGS) -I. -DUNIX" \
 		CC="$(TARGET_CC)" \
@@ -58,8 +58,8 @@
 endef
 
 define Package/unzip/install
-	$(INSTALL_DIR) $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,unzip))
