--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/fakeidentd/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/fakeidentd/Makefile
@@ -37,10 +37,10 @@
 endef
 
 define Package/fakeidentd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_BUILD_DIR)/$(PKG_NAME) $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/$(PKG_NAME).init $(1)/etc/init.d/$(PKG_NAME)
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(CP) $(PKG_BUILD_DIR)/$(PKG_NAME) $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S85fakeidentd $(1)/opt/etc/init.d/
 endef
 
 $(eval $(call BuildPackage,fakeidentd))
