--- ../../../openwrt_trunk.orig/feeds/packages/utils/unrar/Makefile	2012-08-06 09:54:13.407011761 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/unrar/Makefile	2012-08-06 09:56:44.059009012 +0400
@@ -41,8 +41,8 @@
 endef
 
 define Package/unrar/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/unrar $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/unrar $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,unrar))
