--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/serdisplib/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/serdisplib/Makefile
@@ -38,20 +38,20 @@
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
-		CFLAGS="$(TARGET_CFLAGS) -Wall -Wno-implicit -I../include -I.." \
+		CFLAGS="$(TARGET_CFLAGS) -Wall -Wno-implicit -I../include -I.. -I${STAGING_DIR}/opt/include" \
 		all
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/include/serdisplib $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/libserdisp.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_BUILD_DIR)/include/serdisplib $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/libserdisp.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/serdisplib/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/lib/libserdisp.so.* $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/lib/libserdisp.so.* $(1)/opt/lib
 endef
 
 $(eval $(call BuildPackage,serdisplib))
