--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/zip/Makefile	2014-10-06 15:46:23.777250645 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/zip/Makefile	2014-10-09 13:17:48.501190297 +0400
@@ -46,8 +46,9 @@
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) -f unix/Makefile generic \
-		prefix="$(PKG_INSTALL_DIR)/usr" \
+		prefix="$(PKG_INSTALL_DIR)/opt" \
 		CFLAGS="$(TARGET_CFLAGS)" \
+		LFLAGS2="$(TARGET_LDFLAGS)" \
 		CC="$(TARGET_CC) $(TARGET_CFLAGS) -O $(TARGET_CPPFLAGS) -I. -DUNIX" \
 		LD="$(TARGET_CC) $(TARGET_LDFLAGS)" \
 		IZ_BZIP2="no" \
@@ -55,8 +56,8 @@
 endef
 
 define Package/zip/install
-	$(INSTALL_DIR) $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,zip))
