--- ../../../downloads/openwrt_trunk-r44497/feeds/packages/utils/zip/Makefile	2015-02-24 12:08:56.873768246 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/zip/Makefile	2015-02-27 13:29:24.484951933 +0300
@@ -46,17 +46,17 @@
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) -f unix/Makefile generic \
-		prefix="$(PKG_INSTALL_DIR)/usr" \
+		prefix="$(PKG_INSTALL_DIR)/opt" \
 		CFLAGS="$(TARGET_CFLAGS)" \
-		CC="$(TARGET_CC) $(TARGET_CFLAGS) -O $(TARGET_CPPFLAGS) -I. -DUNIX" \
+		CC="$(TARGET_CC) $(TARGET_CFLAGS) -O $(TARGET_CPPFLAGS) -I. -DUNIX $(TARGET_LDFLAGS)" \
 		LD="$(TARGET_CC) $(TARGET_LDFLAGS)" \
 		IZ_BZIP2="no" \
 		install
 endef
 
 define Package/zip/install
-	$(INSTALL_DIR) $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,zip))
