--- ../../../downloads/openwrt_trunk-r44497/feeds/packages/utils/zip/Makefile	2015-02-20 12:39:13.449134248 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/zip/Makefile	2015-02-21 19:38:35.806004518 +0300
@@ -46,8 +46,9 @@
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) -f unix/Makefile generic \
-		prefix="$(PKG_INSTALL_DIR)/usr" \
+		prefix="$(PKG_INSTALL_DIR)/opt" \
 		CFLAGS="$(TARGET_CFLAGS)" \
+		LFLAGS1="$(TARGET_LDFLAGS)" \
 		CC="$(TARGET_CC) $(TARGET_CFLAGS) -O $(TARGET_CPPFLAGS) -I. -DUNIX" \
 		LD="$(TARGET_CC) $(TARGET_LDFLAGS)" \
 		IZ_BZIP2="no" \
@@ -55,8 +56,8 @@
 endef
 
 define Package/zip/install
-	$(INSTALL_DIR) $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,zip))
