--- ../../../downloads/openwrt_trunk-r43718/package/libs/libconfig/Makefile	2014-11-07 23:35:24.890700705 +0300
+++ ../../../openwrt_trunk/package/libs/libconfig/Makefile	2014-11-29 16:59:52.275442476 +0300
@@ -47,12 +47,12 @@
 	--disable-cxx
 
 define Build/InstallDev
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/* $(1)/
 endef
 
 define Package/libconfig/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libconfig.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libconfig.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libconfig))
