--- ../../../openwrt_trunk/package/libs/libconfig/Makefile.orig
+++ ../../../openwrt_trunk/package/libs/libconfig/Makefile
@@ -51,8 +51,8 @@
 endef
 
 define Package/libconfig/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libconfig.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libconfig.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libconfig))
