--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/libs/libmms/Makefile	2014-04-15 16:11:47.624849362 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libmms/Makefile	2014-04-16 10:01:16.831682348 +0400
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libmms
-PKG_VERSION:=0.6
+PKG_VERSION:=0.6.4
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@SF/libmms
-PKG_MD5SUM:=650ad04a4c8bd79246390b81b29680b6
+PKG_MD5SUM:=d6b665b335a6360e000976e770da7691
 
 PKG_LICENSE:=LGPLv2.1
 PKG_LICENSE_FILES:=COPYING.LIB
@@ -47,19 +47,19 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libmms $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libmms.{a,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libmms $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libmms.{a,so*} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
 	$(INSTALL_DATA) \
-		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/*.pc \
-		$(1)/usr/lib/pkgconfig
+		$(PKG_INSTALL_DIR)/opt/lib/pkgconfig/*.pc \
+		$(1)/opt/lib/pkgconfig
 endef
 
 define Package/libmms/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libmms.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libmms.so.* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libmms))
