--- ../../../downloads/openwrt_trunk-r35474/feeds/oldpackages/net/netcat/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/netcat/Makefile
@@ -32,26 +32,9 @@
 endef
 
 define Package/netcat/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/netcat \
-		$(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/netcat \
+		$(1)/opt/bin
 endef
 
-define Package/netcat/postinst
-#!/bin/sh
-if [ -e $${IPKG_INSTROOT}/usr/bin/nc ]; then
-  rm -rf $${IPKG_INSTROOT}/usr/bin/nc;
-fi
-ln -s ./netcat $${IPKG_INSTROOT}/usr/bin/nc
-endef
-
-define Package/netcat/postrm
-#!/bin/sh
-rm $${IPKG_INSTROOT}/usr/bin/nc
-ln -s ../../bin/busybox $${IPKG_INSTROOT}/usr/bin/nc
-$${IPKG_INSTROOT}/usr/bin/nc 2>&1 | grep 'applet not found' > /dev/null 2>&1 && rm $${IPKG_INSTROOT}/usr/bin/nc
-exit 0
-endef
-
-
 $(eval $(call BuildPackage,netcat))
