--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = libsodium libcork libipset libudns src
+SUBDIRS = libcork libipset libudns src
 
 ACLOCAL_AMFLAGS = -I m4
 man_MANS = shadowsocks-libev.8
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -4,14 +4,12 @@
 AM_CFLAGS += -I$(top_srcdir)/libudns
 AM_CFLAGS += -I$(top_srcdir)/libipset/include
 AM_CFLAGS += -I$(top_srcdir)/libcork/include
-AM_CFLAGS += -I$(top_srcdir)/libsodium/src/libsodium/include
 
 AM_LDFLAGS = -static
 
 SS_COMMON_LIBS = \
 				 $(top_builddir)/libipset/libipset.la \
 				 $(top_builddir)/libcork/libcork.la \
-				 $(top_builddir)/libsodium/src/libsodium/libsodium.la \
 				 $(INET_NTOP_LIB)
 
 bin_PROGRAMS = ss-local ss-tunnel
