diff -urx .svn ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/cups/files/etc/cups/cupsd.conf ../../../openwrt_trunk/feeds/oldpackages/net/cups/files/etc/cups/cupsd.conf
--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/cups/files/etc/cups/cupsd.conf	2014-04-09 16:17:24.178286483 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/cups/files/etc/cups/cupsd.conf	2014-04-09 16:53:25.978247102 +0400
@@ -21,12 +21,12 @@
 MaxPrinterHistory 10
 #Printcap /etc/printcap
 #PrintcapFormat BSD
-RequestRoot /var/cups
+RequestRoot /opt/var/cups
 #RemoteRoot remroot
 User nobody
 Group nogroup
 RIPCache 512k
-TempDir /var/cups
+TempDir /opt/var/cups
 Port 631
 HostNameLookups Off
 KeepAlive On
diff -urx .svn ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/cups/Makefile ../../../openwrt_trunk/feeds/oldpackages/net/cups/Makefile
--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/cups/Makefile	2014-04-09 16:17:24.178286483 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/cups/Makefile	2014-04-09 16:53:25.978247102 +0400
@@ -15,7 +15,7 @@
 PKG_SOURCE_URL:= http://www.cups.org/software/$(PKG_VERSION)
 PKG_MD5SUM:=de3006e5cf1ee78a9c6145ce62c4e982
 
-TARGET_LDFLAGS+=-Wl,-rpath-link=$(STAGING_DIR)/usr/lib
+TARGET_LDFLAGS+=-Wl,-rpath-link=$(STAGING_DIR)/opt/lib
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -37,9 +37,9 @@
 endef
 
 define Package/cups/conffiles
-/etc/cups/classes.conf
-/etc/cups/cupsd.conf
-/etc/cups/printers.conf
+/opt/etc/cups/classes.conf
+/opt/etc/cups/cupsd.conf
+/opt/etc/cups/printers.conf
 endef
 
 define Package/cups-bsd
@@ -63,7 +63,7 @@
 endef
 
 define Package/cups-client/conffiles
-/etc/cups/client.conf
+/opt/etc/cups/client.conf
 endef
 
 define Package/cups-client/description
@@ -98,7 +98,7 @@
 $(call Package/cups/Default)
   SECTION:=libs
   CATEGORY:=Libraries
-  DEPENDS:=+zlib +libpthread +libpng +libjpeg
+  DEPENDS:=+zlib +libpthread +libpng +libjpeg +libssp
   TITLE:=Common UNIX Printing System - Core library
 endef
 
@@ -424,247 +424,247 @@
 endef
 
 define Package/cups/install
-	rm -rf $(1)/etc/cups
-	$(INSTALL_DIR) $(1)/etc/cups
-	$(CP) $(PKG_INSTALL_DIR)/etc/cups/* $(1)/etc/cups/
-	rm -rf $(1)/etc/cups/certs
-	ln -sf /tmp $(1)/etc/cups/certs
-	rm -f $(1)/usr/bin/cups-config
-	$(INSTALL_DIR) $(1)/usr/lib/cups
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/cups/backend $(1)/usr/lib/cups
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/cups/cgi-bin $(1)/usr/lib/cups
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/cups/daemon $(1)/usr/lib/cups
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/cups/driver $(1)/usr/lib/cups
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/cups/monitor $(1)/usr/lib/cups
-	$(INSTALL_DIR) $(1)/usr/lib/cups/filter
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/cups/filter/{commandtops,pstops} \
-		$(PKG_INSTALL_DIR)/usr/lib/cups/filter/gziptoany \
-		$(1)/usr/lib/cups/filter
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/*.tmpl \
-		$(1)/usr/share/cups/templates/
-	$(INSTALL_DIR) $(1)/usr/share/cups/mime
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/mime/* $(1)/usr/share/cups/mime/
-	$(INSTALL_DIR) $(1)/usr/share/doc/cups
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/doc/cups/index.html \
-		$(1)/usr/share/doc/cups/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/doc/cups/*.css \
-		$(1)/usr/share/doc/cups/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/doc/cups/images \
-		$(1)/usr/share/doc/cups/
-	$(INSTALL_DIR) $(1)/usr/sbin
+	rm -rf $(1)/opt/etc/cups
+	$(INSTALL_DIR) $(1)/opt/etc/cups
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/cups/* $(1)/opt/etc/cups/
+	rm -rf $(1)/opt/etc/cups/certs
+	ln -sf /opt/tmp $(1)/opt/etc/cups/certs
+	rm -f $(1)/opt/bin/cups-config
+	$(INSTALL_DIR) $(1)/opt/lib/cups
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/cups/backend $(1)/opt/lib/cups
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/cups/cgi-bin $(1)/opt/lib/cups
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/cups/daemon $(1)/opt/lib/cups
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/cups/driver $(1)/opt/lib/cups
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/cups/monitor $(1)/opt/lib/cups
+	$(INSTALL_DIR) $(1)/opt/lib/cups/filter
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/cups/filter/{commandtops,pstops} \
+		$(PKG_INSTALL_DIR)/opt/lib/cups/filter/gziptoany \
+		$(1)/opt/lib/cups/filter
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/*.tmpl \
+		$(1)/opt/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/cups/mime
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/mime/* $(1)/opt/share/cups/mime/
+	$(INSTALL_DIR) $(1)/opt/share/doc/cups
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/doc/cups/index.html \
+		$(1)/opt/share/doc/cups/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/doc/cups/*.css \
+		$(1)/opt/share/doc/cups/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/doc/cups/images \
+		$(1)/opt/share/doc/cups/
+	$(INSTALL_DIR) $(1)/opt/sbin
 	# overwrite default config with our own
-	$(CP) ./files/etc/cups/* $(1)/etc/cups/
+	$(CP) ./files/etc/cups/* $(1)/opt/etc/cups/
 	# install initscript with priority 60
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/cupsd.init $(1)/etc/init.d/cupsd
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S60cupsd $(1)/opt/etc/init.d/
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/sbin/{cupsctl,cupsd} \
-		$(1)/usr/sbin/
+		$(PKG_INSTALL_DIR)/opt/sbin/{cupsctl,cupsd} \
+		$(1)/opt/sbin/
 endef
 
 define Package/cups-bsd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/{lprm,lpq,lpr} $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/lpc $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/{lprm,lpq,lpr} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/lpc $(1)/opt/sbin/
 endef
 
 define Package/cups-client/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/{lp,cancel,cupstestppd,cupstestdsc} \
-		$(PKG_INSTALL_DIR)/usr/bin/{ipptool,lpoptions,lpstat,lppasswd} \
-		$(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/sbin
+		$(PKG_INSTALL_DIR)/opt/bin/{lp,cancel,cupstestppd,cupstestdsc} \
+		$(PKG_INSTALL_DIR)/opt/bin/{ipptool,lpoptions,lpstat,lppasswd} \
+		$(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/sbin/{cupsaccept,cupsaddsmb,cupsfilter} \
-		$(PKG_INSTALL_DIR)/usr/sbin/{lpadmin,lpinfo,lpmove} \
-		$(1)/usr/sbin/
-	(cd $(1)/usr/sbin; ln -sf cupsaccept accept; ln -sf cupsaccept cupsenable; ln -sf cupsaccept cupsdisable; ln -sf cupsaccept reject; ln -sf cupsaccept cupsreject;)
+		$(PKG_INSTALL_DIR)/opt/sbin/{cupsaccept,cupsaddsmb,cupsfilter} \
+		$(PKG_INSTALL_DIR)/opt/sbin/{lpadmin,lpinfo,lpmove} \
+		$(1)/opt/sbin/
+	(cd $(1)/opt/sbin; ln -sf cupsaccept accept; ln -sf cupsaccept cupsenable; ln -sf cupsaccept cupsdisable; ln -sf cupsaccept reject; ln -sf cupsaccept cupsreject;)
 endef
 
 define Package/cups-filters/install
-	$(INSTALL_DIR) $(1)/usr/lib/cups/filter
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/cups/filter/{bannertops,commandtoescpx,commandtopclx,imagetops,imagetoraster,rastertoepson,rastertopwg,rastertohp,rastertoescpx,rastertopclx,rastertolabel,texttops} \
-		$(1)/usr/lib/cups/filter
-	(cd $(1)/usr/lib/cups/filter; ln -sf rastertolabel rastertodymo;)
+	$(INSTALL_DIR) $(1)/opt/lib/cups/filter
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/cups/filter/{bannertops,commandtoescpx,commandtopclx,imagetops,imagetoraster,rastertoepson,rastertopwg,rastertohp,rastertoescpx,rastertopclx,rastertolabel,texttops} \
+		$(1)/opt/lib/cups/filter
+	(cd $(1)/opt/lib/cups/filter; ln -sf rastertolabel rastertodymo;)
 endef
 
 define Package/cups-ppdc/install
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/{ppdc,ppdhtml,ppdi,ppdmerge,ppdpo} \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/{ppdc,ppdhtml,ppdi,ppdmerge,ppdpo} \
+		$(1)/opt/bin/
 endef
 
 define Package/libcups/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcups.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcups.so* $(1)/opt/lib/
 endef
 
 define Package/libcupscgi/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcupscgi.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcupscgi.so* $(1)/opt/lib/
 endef
 
 define Package/libcupsdriver/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcupsdriver.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcupsdriver.so* $(1)/opt/lib/
 endef
 
 define Package/libcupsimage/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcupsimage.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcupsimage.so* $(1)/opt/lib/
 endef
 
 define Package/libcupsmime/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcupsmime.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcupsmime.so* $(1)/opt/lib/
 endef
 
 define Package/libcupsppdc/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcupsppdc.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcupsppdc.so* $(1)/opt/lib/
 endef
 
 define Package/cups-locale-de/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/de $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/de \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/de $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/de \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-nl/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/nl $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/nl $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-hu/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/hu $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/hu \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/hu $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/hu \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-pt/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/pt $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/pt $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-pt_BR/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/pt_BR $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/pt_BR $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-fi/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/fi $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/fi $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-sv/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/sv $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/sv $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-da/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/da $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/da $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-ko/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/ko $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/ko $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-zh/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/zh $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/zh $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-ja/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/ja $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/ja \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/ja $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/ja \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-es/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/es $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/es \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/es $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/es \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-zh_TW/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/zh_TW $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/zh_TW $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-pl/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/pl $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/pl \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/pl $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/pl \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-no/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/no $(1)/usr/share/locale/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/no $(1)/opt/share/locale/
 endef
 
 define Package/cups-locale-ru/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/ru $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/ru \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/ru $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/ru \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-eu/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/eu $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/eu \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/eu $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/eu \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-fr/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/fr $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/fr \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/fr $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/fr \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-it/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/it $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/it \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/it $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/it \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Package/cups-locale-id/install
-	$(INSTALL_DIR) $(1)/usr/share/locale
-	$(INSTALL_DIR) $(1)/usr/share/cups/templates
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/locale/id $(1)/usr/share/locale/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/cups/templates/id \
-		$(1)/usr/share/cups/templates/
+	$(INSTALL_DIR) $(1)/opt/share/locale
+	$(INSTALL_DIR) $(1)/opt/share/cups/templates
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/locale/id $(1)/opt/share/locale/
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/cups/templates/id \
+		$(1)/opt/share/cups/templates/
 endef
 
 define Build/InstallDev
 	$(INSTALL_DIR) $(2)/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/cups-config $(2)/bin/
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/cups $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libcups*.so* $(1)/usr/lib/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/cups-config $(2)/bin/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/cups $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libcups*.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,cups))
