--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/telephony/admin/asterisk18-gui/Makefile
+++ ../../../openwrt_trunk/feeds/telephony/admin/asterisk18-gui/Makefile
@@ -47,11 +47,11 @@
 			--build=$(GNU_HOST_NAME) \
 			--host=$(GNU_TARGET_NAME) \
 			--target=$(GNU_TARGET_NAME) \
-			--prefix=/usr \
+			--prefix=/opt \
 			--disable-nls \
 			--disable-static \
-			--localstatedir="/usr" \
-			--sysconfdir="/etc" \
+			--localstatedir="/opt" \
+			--sysconfdir="/opt/etc" \
 		)
 endef
 
@@ -61,15 +61,13 @@
 endef
 
 define Package/asterisk18-gui/install
-	$(INSTALL_DIR) $(1)/etc/asterisk/
-	$(CP) ./files/gui_* $(1)/etc/asterisk/
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) ./files/asterisk-gui.init $(1)/etc/init.d/asterisk-gui
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/static-http
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/asterisk/static-http/* $(1)/usr/lib/asterisk/static-http
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/scripts
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/asterisk/scripts/* $(1)/usr/lib/asterisk/scripts
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/gui_backups
+	$(INSTALL_DIR) $(1)/opt/etc/asterisk/
+	$(CP) ./files/gui_* $(1)/opt/etc/asterisk/
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/static-http
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/asterisk/static-http/* $(1)/opt/lib/asterisk/static-http
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/scripts
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/asterisk/scripts/* $(1)/opt/lib/asterisk/scripts
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/gui_backups
 endef
 
 $(eval $(call BuildPackage,asterisk18-gui))
