--- ../../../openwrt_trunk/feeds/routing/bird/Makefile.orig
+++ ../../../openwrt_trunk/feeds/routing/bird/Makefile
@@ -131,22 +131,20 @@
 $(STAMP_BUILT): $(STAMP_BUILT)-$(2)
 
 define Package/bird$(2)/install
-	$(INSTALL_DIR)  $$(1)/usr/sbin
-	$(INSTALL_BIN)  $$(PKG_BUILD_DIR)/bird$(2) $$(1)/usr/sbin/
-	$(INSTALL_DIR)  $$(1)/etc
-	$(INSTALL_DATA) ./files/bird$(2).conf $$(1)/etc/
-	$(INSTALL_DIR)  $$(1)/etc/init.d
-	$(INSTALL_BIN)  ./files/bird$(2).init $$(1)/etc/init.d/bird$(2)
+	$(INSTALL_DIR)  $$(1)/opt/sbin
+	$(INSTALL_BIN)  $$(PKG_BUILD_DIR)/bird$(2) $$(1)/opt/sbin/
+	$(INSTALL_DIR)  $$(1)/opt/etc
+	$(INSTALL_DATA) ./files/bird$(2).conf $$(1)/opt/etc/
 
 endef
 
 define Package/bird$(2)/conffiles
-/etc/bird$(2).conf
+/opt/etc/bird$(2).conf
 endef
 
 define Package/birdc$(2)/install
-	$(INSTALL_DIR) $$(1)/usr/sbin
-	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/birdc$(2) $$(1)/usr/sbin/
+	$(INSTALL_DIR) $$(1)/opt/sbin
+	$(INSTALL_BIN) $$(PKG_BUILD_DIR)/birdc$(2) $$(1)/opt/sbin/
 endef
 
 endef
