--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/minicom/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/minicom/Makefile
@@ -37,9 +37,9 @@
 endef
 
 define Package/minicom/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/$(PKG_NAME) $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/ascii-xfr $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/$(PKG_NAME) $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/ascii-xfr $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,minicom))
