--- ../../../downloads/openwrt_trunk-r38381/feeds/oldpackages/utils/cksfv/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/cksfv/Makefile
@@ -29,8 +29,8 @@
 endef
 
 define Package/cksfv/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/cksfv $(1)/usr/sbin/cksfv
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/cksfv $(1)/opt/sbin/cksfv
 endef
 
 $(eval $(call BuildPackage,cksfv))
