--- ../../../openwrt_trunk.orig/package/network/services/openvpn-easy-rsa/Makefile
+++ ../../../openwrt_trunk/package/network/services/openvpn-easy-rsa/Makefile
@@ -11,7 +11,7 @@
 
 PKG_REV:=ff5bfd1dd8e548cb24d302742af3894f893ef92f
 PKG_VERSION:=2013-01-30
-PKG_RELEASE=2
+PKG_RELEASE=2a
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/OpenVPN/easy-rsa.git
@@ -33,9 +33,9 @@
 endef
 
 define Package/openvpn-easy-rsa/conffiles
-/etc/easy-rsa/keys/serial
-/etc/easy-rsa/keys/index.txt
-/etc/easy-rsa/vars
+/opt/etc/easy-rsa/keys/serial
+/opt/etc/easy-rsa/keys/index.txt
+/opt/etc/easy-rsa/vars
 endef
 
 define Build/Configure
@@ -47,14 +47,14 @@
 endef
 
 define Package/openvpn-easy-rsa/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_BUILD_DIR)/easy-rsa/2.0/{build-*,clean-all,inherit-inter,list-crl,pkitool,revoke-full,sign-req,whichopensslcnf} $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/easy-rsa
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/openssl-1.0.0.cnf $(1)/etc/easy-rsa/openssl-1.0.0.cnf
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/vars $(1)/etc/easy-rsa/vars
-	$(INSTALL_DIR) $(1)/etc/easy-rsa/keys
-	$(INSTALL_DATA) files/easy-rsa.index $(1)/etc/easy-rsa/keys/index.txt
-	$(INSTALL_DATA) files/easy-rsa.serial $(1)/etc/easy-rsa/keys/serial
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(CP) $(PKG_BUILD_DIR)/easy-rsa/2.0/{build-*,clean-all,inherit-inter,list-crl,pkitool,revoke-full,sign-req,whichopensslcnf} $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/easy-rsa
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/openssl-1.0.0.cnf $(1)/opt/etc/easy-rsa/openssl-1.0.0.cnf
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/vars $(1)/opt/etc/easy-rsa/vars
+	$(INSTALL_DIR) $(1)/opt/etc/easy-rsa/keys
+	$(INSTALL_DATA) files/easy-rsa.index $(1)/opt/etc/easy-rsa/keys/index.txt
+	$(INSTALL_DATA) files/easy-rsa.serial $(1)/opt/etc/easy-rsa/keys/serial
 endef
 
 $(eval $(call BuildPackage,openvpn-easy-rsa))
--- ../../../openwrt_trunk.orig/package/network/services/openvpn-easy-rsa/patches/100-run-ootb.patch	2013-11-20 21:41:47.774441278 +0400
+++ ../../../openwrt_trunk/package/network/services/openvpn-easy-rsa/patches/100-run-ootb.patch	2013-12-15 03:33:16.417440787 +0400
@@ -8 +8 @@
-+"/usr/sbin/pkitool" --interact --initca $*
++"/opt/sbin/pkitool" --interact --initca $*
@@ -14 +14 @@
-+. /etc/easy-rsa/vars
++. /opt/etc/easy-rsa/vars
@@ -26 +26 @@
-+"/usr/sbin/pkitool" --interact --inter $*
++"/opt/sbin/pkitool" --interact --inter $*
@@ -34 +34 @@
-+"/usr/sbin/pkitool" --interact $*
++"/opt/sbin/pkitool" --interact $*
@@ -42 +42 @@
-+"/usr/sbin/pkitool" --interact --pass $*
++"/opt/sbin/pkitool" --interact --pass $*
@@ -50 +50 @@
-+"/usr/sbin/pkitool" --interact --pkcs12 $*
++"/opt/sbin/pkitool" --interact --pkcs12 $*
@@ -58 +58 @@
-+"/usr/sbin/pkitool" --interact --server $*
++"/opt/sbin/pkitool" --interact --server $*
@@ -66 +66 @@
-+"/usr/sbin/pkitool" --interact --csr $*
++"/opt/sbin/pkitool" --interact --csr $*
@@ -74 +74 @@
-+"/usr/sbin/pkitool" --interact --csr --pass $*
++"/opt/sbin/pkitool" --interact --csr --pass $*
@@ -80 +80 @@
-+. /etc/easy-rsa/vars
++. /opt/etc/easy-rsa/vars
@@ -90 +90 @@
-+. /etc/easy-rsa/vars
++. /opt/etc/easy-rsa/vars
@@ -100 +100 @@
-+. /etc/easy-rsa/vars
++. /opt/etc/easy-rsa/vars
@@ -110 +110 @@
-+. /etc/easy-rsa/vars
++. /opt/etc/easy-rsa/vars
@@ -120 +120 @@
-+. /etc/easy-rsa/vars
++. /opt/etc/easy-rsa/vars
@@ -132 +132 @@
-+"/usr/sbin/pkitool" --interact --sign $*
++"/opt/sbin/pkitool" --interact --sign $*
@@ -140 +140 @@
-+export EASY_RSA="/etc/easy-rsa"
++export EASY_RSA="/opt/etc/easy-rsa"
@@ -149 +149 @@
-+export KEY_CONFIG=`/usr/sbin/whichopensslcnf $EASY_RSA`
++export KEY_CONFIG=`/opt/sbin/whichopensslcnf $EASY_RSA`
