--- ../../../downloads/openwrt_trunk-r40423/feeds/packages/net/polipo/Makefile	2014-05-17 10:46:06.133735902 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/polipo/Makefile	2014-05-22 10:31:01.157885256 +0400
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=polipo
-PKG_VERSION:=1.0.4.1
-PKG_RELEASE:=3
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://freehaven.net/~chrisd/polipo/
-PKG_MD5SUM:=bfc5c85289519658280e093a270d6703
+PKG_SOURCE_URL:=http://www.pps.univ-paris-diderot.fr/~jch/software/files/polipo/
+PKG_MD5SUM:=86939e331e656f638271f578b6e3f893
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -37,23 +37,21 @@
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		CC="$(TARGET_CC)" \
 		CFLAGS="$(TARGET_CFLAGS)" \
+		LDFLAGS="$(TARGET_LDFLAGS)" \
 		polipo
 endef
 
 define Package/polipo/conffiles
-/etc/config/polipo
-/etc/polipo/config
+/opt/etc/polipo/config
 endef
 
 define Package/polipo/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/polipo $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/polipo
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/config.sample $(1)/etc/polipo/config
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/polipo.config $(1)/etc/config/polipo
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/polipo.init $(1)/etc/init.d/polipo
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/polipo $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/polipo
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/config.sample $(1)/opt/etc/polipo/config
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S28polipo $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,polipo))
