--- ../../../openwrt_trunk.orig/package/libs/lzo/Makefile
+++ ../../../openwrt_trunk/package/libs/lzo/Makefile
@@ -45,14 +45,14 @@
 MAKE_FLAGS += CFLAGS_O="$(TARGET_CFLAGS)"
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/lzo $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblzo2.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/lzo $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/liblzo2.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/liblzo/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblzo2.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/liblzo2.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,liblzo))
