--- ../../../downloads/openwrt_trunk-r43593/package/libs/lzo/Makefile	2014-12-10 12:26:33.282951674 +0300
+++ ../../../openwrt_trunk/package/libs/lzo/Makefile	2014-12-10 12:49:06.838925485 +0300
@@ -50,14 +50,14 @@
 endif
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/lzo $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblzo2.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/lzo $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/liblzo2.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/liblzo/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblzo2.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/liblzo2.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,liblzo))
