--- a/rsnapshot.conf.default.in
+++ b/rsnapshot.conf.default.in
@@ -26,3 +26,3 @@
 #
-snapshot_root	/.snapshots/
+snapshot_root	/opt/tmp/backups
 
@@ -43,3 +43,3 @@
 #
-#@CMD_CP@
+@CMD_CP@
 
@@ -56,3 +56,3 @@
 #
-#@CMD_SSH@
+@CMD_SSH@
 
@@ -60,3 +60,3 @@
 #
-@CMD_LOGGER@
+#@CMD_LOGGER@
 
@@ -66,3 +66,3 @@
 #
-#@CMD_DU@
+@CMD_DU@
 
@@ -70,3 +70,3 @@
 #
-#cmd_rsnapshot_diff	/usr/local/bin/rsnapshot-diff
+cmd_rsnapshot_diff	/opt/bin/rsnapshot-diff
 
@@ -88,3 +88,3 @@
 
-interval	hourly	6
+interval	hourly	12
 interval	daily	7
@@ -115,3 +115,3 @@
 #
-#logfile	/var/log/rsnapshot
+logfile	/opt/var/log/rsnapshot
 
@@ -122,3 +122,3 @@
 #
-lockfile	/var/run/rsnapshot.pid
+lockfile	/opt/var/run/rsnapshot.pid
 
@@ -194,6 +194,18 @@
 
+# ROUTER_STATE
+backup	/opt/etc/	router_state/
+backup	/opt/var/	router_state/
+#backup	/tmp/	router_state/
+
+exclude	/opt/var/run/*
+exclude	/tmp/var/run/*
+exclude	/opt/var/lock/*
+exclude	/tmp/var/lock/*
+exclude	/tmp/*.script
+exclude	/tmp/ppp/*
+
 # LOCALHOST
-backup	/home/		localhost/
-backup	/etc/		localhost/
-backup	/usr/local/	localhost/
+#backup	/home/		localhost/
+#backup	/etc/		localhost/
+#backup	/usr/local/	localhost/
 #backup	/var/log/rsnapshot		localhost/
