--- ../../../downloads/openwrt_trunk-r34568/feeds/oldpackages/utils/rsnapshot/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/rsnapshot/Makefile
@@ -40,7 +40,10 @@
 		+perlbase-selectsaver \
 		+perlbase-symbol \
 		+perlbase-tie \
-		+perlbase-xsloader
+		+perlbase-xsloader \
+		+coreutils-cp \
+		+coreutils-du \
+		+coreutils-rm
 endef
 
 define Package/rsnapshot/description
@@ -52,22 +55,25 @@
   space of one full backup, plus incrementals.
 endef
 
-CONFIGURE_ARGS += --with-cp=/bin/cp \
-		  --with-rsync=/usr/bin/rsync \
-		  --with-rm=/bin/rm \
-		  --with-ssh=/usr/bin/ssh \
-		  --with-logger=/usr/bin/logger \
-		  --with-du=/usr/bin/du
+CONFIGURE_ARGS += --without-logger \
+
+CONFIGURE_VARS += ac_cv_path_PERL=/opt/bin/perl \
+		  ac_cv_path_RSYNC=/opt/bin/rsync \
+		  ac_cv_path_CP=/opt/bin/cp \
+		  ac_cv_path_RM=/opt/bin/rm \
+		  ac_cv_path_DU=/opt/bin/du \
+		  ac_cv_path_SSH=/opt/bin/ssh \
 
 define Package/rsnapshot/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/$(PKG_NAME) $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/etc/$(PKG_NAME).conf.default $(1)/etc/$(PKG_NAME).conf
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME) $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/$(PKG_NAME)-diff $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/etc/$(PKG_NAME).conf.default $(1)/opt/etc/$(PKG_NAME).conf
 endef
 
 define Package/rsnapshot/conffiles
-/etc/$(PKG_NAME).conf
+/opt/etc/$(PKG_NAME).conf
 endef
 
 $(eval $(call BuildPackage,rsnapshot))
