--- ../../../downloads/openwrt_trunk-r44497/feeds/oldpackages/net/stunnel/Makefile	2015-02-24 12:08:57.265768241 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/net/stunnel/Makefile	2015-02-27 12:35:37.897020244 +0300
@@ -8,14 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=stunnel
-PKG_VERSION:=4.33
+PKG_VERSION:=5.10
 PKG_RELEASE:=1
 
-PKG_SOURCE_URL:=http://www.stunnel.org/download/stunnel/src/
+PKG_SOURCE_URL:=https://www.stunnel.org/downloads
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_MD5SUM:=559a864066d8cc4afd8a97682c90d41c
+PKG_MD5SUM:=a0edda805eb7d6ea600a230fb0979ea1
 
-PKG_FIXUP:=autoreconf
+#PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
@@ -23,7 +23,7 @@
 define Package/stunnel
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libopenssl +libwrap
+  DEPENDS:=+libopenssl +libwrap +libssp
   TITLE:=SSL TCP Wrapper
   URL:=http://www.stunnel.org/
 endef
@@ -37,29 +37,27 @@
 endef
 
 define Package/stunnel/conffiles
-/etc/stunnel/stunnel.conf
+/opt/etc/stunnel/stunnel.conf
 endef
 
 CONFIGURE_ARGS+= \
 	--with-random=/dev/urandom \
 	--with-threads=fork \
-	--with-ssl=$(STAGING_DIR)/usr \
+	--with-ssl=$(STAGING_DIR)/opt \
 
 define Build/Compile
-	mkdir -p $(PKG_INSTALL_DIR)/etc/stunnel
-	echo '#dummy' > $(PKG_INSTALL_DIR)/etc/stunnel/stunnel.pem
+	mkdir -p $(PKG_INSTALL_DIR)/opt/etc/stunnel
+	echo '#dummy' > $(PKG_INSTALL_DIR)/opt/etc/stunnel/stunnel.pem
 	$(call Build/Compile/Default)
 endef
 
 define Package/stunnel/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/stunnel $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/stunnel
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/stunnel/libstunnel.so $(1)/usr/lib/stunnel/
-	$(INSTALL_DIR) $(1)/etc/stunnel
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/stunnel/stunnel.conf-sample $(1)/etc/stunnel/stunnel.conf
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/stunnel.init $(1)/etc/init.d/stunnel
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/stunnel $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib/stunnel
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/stunnel/libstunnel.so $(1)/opt/lib/stunnel/
+	$(INSTALL_DIR) $(1)/opt/etc/stunnel
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/stunnel/stunnel.conf-sample $(1)/opt/etc/stunnel/stunnel.conf
 endef
 
 $(eval $(call BuildPackage,stunnel))
