--- ../../../downloads/openwrt_trunk-r45513/feeds/oldpackages/net/weechat/Makefile	2015-04-23 10:21:55.665457452 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/net/weechat/Makefile	2015-05-22 14:44:21.528487363 +0300
@@ -8,14 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=weechat
-PKG_VERSION:=0.4.3
+PKG_VERSION:=1.1.1
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=http://www.weechat.org/files/src
-PKG_MD5SUM:=07500f0d1054691d88f21dd12b7c5871
-
-PKG_FIXUP:=autoreconf
+PKG_MD5SUM:=3bb8d27301262bc434de9bcf7d77dce7
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/nls.mk
@@ -27,7 +25,7 @@
   CATEGORY:=Network
   TITLE:=Lightweight IRC client
   URL:=http://www.weechat.org/
-  DEPENDS:=+libcurl +libgnutls +libgcrypt +zlib +libncursesw $(ICONV_DEPENDS)
+  DEPENDS:=+libcurl +libgnutls +libgcrypt +zlib +libncursesw $(ICONV_DEPENDS) $(INTL_DEPENDS) +libaspell +perl +libpthread +python-base
 endef
 
 define Package/weechat/description
@@ -35,24 +33,34 @@
 endef
 
 CMAKE_OPTIONS+= \
-	-DENABLE_PYTHON=OFF \
-	-DENABLE_PERL=OFF \
+	-DENABLE_PYTHON=ON \
+	-DENABLE_PYTHON3=OFF \
+	-DPYTHON_EXECUTABLE="$(STAGING_DIR)/host/bin/python2.7" \
+	-DPYTHON_INCLUDE_PATH="$(STAGING_DIR)/opt/include/python2.7" \
+	-DPYTHON_LIBRARY="$(STAGING_DIR)/opt/lib/libpython2.7.so" \
+	-DENABLE_PERL=ON \
+	-DPERL_EXECUTABLE:FILEPATH="$(STAGING_DIR)/host/bin/perl" \
+	-DPERL_INCLUDE_PATH="$(STAGING_DIR)/opt/lib/perl5/5.20/CORE" \
+	-DPERL_LIBRARY="$(STAGING_DIR)/opt/lib/perl5/5.20/CORE/libperl.so" \
 	-DENABLE_LUA=OFF \
 	-DENABLE_GUILE=OFF \
 	-DENABLE_RUBY=OFF \
 	-DENABLE_TCL=OFF
 
 TARGET_LDFLAGS += \
-	-L$(ICONV_PREFIX)/lib -ldl \
-	-Wl,-rpath-link,$(STAGING_DIR)/usr/lib
+	-L$(ICONV_PREFIX)/lib -ldl -lintl -liconv
+
+CONFIGURE_ARGS+= \
+	--with-libintl-prefix="$(INTL_PREFIX)"
 
 define Package/weechat/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/weechat-curses $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/weechat/plugins
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/weechat-curses $(1)/opt/bin/
+	ln -sf weechat-curses $(1)/opt/bin/weechat
+	$(INSTALL_DIR) $(1)/opt/lib/weechat/plugins
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/weechat/plugins/* \
-		$(1)/usr/lib/weechat/plugins/
+		$(PKG_INSTALL_DIR)/opt/lib/weechat/plugins/* \
+		$(1)/opt/lib/weechat/plugins/
 endef
 
 $(eval $(call BuildPackage,weechat))
