--- ../../../openwrt_trunk.orig/feeds/packages/libs/st2205tool/Makefile	2011-12-05 10:46:45.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/st2205tool/Makefile	2011-12-21 10:45:38.064415923 +0400
@@ -42,18 +42,18 @@
 
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/libst2205/st2205.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libst2205/libst2205.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_BUILD_DIR)/libst2205/st2205.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libst2205/libst2205.{a,so*} $(1)/opt/lib/
 endef
 
 
 define Package/st2205tool/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/setpic/setpic $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/libst2205/libst2205.so $(1)/usr/lib/libst2205.so.1
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/setpic/setpic $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/libst2205/libst2205.so $(1)/opt/lib/libst2205.so.1
 endef
 
 $(eval $(call BuildPackage,st2205tool))
