--- ../../../trunk_clean/feeds/packages/net/opensips/Makefile	2012-04-02 11:00:38.284164879 +0400
+++ ../../../trunk/feeds/packages/net/opensips/Makefile	2012-04-03 11:37:19.285151331 +0400
@@ -58,8 +58,8 @@
 endef
 
 define Package/opensips-example/conffiles
-/etc/default/opensips
-/etc/opensips/opensips.cfg
+/opt/etc/default/opensips
+/opt/etc/opensips/opensips.cfg
 endef
 
 
@@ -70,7 +70,7 @@
 endef
 
 define Package/opensips-tools/conffiles
-/etc/opensips/opensipsctlrc
+/opt/etc/opensips/opensipsctlrc
 endef
 
 
@@ -89,11 +89,11 @@
 	snmpstats \
 
 PKG_MAKE_ARGS:= \
-	cfg-dir="etc/opensips/" \
-	bin-dir="usr/sbin/" \
-	data-dir="var/lib/opensips/" \
-	lib-dir="usr/lib/opensips/" \
-	modules-dir="usr/lib/opensips/modules/" \
+	cfg-dir="opt/etc/opensips/" \
+	bin-dir="opt/sbin/" \
+	data-dir="opt/var/lib/opensips/" \
+	lib-dir="opt/lib/opensips/" \
+	modules-dir="opt/lib/opensips/modules/" \
 	exclude_modules="$(PKG_EXCL_MODULES)" \
 
 
@@ -104,9 +104,10 @@
 		CC="$(TARGET_CC)" \
 		ARCH="$(ARCH)"  \
 		CFLAGS="$(TARGET_CFLAGS)" \
+		LD_EXTRA_OPTS="$(TARGET_LDFLAGS)" \
 		ISSMP="no" \
-		LOCALBASE="$(STAGING_DIR)/usr" \
-		SYSBASE="$(STAGING_DIR)/usr" \
+		LOCALBASE="$(STAGING_DIR)/opt" \
+		SYSBASE="$(STAGING_DIR)/opt" \
 		PCRECFG= \
 		all
 	$(MAKE) -C $(PKG_BUILD_DIR) $(PKG_MAKE_ARGS) \
@@ -115,42 +116,42 @@
 endef
 
 define Package/opensips/install
-	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/sbin/opensips \
-		$(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib/opensips/modules
+		$(PKG_INSTALL_DIR)/opt/sbin/opensips \
+		$(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib/opensips/modules
 endef
 
 define Package/opensips-example/install
-	$(INSTALL_DIR) $(1)/etc/opensips
+	$(INSTALL_DIR) $(1)/opt/etc/opensips
 	$(CP) \
-		$(PKG_INSTALL_DIR)/etc/opensips/opensips.cfg \
-		$(1)/etc/opensips/
+		$(PKG_INSTALL_DIR)/opt/etc/opensips/opensips.cfg \
+		$(1)/opt/etc/opensips/
 	$(SED) 's,$(PKG_INSTALL_DIR),,g' \
-		$(1)/etc/opensips/opensips.cfg
-	$(INSTALL_DIR) $(1)/etc/default
-	$(INSTALL_DATA) ./files/opensips.default $(1)/etc/default/opensips
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/opensips.init $(1)/etc/init.d/opensips
+		$(1)/opt/etc/opensips/opensips.cfg
+#	$(INSTALL_DIR) $(1)/opt/etc/default
+#	$(INSTALL_DATA) ./files/opensips.default $(1)/opt/etc/default/opensips
+#	$(INSTALL_DIR) $(1)/opt/etc/init.d
+#	$(INSTALL_BIN) ./files/opensips.init $(1)/opt/etc/init.d/opensips
 endef
 
 define Package/opensips-tools/install
-	$(INSTALL_DIR) $(1)/etc/opensips
+	$(INSTALL_DIR) $(1)/opt/etc/opensips
 	$(CP) \
-		$(PKG_INSTALL_DIR)/etc/opensips/opensipsctlrc \
-		$(1)/etc/opensips/
-	$(INSTALL_DIR) $(1)/usr/sbin
+		$(PKG_INSTALL_DIR)/opt/etc/opensips/opensipsctlrc \
+		$(1)/opt/etc/opensips/
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/sbin/opensips{ctl,dbctl,unix} \
-		$(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/lib/opensips
+		$(PKG_INSTALL_DIR)/opt/sbin/opensips{ctl,dbctl,unix} \
+		$(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/lib/opensips
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/opensips/opensipsctl \
-		$(1)/usr/lib/opensips/
+		$(PKG_INSTALL_DIR)/opt/lib/opensips/opensipsctl \
+		$(1)/opt/lib/opensips/
 	$(SED) 's,/bin/bash,/bin/sh,g' \
 		-e 's,$(PKG_INSTALL_DIR),,g' \
-		$(1)/usr/sbin/opensips{ctl,dbctl}
+		$(1)/opt/sbin/opensips{ctl,dbctl}
 endef
 
 define BuildPlugin
@@ -161,9 +162,9 @@
   endef
 
   define Package/opensips-mod-$(1)/install
-	[ -z "$(2)" ] || $(INSTALL_DIR) $$(1)/usr/lib/opensips/modules/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/opensips/modules/$(2).so \
-			$$(1)/usr/lib/opensips/modules/ ;
+	[ -z "$(2)" ] || $(INSTALL_DIR) $$(1)/opt/lib/opensips/modules/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/opensips/modules/$(2).so \
+			$$(1)/opt/lib/opensips/modules/ ;
   endef
 
   $$(eval $$(call BuildPackage,opensips-mod-$(1)))
