--- ../../../downloads/openwrt_trunk-r43718/feeds/oldpackages/lang/python-cjson/Makefile	2014-11-07 23:35:24.502715101 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/python-cjson/Makefile	2014-11-28 11:39:40.718705877 +0300
@@ -35,13 +35,13 @@
 endef
 
 define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix="$(PKG_INSTALL_DIR)/usr")
+	$(call Build/Compile/PyMod,,install --prefix="$(PKG_INSTALL_DIR)/Apps/opt")
 endef
 
 define Package/python-cjson/install
 	$(INSTALL_DIR) $(1)$(PYTHON_PKG_DIR)
 	$(CP) \
-	    $(PKG_INSTALL_DIR)$(PYTHON_PKG_DIR)/* \
+	    $(PKG_INSTALL_DIR)/Apps$(PYTHON_PKG_DIR)/* \
 	    $(1)$(PYTHON_PKG_DIR)
 endef
 
