--- ../../../downloads/openwrt_trunk-q46338/feeds/packages/net/squid/Makefile	2015-07-21 13:57:43.000000000 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/squid/Makefile	2015-07-22 17:25:47.367679546 +0300
@@ -51,9 +51,9 @@
 
 CONFIGURE_ARGS += \
 	--config-cache \
-	--datadir=/usr/share/squid \
-	--libexecdir=/usr/lib/squid \
-	--sysconfdir=/etc/squid \
+	--datadir=/Apps/opt/share/squid \
+	--libexecdir=/Apps/opt/lib/squid \
+	--sysconfdir=/Apps/opt/etc/squid \
 	--enable-shared \
 	--disable-static \
 	--enable-icmp \
@@ -78,7 +78,7 @@
 	--without-expat \
 	--without-libxml2 \
 	--without-nettle \
-	--with-openssl=$(STAGING_DIR)/usr \
+	--with-openssl=$(STAGING_DIR)/opt \
 	--enable-epoll \
 	--with-maxfd=4096 \
 	--disable-external-acl-helpers \
@@ -89,43 +89,45 @@
 	--disable-arch-native \
 	--with-krb5-config=no \
 	--without-libcap \
-	--without-netfilter-conntrack
+	--without-netfilter-conntrack \
 
 CONFIGURE_VARS += \
 	ac_cv_header_linux_netfilter_ipv4_h=yes \
-	ac_cv_epoll_works=yes
+	ac_cv_epoll_works=yes \
+
+TARGET_CXXFLAGS += -std=c++0x
 
 define Build/Compile
-	+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR)/lib all
-	+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) \
+	$(MAKE) -C $(PKG_BUILD_DIR)/lib \
+		$(MAKE_VARS) \
+		all
+	$(MAKE) -C $(PKG_BUILD_DIR) \
+		$(MAKE_VARS) \
 		DESTDIR="$(PKG_INSTALL_DIR)" \
 		install
 endef
 
 define Package/squid/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/squid $(1)/usr/sbin/
-
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/squid.config $(1)/etc/config/squid
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/sbin/squid $(1)/opt/sbin/
 
-	$(INSTALL_DIR) $(1)/etc/squid
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/squid/mime.conf $(1)/etc/squid/
-	$(INSTALL_CONF) ./files/squid.conf $(1)/etc/squid/
+	$(INSTALL_DIR) $(1)/opt/etc/squid
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/Apps/opt/etc/squid/mime.conf $(1)/opt/etc/squid/
+	$(INSTALL_CONF) ./files/squid.conf $(1)/opt/etc/squid/
 
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) ./files/squid.init $(1)/etc/init.d/squid
+	$(INSTALL_DIR) $(1)/opt/etc/init.d/
+	$(INSTALL_BIN) ./files/S22squid $(1)/opt/etc/init.d/
 
-	$(INSTALL_DIR) $(1)/usr/share/squid/icons/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/squid/icons/* $(1)/usr/share/squid/icons/
+	$(INSTALL_DIR) $(1)/opt/share/squid/icons/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/share/squid/icons/* $(1)/opt/share/squid/icons/
 
-	$(INSTALL_DIR) $(1)/usr/share/squid/errors/templates/
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/squid/errors/templates/* $(1)/usr/share/squid/errors/templates/
+	$(INSTALL_DIR) $(1)/opt/share/squid/errors/templates/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/share/squid/errors/templates/* $(1)/opt/share/squid/errors/templates/
 endef
 
 define Package/squid-mod-cachemgr/install
-	$(INSTALL_DIR) $(1)/www/cgi-bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/squid/cachemgr.cgi $(1)/www/cgi-bin/
+	$(INSTALL_DIR) $(1)/opt/share/www/cgi-bin/
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/squid/cachemgr.cgi $(1)/opt/share/www/cgi-bin/
 endef
 
 $(eval $(call BuildPackage,squid))
