diff -urx .svn ../../../downloads/openwrt_trunk-r42803/feeds/packages/admin/zabbix/Makefile ../../../openwrt_trunk/feeds/packages/admin/zabbix/Makefile
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/admin/zabbix/Makefile	2014-10-06 15:46:23.769250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/Makefile	2014-10-07 12:25:25.959896439 +0400
@@ -29,6 +29,7 @@
   SECTION:=admin
   CATEGORY:=Administration
   TITLE:=Zabbix
+  DEPENDS:=$(ICONV_DEPENDS)
   URL:=http://www.zabbix.com/
   SUBMENU:=zabbix
   MAINTAINER:=Etienne CHAMPETIER <champetier.etienne@gmail.com>
@@ -92,7 +93,7 @@
 
 define Package/zabbix-extra-network/description
 An extra package for zabbix-agentd that adds a discovery rule for openwrt network interfaces.
-The idea here is to discover only interfaces listed in /etc/config/network (discover br-lan and not eth0.1 and wlan0)
+The idea here is to discover only interfaces listed in /opt/etc/config/network (discover br-lan and not eth0.1 and wlan0)
 See http://wiki.openwrt.org/doc/howto/zabbix for ready to use zabbix templates.
 endef
 
@@ -107,66 +108,66 @@
 	--enable-server \
 	--enable-proxy \
 	--disable-java \
-	--with-sqlite3="$(STAGING_DIR)/usr"
+	--with-sqlite3="$(STAGING_DIR)/opt"
 
 MAKE_FLAGS += ARCH="linux"
 
 define Package/zabbix/install/sbin
 	$(INSTALL_DIR) \
-		$(1)/usr/sbin
+		$(1)/opt/sbin
 
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/sbin/zabbix_$(2) \
-		$(1)/usr/sbin/
+		$(PKG_INSTALL_DIR)/opt/sbin/zabbix_$(2) \
+		$(1)/opt/sbin/
 endef
 
 define Package/zabbix/install/bin
 	$(INSTALL_DIR) \
-		$(1)/usr/bin
+		$(1)/opt/bin
 
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/bin/zabbix_$(2) \
-		$(1)/usr/bin/
+		$(PKG_INSTALL_DIR)/opt/bin/zabbix_$(2) \
+		$(1)/opt/bin/
 endef
 
 define Package/zabbix/install/etc
 	$(INSTALL_DIR) \
-		$(1)/etc
+		$(1)/opt/etc
 
 	$(INSTALL_CONF) \
-		$(PKG_INSTALL_DIR)/etc/zabbix_$(2).conf \
-		$(1)/etc/
+		$(PKG_INSTALL_DIR)/opt/etc/zabbix_$(2).conf \
+		$(1)/opt/etc/
 endef
 
 define Package/zabbix/install/init.d
 	$(INSTALL_DIR) \
-		$(1)/etc/init.d
+		$(1)/opt/etc/init.d
 
 	$(INSTALL_BIN) \
-		./files/zabbix_$(2).init \
-		$(1)/etc/init.d/zabbix_$(2)
+		./files/S07zabbix_$(2) \
+		$(1)/opt/etc/init.d
 endef
 
 define Package/zabbix/install/zabbix.conf.d
 	$(INSTALL_DIR) \
-		$(1)/etc/zabbix_agentd.conf.d
+		$(1)/opt/etc/zabbix_agentd.conf.d
 
 	$(INSTALL_BIN) \
 		./files/$(2) \
-		$(1)/etc/zabbix_agentd.conf.d/$(2)
+		$(1)/opt/etc/zabbix_agentd.conf.d/$(2)
 endef
 
 define Package/zabbix-agent/conffiles
-/etc/zabbix_agent.conf
+/opt/etc/zabbix_agent.conf
 endef
 define Package/zabbix-agentd/conffiles
-/etc/zabbix_agentd.conf
+/opt/etc/zabbix_agentd.conf
 endef
 define Package/zabbix-server/conffiles
-/etc/zabbix_server.conf
+/opt/etc/zabbix_server.conf
 endef
 define Package/zabbix-proxy/conffiles
-/etc/zabbix_proxy.conf
+/opt/etc/zabbix_proxy.conf
 endef
 
 ifdef CONFIG_PACKAGE_zabbix-extra-mac80211
@@ -196,7 +197,7 @@
 endef
 
 define Package/zabbix-agentd/install
-	$(INSTALL_DIR) $(1)/etc/zabbix_agentd.conf.d
+	$(INSTALL_DIR) $(1)/opt/etc/zabbix_agentd.conf.d
 	$(call Package/zabbix/install/sbin,$(1),agentd)
 	$(call Package/zabbix/install/etc,$(1),agentd)
 	$(call Package/zabbix/install/init.d,$(1),agentd)
@@ -204,9 +205,9 @@
 
 define Package/zabbix-extra-mac80211/install
 	$(call Package/zabbix/install/zabbix.conf.d,$(1),mac80211)
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/zabbix-extra-mac80211/zabbix_helper_mac80211 $(1)/usr/bin/
-	chmod 4755 $(1)/usr/bin/zabbix_helper_mac80211
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/zabbix-extra-mac80211/zabbix_helper_mac80211 $(1)/opt/bin/
+	chmod 4755 $(1)/opt/bin/zabbix_helper_mac80211
 endef
 
 define Package/zabbix-extra-network/install
@@ -237,9 +238,9 @@
 
 $(eval $(call BuildPackage,zabbix-agent))
 $(eval $(call BuildPackage,zabbix-agentd))
-$(eval $(call BuildPackage,zabbix-extra-mac80211))
-$(eval $(call BuildPackage,zabbix-extra-network))
-$(eval $(call BuildPackage,zabbix-extra-wifi))
+#$(eval $(call BuildPackage,zabbix-extra-mac80211))
+#$(eval $(call BuildPackage,zabbix-extra-network))
+#$(eval $(call BuildPackage,zabbix-extra-wifi))
 $(eval $(call BuildPackage,zabbix-sender))
 $(eval $(call BuildPackage,zabbix-server))
 $(eval $(call BuildPackage,zabbix-proxy))
diff -urx .svn ../../../downloads/openwrt_trunk-r42803/feeds/packages/admin/zabbix/patches/010-change-agentd-config.patch ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/010-change-agentd-config.patch
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/admin/zabbix/patches/010-change-agentd-config.patch	2014-10-06 15:46:23.769250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/admin/zabbix/patches/010-change-agentd-config.patch	2014-10-09 13:15:23.109202608 +0400
@@ -10,8 +10,8 @@
 -# Mandatory: no
 -# Default:
 -# PidFile=/tmp/zabbix_agentd.pid
-+# Only /var/run/zabbix_agentd.pid supported
-+PidFile=/var/run/zabbix_agentd.pid
++# Only /opt/var/run/zabbix_agentd.pid supported
++PidFile=/opt/var/run/zabbix_agentd.pid
  
  ### Option: LogFile
  #	Name of log file.
@@ -56,7 +56,7 @@
  # Include=/usr/local/etc/zabbix_agentd.userparams.conf
 -# Include=/usr/local/etc/zabbix_agentd.conf.d/
  # Include=/usr/local/etc/zabbix_agentd.conf.d/*.conf
-+Include=/etc/zabbix_agentd.conf.d/
++Include=/opt/etc/zabbix_agentd.conf.d/
  
  ####### USER-DEFINED MONITORED PARAMETERS #######
  
