--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/sshfs/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/sshfs/Makefile
@@ -9,7 +9,7 @@
 
 PKG_NAME:=sshfs
 PKG_VERSION:=2.2
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-fuse-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@SF/fuse
@@ -36,12 +36,12 @@
 CONFIGURE_VARS += \
 	SSHFS_CFLAGS=" \
 		-D_FILE_OFFSET_BITS=64 \
-		-I$(STAGING_DIR)/usr/include/glib-2.0 \
-		-I$(STAGING_DIR)/usr/lib/glib-2.0/include \
-		-I$(STAGING_DIR)/usr/include/fuse" \
+		-I$(STAGING_DIR)/opt/include/glib-2.0 \
+		-I$(STAGING_DIR)/opt/lib/glib-2.0/include \
+		-I$(STAGING_DIR)/opt/include/fuse" \
 	SSHFS_LIBS=" \
 		-lglib-2.0 -liconv $(if $(INTL_FULL),-lintl) -lfuse -pthread -lgthread-2.0 \
-		-L$(STAGING_DIR)/usr/lib"
+		-L$(STAGING_DIR)/opt/lib"
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
@@ -52,8 +52,8 @@
 endef
 
 define Package/sshfs/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/sshfs $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/sshfs $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,sshfs))
