--- ../../../openwrt_trunk.orig/feeds/oldpackages/lang/pyyaml/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/pyyaml/Makefile
@@ -34,7 +34,7 @@
 endef
 
 define Build/Compile
-	$(call Build/Compile/PyMod,,--with-libyaml install --prefix="$(PKG_INSTALL_DIR)/usr")
+	$(call Build/Compile/PyMod,,--with-libyaml install --prefix=/opt --root=$(PKG_INSTALL_DIR))
 endef
 
 define Package/pyyaml/install
