--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/autossh/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/autossh/Makefile
@@ -27,24 +27,17 @@
 
 define Build/Compile
 	$(call Build/Compile/Default, -f Makefile \
-		CFLAGS="$(TARGET_CFLAGS) -Wall -D\"SSH_PATH=\\\"\$$$$(SSH)\\\"\" -D\"VER=\\\"\$$$$(VER)\\\"\"" \
+		LIBS=" -lnsl $(TARGET_LDFLAGS)" \
+		CFLAGS="$(TARGET_CFLAGS)  -Wall -D\"SSH_PATH=\\\"\$$$$(SSH)\\\"\" -D\"VER=\\\"\$$$$(VER)\\\"\"" \
 		all \
 	)
 endef
 
 define Package/autossh/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/autossh $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/autossh.init $(1)/etc/init.d/autossh
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/autossh.config $(1)/etc/config/autossh
-	$(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	$(INSTALL_DATA) ./files/autossh.hotplug $(1)/etc/hotplug.d/iface/20-autossh
-endef
-
-define Package/autossh/conffiles
-/etc/config/autossh
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/autossh $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S41autossh $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,autossh))
