--- ../../../downloads/openwrt_trunk-r40859/feeds/packages/multimedia/xupnpd/Makefile
+++ ../../../openwrt_trunk/feeds/packages/multimedia/xupnpd/Makefile
@@ -36,9 +36,9 @@
 endef
 
 define Package/xupnpd/conffiles
-/usr/share/xupnpd/xupnpd.lua
-/usr/share/xupnpd/config
-/usr/share/xupnpd/playlists
+/opt/share/xupnpd/xupnpd.lua
+/opt/share/xupnpd/config
+/opt/share/xupnpd/playlists
 endef
 
 define Package/xupnpd/description
@@ -49,23 +49,24 @@
 endef
 
 define Package/xupnpd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/xupnpd $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/xupnpd.init $(1)/etc/init.d/xupnpd
-	$(INSTALL_DIR) $(1)/usr/share/xupnpd
-	$(CP) $(PKG_BUILD_DIR)/*.lua $(1)/usr/share/xupnpd
-	$(INSTALL_DIR) $(1)/usr/share/xupnpd/ui
-	$(CP) $(PKG_BUILD_DIR)/ui/* $(1)/usr/share/xupnpd/ui
-	$(INSTALL_DIR) $(1)/usr/share/xupnpd/www
-	$(CP) $(PKG_BUILD_DIR)/www/* $(1)/usr/share/xupnpd/www
-	$(INSTALL_DIR) $(1)/usr/share/xupnpd/playlists
-	$(INSTALL_DIR) $(1)/usr/share/xupnpd/plugins
-	$(CP) $(PKG_BUILD_DIR)/plugins/* $(1)/usr/share/xupnpd/plugins
-	$(INSTALL_DIR) $(1)/usr/share/xupnpd/config
-	$(INSTALL_DIR) $(1)/etc/xupnpd
-	(cd $(1)/etc/xupnpd; ln -s ../../usr/share/xupnpd/config ./; ln -s ../../usr/share/xupnpd/xupnpd.lua ./)
-	$(INSTALL_DIR) $(1)/usr/share/xupnpd/localmedia
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/xupnpd $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S94xupnpd $(1)/opt/etc/init.d
+	$(INSTALL_DIR) $(1)/opt/share/xupnpd
+	$(CP) $(PKG_BUILD_DIR)/*.lua $(1)/opt/share/xupnpd
+	$(INSTALL_DIR) $(1)/opt/share/xupnpd/ui
+	$(CP) $(PKG_BUILD_DIR)/ui/* $(1)/opt/share/xupnpd/ui
+	$(INSTALL_DIR) $(1)/opt/share/xupnpd/www
+	$(CP) $(PKG_BUILD_DIR)/www/* $(1)/opt/share/xupnpd/www
+	$(INSTALL_DIR) $(1)/opt/share/xupnpd/playlists
+	$(CP) $(PKG_BUILD_DIR)/playlists/* $(1)/opt/share/xupnpd/playlists
+	$(INSTALL_DIR) $(1)/opt/share/xupnpd/plugins
+	$(CP) $(PKG_BUILD_DIR)/plugins/* $(1)/opt/share/xupnpd/plugins
+	$(INSTALL_DIR) $(1)/opt/share/xupnpd/config
+	$(INSTALL_DIR) $(1)/opt/share/xupnpd/localmedia
+	$(INSTALL_DIR) $(1)/opt/share/xupnpd/profiles
+	$(CP) $(PKG_BUILD_DIR)/profiles/* $(1)/opt/share/xupnpd/profiles
 endef
 
 $(eval $(call BuildPackage,xupnpd))
