--- ../../../backup/trunk_clean/feeds/packages/net/dansguardian/Makefile	2012-02-21 11:20:06.000000000 +0400
+++ ../../../trunk/feeds/packages/net/dansguardian/Makefile	2012-02-21 11:23:43.990937351 +0400
@@ -29,13 +29,13 @@
 endef
 
 define Package/dansguardian/conffiles
-/etc/dansguardian/dansguardian.conf
+/opt/etc/dansguardian/dansguardian.conf
 endef
 
 CONFIGURE_VARS += \
 	INCLUDES="" \
 	CXXFLAGS="$$$$CXXFLAGS -fno-builtin -fno-rtti -nostdinc++"  \
-	CPPFLAGS="$$$$CPPFLAGS -I$(STAGING_DIR)/usr/include/uClibc++" \
+	CPPFLAGS="$$$$CPPFLAGS -I$(STAGING_DIR)/opt/include/uClibc++" \
 	LDFLAGS="$$$$LDFLAGS" \
 	LIBS="-nodefaultlibs -lc -luClibc++ -lpthread $(LIBGCC_S)" \
 
@@ -50,14 +50,15 @@
 endef
 
 define Package/dansguardian/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/dansguardian $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/dansguardian $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/share/dansguardian
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/share/dansguardian/transparent1x1.gif $(1)/usr/share/dansguardian/
-	$(INSTALL_DIR) $(1)/usr/share/dansguardian/languages/ukenglish
-	$(CP) $(PKG_INSTALL_DIR)/usr/share/dansguardian/languages/ukenglish/* $(1)/usr/share/dansguardian/languages/ukenglish/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/dansguardian $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/dansguardian $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/share/dansguardian
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/share/dansguardian/transparent1x1.gif $(1)/opt/share/dansguardian/
+	$(INSTALL_DIR) $(1)/opt/share/dansguardian/languages/ukenglish
+	$(CP) $(PKG_INSTALL_DIR)/opt/share/dansguardian/languages/ukenglish/* $(1)/opt/share/dansguardian/languages/ukenglish/
+	$(INSTALL_DIR) $(1)/opt/var/run
 endef
 
 $(eval $(call BuildPackage,dansguardian))
