--- ../../../downloads/openwrt_trunk-r35474/feeds/oldpackages/libs/libvncserver/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libvncserver/Makefile
@@ -45,15 +45,15 @@
 	--without-avahi
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/{lib,include,bin}
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib* $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/{lib,include,bin}
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib* $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
 endef
 
 define Package/libvncserver/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib*so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib*so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libvncserver))
