diff -urx .svn ../../../downloads/openwrt_trunk-r43593/feeds/packages/net/privoxy/files/privoxy.config ../../../openwrt_trunk/feeds/packages/net/privoxy/files/privoxy.config
--- ../../../downloads/openwrt_trunk-r43593/feeds/packages/net/privoxy/files/privoxy.config	2014-12-10 12:26:33.006951679 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/privoxy/files/privoxy.config	2014-12-11 11:45:53.973415884 +0300
@@ -15,8 +15,8 @@
 # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 #
 config	privoxy	'privoxy'
-	option	confdir		'/etc/privoxy'
-	option	logdir		'/var/log'
+	option	confdir		'/opt/etc/privoxy'
+	option	logdir		'/opt/var/log'
 	option	logfile		'privoxy.log'
 	list	filterfile	'default.filter'
 	list	actionsfile	'match-all.action'
diff -urx .svn ../../../downloads/openwrt_trunk-r43593/feeds/packages/net/privoxy/files/privoxy.oldconfig ../../../openwrt_trunk/feeds/packages/net/privoxy/files/privoxy.oldconfig
--- ../../../downloads/openwrt_trunk-r43593/feeds/packages/net/privoxy/files/privoxy.oldconfig	2014-12-10 12:26:33.006951679 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/privoxy/files/privoxy.oldconfig	2014-12-11 11:47:10.369413885 +0300
@@ -4,8 +4,8 @@
 # it's converted and moved to uci configuration
 # please look at /etc/config/privoxy
 #
-confdir /etc/privoxy
-logdir /var/log
+confdir /opt/etc/privoxy
+logdir /opt/var/log
 logfile privoxy.log
 filterfile default.filter
 actionsfile match-all.action # Actions that are applied to all sites and maybe overruled later on.
diff -urx .svn ../../../downloads/openwrt_trunk-r43593/feeds/packages/net/privoxy/Makefile ../../../openwrt_trunk/feeds/packages/net/privoxy/Makefile
--- ../../../downloads/openwrt_trunk-r43593/feeds/packages/net/privoxy/Makefile	2014-12-10 12:26:33.006951679 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/privoxy/Makefile	2014-12-11 11:52:50.001414553 +0300
@@ -47,122 +47,26 @@
 endef
 
 CONFIGURE_ARGS += \
-	--sysconfdir=/etc/privoxy \
+	--sysconfdir=/opt/etc/privoxy \
 
 # needed otherwise errors during compile
 MAKE_FLAGS:=
 
-define Build/Install
-	$(call Build/Install/Default,)
-	# rename original sample config from pkg_source to save existing one during install
-	mv $(PKG_INSTALL_DIR)/etc/privoxy/config $(PKG_INSTALL_DIR)/etc/privoxy/config.privoxy
-endef
-
 define Package/privoxy/conffiles
-/etc/config/privoxy
-/etc/privoxy/config	# temporary needed if updating from old version
-endef
-
-define Package/privoxy/preinst
-	#!/bin/sh
-	# if run within buildroot exit
-	[ -n "$${IPKG_INSTROOT}" ] && exit 0
-	# stop service # if PKG_UPGRADE NOT WORKING
-	# [ "$${PKG_UPGRADE}" = "1" ] && /etc/init.d/privoxy stop
-	/etc/init.d/privoxy stop >/dev/null 2>&1
-	exit 0	# supress errors from stop command
+/opt/etc/privoxy/config	# temporary needed if updating from old version
 endef
 
 define Package/privoxy/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/privoxy $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/privoxy.init $(1)/etc/init.d/privoxy
-	# no longer needed because running with procd
-	# $(INSTALL_DIR) $(1)/etc/hotplug.d/iface
-	# $(INSTALL_BIN) ./files/privoxy.hotplug $(1)/etc/hotplug.d/iface/80-privoxy
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(CP) $(PKG_INSTALL_DIR)/opt/sbin/privoxy $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S24privoxy $(1)/opt/etc/init.d/
 
-	$(INSTALL_DIR) $(1)/etc/privoxy
-	$(CP) $(PKG_INSTALL_DIR)/etc/privoxy/* $(1)/etc/privoxy/
+	$(INSTALL_DIR) $(1)/opt/etc/privoxy
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/privoxy/* $(1)/opt/etc/privoxy/
 	# temporary needed if updating from old version
 	# otherwise old config file will be delete by opkg
-	$(INSTALL_CONF) ./files/privoxy.oldconfig $(1)/etc/privoxy/config
-	# create .old file to be removed with next pacakge builds during update
-	$(INSTALL_CONF) ./files/privoxy.oldconfig $(1)/etc/privoxy/config.old
-
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/privoxy.config $(1)/etc/config/privoxy
-endef
-
-define Package/privoxy/postinst
-	#!/bin/sh
-
-	# if fresh install we don't need old config file in privoxy directory
-	[ "$${PKG_UPGRADE}" = "0" ] && rm -f /etc/privoxy/config
-
-	# if run within buildroot exit here
-	[ -n "$${IPKG_INSTROOT}" ] && exit 0
-
-	# if PKG_UPGRADE then build uci configuration
-	# from existing(?) old /etc/privoxy/config file
-	if [ "$${PKG_UPGRADE}" = "1" -a -f /etc/privoxy/config ]; then
-
-		echo "converting OLD config to NEW uci configuration"
-
-		SECTION="privoxy.privoxy"
-		CFGFILE=/etc/privoxy/config
-
-		echo -n > /etc/config/privoxy		# clear/create uci configuration file
-		cp -f $${CFGFILE} $${CFGFILE}.old	# save old configuration
-
-		# cleanup
-		sed -i 's/^[ \t]*//;s/[ \t]*$$//' $${CFGFILE}	# remove invisible chars at beginning and end of lines
-		sed -i '/^#/d' $${CFGFILE}			# remove lines with "#"
-		sed -i '/^$$/d' $${CFGFILE}			# remove empty lines
-
-		uci -q set $${SECTION}="privoxy"		# create section
-
-		cat $${CFGFILE} | while read LINE; do
-			# option is first parameter; uci did not like "-" in option names
-			OPT=$$(echo $${LINE} | awk '{print $$1}' | sed 's/-/_/g')
-			VAL=$$(echo $${LINE} | awk '{print $$2}')
-			case $${OPT} in
-				# debug 1024 => debug_1024 '1'
-				debug)
-					uci -q set $${SECTION}.debug_$${VAL}="1"
-					;;
-				# handle list values; splitted case for better reading
-				actionsfile|filterfile|listen_address)
-					uci -q add_list $${SECTION}.$${OPT}="$${VAL}"
-					;;
-				permit_access|deny_access)
-					uci -q add_list $${SECTION}.$${OPT}="$${VAL}"
-					;;
-				trust_info_url|forward)
-					uci -q add_list $${SECTION}.$${OPT}="$${VAL}"
-					;;
-				forward_socks4|forward_socks4a)
-					uci -q add_list $${SECTION}.$${OPT}="$${VAL}"
-					;;
-				forward_socks5|forward_socks5t)
-					uci -q add_list $${SECTION}.$${OPT}="$${VAL}"
-					;;
-				# all others are normal options
-				*)
-					uci -q set $${SECTION}.$${OPT}="$${VAL}"
-					;;
-			esac
-		done
-		uci -q commit privoxy	# commit changes
-		rm -f $${CFGFILE}	# remove old configuration file
-	fi
-
-	# set permissions to privoxy group
-	echo "setting permissions"
-	chgrp -R privoxy /etc/privoxy/*
-	chmod 664 /etc/privoxy/*
-	chmod 755 /etc/privoxy/templates
-	chmod 644 /etc/privoxy/templates/*
+	$(INSTALL_CONF) ./files/privoxy.oldconfig $(1)/opt/etc/privoxy/config
 endef
 
 $(eval $(call BuildPackage,privoxy))
