diff -urx .svn ../../../downloads/openwrt_trunk-r36556/feeds/oldpackages/net/privoxy/files/privoxy.config ../../../openwrt_trunk/feeds/oldpackages/net/privoxy/files/privoxy.config
--- ../../../downloads/openwrt_trunk-r36556/feeds/oldpackages/net/privoxy/files/privoxy.config	2013-05-06 15:25:41.763204416 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/privoxy/files/privoxy.config	2013-05-06 15:36:22.059192743 +0400
@@ -1,5 +1,5 @@
-confdir /etc/privoxy
-logdir /var/log
+confdir /opt/etc/privoxy
+logdir /opt/var/log
 filterfile default.filter
 logfile privoxy
 actionsfile match-all.action # Actions that are applied to all sites and maybe overruled later on.
diff -urx .svn ../../../downloads/openwrt_trunk-r36556/feeds/oldpackages/net/privoxy/Makefile ../../../openwrt_trunk/feeds/oldpackages/net/privoxy/Makefile
--- ../../../downloads/openwrt_trunk-r36556/feeds/oldpackages/net/privoxy/Makefile	2013-05-06 15:25:41.763204416 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/net/privoxy/Makefile	2013-05-06 15:36:22.059192743 +0400
@@ -43,22 +43,22 @@
 endef
 
 CONFIGURE_ARGS += \
-	--sysconfdir=/etc/privoxy \
+	--sysconfdir=/opt/etc/privoxy \
 
 MAKE_FLAGS :=
 
 define Package/privoxy/conffiles
-/etc/privoxy/config
+/opt/etc/privoxy/config
 endef
 
 define Package/privoxy/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/privoxy $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/privoxy
-	$(CP) $(PKG_INSTALL_DIR)/etc/privoxy/* $(1)/etc/privoxy/
-	$(INSTALL_CONF) ./files/privoxy.config $(1)/etc/privoxy/config
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/privoxy.init $(1)/etc/init.d/privoxy
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(CP) $(PKG_INSTALL_DIR)/opt/sbin/privoxy $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/privoxy
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/privoxy/* $(1)/opt/etc/privoxy/
+	$(INSTALL_CONF) ./files/privoxy.config $(1)/opt/etc/privoxy/config
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S24privoxy $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,privoxy))
