--- ../../../downloads/openwrt_trunk-r44497/feeds/packages/net/privoxy/Makefile	2015-02-24 12:08:56.437768256 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/privoxy/Makefile	2015-02-25 10:17:39.204317181 +0300
@@ -9,7 +9,7 @@
 
 PKG_NAME:=privoxy
 PKG_VERSION:=3.0.23
-PKG_RELEASE:=3
+PKG_RELEASE:=3a
 
 PKG_SOURCE:=privoxy-$(PKG_VERSION)-stable-src.tar.gz
 PKG_SOURCE_URL:=@SF/ijbswa
@@ -58,52 +58,39 @@
 endef
 
 CONFIGURE_ARGS += \
-	--sysconfdir=/etc/privoxy
+	--sysconfdir=/opt/etc/privoxy
 
 # needed otherwise errors during compile
 MAKE_FLAGS:=
 
 define Package/$(PKG_NAME)/conffiles
-/etc/config/privoxy
-/etc/privoxy/user.action
-/etc/privoxy/user.filter
-/etc/privoxy/user.trust
-endef
-
-define Package/$(PKG_NAME)/preinst
-	#!/bin/sh
-	[ -n "$${IPKG_INSTROOT}" ] && exit 0	# if run within buildroot exit
-
-	# stop service if PKG_UPGRADE
-	[ "$${PKG_UPGRADE}" = "1" ] && /etc/init.d/privoxy stop >/dev/null 2>&1
-		
-	exit 0	# supress errors from stop command
+/opt/etc/config/privoxy
+/opt/etc/privoxy/user.action
+/opt/etc/privoxy/user.filter
+/opt/etc/privoxy/user.trust
 endef
 
 define Package/$(PKG_NAME)/install
-	if [ -f $(PKG_INSTALL_DIR)/etc/privoxy/trust ]; then \
-		mv -f $(PKG_INSTALL_DIR)/etc/privoxy/trust $(PKG_INSTALL_DIR)/etc/privoxy/user.trust; \
+	if [ -f $(PKG_INSTALL_DIR)/opt/etc/privoxy/trust ]; then \
+		mv -f $(PKG_INSTALL_DIR)/opt/etc/privoxy/trust $(PKG_INSTALL_DIR)/opt/etc/privoxy/user.trust; \
 	fi
-	if [ -f $(PKG_INSTALL_DIR)/etc/privoxy/config ]; then \
-		rm -f $(PKG_INSTALL_DIR)/etc/privoxy/config; \
+	if [ -f $(PKG_INSTALL_DIR)/opt/etc/privoxy/config ]; then \
+		rm -f $(PKG_INSTALL_DIR)/opt/etc/privoxy/config; \
 	fi
 
 	$(INSTALL_DIR) \
-		$(1)/usr/sbin\
-		$(1)/etc/privoxy/templates
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/privoxy $(1)/usr/sbin/
-	find $(PKG_INSTALL_DIR)/etc/privoxy/templates -maxdepth 1 -type f \
-		-exec $(INSTALL_DATA) -t $(1)/etc/privoxy/templates {} \;
-	find $(PKG_INSTALL_DIR)/etc/privoxy -maxdepth 1 -type f \
-		-exec install -m0664 -t $(1)/etc/privoxy {} \;
+		$(1)/opt/sbin\
+		$(1)/opt/etc/privoxy/templates
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/privoxy $(1)/opt/sbin/
+	find $(PKG_INSTALL_DIR)/opt/etc/privoxy/templates -maxdepth 1 -type f \
+		-exec $(INSTALL_DATA) -t $(1)/opt/etc/privoxy/templates {} \;
+	find $(PKG_INSTALL_DIR)/opt/etc/privoxy -maxdepth 1 -type f \
+		-exec install -m0664 -t $(1)/opt/etc/privoxy {} \;
+	$(INSTALL_CONF) ./files/privoxy.oldconfig $(1)/opt/etc/privoxy/config
 
 	$(INSTALL_DIR) \
-		$(1)/etc/init.d \
-		$(1)/etc/hotplug.d/iface \
-		$(1)/etc/config
-	$(INSTALL_BIN)  ./files/privoxy.init $(1)/etc/init.d/privoxy
-	$(INSTALL_BIN)  ./files/privoxy.hotplug $(1)/etc/hotplug.d/iface/80-privoxy
-	$(INSTALL_CONF) ./files/privoxy.config $(1)/etc/config/privoxy
+		$(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S24privoxy $(1)/opt/etc/init.d/
 endef
 
 $(eval $(call BuildPackage,$(PKG_NAME)))
