--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/git/Makefile	2014-10-06 15:46:23.757250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/git/Makefile	2014-10-07 11:59:10.135925121 +0400
@@ -36,35 +36,27 @@
  with speed and efficiency.
 endef
 
-MAKE_FLAGS := \
-	CC="$(TARGET_CC)" \
-	CFLAGS="$(TARGET_CFLAGS)" \
-	CPPFLAGS="$(TARGET_CPPFLAGS)" \
-	LDFLAGS="$(TARGET_LDFLAGS)" \
-	NO_CURL="YesPlease" \
-	NO_EXPAT="YesPlease" \
-	NO_MKSTEMPS="YesPlease" \
-	NO_GETTEXT="YesPlease" \
-	NO_UNIX_SOCKETS="YesPlease" \
-	NO_IPV6="YesPlease" \
-	NO_ICONV="YesPlease" \
-	NO_NSEC="YesPlease" \
-	NO_PERL="YesPlease" \
-	NO_PYTHON="YesPlease" \
-	NO_TCLTK="YesPlease"
+CONFIGURE_ARGS += \
+	--without-iconv \
+	--with-gitconfig=/opt/etc/gitconfig \
+	--with-gitattributes=/opt/etc/gitattributes \
+	--without-python \
+	--without-tcltk \
 
-define Build/Configure
-	$(MAKE) -C $(PKG_BUILD_DIR) \
-		configure
+CONFIGURE_VARS += \
+	ac_cv_fread_reads_directories=no \
+	ac_cv_snprintf_returns_bogus=no
 
-	( cd $(PKG_BUILD_DIR); \
-		./configure --prefix=/usr \
-	);
-endef
+MAKE_FLAGS += \
+	NO_INSTALL_HARDLINKS="Yes" \
+	NO_PERL="Yes"
 
 define Package/git/install
 	$(INSTALL_DIR) $(1)
-	$(RM) $(PKG_INSTALL_DIR)/usr/bin/git-cvsserver
+	for app in git git-cvsserver git-shell git-upload-pack; do \
+	    $(RM) $(PKG_INSTALL_DIR)/opt/bin/$$$${app} ; \
+	    $(LN) -s /opt/lib/git-core/$$$${app} $(PKG_INSTALL_DIR)/opt/bin/$$$${app} ; \
+	done
 	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
 endef
 
