--- a/magick/blob.c	2011-01-23 06:42:39.000000000 +0300
+++ b/magick/blob.c	2012-07-26 13:29:13.781798109 +0400
@@ -506,7 +506,7 @@
     case ZipStream:
     {
 #if defined(MAGICKCORE_ZLIB_DELEGATE)
-      (void) gzerror(image->blob->file,&status);
+      (void) gzerror((gzFile) image->blob->file,&status);
 #endif
       break;
     }
@@ -547,7 +547,7 @@
     case ZipStream:
     {
 #if defined(MAGICKCORE_ZLIB_DELEGATE)
-      status=gzclose(image->blob->file);
+      status=gzclose((gzFile) image->blob->file);
 #endif
       break;
     }
@@ -2768,12 +2768,13 @@
       {
         default:
         {
-          count=(ssize_t) gzread(image->blob->file,q,(unsigned int) length);
+          count=(ssize_t) gzread((gzFile) image->blob->file,q,(unsigned int)
+            length);
           break;
         }
         case 2:
         {
-          c=gzgetc(image->blob->file);
+          c=gzgetc((gzFile) image->blob->file);
           if (c == EOF)
             break;
           *q++=(unsigned char) c;
@@ -2781,7 +2782,7 @@
         }
         case 1:
         {
-          c=gzgetc(image->blob->file);
+          c=gzgetc((gzFile) image->blob->file);
           if (c == EOF)
             break;
           *q++=(unsigned char) c;
@@ -3525,7 +3526,7 @@
     case ZipStream:
     {
 #if defined(MAGICKCORE_ZLIB_DELEGATE)
-      if (gzseek(image->blob->file,(off_t) offset,whence) < 0)
+      if (gzseek((gzFile) image->blob->file,(off_t) offset,whence) < 0)
         return(-1);
 #endif
       image->blob->offset=TellBlob(image);
@@ -3789,7 +3790,7 @@
     case ZipStream:
     {
 #if defined(MAGICKCORE_ZLIB_DELEGATE)
-      status=gzflush(image->blob->file,Z_SYNC_FLUSH);
+      status=gzflush((gzFile) image->blob->file,Z_SYNC_FLUSH);
 #endif
       break;
     }
@@ -3863,7 +3864,7 @@
     case ZipStream:
     {
 #if defined(MAGICKCORE_ZLIB_DELEGATE)
-      offset=(MagickOffsetType) gztell(image->blob->file);
+      offset=(MagickOffsetType) gztell((gzFile) image->blob->file);
 #endif
       break;
     }
@@ -4012,20 +4013,20 @@
       {
         default:
         {
-          count=(ssize_t) gzwrite(image->blob->file,(void *) data,
+          count=(ssize_t) gzwrite((gzFile) image->blob->file,(void *) data,
             (unsigned int) length);
           break;
         }
         case 2:
         {
-          c=gzputc(image->blob->file,(int) *p++);
+          c=gzputc((gzFile) image->blob->file,(int) *p++);
           if (c == EOF)
             break;
           count++;
         }
         case 1:
         {
-          c=gzputc(image->blob->file,(int) *p++);
+          c=gzputc((gzFile) image->blob->file,(int) *p++);
           if (c == EOF)
             break;
           count++;
