--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/libs/protobuf/Makefile	2014-04-09 16:17:24.302286481 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/protobuf/Makefile	2014-04-11 14:55:22.055224648 +0400
@@ -41,33 +41,33 @@
 
 define Build/InstallDev
 	$(INSTALL_DIR) \
-		$(1)/usr/lib \
-		$(1)/usr/include
+		$(1)/opt/lib \
+		$(1)/opt/include
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/* \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/* \
+		$(1)/opt/include/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/* \
+		$(1)/opt/lib/
 endef
 
 define Package/protobuf/install
 	$(INSTALL_DIR) \
-		$(1)/usr/lib
+		$(1)/opt/lib
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotoc.so*  \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotoc.so*  \
+		$(1)/opt/lib/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotobuf-lite.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotobuf-lite.so* \
+		$(1)/opt/lib/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotobuf.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotobuf.so* \
+		$(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,protobuf))
