--- ../../../downloads/openwrt_trunk-r43708/feeds/packages/libs/protobuf/Makefile	2014-12-15 12:19:58.008388958 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/protobuf/Makefile	2014-12-15 14:37:00.736242757 +0300
@@ -50,33 +50,33 @@
 
 define Build/InstallDev
 	$(INSTALL_DIR) \
-		$(1)/usr/lib \
-		$(1)/usr/include
+		$(1)/opt/lib \
+		$(1)/opt/include
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/* \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/* \
+		$(1)/opt/include/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/* \
+		$(1)/opt/lib/
 endef
 
 define Package/protobuf/install
 	$(INSTALL_DIR) \
-		$(1)/usr/lib
+		$(1)/opt/lib
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotoc.so*  \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotoc.so*  \
+		$(1)/opt/lib/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotobuf-lite.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotobuf-lite.so* \
+		$(1)/opt/lib/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotobuf.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotobuf.so* \
+		$(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,protobuf))
