--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/libs/protobuf/Makefile
+++ ../../../openwrt_trunk/feeds/packages/libs/protobuf/Makefile
@@ -43,39 +43,40 @@
 endef
 
 define Host/Install
+	$(INSTALL_BIN) $(HOST_BUILD_DIR)/src/protoc $(STAGING_DIR_HOST)/bin
 endef
 
 CONFIGURE_ARGS += --with-protoc=$(HOST_BUILD_DIR)/src/protoc
 
 define Build/InstallDev
 	$(INSTALL_DIR) \
-		$(1)/usr/lib \
-		$(1)/usr/include
+		$(1)/opt/lib \
+		$(1)/opt/include
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/include/* \
-		$(1)/usr/include/
+		$(PKG_INSTALL_DIR)/opt/include/* \
+		$(1)/opt/include/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/* \
+		$(1)/opt/lib/
 endef
 
 define Package/protobuf/install
 	$(INSTALL_DIR) \
-		$(1)/usr/lib
+		$(1)/opt/lib
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotoc.so*  \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotoc.so*  \
+		$(1)/opt/lib/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotobuf-lite.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotobuf-lite.so* \
+		$(1)/opt/lib/
 
 	$(CP) \
-		$(PKG_INSTALL_DIR)/usr/lib/libprotobuf.so* \
-		$(1)/usr/lib/
+		$(PKG_INSTALL_DIR)/opt/lib/libprotobuf.so* \
+		$(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,protobuf))
