--- ../../../downloads/openwrt_trunk-r42803/package/utils/util-linux/Makefile
+++ ../../../openwrt_trunk/package/utils/util-linux/Makefile
@@ -45,22 +45,22 @@
 		DESTDIR="$(1)" \
 		installdirs install-data
 
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_BUILD_DIR)/libblkid/blkid.pc $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_BUILD_DIR)/libmount/mount.pc $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_BUILD_DIR)/libuuid/uuid.pc $(1)/usr/lib/pkgconfig
-
-	$(INSTALL_DIR) $(1)/usr/include/blkid
-	$(CP) $(PKG_BUILD_DIR)/libblkid/src/blkid.h $(1)/usr/include/blkid
-	$(INSTALL_DIR) $(1)/usr/include/libmount
-	$(CP) $(PKG_BUILD_DIR)/libmount/src/libmount.h $(1)/usr/include/libmount
-	$(INSTALL_DIR) $(1)/usr/include/uuid
-	$(CP) $(PKG_BUILD_DIR)/libuuid/src/uuid.h $(1)/usr/include/uuid
-
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libblkid.{a,so*} $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libmount.{a,so*} $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libuuid.{a,so*} $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_BUILD_DIR)/libblkid/blkid.pc $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_BUILD_DIR)/libmount/mount.pc $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_BUILD_DIR)/libuuid/uuid.pc $(1)/opt/lib/pkgconfig
+
+	$(INSTALL_DIR) $(1)/opt/include/blkid
+	$(CP) $(PKG_BUILD_DIR)/libblkid/src/blkid.h $(1)/opt/include/blkid
+	$(INSTALL_DIR) $(1)/opt/include/libmount
+	$(CP) $(PKG_BUILD_DIR)/libmount/src/libmount.h $(1)/opt/include/libmount
+	$(INSTALL_DIR) $(1)/opt/include/uuid
+	$(CP) $(PKG_BUILD_DIR)/libuuid/src/uuid.h $(1)/opt/include/uuid
+
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libblkid.{a,so*} $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libmount.{a,so*} $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libuuid.{a,so*} $(1)/opt/lib
 endef
 
 define Package/libblkid
@@ -336,7 +336,7 @@
 define Package/swap-utils
 $(call Package/util-linux/Default)
   TITLE:=swap space management utilities
-  DEPENDS+= +libblkid
+  DEPENDS+= +libblkid +libmount
   SUBMENU:=disc
 endef
 
@@ -404,159 +404,161 @@
 endef
 
 define Package/libblkid/install
-        $(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libblkid.so.* $(1)/usr/lib/
+        $(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libblkid.so.* $(1)/opt/lib/
 endef
 
 define Package/libmount/install
-        $(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libmount.so.* $(1)/usr/lib/
+        $(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libmount.so.* $(1)/opt/lib/
 endef
 
 define Package/libuuid/install
-        $(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/.libs/libuuid.so.* $(1)/usr/lib/
+        $(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/.libs/libuuid.so.* $(1)/opt/lib/
 endef
 
 define Package/agetty/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/agetty $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/agetty $(1)/opt/sbin/
 endef
 
 define Package/blkid/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/blkid $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/blkid $(1)/opt/sbin/
 endef
 
 define Package/cal/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/cal $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/cal $(1)/opt/bin/
 endef
 
 define Package/cfdisk/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/cfdisk $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/cfdisk $(1)/opt/sbin/
 endef
 
 define Package/fdisk/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/fdisk $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/fdisk $(1)/opt/sbin/
 endef
 
 define Package/findfs/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/findfs $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/findfs $(1)/opt/sbin/
 endef
 
 define Package/flock/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/flock $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/flock $(1)/opt/bin/
 endef
 
 define Package/getopt/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/getopt $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/getopt $(1)/opt/bin/
 endef
 
 define Package/hwclock/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/hwclock $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/hwclock $(1)/opt/sbin/
 endef
 
 define Package/logger/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/logger $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/logger $(1)/opt/bin/
 endef
 
 define Package/look/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/look $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/look $(1)/opt/bin/
 endef
 
 define Package/losetup/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/losetup $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/losetup $(1)/opt/sbin/
 endef
 
 define Package/lsblk/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/lsblk $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/lsblk $(1)/opt/bin/
 endef
 
 define Package/mcookie/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mcookie $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/mcookie $(1)/opt/bin/
 endef
 
 define Package/mount-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/{u,}mount $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/mountpoint $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/findmnt $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/{u,}mount $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/mountpoint $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/findmnt $(1)/opt/bin/
 endef
 
 define Package/namei/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/namei $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/namei $(1)/opt/bin/
 endef
 
 define Package/rename/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rename $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rename $(1)/opt/bin/
 endef
 
 define Package/partx-utils/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/partx $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/addpart $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/delpart $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/partx $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/addpart $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/delpart $(1)/opt/sbin/
 endef
 
 define Package/script-utils/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/script $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scriptreplay $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/script $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scriptreplay $(1)/opt/bin/
 endef
 
 define Package/setterm/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/setterm $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/setterm $(1)/opt/bin/
 endef
 
 define Package/sfdisk/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sfdisk $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sfdisk $(1)/opt/sbin/
 endef
 
 define Package/swap-utils/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/mkswap $(1)/usr/sbin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/swaplabel $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/mkswap $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/swaplabel $(1)/opt/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/swapon $(1)/opt/sbin/
+	$(LN) swapon $(1)/opt/sbin/swapoff
 endef
 
 define Package/uuidd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/uuidd $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/uuidd $(1)/opt/sbin/
 endef
 
 define Package/uuidgen/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/uuidgen $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/uuidgen $(1)/opt/bin/
 endef
 
 define Package/wall/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/wall $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/wall $(1)/opt/bin/
 endef
 
 define Package/whereis/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/whereis $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/whereis $(1)/opt/bin/
 endef
 
 define Package/wipefs/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/wipefs $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/.libs/wipefs $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,libblkid))
