--- ../../../downloads/openwrt_trunk-r42803/feeds/telephony/net/asterisk-11.x-chan-sccp-b/Makefile
+++ ../../../openwrt_trunk/feeds/telephony/net/asterisk-11.x-chan-sccp-b/Makefile
@@ -38,7 +38,7 @@
 endef
 
 define Build/Configure
-	$(call Build/Configure/Default,--with-asterisk=$(STAGING_DIR)/usr/include/asterisk-11)
+	$(call Build/Configure/Default,--with-asterisk=$(STAGING_DIR)/opt/include/asterisk-11)
 endef
 
 define Build/Compile
@@ -49,14 +49,14 @@
 endef
 
 define Package/asterisk11-chan-sccp-b/conffiles
-/etc/asterisk/sccp.conf
+/opt/etc/asterisk/sccp.conf
 endef
 
 define Package/asterisk11-chan-sccp-b/install
-	$(INSTALL_DIR) $(1)/etc/asterisk
-	$(CP) ./files/sccp.openwrt.conf $(1)/etc/asterisk/sccp.conf
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/modules
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/asterisk/modules/chan_sccp.so $(1)/usr/lib/asterisk/modules/
+	$(INSTALL_DIR) $(1)/opt/etc/asterisk
+	$(CP) ./files/sccp.openwrt.conf $(1)/opt/etc/asterisk/sccp.conf
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/modules
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/asterisk/modules/chan_sccp.so $(1)/opt/lib/asterisk/modules/
 endef
 
 $(eval $(call BuildPackage,asterisk11-chan-sccp-b))
