--- ../../../downloads/openwrt_trunk-r43718/feeds/oldpackages/net/matrixtunnel/Makefile	2015-01-09 19:51:33.795205887 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/net/matrixtunnel/Makefile	2015-01-09 22:44:05.189058089 +0300
@@ -37,6 +37,8 @@
 	based on matrixssl library.
 endef
 
+TARGET_CFLAGS += -DLINUX
+
 define Build/Configure
 	$(call Build/Configure/Default, \
 		--without-libiconv-prefix \
@@ -46,8 +48,8 @@
 endef
 
 define Package/matrixtunnel/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/matrixtunnel $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/bin/matrixtunnel $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,matrixtunnel))
