--- ../../../openwrt_trunk.orig/feeds/oldpackages/lang/php-pear/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/php-pear/Makefile
@@ -35,9 +35,9 @@
 endef
 
 define Package/php-pear/install
-	$(INSTALL_DIR) $(1)/usr/lib/php
+	$(INSTALL_DIR) $(1)/opt/lib/php
 	$(CP)	$(PKG_BUILD_DIR)/PEAR.php $(PKG_BUILD_DIR)/PEAR \
-		$(1)/usr/lib/php/
+		$(1)/opt/lib/php/
 endef
 
 $(eval $(call BuildPackage,php-pear))
