--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/curlftpfs/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/curlftpfs/Makefile
@@ -33,7 +33,7 @@
 
 EXTRA_CFLAGS += \
 	-I$(ICONV_PREFIX)/include \
-	-I$(STAGING_DIR)/usr/include/fuse \
+	-I$(STAGING_DIR)/opt/include/fuse \
 	-DICONV_CONST=const
 
 EXTRA_LDFLAGS += \
@@ -42,8 +42,8 @@
 	-lpthread
 
 define Package/curlftpfs/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/curlftpfs $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/curlftpfs $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,curlftpfs))
