--- ../../../downloads/openwrt_trunk-q47008/feeds/packages/lang/python-setuptools/Makefile	2015-09-24 20:31:27.302958283 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/python-setuptools/Makefile	2015-09-27 18:10:20.710449431 +0300
@@ -36,7 +36,7 @@
 
 define Build/Compile
 	$(call Build/Compile/PyMod,,\
-		install --root="$(PKG_INSTALL_DIR)" --prefix="/usr" \
+		install --root="$(PKG_INSTALL_DIR)" --prefix="/opt" \
 		--single-version-externally-managed \
 	)
 endef
@@ -49,8 +49,11 @@
 endef
 
 define PyPackage/python-setuptools/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/* $(1)/opt/lib/
+	
 endef
 
 $(eval $(call PyPackage,python-setuptools))
