--- ../../../downloads/openwrt_trunk-r46940/feeds/packages/lang/python-setuptools/Makefile	2015-09-15 12:24:40.722451661 +0300
+++ ../../../openwrt_trunk/feeds/packages/lang/python-setuptools/Makefile	2015-09-21 11:04:04.490999386 +0300
@@ -36,7 +36,7 @@
 
 define Build/Compile
 	$(call Build/Compile/PyMod,,\
-		install --root="$(PKG_INSTALL_DIR)" --prefix="/usr" \
+		install --root="$(PKG_INSTALL_DIR)" --prefix="/opt" \
 		--single-version-externally-managed \
 	)
 endef
@@ -49,8 +49,8 @@
 endef
 
 define PyPackage/python-setuptools/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin/
 endef
 
 $(eval $(call PyPackage,python-setuptools))
