--- ../../../downloads/openwrt_trunk-r35474/package/network/services/openvpn/Makefile	2013-02-05 10:42:43.814945812 +0400
+++ ../../../openwrt_trunk/package/network/services/openvpn/Makefile	2013-02-06 20:42:01.148716861 +0400
@@ -9,7 +9,6 @@
 
 PKG_NAME:=openvpn
 
-PKG_REV:=56cffde5d5320e0b1ec364d3e486aca9
 PKG_VERSION:=2.3.0
 PKG_RELEASE=1
 
@@ -31,7 +30,7 @@
   URL:=http://openvpn.net
   SUBMENU:=VPN
   MENU:=1
-  DEPENDS:=+kmod-tun +OPENVPN_$(1)_ENABLE_LZO:liblzo +OPENVPN_$(1)_ENABLE_IPROUTE2:ip $(3)
+  DEPENDS:=+OPENVPN_$(1)_ENABLE_LZO:liblzo +OPENVPN_$(1)_ENABLE_IPROUTE2:ip $(3)
   VARIANT:=$(1)
   MAINTAINER:=Mirko Vogt <mirko@openwrt.org>
 endef
@@ -48,19 +47,6 @@
 Package/openvpn-polarssl/config=$(call Package/openvpn/config/Default,polarssl)
 Package/openvpn-nossl/config=$(call Package/openvpn/config/Default,nossl)
 
-define Package/openvpn-easy-rsa
-  $(call Package/openvpn)
-  DEPENDS:=+openssl-util
-  TITLE:=Simple shell scripts to manage a Certificate Authority
-endef
-
-define Package/openvpn-easy-rsa/conffiles
-/etc/easy-rsa/keys/serial
-/etc/easy-rsa/keys/index.txt
-/etc/easy-rsa/vars
-endef
-
-
 ifeq ($(BUILD_VARIANT),polarssl)
 CONFIG_OPENVPN_POLARSSL:=y
 endif
@@ -75,7 +61,7 @@
 	IFCONFIG=/sbin/ifconfig \
 	ROUTE=/sbin/route \
 	IPROUTE=/usr/sbin/ip \
-	NETSTAT=/sbin/netstat
+	NETSTAT=/bin/netstat
 
 define Build/Configure
 	$(call Build/Configure/Default, \
@@ -104,45 +90,26 @@
 endef
 
 define Package/openvpn-$(BUILD_VARIANT)/conffiles
-/etc/config/openvpn
+/opt/etc/config/openvpn
 endef
 
 define Package/openvpn-$(BUILD_VARIANT)/install
 	$(INSTALL_DIR) \
-		$(1)/usr/sbin \
-		$(1)/etc/init.d \
-		$(1)/etc/config \
-		$(1)/etc/openvpn \
-		$(1)/lib/upgrade/keep.d
+		$(1)/opt/sbin \
+		$(1)/opt/etc/init.d \
+		$(1)/opt/etc/config \
+		$(1)/opt/etc/openvpn
 
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/sbin/openvpn \
-		$(1)/usr/sbin/
+		$(PKG_INSTALL_DIR)/opt/sbin/openvpn \
+		$(1)/opt/sbin/
 
 	$(INSTALL_BIN) \
-		files/openvpn.init \
-		$(1)/etc/init.d/openvpn
-
-	$(INSTALL_CONF) files/openvpn.config \
-		$(1)/etc/config/openvpn
-
-	$(INSTALL_DATA) \
-		files/openvpn.upgrade \
-		$(1)/lib/upgrade/keep.d/openvpn
-endef
+		files/S20openvpn \
+		$(1)/opt/etc/init.d
 
-define Package/openvpn-easy-rsa/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_BUILD_DIR)/easy-rsa/2.0/{build-*,clean-all,inherit-inter,list-crl,pkitool,revoke-full,sign-req,whichopensslcnf} $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/easy-rsa
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/openssl-1.0.0.cnf $(1)/etc/easy-rsa/openssl-1.0.0.cnf
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/vars $(1)/etc/easy-rsa/vars
-	$(INSTALL_DIR) $(1)/etc/easy-rsa/keys
-	$(INSTALL_DATA) files/easy-rsa.index $(1)/etc/easy-rsa/keys/index.txt
-	$(INSTALL_DATA) files/easy-rsa.serial $(1)/etc/easy-rsa/keys/serial
 endef
 
 $(eval $(call BuildPackage,openvpn-openssl))
 $(eval $(call BuildPackage,openvpn-polarssl))
 $(eval $(call BuildPackage,openvpn-nossl))
-$(eval $(call BuildPackage,openvpn-easy-rsa))
