--- ../../../downloads/openwrt_trunk-r35474/package/network/services/openvpn/Makefile	2013-02-05 09:11:14.427045805 +0400
+++ ../../../openwrt_trunk/package/network/services/openvpn/Makefile	2013-02-05 10:08:46.634998773 +0400
@@ -31,7 +31,7 @@
   URL:=http://openvpn.net
   SUBMENU:=VPN
   MENU:=1
-  DEPENDS:=+kmod-tun +OPENVPN_$(1)_ENABLE_LZO:liblzo +OPENVPN_$(1)_ENABLE_IPROUTE2:ip $(3)
+  DEPENDS:=+OPENVPN_$(1)_ENABLE_LZO:liblzo +OPENVPN_$(1)_ENABLE_IPROUTE2:ip $(3)
   VARIANT:=$(1)
   MAINTAINER:=Mirko Vogt <mirko@openwrt.org>
 endef
@@ -55,9 +55,9 @@
 endef
 
 define Package/openvpn-easy-rsa/conffiles
-/etc/easy-rsa/keys/serial
-/etc/easy-rsa/keys/index.txt
-/etc/easy-rsa/vars
+/opt/opt/etc/easy-rsa/keys/serial
+/opt/opt/etc/easy-rsa/keys/index.txt
+/opt/opt/etc/easy-rsa/vars
 endef
 
 
@@ -75,7 +75,7 @@
 	IFCONFIG=/sbin/ifconfig \
 	ROUTE=/sbin/route \
 	IPROUTE=/usr/sbin/ip \
-	NETSTAT=/sbin/netstat
+	NETSTAT=/bin/netstat
 
 define Build/Configure
 	$(call Build/Configure/Default, \
@@ -104,42 +104,35 @@
 endef
 
 define Package/openvpn-$(BUILD_VARIANT)/conffiles
-/etc/config/openvpn
+/opt/etc/config/openvpn
 endef
 
 define Package/openvpn-$(BUILD_VARIANT)/install
 	$(INSTALL_DIR) \
-		$(1)/usr/sbin \
-		$(1)/etc/init.d \
-		$(1)/etc/config \
-		$(1)/etc/openvpn \
-		$(1)/lib/upgrade/keep.d
+		$(1)/opt/sbin \
+		$(1)/opt/etc/init.d \
+		$(1)/opt/etc/config \
+		$(1)/opt/etc/openvpn
 
 	$(INSTALL_BIN) \
-		$(PKG_INSTALL_DIR)/usr/sbin/openvpn \
-		$(1)/usr/sbin/
+		$(PKG_INSTALL_DIR)/opt/sbin/openvpn \
+		$(1)/opt/sbin/
 
 	$(INSTALL_BIN) \
-		files/openvpn.init \
-		$(1)/etc/init.d/openvpn
+		files/S20openvpn \
+		$(1)/opt/etc/init.d
 
-	$(INSTALL_CONF) files/openvpn.config \
-		$(1)/etc/config/openvpn
-
-	$(INSTALL_DATA) \
-		files/openvpn.upgrade \
-		$(1)/lib/upgrade/keep.d/openvpn
 endef
 
 define Package/openvpn-easy-rsa/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_BUILD_DIR)/easy-rsa/2.0/{build-*,clean-all,inherit-inter,list-crl,pkitool,revoke-full,sign-req,whichopensslcnf} $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/easy-rsa
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/openssl-1.0.0.cnf $(1)/etc/easy-rsa/openssl-1.0.0.cnf
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/vars $(1)/etc/easy-rsa/vars
-	$(INSTALL_DIR) $(1)/etc/easy-rsa/keys
-	$(INSTALL_DATA) files/easy-rsa.index $(1)/etc/easy-rsa/keys/index.txt
-	$(INSTALL_DATA) files/easy-rsa.serial $(1)/etc/easy-rsa/keys/serial
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(CP) $(PKG_BUILD_DIR)/easy-rsa/2.0/{build-*,clean-all,inherit-inter,list-crl,pkitool,revoke-full,sign-req,whichopensslcnf} $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/easy-rsa
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/openssl-1.0.0.cnf $(1)/opt/etc/easy-rsa/openssl-1.0.0.cnf
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/easy-rsa/2.0/vars $(1)/opt/etc/easy-rsa/vars
+	$(INSTALL_DIR) $(1)/opt/etc/easy-rsa/keys
+	$(INSTALL_DATA) files/easy-rsa.index $(1)/opt/etc/easy-rsa/keys/index.txt
+	$(INSTALL_DATA) files/easy-rsa.serial $(1)/opt/etc/easy-rsa/keys/serial
 endef
 
 $(eval $(call BuildPackage,openvpn-openssl))
