--- ../../../openwrt_trunk.orig/feeds/telephony/libs/spandsp/Makefile	2012-07-11 14:54:00.690752516 +0400
+++ ../../../openwrt_trunk/feeds/telephony/libs/spandsp/Makefile	2012-07-11 21:27:58.545584546 +0400
@@ -33,14 +33,14 @@
 TARGET_CFLAGS += $(FPIC)
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/{lib,include}
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libspandsp* $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/{lib,include}
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libspandsp* $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
 endef
 
 define Package/libspandsp/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libspandsp*so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libspandsp*so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libspandsp))
