diff -ur ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/netatalk/files/afpd.conf ../../../openwrt_trunk/feeds/packages/net/netatalk/files/afpd.conf
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/netatalk/files/afpd.conf	2014-10-06 15:46:23.753250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/netatalk/files/afpd.conf	2014-10-07 11:59:31.599924730 +0400
@@ -1 +1 @@
-- -noddp -uampath /usr/lib/uams -uamlist uams_guest.so,uams_passwd.so,uams_dhx_passwd.so,uams_randnum.so,uams_dhx2.so -passwdfile /etc/netatalk/afppasswd -savepassword -passwdminlen 0 -nosetpassword -defaultvol /etc/netatalk/AppleVolumes.default -systemvol /etc/netatalk/AppleVolumes.system -nouservol -guestname "nobody" -sleep 1 -icon
+- -noddp -uampath /opt/lib/uams -uamlist uams_guest.so,uams_passwd.so,uams_dhx_passwd.so,uams_randnum.so,uams_dhx2.so -passwdfile /etc/netatalk/afppasswd -savepassword -passwdminlen 0 -nosetpassword -defaultvol /etc/netatalk/AppleVolumes.default -systemvol /etc/netatalk/AppleVolumes.system -nouservol -guestname "nobody" -sleep 1 -icon
diff -ur ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/netatalk/Makefile ../../../openwrt_trunk/feeds/packages/net/netatalk/Makefile
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/netatalk/Makefile	2014-10-06 15:46:23.753250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/netatalk/Makefile	2014-10-08 14:18:15.934707235 +0400
@@ -38,11 +38,11 @@
 endef
 
 define Package/netatalk/conffiles
-/etc/netatalk/afpd.conf
+/opt/etc/netatalk/afpd.conf
 endef
 
 TARGET_CFLAGS += -std=gnu99
-TARGET_LDFLAGS += $(LIBRPC)
+#TARGET_LDFLAGS += $(LIBRPC)
 
 CONFIGURE_ARGS += \
 	--disable-afs \
@@ -54,10 +54,10 @@
 	--disable-cups \
 	--disable-tcp-wrappers \
 	--with-cnid-default-backend=dbd \
-	--with-bdb="$(STAGING_DIR)/usr/" \
-	--with-libgcrypt-dir="$(STAGING_DIR)/usr" \
-	--with-ssl-dir="$(STAGING_DIR)/usr" \
-	--with-uams-path="/usr/lib/uams" \
+	--with-bdb="$(STAGING_DIR)/opt/" \
+	--with-libgcrypt-dir="$(STAGING_DIR)/opt" \
+	--with-ssl-dir="$(STAGING_DIR)/opt" \
+	--with-uams-path="/opt/lib/uams" \
 	--without-acls \
 	--without-pam \
 	--disable-admin-group \
@@ -67,20 +67,20 @@
 	--without-ldap
 
 define Package/netatalk/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_DIR) $(1)/usr/lib/uams
-	$(INSTALL_DIR) $(1)/etc/netatalk
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/afppasswd $(1)/usr/bin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/afpd $(1)/usr/sbin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/cnid_dbd $(1)/usr/sbin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/cnid_metad $(1)/usr/sbin/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/uams/*.so $(1)/usr/lib/uams/
-	$(CP) ./files/AppleVolumes.default $(1)/etc/netatalk/
-	$(CP) $(PKG_INSTALL_DIR)/etc/netatalk/AppleVolumes.system $(1)/etc/netatalk/
-	$(INSTALL_CONF) ./files/afpd.conf $(1)/etc/netatalk/
-	$(INSTALL_BIN) ./files/afpd.init $(1)/etc/init.d/afpd
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_DIR) $(1)/opt/lib/uams
+	$(INSTALL_DIR) $(1)/opt/etc/netatalk
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/afppasswd $(1)/opt/bin/
+	$(CP) $(PKG_INSTALL_DIR)/opt/sbin/afpd $(1)/opt/sbin/
+	$(CP) $(PKG_INSTALL_DIR)/opt/sbin/cnid_dbd $(1)/opt/sbin/
+	$(CP) $(PKG_INSTALL_DIR)/opt/sbin/cnid_metad $(1)/opt/sbin/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/uams/*.so $(1)/opt/lib/uams/
+	$(CP) ./files/AppleVolumes.default $(1)/opt/etc/netatalk/
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/netatalk/AppleVolumes.system $(1)/opt/etc/netatalk/
+	$(INSTALL_CONF) ./files/afpd.conf $(1)/opt/etc/netatalk/
+	$(INSTALL_BIN) ./files/{S26cnid_metad,S27afpd} $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,netatalk))
