--- a/src/oidentd_options.c
+++ b/src/oidentd_options.c
@@ -381,7 +381,7 @@
 "\nUsage: oidentd [options]\n"
 "-a or --address <address>    Bind to <address>\n"
 "-c or --charset <charset>    Specify an alternate charset\n"
-"-C or --config <config file> Use the specifed file instead of /etc/oidentd.conf\n"
+"-C or --config <config file> Use the specifed file instead of /opt/etc/oidentd.conf\n"
 "-d or --debug                Enable debugging\n"
 "-e or --error                Return \"UNKNOWN-ERROR\" for all errors\n"
 
--- a/src/oidentd.h	2012-08-28 10:22:34.429402419 +0400
+++ b/src/oidentd.h	2012-08-28 10:22:40.985387172 +0400
@@ -36,7 +36,7 @@
 ** System-wide configuration file.
 */
 
-#define CONFFILE		"/etc/oidentd.conf"
+#define CONFFILE		"/opt/etc/oidentd.conf"
 
 /*
 ** Per-user configuration file.  This file is relative to the user's
