--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/oidentd/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/oidentd/Makefile
@@ -35,10 +35,10 @@
 endef
 
 define Package/oidentd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(CP) $(PKG_BUILD_DIR)/src/oidentd $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/oidentd.init $(1)/etc/init.d/oidentd
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/oidentd $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S39oidentd $(1)/opt/etc/init.d/
 endef
 
 $(eval $(call BuildPackage,oidentd))
