--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/mosh/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mosh/Makefile
@@ -8,12 +8,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mosh
-PKG_VERSION:=1.2.4
+PKG_VERSION:=1.2.4a-20140108
+PKG_REV:=50152bfcba872dd2ef5318877823005ad0c4593f
 PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://mosh.mit.edu/
-PKG_MD5SUM:=c2d918f4d91fdc32546e2e089f9281b2
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_URL:=git://github.com/keithw/mosh.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=$(PKG_REV)
 
 PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
@@ -83,23 +86,23 @@
 
 
 CONFIGURE_ARGS += \
-	--with-curses=$(STAGING_DIR)/usr \
+	--with-curses=$(STAGING_DIR)/opt \
 	--disable-hardening \
 	--without-utempter \
 
 define Package/mosh-client/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/mosh-client $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/mosh-client $(1)/opt/bin/
 endef
 
 define Package/mosh-server/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/mosh-server $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/mosh-server $(1)/opt/bin/
 endef
 
 define Package/mosh-full/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/mosh $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/mosh $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,mosh-client))
