--- ../../../downloads/openwrt_trunk-r42803/feeds/oldpackages/admin/nagios/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/admin/nagios/Makefile
@@ -20,7 +20,7 @@
 define Package/nagios
   SECTION:=admin
   CATEGORY:=Administration
-  DEPENDS:=+libpthread +microperl +libltdl
+  DEPENDS:=+libpthread +perl +libltdl
   TITLE:=service and network monitoring program
   URL:=http://www.nagios.org/
 endef
@@ -40,7 +40,7 @@
 CONFIGURE_ARGS += \
 	--with-nagios-user="root" \
 	--with-nagios-group="root" \
-	--sysconfdir="/etc/nagios" \
+	--sysconfdir="/opt/etc/nagios" \
 	--without-gd-lib \
 	--without-gd-inc \
 	CFLAGS="$(TARGET_CFLAGS) $(EXTRA_CFLAGS)"
@@ -54,22 +54,22 @@
 endef
 
 define Package/nagios/install
-	$(INSTALL_DIR)	$(1)/etc/nagios
-	$(INSTALL_DATA)	$(PKG_INSTALL_DIR)/etc/nagios/* $(1)/etc/nagios/
-	$(INSTALL_DIR)	$(1)/usr/bin
-	$(INSTALL_BIN)	$(PKG_INSTALL_DIR)/usr/bin/{nagios,nagiostats} $(1)/usr/bin/
-	$(INSTALL_DIR)	$(1)/usr/sbin
-	$(INSTALL_BIN)	$(PKG_INSTALL_DIR)/usr/sbin/* $(1)/usr/sbin/
-	$(INSTALL_DIR)	$(1)/usr/share
-	$(CP)	$(PKG_INSTALL_DIR)/usr/share/* $(1)/usr/share/
+	$(INSTALL_DIR)	$(1)/opt/etc/nagios
+	$(INSTALL_DATA)	$(PKG_INSTALL_DIR)/opt/etc/nagios/* $(1)/opt/etc/nagios/
+	$(INSTALL_DIR)	$(1)/opt/bin
+	$(INSTALL_BIN)	$(PKG_INSTALL_DIR)/opt/bin/{nagios,nagiostats} $(1)/opt/bin/
+	$(INSTALL_DIR)	$(1)/opt/sbin
+	$(INSTALL_BIN)	$(PKG_INSTALL_DIR)/opt/sbin/* $(1)/opt/sbin/
+	$(INSTALL_DIR)	$(1)/opt/share
+	$(CP)	$(PKG_INSTALL_DIR)/opt/share/* $(1)/opt/share/
 endef
 
 define Package/nagios/conffiles
-/etc/nagios/cgi.cfg
-/etc/nagios/commands.cfg
-/etc/nagios/localhost.cfg
-/etc/nagios/nagios.cfg
-/etc/nagios/resource.cfg
+/opt/etc/nagios/cgi.cfg
+/opt/etc/nagios/commands.cfg
+/opt/etc/nagios/localhost.cfg
+/opt/etc/nagios/nagios.cfg
+/opt/etc/nagios/resource.cfg
 endef
 
 $(eval $(call BuildPackage,nagios))
