--- ../../../downloads/openwrt_trunk-r35474/feeds/oldpackages/utils/lcd4linux/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/lcd4linux/Makefile
@@ -129,9 +129,10 @@
 	libnmeap \
 	libsqlite3 \
 	libvncserver \
-	ppp \
 	serdisplib \
 	st2205tool \
+
+#	ppp \
 #	libftdi \
 #	libX11 \
 #	python \
@@ -161,7 +162,6 @@
 $(call Package/lcd4linux/Default)
   DEPENDS:= \
 	+LCD4LINUX_CUSTOM_NEEDS_libdbus:libdbus \
-	+LCD4LINUX_CUSTOM_NEEDS_libdpf:libdpf \
 	+LCD4LINUX_CUSTOM_NEEDS_libgd:libgd \
 	$(if $(ICONV_FULL),+LCD4LINUX_CUSTOM_NEEDS_libiconv:libiconv-full) \
 	+LCD4LINUX_CUSTOM_NEEDS_libjpeg:libjpeg \
@@ -196,7 +196,6 @@
 $(call Package/lcd4linux/Default)
   DEPENDS:= @DEVEL \
 	+libdbus \
-	+libdpf \
 	+libgd \
 	$(if $(ICONV_FULL),+libiconv-full) \
 	+libmpdclient \
@@ -225,8 +224,6 @@
 CONFIGURE_ARGS+= \
 	--disable-rpath \
 
-EXTRA_LDFLAGS+= -Wl,-rpath-link,$(STAGING_DIR)/usr/lib
-
 ifeq ($(BUILD_VARIANT),custom)
 
   LCD4LINUX_CUSTOM_DRIVERS:= $(strip $(foreach c, $(LCD4LINUX_DRIVERS), \
@@ -254,7 +251,7 @@
   endif
 
   ifneq ($(CONFIG_LCD4LINUX_CUSTOM_NEEDS_libmysqlclient),)
-    EXTRA_LDFLAGS+= -L$(STAGING_DIR)/usr/lib/mysql
+    EXTRA_LDFLAGS+= -L$(STAGING_DIR)/opt/lib/mysql
   endif
 
 #  ifneq ($(CONFIG_LCD4LINUX_CUSTOM_NEEDS_python),)
@@ -288,27 +285,27 @@
 	--without-python \
 	--without-x \
 
-  EXTRA_LDFLAGS+= -L$(STAGING_DIR)/usr/lib/mysql
+  EXTRA_LDFLAGS+= -L$(STAGING_DIR)/opt/lib/mysql
 
 endif
 
 
 define Package/lcd4linux/conffiles
-/etc/lcd4linux.conf
+/opt/etc/lcd4linux.conf
 endef
 
 define Package/lcd4linux/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/lcd4linux $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/lcd4linux.conf.sample $(1)/etc/lcd4linux.conf
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/lcd4linux.init $(1)/etc/init.d/lcd4linux
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/lcd4linux $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/lcd4linux.conf.sample $(1)/opt/etc/lcd4linux.conf
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S91lcd4linux $(1)/opt/etc/init.d/
 	$(SED) "s|^\(Display 'GLCD2USB'\)|#\1|g" \
 	    -e "s|^\(Layout 'TestLayer'\)|#\1|g" \
 	    -e "s|^#\(Display 'Image'\)|\1|g" \
 	    -e "s|^#\(Layout 'Default'\)|\1|g" \
-	     $(1)/etc/lcd4linux.conf
+	     $(1)/opt/etc/lcd4linux.conf
 endef
 
 Package/lcd4linux-custom/conffiles = $(Package/lcd4linux/conffiles)
