--- ../../../downloads/openwrt_trunk-r36151/feeds/oldpackages/net/arpwatch/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/arpwatch/Makefile
@@ -34,7 +34,7 @@
 endef
 
 define Package/arpwatch/conffiles
-/etc/arpwatch/arp.dat
+/opt/etc/arpwatch/arp.dat
 endef
 
 define Build/Prepare
@@ -42,21 +42,25 @@
 	chmod -R u+w $(PKG_BUILD_DIR)
 endef
 
+CONFIGURE_VARS += \
+	ac_cv_path_V_SENDMAIL="/usr/sbin/sendmail"
+
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
-		ARPDIR=/etc/arpwatch \
+		ARPDIR=/opt/etc/arpwatch \
 		CCOPT="$(TARGET_CFLAGS)" \
 		INCLS="-I. $(TARGET_CPPFLAGS)" \
-		LIBS="$(TARGET_LDFLAGS) -lpcap"
+		LIBS="$(TARGET_LDFLAGS) -lpcap" \
+		SLIBS="$(TARGET_LDFLAGS)"
 endef
 
 define Package/arpwatch/install
-	$(INSTALL_DIR) $(1)/etc/arpwatch
-	$(CP) $(PKG_BUILD_DIR)/arp.dat $(1)/etc/arpwatch/
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/arp{watch,snmp} $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/arpwatch.init $(1)/etc/init.d/arpwatch
+	$(INSTALL_DIR) $(1)/opt/etc/arpwatch
+	$(CP) $(PKG_BUILD_DIR)/arp.dat $(1)/opt/etc/arpwatch/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/arp{watch,snmp} $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S04arpwatch $(1)/opt/etc/init.d/
 endef
 
 $(eval $(call BuildPackage,arpwatch))
