--- ../../../openwrt_trunk/feeds/packages/net/nginx/Config.in.prev	2012-07-18 00:48:36.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/nginx/Config.in	2012-07-18 01:35:42.000000000 +0400
@@ -25,14 +25,14 @@
 config NGINX_SSL
 	bool
 	prompt "Enable SSL module"
-	default n
+	default y
 	help
 		Enable HTTPS/SSL support.
 
 config NGINX_DAV
 	bool
 	prompt "Enable WebDAV module"
-	default n
+	default y
 	help
 		Enable the HTTP and WebDAV methods PUT, DELETE, MKCOL, COPY and MOVE.
 
--- ../../../openwrt_trunk/feeds/packages/net/nginx/Makefile.prev	2012-07-18 01:48:33.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/nginx/Makefile	2012-07-18 01:38:53.000000000 +0400
@@ -32,7 +32,7 @@
   SUBMENU:=Web Servers/Proxies
   TITLE:=Nginx web server
   URL:=http://nginx.org/
-  DEPENDS:=+libpcre +libopenssl +zlib +libpthread
+  DEPENDS:=+libpcre +libopenssl +zlib +libpthread +libexpat
   MENU:=1
 endef
 
@@ -88,6 +88,7 @@
 			--with-cc="$(TARGET_CC)" \
 			--with-cc-opt="$(TARGET_CPPFLAGS) $(TARGET_CFLAGS)" \
 			--with-ld-opt="$(TARGET_LDFLAGS)" \
+			--add-module=src/http_dav_ext \
 			$(ADDITIONAL_MODULES) )
 endef
 
