--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/packages/utils/dtach/Makefile	2014-04-09 16:17:24.218286483 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/dtach/Makefile	2014-04-11 12:06:03.403415327 +0400
@@ -41,8 +41,8 @@
 endef
 
 define Package/dtach/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dtach $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/dtach $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,dtach))
