--- ../../../downloads/openwrt_trunk-r43718/feeds/packages/utils/smartmontools/Makefile	2014-12-08 20:56:36.449259284 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/smartmontools/Makefile	2014-12-11 20:42:02.491890932 +0300
@@ -18,6 +18,8 @@
 PKG_MAINTAINER:=Maxim Storchak <m.storchak@gmail.com>
 PKG_LICENSE:=GPL-2.0+
 
+PKG_INSTALL:=1
+
 include $(INCLUDE_DIR)/package.mk
 
 define Package/smartmontools/Default
@@ -54,12 +56,6 @@
 
 # uses GNU configure
 
-CONFIGURE_VARS += \
-	CXXFLAGS="$$$$CXXFLAGS -fno-builtin -fno-rtti -nostdinc++" \
-	CPPFLAGS="$$$$CPPFLAGS -I$(STAGING_DIR)/usr/include/uClibc++ -I$(LINUX_DIR)/include" \
-	LDFLAGS="$$$$LDFLAGS" \
-	LIBS="-nodefaultlibs -lc -luClibc++ -lm $(LIBGCC_S) -lc" \
-
 define Build/Configure
 	(cd $(PKG_BUILD_DIR); rm -rf config.{cache,status} ; \
 		./autogen.sh \
@@ -74,21 +70,23 @@
 endef
 
 define Package/smartmontools/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/smartctl $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/sbin/smartctl $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/sbin/update-smart-drivedb $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share/smartmontools
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/Apps/opt/share/smartmontools/drivedb.h $(1)/opt/share/smartmontools/
 endef
 
 define Package/smartd/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/smartd $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/smartd.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/smartd.init $(1)/etc/init.d/smartd
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/sbin/smartd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/Apps/opt/etc/smartd.conf $(1)/opt/etc/
+	$(INSTALL_BIN) ./files/S20smartmontools $(1)/opt/etc/init.d
 endef
 
 define Package/smartd/conffiles
-/etc/smartd.conf
+/opt/etc/smartd.conf
 endef
 
 $(eval $(call BuildPackage,smartmontools))
