--- ../../../downloads/openwrt_trunk-r42803/package/utils/busybox/Makefile
+++ ../../../openwrt_trunk/package/utils/busybox/Makefile
@@ -10,7 +10,6 @@
 PKG_NAME:=busybox
 PKG_VERSION:=1.22.1
 PKG_RELEASE:=2
-PKG_FLAGS:=essential
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=http://www.busybox.net/downloads \
@@ -75,13 +74,13 @@
 
 LDLIBS:=m crypt
 ifdef CONFIG_BUSYBOX_USE_LIBRPC
-  TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
+  TARGET_CFLAGS += -I$(STAGING_DIR)/opt/include
   export LDFLAGS=$(TARGET_LDFLAGS)
   LDLIBS += rpc
 endif
 
 ifdef CONFIG_BUSYBOX_CONFIG_PAM
-  TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
+  TARGET_CFLAGS += -I$(STAGING_DIR)/opt/include
   export LDFLAGS=$(TARGET_LDFLAGS)
   LDLIBS += pam pam_misc pthread
 endif
@@ -92,6 +91,7 @@
 		CROSS_COMPILE="$(TARGET_CROSS)" \
 		KBUILD_HAVE_NLS=no \
 		EXTRA_CFLAGS="$(TARGET_CFLAGS)" \
+		EXTRA_LDFLAGS="$(TARGET_LDFLAGS)" \
 		ARCH="$(ARCH)" \
 		SKIP_STRIP=y \
 		LDLIBS="$(LDLIBS)" \
@@ -103,20 +103,53 @@
 		CC="$(TARGET_CC)" \
 		CROSS_COMPILE="$(TARGET_CROSS)" \
 		EXTRA_CFLAGS="$(TARGET_CFLAGS)" \
+		EXTRA_LDFLAGS="$(TARGET_LDFLAGS)" \
 		ARCH="$(ARCH)" \
-		CONFIG_PREFIX="$(PKG_INSTALL_DIR)" \
+		CONFIG_PREFIX="$(PKG_INSTALL_DIR)/opt" \
 		LDLIBS="$(LDLIBS)" \
 		$(BB_MAKE_VERBOSE) \
 		install
 endef
 
 define Package/busybox/install
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(CP) $(PKG_INSTALL_DIR)/* $(1)/
-	$(INSTALL_BIN) ./files/cron $(1)/etc/init.d/cron
-	$(INSTALL_BIN) ./files/telnet $(1)/etc/init.d/telnet
-	$(INSTALL_BIN) ./files/sysntpd $(1)/etc/init.d/sysntpd
-	-rm -rf $(1)/lib64
+	$(INSTALL_DIR) $(1)/opt/{bin,sbin}
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/busybox $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share
+	(cd $(PKG_INSTALL_DIR); \
+	    find ./opt -lname "*" | sed "s|^./opt/||g" > $(1)/opt/share/busybox_applets.lst.dist)
+endef
+
+define Package/busybox/postinst
+#!/bin/sh
+
+cd /opt
+rm -f share/busybox_applets.lst
+for applet in $$(cat share/busybox_applets.lst.dist)
+do
+    if [ ! -e $$applet ]
+    then
+	ln -s /opt/bin/busybox $$applet
+	echo "$$applet" >> share/busybox_applets.lst
+    else
+	echo "Warning: '/opt/$$applet' is already exists, skipping."
+    fi
+done
+endef
+
+define Package/busybox/postrm
+#!/bin/sh
+
+cd /opt
+for applet in $$(cat share/busybox_applets.lst)
+do
+    if [ -L $$applet ]
+    then
+	rm -f $$applet
+    else
+	echo "Warning: '/opt/$$applet' has been deleted\modified before, skipping."
+    fi
+done
+rm -f share/busybox_applets.lst
 endef
 
 $(eval $(call BuildPackage,busybox))
