--- ../../../downloads/openwrt_trunk-r37769/feeds/oldpackages/libs/radlib/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/radlib/Makefile
@@ -38,21 +38,21 @@
 
 EXTRA_LDFLAGS=-lsqlite3
 CONFIGURE_ARGS += \
-	--prefix="$(TOOLCHAIN_DIR)/usr" \
+	--prefix="$(TOOLCHAIN_DIR)/opt" \
 	--enable-sqlite
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_BUILD_DIR)/h/rad*.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/src/.libs/librad.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_BUILD_DIR)/h/rad*.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/src/.libs/librad.so* $(1)/opt/lib/
 endef
 
 define Package/radlib/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/.libs/librad.so* $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/msgRouter/.libs/radmrouted $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/src/.libs/librad.so* $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/msgRouter/.libs/radmrouted $(1)/opt/bin/
 endef
 
 define Package/radlib-sqlite/install
