--- ../../../openwrt_trunk.orig/package/base-files/Makefile	2012-09-03 12:36:07.951891454 +0400
+++ ../../../openwrt_trunk/package/base-files/Makefile	2012-09-03 12:37:30.467889951 +0400
@@ -28,7 +28,7 @@
 define Package/base-files
   SECTION:=base
   CATEGORY:=Base system
-  DEPENDS:=+netifd +libc
+  DEPENDS:=+libc
   TITLE:=Base filesystem for OpenWrt
   URL:=http://openwrt.org/
   VERSION:=$(PKG_RELEASE)-$(REVISION)
