--- ../../../openwrt_trunk/package/base-files/Makefile.orig	2013-01-28 14:40:44.000000000 +0400
+++ ../../../openwrt_trunk/package/base-files/Makefile	2013-02-04 11:24:05.028473991 +0400
@@ -29,7 +29,7 @@
 define Package/base-files
   SECTION:=base
   CATEGORY:=Base system
-  DEPENDS:=+netifd +libc +procd
+  DEPENDS:=+libc
   TITLE:=Base filesystem for OpenWrt
   URL:=http://openwrt.org/
   VERSION:=$(PKG_RELEASE)-$(REVISION)
