--- ../../../downloads/openwrt_trunk-r43719/feeds/oldpackages/net/bitlbee/Makefile	2015-01-17 14:40:34.087631819 +0300
+++ ../../../openwrt_trunk/feeds/oldpackages/net/bitlbee/Makefile	2015-02-25 16:15:24.673041413 +0300
@@ -33,13 +33,19 @@
 endef
 
 define Package/bitlbee/conffiles
-/etc/bitlbee/motd.txt
-/etc/bitlbee/bitlbee.conf
+/opt/etc/bitlbee/motd.txt
+/opt/etc/bitlbee/bitlbee.conf
 endef
 
+DYNLINK=ld-linux.so.2
+ifeq ($(ARCH),arm)
+  DYNLINK:=ld-linux.so.3
+endif
+
 define Build/Configure
 	# it's not GNU autoconf stuff
 	(cd $(PKG_BUILD_DIR); \
+		$(SED) 's,ld-linux.so.0,$(DYNLINK),g' $(PKG_BUILD_DIR)/Makefile; \
 		$(TARGET_CONFIGURE_OPTS) \
 		CFLAGS="$(TARGET_CFLAGS)" \
 		CPPFLAGS="$(TARGET_CPPFLAGS)" \
@@ -47,29 +53,38 @@
 		STAGING_DIR="$(STAGING_DIR)" \
 		STRIP="$(STRIP)" \
 		./configure \
-			--prefix=/usr/ \
-			--bindir=/usr/sbin/ \
-			--etcdir=/etc/bitlbee/ \
-			--datadir=/usr/share/bitlbee/ \
-			--config=/var/lib/bitlbee/ \
+			--prefix=/Apps/opt/ \
+			--bindir=/Apps/opt/sbin/ \
+			--etcdir=/Apps/opt/etc/bitlbee/ \
+			--datadir=/Apps/opt/share/bitlbee/ \
+			--config=/Apps/opt/var/lib/bitlbee/ \
 			--arch=OpenWrt \
 			--cpu="$(ARCH)" \
 			--debug=0 \
 			--strip=1 \
-			--sizeopt=1 \
 			--ipv6=1 \
 			--ssl=openssl \
 			--glib1=1 \
-			--glib1prefix="$(STAGING_DIR)/usr" \
+			--glib1prefix="$(STAGING_DIR)/opt" \
 	);
 	$(SED) 's,^CFLAGS=.*,CFLAGS=$(TARGET_CFLAGS),g'\
 		$(PKG_BUILD_DIR)/Makefile.settings
 	@echo 'CFLAGS+=-I$(ICONV_PREFIX)/include' >> \
 		$(PKG_BUILD_DIR)/Makefile.settings
+	@echo 'CFLAGS+=-I$(STAGING_DIR)/opt/include' >> \
+		$(PKG_BUILD_DIR)/Makefile.settings
 	@echo 'LFLAGS+=$(TARGET_LDFLAGS)' >> \
 		$(PKG_BUILD_DIR)/Makefile.settings
 	@echo 'LFLAGS+=-L$(ICONV_PREFIX)/lib' >> \
 		$(PKG_BUILD_DIR)/Makefile.settings
+	$(SED) 's|-Wl,-rpath,/Apps/opt/lib||g'\
+		$(PKG_BUILD_DIR)/Makefile.settings
+	$(SED) 's|-Wl,--dynamic-linker=/Apps/opt/lib/ld-linux.so.[0-9]||g'\
+		$(PKG_BUILD_DIR)/Makefile.settings
+	$(SED) 's|-Wl,-rpath-link=$(STAGING_DIR)/opt/lib||g'\
+		$(PKG_BUILD_DIR)/Makefile.settings
+	$(SED) 's| /libiconv-full/lib||g'\
+		$(PKG_BUILD_DIR)/Makefile.settings
 endef
 
 define Build/Compile
@@ -78,11 +93,13 @@
 
 define Package/bitlbee/install
 	$(MAKE) -C $(PKG_BUILD_DIR) \
-		DESTDIR="$(1)" install-etc install-bin
+		DESTDIR="$(PKG_INSTALL_DIR)" install-etc install-bin
 	$(MAKE) -C $(PKG_BUILD_DIR)/doc/user-guide \
-		DESTDIR="$(1)" install
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/bitlbee.init $(1)/etc/init.d/bitlbee
+		DESTDIR="$(PKG_INSTALL_DIR)" install
+	$(INSTALL_DIR) $(1)/opt
+	cp -r $(PKG_INSTALL_DIR)/Apps/opt/* $(1)/opt
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S72bitlbee $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,bitlbee))
