--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/net/bitlbee/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/bitlbee/Makefile
@@ -33,8 +33,8 @@
 endef
 
 define Package/bitlbee/conffiles
-/etc/bitlbee/motd.txt
-/etc/bitlbee/bitlbee.conf
+/opt/etc/bitlbee/motd.txt
+/opt/etc/bitlbee/bitlbee.conf
 endef
 
 define Build/Configure
@@ -47,29 +47,38 @@
 		STAGING_DIR="$(STAGING_DIR)" \
 		STRIP="$(STRIP)" \
 		./configure \
-			--prefix=/usr/ \
-			--bindir=/usr/sbin/ \
-			--etcdir=/etc/bitlbee/ \
-			--datadir=/usr/share/bitlbee/ \
-			--config=/var/lib/bitlbee/ \
+			--prefix=/opt/ \
+			--bindir=/opt/sbin/ \
+			--etcdir=/opt/etc/bitlbee/ \
+			--datadir=/opt/share/bitlbee/ \
+			--config=/opt/var/lib/bitlbee/ \
 			--arch=OpenWrt \
 			--cpu="$(ARCH)" \
 			--debug=0 \
 			--strip=1 \
-			--sizeopt=1 \
 			--ipv6=1 \
 			--ssl=openssl \
 			--glib1=1 \
-			--glib1prefix="$(STAGING_DIR)/usr" \
+			--glib1prefix="$(STAGING_DIR)/opt" \
 	);
 	$(SED) 's,^CFLAGS=.*,CFLAGS=$(TARGET_CFLAGS),g'\
 		$(PKG_BUILD_DIR)/Makefile.settings
 	@echo 'CFLAGS+=-I$(ICONV_PREFIX)/include' >> \
 		$(PKG_BUILD_DIR)/Makefile.settings
+	@echo 'CFLAGS+=-I$(STAGING_DIR)/opt/include' >> \
+		$(PKG_BUILD_DIR)/Makefile.settings
 	@echo 'LFLAGS+=$(TARGET_LDFLAGS)' >> \
 		$(PKG_BUILD_DIR)/Makefile.settings
 	@echo 'LFLAGS+=-L$(ICONV_PREFIX)/lib' >> \
 		$(PKG_BUILD_DIR)/Makefile.settings
+	$(SED) 's|-Wl,-rpath,/opt/lib||g'\
+		$(PKG_BUILD_DIR)/Makefile.settings
+	$(SED) 's|-Wl,--dynamic-linker=/opt/lib/ld-uClibc.so.0||g'\
+		$(PKG_BUILD_DIR)/Makefile.settings
+	$(SED) 's|-Wl,-rpath-link=$(STAGING_DIR)/opt/lib||g'\
+		$(PKG_BUILD_DIR)/Makefile.settings
+	$(SED) 's| /libiconv-full/lib||g'\
+		$(PKG_BUILD_DIR)/Makefile.settings
 endef
 
 define Build/Compile
@@ -81,8 +90,8 @@
 		DESTDIR="$(1)" install-etc install-bin
 	$(MAKE) -C $(PKG_BUILD_DIR)/doc/user-guide \
 		DESTDIR="$(1)" install
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/bitlbee.init $(1)/etc/init.d/bitlbee
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S72bitlbee $(1)/opt/etc/init.d
 endef
 
 $(eval $(call BuildPackage,bitlbee))
