--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/bluez-hcidump/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/bluez-hcidump/Makefile
@@ -29,13 +29,13 @@
 
 define Build/Configure
 	$(call Build/Configure/Default, \
-		--with-bluez="$(STAGING_DIR)/usr/include" \
+		--with-bluez="$(STAGING_DIR)/opt/include" \
 	)
 endef
 
 define Package/bluez-hcidump/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/hcidump $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/hcidump $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,bluez-hcidump))
