--- ../../../openwrt_trunk.orig/feeds/packages/lang/python-curl/Makefile	2012-01-13 12:14:28.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/lang/python-curl/Makefile	2012-01-13 22:09:42.575340468 +0400
@@ -32,7 +32,7 @@
 endef
 
 define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix=/usr --root=$(PKG_INSTALL_DIR))
+	$(call Build/Compile/PyMod,,install --prefix=/opt --root=$(PKG_INSTALL_DIR))
 endef
 
 define Package/python-curl/install
