--- ../../../openwrt_trunk/feeds/packages/net/bind/Makefile.prev	2014-01-27 02:17:08.759860691 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/bind/Makefile	2014-01-30 23:41:29.289733674 +0400
@@ -11,3 +11,3 @@
 PKG_VERSION:=9.9.4
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
@@ -38,3 +38,3 @@
   CATEGORY:=Libraries
-  DEPENDS:=+libopenssl
+  DEPENDS:=+libopenssl +libxml2
   TITLE:=bind shared libraries
@@ -98,3 +98,3 @@
 	--with-libtool \
-	--with-libxml2=no \
+	--with-libxml2=yes \
 	--enable-epoll=yes \
