--- a/tspc-advanced/src/tsp/Makefile
+++ b/tspc-advanced/src/tsp/Makefile
@@ -14,9 +14,9 @@
 
 
 ifdef DEBUG
-CFLAGS=-g -Wall $(CC_INC_PATHS) $(PLATFORM_CFLAGS) -D_REENTRANT -DDEBUG
+CFLAGS+=-g -Wall $(CC_INC_PATHS) $(PLATFORM_CFLAGS) -D_REENTRANT -DDEBUG
 else
-CFLAGS=-O2 -Wall $(CC_INC_PATHS) $(PLATFORM_CFLAGS) -D_REENTRANT
+CFLAGS+=-O2 -Wall $(CC_INC_PATHS) $(PLATFORM_CFLAGS) -D_REENTRANT
 endif
 CC=gcc
 
