--- a/tspc-advanced/template/linux.sh
+++ b/tspc-advanced/template/linux.sh
@@ -74,10 +74,9 @@
 
 ifconfig=/sbin/ifconfig
 route=/sbin/route
-ipconfig=/sbin/ip
+ipconfig=/usr/sbin/ip
 rtadvd=/usr/sbin/radvd
-rtadvd_pid=/var/run/radvd/radvd.pid
-sysctl=/sbin/sysctl
+rtadvd_pid=/var/run/radvd.pid
 rtadvdconfigfilename=gw6c-rtadvd.conf
 rtadvdconfigfile=$TSP_HOME_DIR/$rtadvdconfigfilename
 
@@ -124,7 +123,7 @@
     fi
 
     # Remove address from TSP HOME INTERFACE
-    ExecNoCheck $ifconfig $TSP_HOME_INTERFACE inet6 del $TSP_PREFIX::1/64
+    ExecNoCheck $ifconfig $TSP_HOME_INTERFACE del $TSP_PREFIX::1/64
   fi
 
   # Delete default IPv6 route(s).
@@ -144,7 +143,7 @@
       PREF=`echo $TSP_CLIENT_ADDRESS_IPV6 | sed "s/:0*/:/g" |cut -d : -f1-2`
       OLDADDR=`$ifconfig $TSP_TUNNEL_INTERFACE | grep "inet6.* $PREF" | sed -e "s/^.*inet6 addr: //" -e "s/ Scope.*\$//"`
       if [ ! -z $OLDADDR ]; then
-        ExecNoCheck $ifconfig $TSP_TUNNEL_INTERFACE inet6 del $OLDADDR
+        ExecNoCheck $ifconfig $TSP_TUNNEL_INTERFACE del $OLDADDR
       fi
 
       # Bring interface down
@@ -184,10 +183,12 @@
    OLDADDR=`$ifconfig $TSP_TUNNEL_INTERFACE | grep "inet6.* $PREF" | sed -e "s/^.*inet6 addr: //" -e "s/ Scope.*\$//"`
    if [ ! -z $OLDADDR ]; then
       Display 1 "Removing old IPv6 address $OLDADDR"
-      Exec $ifconfig $TSP_TUNNEL_INTERFACE inet6 del $OLDADDR
+      Exec $ifconfig $TSP_TUNNEL_INTERFACE del $OLDADDR
    fi
 
    Display 1 "This host is: $TSP_CLIENT_ADDRESS_IPV6/$TSP_TUNNEL_PREFIXLEN"
+   Exec /bin/echo 0 > /proc/sys/net/ipv6/conf/all/disable_ipv6
+   Exec /bin/sleep 5
    Exec $ifconfig $TSP_TUNNEL_INTERFACE add $TSP_CLIENT_ADDRESS_IPV6/$TSP_TUNNEL_PREFIXLEN
    Exec $ifconfig $TSP_TUNNEL_INTERFACE mtu 1280
 
@@ -206,7 +207,7 @@
    Display 1 "Router configuration"
 
    # Tell kernel to forward IPv6 traffic.
-   Exec $sysctl -w net.ipv6.conf.all.forwarding=1
+   Exec echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
 
    # Blackholing on interface lo, if prefixlen is not 64.
    if [ X"${TSP_PREFIXLEN}" != X"64" ]; then
@@ -218,14 +219,13 @@
    OLDADDR=`$ifconfig $TSP_HOME_INTERFACE | grep "inet6.* $PREF" | sed -e "s/^.*inet6 addr: //" -e "s/ Scope.*\$//"`
    if [ ! -z $OLDADDR ]; then
       Display 1 "Removing old IPv6 address $OLDADDR"
-      Exec $ifconfig $TSP_HOME_INTERFACE inet6 del $OLDADDR
+      Exec $ifconfig $TSP_HOME_INTERFACE del $OLDADDR
    fi
    Display 1 "Adding prefix to $TSP_HOME_INTERFACE"
    Exec $ifconfig $TSP_HOME_INTERFACE add $TSP_PREFIX::1/64
 
 
    # Stop radvd daemon if it was running. Twice.
-   /etc/init.d/radvd stop
    if [ -f $rtadvdconfigfile ]; then
      KillProcess $rtadvdconfigfile
    fi
