--- a/gw6c-messaging/src/servermsgsender.cc
+++ b/gw6c-messaging/src/servermsgsender.cc
@@ -15,6 +15,7 @@
 // **************************************************************************
 #include <gw6cmessaging/servermsgsender.h>
 #include <assert.h>
+#include <stddef.h>
 
 
 namespace gw6cmessaging
--- a/gw6c-messaging/src/clientmsgsender.cc
+++ b/gw6c-messaging/src/clientmsgsender.cc
@@ -15,6 +15,7 @@
 // **************************************************************************
 #include <gw6cmessaging/clientmsgsender.h>
 #include <assert.h>
+#include <stddef.h>
 
 
 namespace gw6cmessaging
