--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r40423/feeds/oldpackages/ipv6/gw6c/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/ipv6/gw6c/Makefile
@@ -1,25 +1,24 @@
-#
-# Copyright (C) 2008-2014 OpenWrt.org
+# 
+# Copyright (C) 2008-2011 OpenWrt.org
 #
 
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=gw6c
-PKG_VERSION:=5.1
-PKG_RELEASE:=5
+PKG_VERSION:=6.0.1
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=http://openwrt.linux-appliance.net/sources
-PKG_MD5SUM:=eeac7292a622681651ec3bd9b2e5b061
-PKG_BUILD_DIR:=$(BUILD_DIR)/gw6-$(PKG_VERSION)
+PKG_SOURCE:=gw6c-6_0-RELEASE-src.tar
+PKG_SOURCE_URL:=http://ipv6now.com.au/download
+PKG_MD5SUM:=0851184a08605e0a2b7395899e55c433
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-6_0_1
 
-include $(INCLUDE_DIR)/uclibc++.mk
 include $(INCLUDE_DIR)/package.mk
 
 define Package/gw6c
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libpthread +ip +kmod-tun +kmod-ipv6 $(CXX_DEPENDS) +radvd
+  DEPENDS:=+libopenssl +libpthread +libstdcpp
   TITLE:=IPv6 tunnel client software
   URL:=http://www.go6.net
 endef
@@ -29,53 +28,30 @@
   Hexago/freenet6/go6.net's migration broker.
 endef
 
-GW6C_OPTS:= \
-	C_COMPILER="$(TARGET_CC) -c -DNO_STDLIBCXX" \
-	COMPILER="$(TARGET_CXX) -c -DNO_STDLIBCXX" \
-	CPP_FLAGS="$(TARGET_CFLAGS) -I. -Wall -DNDEBUG  $(TARGET_CPPFLAGS)" \
-	C_FLAGS="$(TARGET_CFLAGS) -I. -Wall -DNDEBUG $(TARGET_CPPFLAGS)" \
-	RANLIB="$(TARGET_CROSS)ranlib" \
-	ARCHIVER="$(TARGET_CROSS)ar" \
-	C_LINKER="$(TARGET_CC)" \
-	LINKER="$(TARGET_CXX)" \
-	exportdir=$(PKG_BUILD_DIR)/gw6c \
-	export
-
-define Build/Configure
-	$(INSTALL_DIR) $(PKG_BUILD_DIR)/gw6c/include
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/gw6c-config/gw6cconfig/gw6c_c_wrapper.h \
-		$(PKG_BUILD_DIR)/gw6c/include
-endef
-
 define Build/Compile
-	$(MAKE) -C $(PKG_BUILD_DIR)/gw6c-config \
-		$(GW6C_OPTS)
-	$(MAKE) -C $(PKG_BUILD_DIR)/gw6c-messaging \
-		$(GW6C_OPTS)
-	$(MAKE) -C $(PKG_BUILD_DIR)/tspc-advanced \
-		CC="$(TARGET_CC)" \
-		CFLAGS="$(TARGET_CFLAGS) -fno-builtin -Wall -I$(PKG_BUILD_DIR)/tspc-advanced/include \
-			-I$(PKG_BUILD_DIR)/tspc-advanced/platform/openwrt -I$(PKG_BUILD_DIR)/tspc-advanced" \
-		LDFLAGS="-L$(PKG_BUILD_DIR)/tspc-advanced/gw6cconfig \
-			-L$(PKG_BUILD_DIR)/tspc-advanced/gw6cmessaging \
-			$(TARGET_LDFLAGS) $(LIBGCC_S)" \
-		target="openwrt" \
-		installdir="$(PKG_INSTALL_DIR)"
+	$(call Build/Compile/Default,\
+		EXTRA_CFLAGS="$(TARGET_CFLAGS)" \
+		EXTRA_CXXFLAGS="$(TARGET_CFLAGS)" \
+		EXTRA_LDFLAGS="$(TARGET_LDFLAGS)" \
+		platform=linux \
+		installdir=$(PKG_INSTALL_DIR)/opt \
+		all install\
+	)
 endef
 
 define Package/gw6c/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/tspc-advanced/bin/$(PKG_NAME) $(1)/usr/sbin
-	$(INSTALL_DIR) $(1)/usr/share/gw6c/template
-	$(CP) ./files/openwrt.sh $(1)/usr/share/gw6c/template
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/gw6c.init $(1)/etc/init.d/gw6c
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_CONF) ./files/gw6c.config $(1)/etc/config/gw6c
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/gw6c $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/share/gw6c/template
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/template/linux.sh $(1)/opt/share/gw6c/template
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_DATA) ./files/gw6c.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/var/log
+	$(INSTALL_BIN) ./files/S11gw6c $(1)/opt/etc/init.d/
 endef
 
 define Package/gw6c/conffiles
-/etc/config/gw6c
+/opt/etc/gw6c.conf
 endef
 
 $(eval $(call BuildPackage,gw6c))
