--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/nfs-server/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/nfs-server/Makefile
@@ -21,7 +21,7 @@
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=Filesystem
-  DEPENDS:=+portmap $(LIBRPC_DEPENDS)
+  DEPENDS:=+portmap
   TITLE:=User Space NFS server
   URL:=http://packages.debian.org/nfs-user-server
 endef
@@ -34,20 +34,19 @@
 	$(MAKE) -C $(PKG_BUILD_DIR)
 endef
 
-TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
-TARGET_LDFLAGS += $(LIBRPC)
+TARGET_CFLAGS += -I$(STAGING_DIR)/opt/include
 
 define Package/nfs-server/install
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/nfsd.exports $(1)/etc/exports
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/nfsd.init $(1)/etc/init.d/nfsd
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rpc.* $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/nfsd.exports $(1)/opt/etc/exports
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/{S56mountd,S57nfsd} $(1)/opt/etc/init.d/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/rpc.* $(1)/opt/sbin/
 endef
 
 define Package/nfs-server/conffiles
-/etc/exports
+/opt/etc/exports
 endef
 
 $(eval $(call BuildPackage,nfs-server))
