--- ../../../openwrt_trunk.orig/feeds/oldpackages/libs/radiusclient-ng/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/radiusclient-ng/Makefile
@@ -42,10 +42,10 @@
 endef
 
 define Package/libradiusclient-ng/conffiles
-/etc/radiusclient-ng/issue
-/etc/radiusclient-ng/port-id-map
-/etc/radiusclient-ng/radiusclient.conf
-/etc/radiusclient-ng/servers
+/opt/etc/radiusclient-ng/issue
+/opt/etc/radiusclient-ng/port-id-map
+/opt/etc/radiusclient-ng/radiusclient.conf
+/opt/etc/radiusclient-ng/servers
 endef
 
 define Package/radiusclient-ng
@@ -74,22 +74,22 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/radiusclient-ng.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libradiusclient-ng.{a,so}* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/radiusclient-ng.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libradiusclient-ng.{a,so}* $(1)/opt/lib/
 endef
 
 define Package/libradiusclient-ng/install
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/radiusclient-ng $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libradiusclient-ng.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/radiusclient-ng $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libradiusclient-ng.so.* $(1)/opt/lib/
 endef
 
 define Package/radiusclient-ng/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/rad{acct,iusclient,login,status} $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/rad{acct,iusclient,login,status} $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,libradiusclient-ng))
