--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r38784/feeds/oldpackages/libs/libotr/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/libotr/Makefile
@@ -36,16 +36,16 @@
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libotr $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libotr.{a,la,so*} $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libotr.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include $(1)/opt/lib $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libotr $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libotr.{a,la,so*} $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/libotr.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libotr/install
-	$(INSTALL_DIR) $(1)/usr/lib $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libotr.so.* $(1)/usr/lib/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/lib $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libotr.so.* $(1)/opt/lib/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin
 endef
 
 $(eval $(call BuildPackage,libotr))
