--- ../../../openwrt_trunk/feeds/packages/libs/db47/Makefile.orig
+++ ../../../openwrt_trunk/feeds/packages/libs/db47/Makefile
@@ -79,21 +79,21 @@
 endef
 
 define Package/libdb47/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libdb-*.so $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libdb-*.so $(1)/opt/lib/
 endef
 
 define Package/libdb47xx/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libdb_cxx-*.so $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libdb_cxx-*.so $(1)/opt/lib/
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/db.h $(1)/usr/include/
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/db_cxx.h $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libdb*.{a,so} $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/db.h $(1)/opt/include/
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/db_cxx.h $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libdb*.{a,so} $(1)/opt/lib
 endef
 
 $(eval $(call BuildPackage,libdb47))
