--- ../../../downloads/openwrt_trunk-r45513/feeds/telephony/net/chan-sccp-b/Makefile	2015-04-21 10:37:38.356588032 +0300
+++ ../../../openwrt_trunk/feeds/telephony/net/chan-sccp-b/Makefile	2015-04-22 10:26:04.523026481 +0300
@@ -64,19 +64,19 @@
 Package/asterisk13-chan-sccp-b/description = $(Package/description/Default)
 
 ifeq ($(BUILD_VARIANT),asterisk13)
-  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/usr/include/asterisk-13
+  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/opt/include/asterisk-13
 endif
 
 ifeq ($(BUILD_VARIANT),asterisk11)
-  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/usr/include/asterisk-11
+  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/opt/include/asterisk-11
 endif
 
 ifeq ($(BUILD_VARIANT),asterisk18)
-  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/usr/include/asterisk-1.8
+  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/opt/include/asterisk-1.8
 endif
 
 define Package/conffiles/Default
-/etc/asterisk/sccp.conf
+/opt/etc/asterisk/sccp.conf
 endef
 
 Package/asterisk18-chan-sccp-b/conffiles = $(Package/conffiles/Default)
@@ -91,10 +91,10 @@
 endef
 
 define Package/Install/Default
-	$(INSTALL_DIR) $(1)/etc/asterisk
-	$(CP) ./files/sccp.conf $(1)/etc/asterisk/sccp.conf
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/modules
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/asterisk/modules/chan_sccp.so $(1)/usr/lib/asterisk/modules/
+	$(INSTALL_DIR) $(1)/opt/etc/asterisk
+	$(CP) ./files/sccp.conf $(1)/opt/etc/asterisk/sccp.conf
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/modules
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/asterisk/modules/chan_sccp.so $(1)/opt/lib/asterisk/modules/
 endef
 
 Package/asterisk18-chan-sccp-b/install = $(Package/Install/Default)
@@ -103,4 +103,4 @@
 
 $(eval $(call BuildPackage,asterisk18-chan-sccp-b))
 $(eval $(call BuildPackage,asterisk11-chan-sccp-b))
-$(eval $(call BuildPackage,asterisk13-chan-sccp-b))
+#$(eval $(call BuildPackage,asterisk13-chan-sccp-b))
