--- ../../../downloads/openwrt_trunk-r46338/feeds/telephony/net/chan-sccp-b/Makefile	2015-07-14 10:55:19.886308175 +0300
+++ ../../../openwrt_trunk/feeds/telephony/net/chan-sccp-b/Makefile	2015-07-14 11:34:36.158345489 +0300
@@ -64,23 +64,23 @@
 Package/asterisk13-chan-sccp-b/description = $(Package/description/Default)
 
 ifeq ($(BUILD_VARIANT),asterisk13)
-  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/usr/include/asterisk-13
+  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/opt/include/asterisk-13
 endif
 
 ifeq ($(BUILD_VARIANT),asterisk11)
-  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/usr/include/asterisk-11
+  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/opt/include/asterisk-11
 endif
 
 ifeq ($(BUILD_VARIANT),asterisk18)
-  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/usr/include/asterisk-1.8
+  CONFIGURE_ARGS += --with-asterisk=$(STAGING_DIR)/opt/include/asterisk-1.8
 endif
 
 define Package/conffiles/Default
-/etc/asterisk/sccp.conf
-/etc/asterisk/extconfig.conf.sccp_sample
-/etc/asterisk/res_config_sqlite3.conf.sccp_sample
-/etc/asterisk/sccp_sqlite3.sql
-/etc/asterisk/sccp.conf.realtime_sample
+/opt/etc/asterisk/sccp.conf
+/opt/etc/asterisk/extconfig.conf.sccp_sample
+/opt/etc/asterisk/res_config_sqlite3.conf.sccp_sample
+/opt/etc/asterisk/sccp_sqlite3.sql
+/opt/etc/asterisk/sccp.conf.realtime_sample
 endef
 
 Package/asterisk18-chan-sccp-b/conffiles = $(Package/conffiles/Default)
@@ -95,14 +95,14 @@
 endef
 
 define Package/Install/Default
-	$(INSTALL_DIR) $(1)/etc/asterisk
-	$(CP) ./files/sccp.conf $(1)/etc/asterisk/sccp.conf
-	$(INSTALL_DIR) $(1)/usr/lib/asterisk/modules
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/asterisk/modules/chan_sccp.so $(1)/usr/lib/asterisk/modules/
-	$(INSTALL_DATA) ./files/extconfig.conf.sccp_sample $(1)/etc/asterisk/extconfig.conf.sccp_sample
-	$(INSTALL_DATA) ./files/res_config_sqlite3.conf.sccp_sample $(1)/etc/asterisk/res_config_sqlite3.conf.sccp_sample
-	$(INSTALL_DATA) ./files/sccp_sqlite3.sql $(1)/etc/asterisk/sccp_sqlite3.sql
-	$(INSTALL_DATA) ./files/sccp.conf.realtime_sample $(1)/etc/asterisk/sccp.conf.realtime_sample
+	$(INSTALL_DIR) $(1)/opt/etc/asterisk
+	$(CP) ./files/sccp.conf $(1)/opt/etc/asterisk/sccp.conf
+	$(INSTALL_DIR) $(1)/opt/lib/asterisk/modules
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/asterisk/modules/chan_sccp.so $(1)/opt/lib/asterisk/modules/
+	$(INSTALL_DATA) ./files/extconfig.conf.sccp_sample $(1)/opt/etc/asterisk/extconfig.conf.sccp_sample
+	$(INSTALL_DATA) ./files/res_config_sqlite3.conf.sccp_sample $(1)/opt/etc/asterisk/res_config_sqlite3.conf.sccp_sample
+	$(INSTALL_DATA) ./files/sccp_sqlite3.sql $(1)/opt/etc/asterisk/sccp_sqlite3.sql
+	$(INSTALL_DATA) ./files/sccp.conf.realtime_sample $(1)/opt/etc/asterisk/sccp.conf.realtime_sample
 endef
 
 Package/asterisk18-chan-sccp-b/install = $(Package/Install/Default)
@@ -111,4 +111,4 @@
 
 $(eval $(call BuildPackage,asterisk18-chan-sccp-b))
 $(eval $(call BuildPackage,asterisk11-chan-sccp-b))
-$(eval $(call BuildPackage,asterisk13-chan-sccp-b))
+#$(eval $(call BuildPackage,asterisk13-chan-sccp-b))
