--- ../../../downloads/openwrt_trunk-r44497/feeds/packages/utils/smstools3/Makefile	2015-02-20 12:39:13.449134248 +0300
+++ ../../../openwrt_trunk/feeds/packages/utils/smstools3/Makefile	2015-02-21 18:22:53.406078524 +0300
@@ -47,14 +47,14 @@
 
 TARGET_LDFLAGS += -liconv
 
-BINDIR=/usr/local/bin
+BINDIR=/opt/bin
 
 define Build/Compile
 	$(MAKE) -C "$(PKG_BUILD_DIR)"/src \
 		CC="$(TARGET_CC)" \
 		USER_CFLAGS='$(TARGET_CFLAGS) $(EXTRA_CFLAGS) $(TARGET_CPPFLAGS) $(EXTRA_CPPFLAGS) \
-		-I"$(STAGING_DIR)/usr/lib/libiconv-full/include"' \
-		USER_LDFLAGS='$(TARGET_LFLAGS) $(EXTRA_LDFLAGS) -L"$(STAGING_DIR)/usr/lib/libiconv-full/lib"' \
+		-I"$(STAGING_DIR)/opt/lib/libiconv-full/include"' \
+		USER_LDFLAGS='$(TARGET_LFLAGS) $(EXTRA_LDFLAGS) -L"$(STAGING_DIR)/opt/lib/libiconv-full/lib"' \
 		all
 endef
 
@@ -65,10 +65,9 @@
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/sms2html $(1)/$(BINDIR)/sms2html
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/sms2unicode $(1)/$(BINDIR)/sms2unicode
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/scripts/unicode2sms $(1)/$(BINDIR)/unicode2sms
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_DATA) -m 0755 ./files/smstools3.init $(1)/etc/init.d/smstools3
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) -m 0755 ./files/smstools3.conf $(1)/etc/smsd.conf
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S53smstools3 $(1)/opt/etc/init.d/
+	$(INSTALL_DATA) -m 0755 ./files/smstools3.conf $(1)/opt/etc/smsd.conf
 endef
 
 $(eval $(call BuildPackage,smstools3))
