--- ../../../openwrt_trunk.orig/feeds/oldpackages/lang/distribute/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/lang/distribute/Makefile
@@ -39,7 +39,7 @@
 endef
 
 define Host/Compile
-	$(call Build/Compile/PyMod,,install --prefix=/usr --root=$(PKG_INSTALL_DIR))
+	$(call Build/Compile/PyMod,,install --prefix=/opt --root=$(PKG_INSTALL_DIR))
 endef
 
 define Host/Install
@@ -50,7 +50,7 @@
 endef
 
 define Build/Compile
-	$(call Build/Compile/PyMod,,install --prefix=/usr --root=$(PKG_INSTALL_DIR))
+	$(call Build/Compile/PyMod,,install --prefix=/opt --root=$(PKG_INSTALL_DIR))
 endef
 
 define Package/distribute/install
@@ -58,11 +58,11 @@
 	$(CP) \
 	    $(PKG_INSTALL_DIR)$(PYTHON_PKG_DIR)/* \
 	    $(1)$(PYTHON_PKG_DIR)/
-	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/opt/bin
 	$(INSTALL_BIN) \
-	    $(PKG_INSTALL_DIR)/usr/bin/easy_install \
-	    $(1)/usr/bin/
-	$(SED) '1s/#\!.*/#\!\/usr\/bin\/python/' $(1)/usr/bin/easy_install
+	    $(PKG_INSTALL_DIR)/opt/bin/easy_install \
+	    $(1)/opt/bin/
+	$(SED) '1s/#\!.*/#\!\/opt\/bin\/python/' $(1)/opt/bin/easy_install
 endef
 
 $(eval $(call HostBuild))
