--- ../../../downloads/openwrt_trunk-r44497/feeds/packages/multimedia/ices/Makefile	2015-02-20 12:39:13.361134248 +0300
+++ ../../../openwrt_trunk/feeds/packages/multimedia/ices/Makefile	2015-02-21 18:02:19.406102209 +0300
@@ -40,12 +40,12 @@
 endef
 
 CONFIGURE_ARGS+= \
-	--with-ogg="$(STAGING_DIR)/usr/include" \
-	--with-vorbis="$(STAGING_DIR)/usr/include" \
+	--with-ogg="$(STAGING_DIR)/opt/include" \
+	--with-vorbis="$(STAGING_DIR)/opt/include" \
 
 define Package/ices/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ices $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/ices $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,ices))
