--- ../../../downloads/openwrt_trunk-q44497/feeds/packages/multimedia/ices/Makefile	2015-02-25 12:52:24.000426160 +0300
+++ ../../../openwrt_trunk/feeds/packages/multimedia/ices/Makefile	2015-03-01 09:18:47.955684019 +0300
@@ -40,12 +40,12 @@
 endef
 
 CONFIGURE_ARGS+= \
-	--with-ogg="$(STAGING_DIR)/usr/include" \
-	--with-vorbis="$(STAGING_DIR)/usr/include" \
+	--with-ogg="$(STAGING_DIR)/opt/include" \
+	--with-vorbis="$(STAGING_DIR)/opt/include" \
 
 define Package/ices/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/ices $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/bin/ices $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,ices))
