--- ../../../downloads/openwrt_trunk-r35474/feeds/packages/libs/libvncserver/Makefile	2013-02-05 10:42:11.382946403 +0400
+++ ../../../openwrt_trunk/feeds/packages/libs/libvncserver/Makefile	2013-02-06 16:39:21.572983492 +0400
@@ -24,7 +24,7 @@
 define Package/libvncserver
   SECTION:=libs
   CATEGORY:=Libraries
-  DEPENDS:=+zlib +libjpeg +libpthread $(ICONV_DEPENDS)
+  DEPENDS:=+zlib +libjpeg-turbo +libpthread $(ICONV_DEPENDS)
   TITLE:=VNC Server Library
   URL:=http://libvncserver.sourceforge.net/
 endef
@@ -45,15 +45,15 @@
 	--without-avahi
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/{lib,include,bin}
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib* $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
+	$(INSTALL_DIR) $(1)/opt/{lib,include,bin}
+	$(CP) $(PKG_INSTALL_DIR)/opt/bin/* $(1)/opt/bin
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib* $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
 endef
 
 define Package/libvncserver/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/lib*so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/lib*so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libvncserver))
