--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/owfs/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/owfs/Makefile
@@ -21,6 +21,7 @@
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
 
 #
 # templates
@@ -74,7 +75,7 @@
 
 define Package/libow
   $(call Package/owfs/Library)
-  DEPENDS:=+libusb-compat +libpthread
+  DEPENDS:=+libusb-compat +libpthread +$(ICONV_DEPENDS)
   TITLE:=OWFS - common shared library
 endef
 
@@ -115,7 +116,7 @@
 define Package/owfs
   $(call Package/owfs/Utility)
   # libfuse depends on kmod-fuse, no need to declare dependency
-  DEPENDS+= +libfuse +fuse-utils
+  DEPENDS+= +libfuse +fuse-utils $(ICONV_DEPENDS) +libusb-compat
   TITLE:=OWFS - fuse file system
 endef
 
@@ -167,8 +168,6 @@
 	--enable-owserver \
 	--enable-owhttpd \
 	--enable-owfs \
-	--with-fuseinclude="$(STAGING_DIR)/usr/include" \
-	--with-fuselib="$(STAGING_DIR)/usr/lib" \
 	--enable-shared \
 	--enable-zero \
 	--disable-parport \
@@ -177,11 +176,21 @@
 	--disable-owphp \
 	--disable-owtcl \
 	--disable-swig \
+	--with-fuseinclude="$(STAGING_DIR)/opt/include" \
+	--with-fuselib="$(STAGING_DIR)/opt/lib" \
+	--enable-usb \
+	--with-libusb-config="$(STAGING_DIR)/opt/bin/libusb-config"
+
+TARGET_LDFLAGS += \
+	-L$(ICONV_PREFIX)/lib \
+	-liconv \
+
+TARGET_CFLAGS += \
+	-I$(ICONV_PREFIX)/include \
 
 CONFIGURE_VARS += \
-	LDFLAGS="$(TARGET_LDFLAGS) -Wl,-rpath-link=$(STAGING_DIR)/usr/lib -Wl,-rpath-link=$(TOOLCHAIN_DIR)/usr/lib" \
-	lt_cv_sys_lib_dlsearch_path_spec="$(STAGING_DIR)/lib $(STAGING_DIR)/usr/lib" \
-	lt_cv_sys_lib_search_path_spec="$(STAGING_DIR)/lib $(STAGING_DIR)/usr/lib" \
+	lt_cv_sys_lib_dlsearch_path_spec="$(STAGING_DIR)/opt/lib" \
+	lt_cv_sys_lib_search_path_spec="$(STAGING_DIR)/opt/lib" \
 	shrext_cmds=".so" \
 
 MAKE_FLAGS += \
@@ -189,51 +198,51 @@
 	HOST_CPU="$(PKGARCH)"
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(STAGING_DIR)/usr/include
-	$(CP)	$(PKG_INSTALL_DIR)/usr/include/ow{capi,fs_config}.h $(STAGING_DIR)/usr/include/
-	$(INSTALL_DIR) $(STAGING_DIR)/usr/lib
-	$(CP)	$(PKG_INSTALL_DIR)/usr/lib/libow{,capi}*.so* $(STAGING_DIR)/usr/lib/
+	$(INSTALL_DIR) $(STAGING_DIR)/opt/include
+	$(CP)	$(PKG_INSTALL_DIR)/opt/include/ow{capi,fs_config}.h $(STAGING_DIR)/opt/include/
+	$(INSTALL_DIR) $(STAGING_DIR)/opt/lib
+	$(CP)	$(PKG_INSTALL_DIR)/opt/lib/libow{,capi}*.so* $(STAGING_DIR)/opt/lib/
 endef
 
 define Package/owfs/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owfs $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owfs $(1)/opt/bin/
 endef
 
 define Package/owshell/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owget $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owread $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owwrite $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owdir $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owpresent $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owget $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owread $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owwrite $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owdir $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owpresent $(1)/opt/bin/
 endef
 
 
 define Package/owserver/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owserver $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owserver $(1)/opt/bin/
 endef
 
 define Package/owhttpd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owhttpd $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owhttpd $(1)/opt/bin/
 endef
 
 define Package/owftpd/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/owftpd $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/owftpd $(1)/opt/bin/
 endef
 
 
 define Package/libow/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libow-*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libow-*.so.* $(1)/opt/lib/
 endef
 
 define Package/libow-capi/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libowcapi-*.so.* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libowcapi-*.so.* $(1)/opt/lib/
 endef
 
 
