--- ../../../downloads/openwrt_trunk-r40859/feeds/packages/utils/screen/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/screen/Makefile
@@ -37,20 +37,22 @@
 define Build/Configure
 	$(call Build/Configure/Default,\
 		--with-sys-screenrc=/etc/screenrc \
+		--with-sys-screenrc=/opt/etc/screenrc \
+		--with-socket-dir=/opt/tmp/screens \
 	)
 	# XXX: memmove() works well with overlapped memory areas
 	echo "#define USEMEMMOVE 1" >>$(PKG_BUILD_DIR)/config.h
 endef
 
 define Package/screen/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/screen $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/etc/screenrc $(1)/etc/screenrc
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/screen $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) ./files/etc/screenrc $(1)/opt/etc/screenrc
 endef
 
 define Package/screen/conffiles
-/etc/screenrc
+/opt/etc/screenrc
 endef
 
 $(eval $(call BuildPackage,screen))
