--- a/src/cfgfile.c
+++ b/src/cfgfile.c
@@ -57,10 +57,10 @@
 #define CONFIG_YP_URL_TIMEOUT 10
 
 #ifndef _WIN32
-#define CONFIG_DEFAULT_BASE_DIR "/usr/local/icecast"
-#define CONFIG_DEFAULT_LOG_DIR "/usr/local/icecast/logs"
-#define CONFIG_DEFAULT_WEBROOT_DIR "/usr/local/icecast/webroot"
-#define CONFIG_DEFAULT_ADMINROOT_DIR "/usr/local/icecast/admin"
+#define CONFIG_DEFAULT_BASE_DIR "/opt/share/icecast"
+#define CONFIG_DEFAULT_LOG_DIR "/opt/var/log"
+#define CONFIG_DEFAULT_WEBROOT_DIR "/opt/share/icecast/web"
+#define CONFIG_DEFAULT_ADMINROOT_DIR "/opt/share/icecast/admin"
 #define MIMETYPESFILE "/etc/mime.types"
 #else
 #define CONFIG_DEFAULT_BASE_DIR ".\\"
