--- ../../../openwrt_trunk/package/libs/gettext-full/Makefile.orig
+++ ../../../openwrt_trunk/package/libs/gettext-full/Makefile
@@ -51,6 +51,7 @@
 	--disable-java \
 	--disable-native-java \
 	--disable-openmp \
+	--disable-curses \
 	--with-included-gettext \
 	--without-libintl-prefix \
 	--without-libexpat-prefix \
@@ -60,19 +61,19 @@
 	--with-included-libxml --without-libxml2-prefix --with-included-libcroco
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/lib/libintl-full/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/libintl.h $(1)/usr/lib/libintl-full/include/
+	$(INSTALL_DIR) $(1)/opt/lib/libintl-full/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/libintl.h $(1)/opt/lib/libintl-full/include/
 
-	$(INSTALL_DIR) $(1)/usr/lib/libintl-full/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libintl.{a,so*} $(1)/usr/lib/libintl-full/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/libintl-full/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libintl.{a,so*} $(1)/opt/lib/libintl-full/lib/
 
-	$(INSTALL_DIR) $(1)/usr/share/aclocal
-	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/share/aclocal/* $(1)/usr/share/aclocal/
+	$(INSTALL_DIR) $(1)/opt/share/aclocal
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/opt/share/aclocal/* $(1)/opt/share/aclocal/
 endef
 
 define Package/libintl-full/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libintl.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libintl.so* $(1)/opt/lib/
 endef
 
 $(eval $(call HostBuild))
