--- ../../../downloads/mipsel-hardfloat/openwrt_trunk-r38800/feeds/oldpackages/libs/loudmouth/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/libs/loudmouth/Makefile
@@ -26,7 +26,7 @@
 define Package/libloudmouth1
 	SECTION:=libs
 	CATEGORY:=Libraries
-	DEPENDS:=+libopenssl +glib2
+	DEPENDS:=+libopenssl +glib2 +libidn
 	TITLE:=Lightweight C Jabber library
 	URL:=http://mcabber.com
 	MAINTAINER:=Peter Wagner <tripolar@gmx.at>
@@ -40,24 +40,24 @@
 
 CONFIGURE_VARS += \
 	ac_cv_path_KRB5CONFIG=no \
-	LIBS="-liconv -lgobject-2.0"
+	LIBS="-liconv -lgobject-2.0 -lintl"
 
 CONFIGURE_ARGS += \
 	--with-compile-warnings=no \
 	--with-ssl=openssl
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/loudmouth-1.0/loudmouth
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/loudmouth-1.0/loudmouth/*.h $(1)/usr/include/loudmouth-1.0/loudmouth
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libloudmouth-1.{a,la,so*} $(1)/usr/lib/
-	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/loudmouth-1.0.pc $(1)/usr/lib/pkgconfig/
+	$(INSTALL_DIR) $(1)/opt/include/loudmouth-1.0/loudmouth
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/loudmouth-1.0/loudmouth/*.h $(1)/opt/include/loudmouth-1.0/loudmouth
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libloudmouth-1.{a,la,so*} $(1)/opt/lib/
+	$(INSTALL_DIR) $(1)/opt/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/pkgconfig/loudmouth-1.0.pc $(1)/opt/lib/pkgconfig/
 endef
 
 define Package/libloudmouth1/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_BUILD_DIR)/loudmouth/.libs/libloudmouth-1.so* $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_BUILD_DIR)/loudmouth/.libs/libloudmouth-1.so* $(1)/opt/lib
 endef
 
 $(eval $(call BuildPackage,libloudmouth1))
