--- ../../../downloads/openwrt_trunk-r43718/feeds/packages/net/mtr/Makefile	2014-11-07 23:35:24.536148262 +0300
+++ ../../../openwrt_trunk/feeds/packages/net/mtr/Makefile	2014-11-29 00:10:37.026199148 +0300
@@ -46,6 +46,11 @@
 CONFIGURE_ARGS += \
 	--without-gtk \
 	--without-glib \
+	--disable-ipv6 \
+
+TARGET_LDFLAGS += \
+        -lresolv \
+
 
 define Build/Configure
 	(cd $(PKG_BUILD_DIR); touch \
@@ -61,8 +66,8 @@
 endef
 
 define Package/mtr/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/mtr $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/Apps/opt/sbin/mtr $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,mtr))
