--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/mtr/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/mtr/Makefile
@@ -60,8 +60,8 @@
 endef
 
 define Package/mtr/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/mtr $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/mtr $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,mtr))
