diff -urx .svn ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/strongswan/Makefile ../../../openwrt_trunk/feeds/packages/net/strongswan/Makefile
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/strongswan/Makefile	2014-10-06 15:46:23.765250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/strongswan/Makefile	2014-10-09 12:19:45.661251204 +0400
@@ -109,10 +109,7 @@
 
 define Package/strongswan
 $(call Package/strongswan/Default)
-  DEPENDS:= +libpthread +ip \
-	+kmod-crypto-authenc \
-	+kmod-ipsec +kmod-ipsec4 +kmod-ipsec6 \
-	+kmod-ipt-ipsec +iptables-mod-ipsec
+  DEPENDS:= +libpthread
 endef
 
 define Package/strongswan/config
@@ -298,17 +295,17 @@
     DEPENDS:= +strongswan $(3)
   endef
 
-  strongswan_mod_conf=$(wildcard $(PKG_INSTALL_DIR)/etc/strongswan.d/charon/$(1).conf)
+  strongswan_mod_conf=$(wildcard $(PKG_INSTALL_DIR)/opt/etc/strongswan.d/charon/$(1).conf)
   define Package/strongswan-mod-$(1)/install
-	$(INSTALL_DIR) $$(1)/etc/strongswan.d/charon
+	$(INSTALL_DIR) $$(1)/opt/etc/strongswan.d/charon
 	$(if $(call strongswan_mod_conf,$(1)), \
 		$(INSTALL_DATA) \
 			$(call strongswan_mod_conf,$(1)) \
-			$$(1)/etc/strongswan.d/charon/ \
+			$$(1)/opt/etc/strongswan.d/charon/ \
 	)
-	$(INSTALL_DIR) $$(1)/usr/lib/ipsec/plugins
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/plugins/libstrongswan-$(1).so \
-		$$(1)/usr/lib/ipsec/plugins/
+	$(INSTALL_DIR) $$(1)/opt/lib/ipsec/plugins
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/plugins/libstrongswan-$(1).so \
+		$$(1)/opt/lib/ipsec/plugins/
 	$(call Plugin/$(1)/install,$$(1))
   endef
 
@@ -331,23 +328,23 @@
 	  $(if $(CONFIG_PACKAGE_strongswan-mod-$(m)),--enable-$(m),--disable-$(m)) \
 	)
 
-EXTRA_LDFLAGS+= -Wl,-rpath-link,$(STAGING_DIR)/usr/lib
+EXTRA_LDFLAGS+= -Wl,-rpath-link,$(STAGING_DIR)/opt/lib
 
 define Package/strongswan/conffiles
-/etc/ipsec.conf
-/etc/ipsec.secrets
-/etc/ipsec.user
-/etc/strongswan.conf
+/opt/etc/ipsec.conf
+/opt/etc/ipsec.secrets
+/opt/etc/ipsec.user
+/opt/etc/strongswan.conf
 endef
 
 define Package/strongswan/install
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/strongswan.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/{libstrongswan.so.*,libhydra.so.*} $(1)/usr/lib/ipsec/
-	$(INSTALL_CONF) ./files/ipsec.secrets $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/ipsec.init $(1)/etc/init.d/ipsec
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/strongswan.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/{libstrongswan.so.*,libhydra.so.*} $(1)/opt/lib/ipsec/
+	$(INSTALL_CONF) ./files/ipsec.secrets $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S90strongswan $(1)/opt/etc/init.d/
 endef
 
 define Package/strongswan-default/install
@@ -363,71 +360,71 @@
 endef
 
 define Package/strongswan-charon/install
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/ipsec/charon $(1)/usr/lib/ipsec/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/libcharon.so.* $(1)/usr/lib/ipsec/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/ipsec/charon $(1)/opt/lib/ipsec/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/libcharon.so.* $(1)/opt/lib/ipsec/
 endef
 
 define Package/strongswan-utils/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/ipsec $(1)/usr/sbin/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/pki $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/ipsec/scepclient $(1)/usr/lib/ipsec/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/sbin/ipsec $(1)/opt/sbin/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/pki $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/ipsec/scepclient $(1)/opt/lib/ipsec/
 endef
 
 define Plugin/duplicheck/install
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec/plugins
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/duplicheck $(1)/usr/lib/ipsec/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/plugins/libstrongswan-duplicheck.so $(1)/usr/lib/ipsec/plugins/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec/plugins
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/duplicheck $(1)/opt/lib/ipsec/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/plugins/libstrongswan-duplicheck.so $(1)/opt/lib/ipsec/plugins/
 endef
 
 define Plugin/eap-radius/install
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec/plugins
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/libradius.so.* $(1)/usr/lib/ipsec/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/plugins/libstrongswan-eap-radius.so $(1)/usr/lib/ipsec/plugins/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec/plugins
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/libradius.so.* $(1)/opt/lib/ipsec/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/plugins/libstrongswan-eap-radius.so $(1)/opt/lib/ipsec/plugins/
 endef
 
 define Plugin/attr-sql/install
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/pool $(1)/usr/lib/ipsec/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/pool $(1)/opt/lib/ipsec/
 endef
 
 define Plugin/stroke/install
-	$(INSTALL_DIR) $(1)/etc/ipsec.d/aacerts
-	$(INSTALL_DIR) $(1)/etc/ipsec.d/acerts
-	$(INSTALL_DIR) $(1)/etc/ipsec.d/cacerts
-	$(INSTALL_DIR) $(1)/etc/ipsec.d/certs
-	$(INSTALL_DIR) $(1)/etc/ipsec.d/crls
-	$(INSTALL_DIR) $(1)/etc/ipsec.d/ocspcerts
-	$(INSTALL_DIR) $(1)/etc/ipsec.d/private
-	$(INSTALL_DIR) $(1)/etc/ipsec.d/reqs
-
-	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/ipsec.conf $(1)/etc/
-
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec/plugins
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/lib/ipsec/{starter,stroke} $(1)/usr/lib/ipsec/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/plugins/libstrongswan-stroke.so $(1)/usr/lib/ipsec/plugins/
+	$(INSTALL_DIR) $(1)/opt/etc/ipsec.d/aacerts
+	$(INSTALL_DIR) $(1)/opt/etc/ipsec.d/acerts
+	$(INSTALL_DIR) $(1)/opt/etc/ipsec.d/cacerts
+	$(INSTALL_DIR) $(1)/opt/etc/ipsec.d/certs
+	$(INSTALL_DIR) $(1)/opt/etc/ipsec.d/crls
+	$(INSTALL_DIR) $(1)/opt/etc/ipsec.d/ocspcerts
+	$(INSTALL_DIR) $(1)/opt/etc/ipsec.d/private
+	$(INSTALL_DIR) $(1)/opt/etc/ipsec.d/reqs
+
+	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/opt/etc/ipsec.conf $(1)/opt/etc/
+
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec/plugins
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/lib/ipsec/{starter,stroke} $(1)/opt/lib/ipsec/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/plugins/libstrongswan-stroke.so $(1)/opt/lib/ipsec/plugins/
 endef
 
 define Plugin/updown/install
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec/plugins
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/{_updown,_updown_espmark} $(1)/usr/lib/ipsec/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/plugins/libstrongswan-updown.so $(1)/usr/lib/ipsec/plugins/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) ./files/ipsec.user $(1)/etc/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec/plugins
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/{_updown,_updown_espmark} $(1)/opt/lib/ipsec/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/plugins/libstrongswan-updown.so $(1)/opt/lib/ipsec/plugins/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_CONF) ./files/ipsec.user $(1)/opt/etc/
 endef
 
 define Plugin/whitelist/install
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec/plugins
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/whitelist $(1)/usr/lib/ipsec/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/plugins/libstrongswan-whitelist.so $(1)/usr/lib/ipsec/plugins/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec/plugins
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/whitelist $(1)/opt/lib/ipsec/
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/plugins/libstrongswan-whitelist.so $(1)/opt/lib/ipsec/plugins/
 endef
 
 define Plugin/kernel-libipsec/install
-	$(INSTALL_DIR) $(1)/usr/lib/ipsec
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ipsec/libipsec.so.* $(1)/usr/lib/ipsec/
+	$(INSTALL_DIR) $(1)/opt/lib/ipsec
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/ipsec/libipsec.so.* $(1)/opt/lib/ipsec/
 endef
 
 $(eval $(call BuildPackage,strongswan))
@@ -438,7 +435,7 @@
 $(eval $(call BuildPackage,strongswan-utils))
 $(eval $(call BuildPlugin,addrblock,RFC 3779 address block constraint support,))
 $(eval $(call BuildPlugin,aes,AES crypto,))
-$(eval $(call BuildPlugin,af-alg,AF_ALG crypto interface to Linux Crypto API,+kmod-crypto-user))
+$(eval $(call BuildPlugin,af-alg,AF_ALG crypto interface to Linux Crypto API,))
 $(eval $(call BuildPlugin,agent,SSH agent signing,))
 $(eval $(call BuildPlugin,attr,file based config,))
 $(eval $(call BuildPlugin,attr-sql,SQL based config,+strongswan-mod-sql))
diff -urx .svn ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/strongswan/patches/201-kmodloader.patch ../../../openwrt_trunk/feeds/packages/net/strongswan/patches/201-kmodloader.patch
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/strongswan/patches/201-kmodloader.patch	2014-10-06 15:46:23.765250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/strongswan/patches/201-kmodloader.patch	2014-10-07 11:59:35.787924655 +0400
@@ -5,7 +5,7 @@
  		if (stat(PROC_MODULES, &stb) == 0)
  		{
 -			ignore_result(system("modprobe -qv af_key"));
-+			ignore_result(system("modprobe af_key 2>&1 >/dev/null"));
++			ignore_result(system("insmod af_key 2>&1 >/dev/null"));
  		}
  
  		/* now test again */
@@ -18,11 +18,11 @@
 -		ignore_result(system("modprobe -qv ipcomp"));
 -		ignore_result(system("modprobe -qv xfrm4_tunnel"));
 -		ignore_result(system("modprobe -qv xfrm_user"));
-+		ignore_result(system("modprobe ah4 2>&1 >/dev/null"));
-+		ignore_result(system("modprobe esp4 2>&1 >/dev/null"));
-+		ignore_result(system("modprobe ipcomp 2>&1 >/dev/null"));
-+		ignore_result(system("modprobe xfrm4_tunnel 2>&1 >/dev/null"));
-+		ignore_result(system("modprobe xfrm_user 2>&1 >/dev/null"));
++		ignore_result(system("insmod ah4 2>&1 >/dev/null"));
++		ignore_result(system("insmod esp4 2>&1 >/dev/null"));
++		ignore_result(system("insmod ipcomp 2>&1 >/dev/null"));
++		ignore_result(system("insmod xfrm4_tunnel 2>&1 >/dev/null"));
++		ignore_result(system("insmod xfrm_user 2>&1 >/dev/null"));
  	}
  
  	DBG2(DBG_APP, "found netkey IPsec stack");
diff -urx .svn ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/strongswan/patches/300-include-ipsec-user-script.patch ../../../openwrt_trunk/feeds/packages/net/strongswan/patches/300-include-ipsec-user-script.patch
--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/net/strongswan/patches/300-include-ipsec-user-script.patch	2014-10-06 15:46:23.765250644 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/strongswan/patches/300-include-ipsec-user-script.patch	2014-10-07 11:59:35.787924655 +0400
@@ -9,9 +9,9 @@
 -# you need changes, make a copy of this under another name, and customize
 -# that, and use the (left/right)updown parameters in ipsec.conf to make
 -# strongSwan use yours instead of this default one.
-+# Add your custom ip rules to the /etc/ipsec.user file if you need that functionality.
++# Add your custom ip rules to the /opt/etc/ipsec.user file if you need that functionality.
 +
-+[ -e /etc/ipsec.user ] && . /etc/ipsec.user "$1"
++[ -e /opt/etc/ipsec.user ] && . /opt/etc/ipsec.user "$1"
  
  # things that this script gets (from ipsec_pluto(8) man page)
  #
