--- ../../../openwrt_trunk.orig/feeds/packages/net/etherwake/Makefile
+++ ../../../openwrt_trunk/feeds/packages/net/etherwake/Makefile
@@ -36,20 +36,12 @@
 endef
 
 define Build/Compile
-	$(TARGET_CC) $(TARGET_CFLAGS) -D__UCLIBC__ $(PKG_BUILD_DIR)/ether-wake.c -o $(PKG_BUILD_DIR)/etherwake
+	$(TARGET_CC) $(TARGET_LDFLAGS) $(TARGET_CFLAGS) -D__UCLIBC__ $(PKG_BUILD_DIR)/ether-wake.c -o $(PKG_BUILD_DIR)/etherwake
 endef
 
 define Package/etherwake/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/etherwake $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) files/$(PKG_NAME).config $(1)/etc/config/$(PKG_NAME)
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) files/$(PKG_NAME).init $(1)/etc/init.d/$(PKG_NAME)
-endef
-
-define Package/etherwake/conffiles
-/etc/config/etherwake
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/etherwake $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,etherwake))
