--- ../../../downloads/openwrt_trunk-q46338/feeds/packages/libs/libev/Makefile	2015-07-21 13:57:42.000000000 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/libev/Makefile	2015-07-23 21:30:49.012723915 +0300
@@ -43,15 +43,15 @@
 	--enable-static \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libev.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/include/* $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libev.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libev/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libev.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/Apps/opt/lib/libev.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libev))
