--- ../../../downloads/openwrt_trunk-r46338/feeds/packages/libs/libev/Makefile	2015-07-14 10:55:19.774308173 +0300
+++ ../../../openwrt_trunk/feeds/packages/libs/libev/Makefile	2015-07-14 13:37:23.962473420 +0300
@@ -43,15 +43,15 @@
 	--enable-static \
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include/
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libev.{a,so*} $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/include
+	$(CP) $(PKG_INSTALL_DIR)/opt/include/* $(1)/opt/include/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libev.{a,so*} $(1)/opt/lib/
 endef
 
 define Package/libev/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libev.so* $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/opt/lib
+	$(CP) $(PKG_INSTALL_DIR)/opt/lib/libev.so* $(1)/opt/lib/
 endef
 
 $(eval $(call BuildPackage,libev))
