--- a/daemon.c
+++ b/daemon.c
@@ -63,7 +63,7 @@
 
 /* options and default values */
 int opt_detach = TRUE;
-char *opt_exports = "/etc/exports";
+char *opt_exports = "/opt/etc/exports";
 int opt_cluster = FALSE;
 char *opt_cluster_path = "/";
 int opt_tcponly = FALSE;
@@ -239,7 +239,7 @@
 		printf("\t-h          display this short option summary\n");
 		printf("\t-u          use unprivileged port for services\n");
 		printf("\t-d          do not detach from terminal\n");
-		printf("\t-e <file>   file to use instead of /etc/exports\n");
+		printf("\t-e <file>   file to use instead of /opt/etc/exports\n");
 		printf("\t-i <file>   write daemon pid to given file\n");
 #ifdef WANT_CLUSTER
 		printf("\t-c          enable cluster extensions\n");
