--- ../../../openwrt_trunk.orig/feeds/oldpackages/net/unfs3/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/unfs3/Makefile
@@ -26,7 +26,7 @@
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=Filesystem
-  DEPENDS:=+portmap $(LIBRPC_DEPENDS)
+  DEPENDS:=+portmap
   TITLE:=User-space NFSv3 Server
   URL:=http://unfs3.sourceforge.net/
 endef
@@ -38,20 +38,18 @@
 	on the server.
 endef
 
-TARGET_CFLAGS += -I$(STAGING_DIR)/usr/include
-TARGET_LDFLAGS += $(LIBRPC)
+TARGET_CFLAGS += -I$(STAGING_DIR)/opt/include
 
 define Package/unfs3/install
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/$(PKG_NAME).init $(1)/etc/init.d/$(PKG_NAME)
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) ./files/$(PKG_NAME).exports $(1)/etc/exports
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/unfsd $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S56unfsd $(1)/opt/etc/init.d/
+	$(INSTALL_DATA) ./files/$(PKG_NAME).exports $(1)/opt/etc/exports
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/unfsd $(1)/opt/sbin
 endef
 
 define Package/unfs3/conffiles
-/etc/exports
+/opt/etc/exports
 endef
 
 $(eval $(call BuildPackage,unfs3))
