--- ../../../downloads/openwrt_trunk-r35474/feeds/oldpackages/net/ipcad/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/net/ipcad/Makefile
@@ -22,7 +22,7 @@
 define Package/ipcad
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libpcap $(LIBRPC_DEPENDS) +libpthread
+  DEPENDS:=+libpcap +libpthread
   TITLE:=listens for traffic on the specified interfaces
   URL:=http://lionet.info/ipcad/
 endef
@@ -34,15 +34,15 @@
 endef
 
 define Package/ipcad/conffiles/
-/etc/ipcad.conf
+/opt/etc/ipcad.conf
 endef
 
 CONFIGURE_ARGS += \
 	--enable-shared \
 	--disable-static \
 	--with-psrc=pcap \
-	--with-pcap-include="$(STAGING_DIR)/usr/include" \
-	--with-pcap-libraries="$(STAGING_DIR)/usr/lib" \
+	--with-pcap-include="$(STAGING_DIR)/opt/include" \
+	--with-pcap-libraries="$(STAGING_DIR)/opt/lib" \
 
 # XXX: don't detect/use netfilter QUEUE/ULOG captures
 CONFIGURE_VARS += \
@@ -51,13 +51,11 @@
 	ac_cv_header_linux_netfilter_ipv4_ip_queue_h=no \
 	ac_cv_header_linux_netfilter_ipv4_ipt_ULOG_h=no \
 
-TARGET_LDFLAGS += $(LIBRPC)
-
 define Package/ipcad/install
-	$(INSTALL_DIR) $(1)/etc
-	$(CP) $(PKG_INSTALL_DIR)/etc/ipcad.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ipcad $(1)/usr/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(CP) $(PKG_INSTALL_DIR)/opt/etc/ipcad.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/ipcad $(1)/opt/bin/
 endef
 
 $(eval $(call BuildPackage,ipcad))
