--- ../../../downloads/openwrt_trunk-r35474/package/boot/kexec-tools/Makefile	2013-02-05 10:42:42.642945832 +0400
+++ ../../../openwrt_trunk/package/boot/kexec-tools/Makefile	2013-02-05 11:47:01.638875513 +0400
@@ -22,7 +22,7 @@
 define Package/kexec-tools
   SECTION:=utils
   CATEGORY:=Utilities
-  DEPENDS:=@armeb||@arm||@i386||@powerpc64||@mipsel||@mips +zlib
+  DEPENDS:=@armeb||@arm||@i386||@powerpc64||@mipsel||@mips +zlib +liblzma
   TITLE:=Kernel boots kernel
   URL:=http://kernel.org/pub/linux/kernel/people/horms/kexec-tools/
   MAINTAINER:=Florian Fainelli <florian@openwrt.org>
@@ -38,7 +38,7 @@
 	source "$(SOURCE)/Config.in"
 endef
 
-KEXEC_TARGET_NAME:=$(call qstrip,$(CONFIG_KEXEC_TOOLS_TARGET_NAME))-linux-$(TARGET_SUFFIX)
+KEXEC_TARGET_NAME:=$(call qstrip,$(CONFIG_KEXEC_TOOLS_TARGET_NAME))-linux-gnu
 
 CONFIGURE_ARGS = \
 		--target=$(KEXEC_TARGET_NAME) \
@@ -46,12 +46,12 @@
 		--build=$(GNU_HOST_NAME) \
 		--program-prefix="" \
 		--program-suffix="" \
-		--prefix=/usr \
-		--exec-prefix=/usr \
-		--bindir=/usr/bin \
-		--sbindir=/usr/sbin \
-		--libexecdir=/usr/lib \
-		--sysconfdir=/etc
+		--prefix=/opt \
+		--exec-prefix=/opt \
+		--bindir=/opt/bin \
+		--sbindir=/opt/sbin \
+		--libexecdir=/opt/lib \
+		--sysconfdir=/opt/etc
 
 CONFIGURE_VARS += \
 	BUILD_CC="$(HOSTCC)" \
@@ -64,17 +64,13 @@
 endef
 
 define Package/kexec-tools/install
-	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/opt/sbin
 	$(INSTALL_BIN) \
-		$(addprefix $(PKG_INSTALL_DIR)/usr/sbin/, \
+		$(addprefix $(PKG_INSTALL_DIR)/opt/sbin/, \
 			$(kexec-extra-sbin-y)) \
 		$(kexec-extra-bin-y) \
-		$(PKG_INSTALL_DIR)/usr/sbin/kexec \
-		$(1)/usr/sbin
-
-# make a link for compatability with other distros
-	$(INSTALL_DIR) $(1)/sbin
-	ln -s /usr/sbin/kexec $(1)/sbin/kexec
+		$(PKG_INSTALL_DIR)/opt/sbin/kexec \
+		$(1)/opt/sbin
 endef
 
 $(eval $(call BuildPackage,kexec-tools))
