--- ../../../openwrt_trunk.orig/feeds/oldpackages/utils/empty/Makefile	2012-09-21 10:59:55.953103763 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/utils/empty/Makefile	2012-09-21 15:57:45.444776479 +0400
@@ -33,9 +33,12 @@
  programming tools.
 endef
 
+MAKE_FLAGS += \
+	LIBS="$(TARGET_LDFLAGS) -lutil"
+
 define Package/empty/install
-	$(INSTALL_DIR) $(1)/usr/sbin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(PKG_NAME) $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/opt/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(PKG_NAME) $(1)/opt/sbin/
 endef
 
 $(eval $(call BuildPackage,empty))
