--- ../../../downloads/openwrt_trunk-r35953/feeds/packages/net/umurmur/Makefile	2013-03-11 09:29:43.058073746 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/umurmur/Makefile	2013-03-12 22:11:00.939659441 +0400
@@ -8,7 +8,7 @@
 
 PKG_NAME:=umurmur
 PKG_VERSION:=0.2.10
-PKG_RELEASE:=1
+PKG_RELEASE:=1a
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://umurmur.googlecode.com/files
@@ -64,19 +64,18 @@
 endef
 
 define Package/umurmur-openssl/conffiles
-/etc/umurmur.conf
+/opt/etc/umurmur.conf
 endef
 
 Package/umurmur-polarssl/conffiles = $(Package/umurmur-openssl/conffiles)
 
 define Package/umurmur-openssl/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/umurmurd $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/openwrt/files/umurmur.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/openwrt/files/umurmur.init $(1)/etc/init.d/umurmur
-	$(INSTALL_DIR) $(1)/etc/umurmur
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/umurmurd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/umurmur
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/openwrt/files/umurmur.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S73umurmurd $(1)/opt/etc/init.d
 endef
 
 Package/umurmur-polarssl/install = $(Package/umurmur-openssl/install)
