--- ../../../openwrt_trunk.orig/feeds/packages/net/umurmur/Makefile	2012-06-01 10:50:34.428282662 +0400
+++ ../../../openwrt_trunk/feeds/packages/net/umurmur/Makefile	2012-06-01 10:51:36.027508255 +0400
@@ -64,19 +64,18 @@
 endef
 
 define Package/umurmur-openssl/conffiles
-/etc/umurmur.conf
+/opt/etc/umurmur.conf
 endef
 
 Package/umurmur-polarssl/conffiles = $(Package/umurmur-openssl/conffiles)
 
 define Package/umurmur-openssl/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/umurmurd $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/openwrt/files/umurmur.conf $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/openwrt/files/umurmur.init $(1)/etc/init.d/umurmur
-	$(INSTALL_DIR) $(1)/etc/umurmur
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/src/umurmurd $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc/umurmur
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/openwrt/files/umurmur.conf $(1)/opt/etc/
+	$(INSTALL_DIR) $(1)/opt/etc/init.d
+	$(INSTALL_BIN) ./files/S73umurmurd $(1)/opt/etc/init.d
 endef
 
 Package/umurmur-polarssl/install = $(Package/umurmur-openssl/install)
