--- ../../../downloads/openwrt_trunk-r42803/feeds/oldpackages/mail/xmail/Makefile	2014-10-15 10:43:09.903917917 +0400
+++ ../../../openwrt_trunk/feeds/oldpackages/mail/xmail/Makefile	2014-10-28 15:02:32.851961346 +0300
@@ -9,7 +9,7 @@
 
 PKG_NAME:=xmail
 PKG_VERSION:=1.27
-PKG_RELEASE:=1
+PKG_RELEASE:=1b
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.xmailserver.org/
@@ -36,24 +36,19 @@
 	CFLAGS="$(TARGET_CFLAGS)" \
 	$(MAKE) -C $(PKG_BUILD_DIR) -f Makefile.lnx \
 		$(TARGET_CONFIGURE_OPTS) \
+		CPPFLAGS="$(TARGET_CPPFLAGS)" \
+		LDFLAGS="$(TARGET_LDFLAGS) -lpthread -lssl -ldl -lcrypto" \
 		LD="$(TARGET_CXX)" \
 		all
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/xmail $(PKG_BUILD_DIR)/MailRoot
 	tar -cjf $(PKG_BUILD_DIR)/MailRoot.tar.bz2 -C $(PKG_BUILD_DIR)/MailRoot --exclude bin .
 endef
 
 define Package/xmail/install
-	$(INSTALL_DIR) $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/{CtrlClnt,MkUsers,XMCrypt,XMail,sendmail} $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/MailRoot.tar.bz2 $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/xmail.config $(1)/etc/config/xmail
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) ./files/xmail.init $(1)/etc/init.d/xmail
-endef
-
-define Package/xmail/conffiles
-/etc/config/xmail
+	$(INSTALL_DIR) $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/{CtrlClnt,MkUsers,XMCrypt,XMail,sendmail} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/MailRoot.tar.bz2 $(1)/opt/etc/
 endef
 
 $(eval $(call BuildPackage,xmail))
