--- ../../../downloads/openwrt_trunk-r40423/feeds/oldpackages/mail/xmail/Makefile
+++ ../../../openwrt_trunk/feeds/oldpackages/mail/xmail/Makefile
@@ -36,24 +36,18 @@
 	CFLAGS="$(TARGET_CFLAGS)" \
 	$(MAKE) -C $(PKG_BUILD_DIR) -f Makefile.lnx \
 		$(TARGET_CONFIGURE_OPTS) \
+		CPPFLAGS="$(TARGET_CPPFLAGS)" \
+		LDFLAGS="$(TARGET_LDFLAGS) -lpthread -lssl -ldl -lcrypto" \
 		LD="$(TARGET_CXX)" \
 		all
 	tar -cjf $(PKG_BUILD_DIR)/MailRoot.tar.bz2 -C $(PKG_BUILD_DIR)/MailRoot --exclude bin .
 endef
 
 define Package/xmail/install
-	$(INSTALL_DIR) $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/{CtrlClnt,MkUsers,XMCrypt,XMail,sendmail} $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/MailRoot.tar.bz2 $(1)/etc/
-	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/xmail.config $(1)/etc/config/xmail
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) ./files/xmail.init $(1)/etc/init.d/xmail
-endef
-
-define Package/xmail/conffiles
-/etc/config/xmail
+	$(INSTALL_DIR) $(1)/opt/bin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/bin/{CtrlClnt,MkUsers,XMCrypt,XMail,sendmail} $(1)/opt/bin/
+	$(INSTALL_DIR) $(1)/opt/etc
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/MailRoot.tar.bz2 $(1)/opt/etc/
 endef
 
 $(eval $(call BuildPackage,xmail))
