--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/bash/Makefile
+++ ../../../openwrt_trunk/feeds/packages/utils/bash/Makefile
@@ -10,7 +10,7 @@
 BASE_VERSION:=4.3
 
 PKG_NAME:=bash
-PKG_VERSION:=$(BASE_VERSION).26
+PKG_VERSION:=$(BASE_VERSION).30
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(BASE_VERSION).tar.gz
@@ -42,7 +42,7 @@
 define Build/Configure
 	$(call Build/Configure/Default, \
 		--without-bash-malloc \
-                --bindir=/bin \
+                --bindir=/opt/bin \
 	)
 endef
 
@@ -55,17 +55,11 @@
 		all install
 endef
 
-define Package/bash/postinst
-#!/bin/sh
-grep bash $${IPKG_INSTROOT}/etc/shells || \
-	echo "/bin/bash" >> $${IPKG_INSTROOT}/etc/shells
-	echo "/bin/rbash" >> $${IPKG_INSTROOT}/etc/shells
-endef
 
 define Package/bash/install
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/bash $(1)/bin/
-	ln -sf bash $(1)/bin/rbash
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/bash $(1)/opt/bin/
+	ln -sf bash $(1)/opt/bin/rbash
 endef
 
 
