--- ../../../openwrt_trunk.orig/feeds/packages/utils/bash/Makefile	2012-01-20 19:21:03.000000000 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/bash/Makefile	2012-01-23 14:40:29.983085821 +0400
@@ -35,7 +35,7 @@
 define Build/Configure
 	$(call Build/Configure/Default, \
 		--without-bash-malloc \
-                --bindir=/bin \
+                --bindir=/opt/bin \
 	)
 endef
 
@@ -48,17 +48,11 @@
 		all install
 endef
 
-define Package/bash/postinst
-#!/bin/sh
-grep bash $${IPKG_INSTROOT}/etc/shells || \
-	echo "/bin/bash" >> $${IPKG_INSTROOT}/etc/shells
-	echo "/bin/rbash" >> $${IPKG_INSTROOT}/etc/shells
-endef
 
 define Package/bash/install
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/bash $(1)/bin/
-	ln -sf bash $(1)/bin/rbash
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/bash $(1)/opt/bin/
+	ln -sf bash $(1)/opt/bin/rbash
 endef
 
 $(eval $(call BuildPackage,bash))
