--- ../../../downloads/openwrt_trunk-r42803/feeds/packages/utils/bash/Makefile	2014-10-06 15:46:23.773250645 +0400
+++ ../../../openwrt_trunk/feeds/packages/utils/bash/Makefile	2014-10-07 11:59:08.875925144 +0400
@@ -42,7 +42,7 @@
 define Build/Configure
 	$(call Build/Configure/Default, \
 		--without-bash-malloc \
-                --bindir=/bin \
+                --bindir=/opt/bin \
 	)
 endef
 
@@ -55,17 +55,11 @@
 		all install
 endef
 
-define Package/bash/postinst
-#!/bin/sh
-grep bash $${IPKG_INSTROOT}/etc/shells || \
-	echo "/bin/bash" >> $${IPKG_INSTROOT}/etc/shells
-	echo "/bin/rbash" >> $${IPKG_INSTROOT}/etc/shells
-endef
 
 define Package/bash/install
-	$(INSTALL_DIR) $(1)/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/bin/bash $(1)/bin/
-	ln -sf bash $(1)/bin/rbash
+	$(INSTALL_DIR) $(1)/opt/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/opt/bin/bash $(1)/opt/bin/
+	ln -sf bash $(1)/opt/bin/rbash
 endef
 
 
